<<<<<<< HEAD
{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Dataframe Shape: (751, 5) \n",
      "\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>element_id</th>\n",
       "      <th>text</th>\n",
       "      <th>page_number</th>\n",
       "      <th>filename</th>\n",
       "      <th>chunk_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>38ce4f336a6096a4e0eb4efc385c6884</td>\n",
       "      <td>admission to the hospital (range 0–24, n = 82)...</td>\n",
       "      <td>8</td>\n",
       "      <td>ALD patent background + target paper.pdf</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>d816fd6f4542d1b8eb4f77190745d144</td>\n",
       "      <td>respective laboratory values were avail- able....</td>\n",
       "      <td>8</td>\n",
       "      <td>ALD patent background + target paper.pdf</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>202</th>\n",
       "      <td>077ff414cee1eb0193dbc2ca6bd1badb</td>\n",
       "      <td>informed consent was obtained from each patient.</td>\n",
       "      <td>8</td>\n",
       "      <td>ALD patent background + target paper.pdf</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>203</th>\n",
       "      <td>8c2cd087d7482ec1cc860453e9de31fb</td>\n",
       "      <td>Mice\\n\\nC57BL/6 mice were purchased from Charl...</td>\n",
       "      <td>8</td>\n",
       "      <td>ALD patent background + target paper.pdf</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>43d130fb7cbac4d01b6a808303dff089</td>\n",
       "      <td>8 | Nature | www.nature.com</td>\n",
       "      <td>8</td>\n",
       "      <td>ALD patent background + target paper.pdf</td>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>595</th>\n",
       "      <td>828e3d7554e59588cca8ddf4d1db363a</td>\n",
       "      <td>GVHD-related mortality Enterococcus ≥ 0.3 (n=5...</td>\n",
       "      <td>2</td>\n",
       "      <td>GvHD-paper.pdf</td>\n",
       "      <td>596</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>596</th>\n",
       "      <td>9699d347d90b3ff6d6ed7bf145b4b245</td>\n",
       "      <td>point represents a fecal sample, and color ind...</td>\n",
       "      <td>2</td>\n",
       "      <td>GvHD-paper.pdf</td>\n",
       "      <td>597</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>597</th>\n",
       "      <td>10ac17b617ed6906ffae7f3f788f203e</td>\n",
       "      <td>between day 0 and +21). n, number of individua...</td>\n",
       "      <td>2</td>\n",
       "      <td>GvHD-paper.pdf</td>\n",
       "      <td>598</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>598</th>\n",
       "      <td>ba25ff6c46b5bba6c5e2795bfcda92d4</td>\n",
       "      <td>with those without domination.</td>\n",
       "      <td>2</td>\n",
       "      <td>GvHD-paper.pdf</td>\n",
       "      <td>599</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>599</th>\n",
       "      <td>f73b871ed5f104c18f8b1bc12bee8eaf</td>\n",
       "      <td>2 of 7</td>\n",
       "      <td>2</td>\n",
       "      <td>GvHD-paper.pdf</td>\n",
       "      <td>600</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>400 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                           element_id  \\\n",
       "200  38ce4f336a6096a4e0eb4efc385c6884   \n",
       "201  d816fd6f4542d1b8eb4f77190745d144   \n",
       "202  077ff414cee1eb0193dbc2ca6bd1badb   \n",
       "203  8c2cd087d7482ec1cc860453e9de31fb   \n",
       "204  43d130fb7cbac4d01b6a808303dff089   \n",
       "..                                ...   \n",
       "595  828e3d7554e59588cca8ddf4d1db363a   \n",
       "596  9699d347d90b3ff6d6ed7bf145b4b245   \n",
       "597  10ac17b617ed6906ffae7f3f788f203e   \n",
       "598  ba25ff6c46b5bba6c5e2795bfcda92d4   \n",
       "599  f73b871ed5f104c18f8b1bc12bee8eaf   \n",
       "\n",
       "                                                  text  page_number  \\\n",
       "200  admission to the hospital (range 0–24, n = 82)...            8   \n",
       "201  respective laboratory values were avail- able....            8   \n",
       "202   informed consent was obtained from each patient.            8   \n",
       "203  Mice\\n\\nC57BL/6 mice were purchased from Charl...            8   \n",
       "204                        8 | Nature | www.nature.com            8   \n",
       "..                                                 ...          ...   \n",
       "595  GVHD-related mortality Enterococcus ≥ 0.3 (n=5...            2   \n",
       "596  point represents a fecal sample, and color ind...            2   \n",
       "597  between day 0 and +21). n, number of individua...            2   \n",
       "598                     with those without domination.            2   \n",
       "599                                             2 of 7            2   \n",
       "\n",
       "                                     filename  chunk_id  \n",
       "200  ALD patent background + target paper.pdf       201  \n",
       "201  ALD patent background + target paper.pdf       202  \n",
       "202  ALD patent background + target paper.pdf       203  \n",
       "203  ALD patent background + target paper.pdf       204  \n",
       "204  ALD patent background + target paper.pdf       205  \n",
       "..                                        ...       ...  \n",
       "595                            GvHD-paper.pdf       596  \n",
       "596                            GvHD-paper.pdf       597  \n",
       "597                            GvHD-paper.pdf       598  \n",
       "598                            GvHD-paper.pdf       599  \n",
       "599                            GvHD-paper.pdf       600  \n",
       "\n",
       "[400 rows x 5 columns]"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import lancedb\n",
    "import pandas as pd\n",
    "\n",
    "df1 = pd.read_csv(rf\"C:\\Users\\vtorr\\Work\\Projects\\aipatent\\app\\data\\tmp\\ALD patent background + target paper.csv\")\n",
    "df2 = pd.read_csv(rf\"C:\\Users\\vtorr\\Work\\Projects\\aipatent\\app\\data\\tmp\\GvHD-paper.csv\")\n",
    "\n",
    "df = pd.concat([df1, df2], ignore_index=True)\n",
    "\n",
    "df['chunk_id'] = range(1, len(df) + 1)\n",
    "\n",
    "print(f\"Dataframe Shape: {df.shape} \\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pydantic import BaseModel\n",
    "\n",
    "class Chunk(BaseModel):\n",
    "    element_id: str\n",
    "    text: str\n",
    "    page_number: int\n",
    "    filename: str\n",
    "    chunk_id: int\n",
    "    \n",
    "class ChunkReview(BaseModel):\n",
    "    relevant: bool\n",
    "    \n",
    "class ReviewedChunk(BaseModel):\n",
    "    element_id: str\n",
    "    text: str\n",
    "    page_number: int\n",
    "    filename: str\n",
    "    chunk_id: int\n",
    "    relevant: bool\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[Chunk(element_id='38dc4ba7c2bf7ed7323042c35312ad5f', text='Article', page_number=1, filename='ALD patent background + target paper.pdf', chunk_id=1)]"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "chunks = []\n",
    "for _,row in df.iterrows():\n",
    "    chunk = [\n",
    "        Chunk(\n",
    "        element_id=row[\"element_id\"],\n",
    "        text=row[\"text\"],\n",
    "        page_number=row[\"page_number\"],\n",
    "        filename=row[\"filename\"],\n",
    "        chunk_id=row[\"chunk_id\"]    \n",
    "    )]\n",
    "    \n",
    "    chunks.append(chunk)\n",
    "\n",
    "chunks[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "import instructor\n",
    "from openai import OpenAI\n",
    "\n",
    "\n",
    "openai = instructor.from_openai(OpenAI())\n",
    "\n",
    "def chunk_review(chunk: Chunk)-> ReviewedChunk:\n",
    "    \n",
    "    prompt  = f\"\"\"\n",
    "        the chunks will be stored.\n",
    "        You are a data annotator, your task is to review a corpus of data consisting of chunks from a source document.\n",
    "        Your perfomance is critical as it ensures that bad or irrelevant data doesn't flood the vector database where\n",
    "        the chunks will be stored\n",
    "        \n",
    "        here's the chunk text: <Text>{chunk.text}</Text>\n",
    "        \n",
    "        <Guidelines>\n",
    "            What makes text irrelevant:\n",
    "            - the text is too short and doesn't develop any idea or argument\n",
    "            - contains only numbers\n",
    "            - text that appears to be a caption for an image\n",
    "            - text that appears to only contain people's names\n",
    "            - text that is out of context\n",
    "            - as a rule of thumb text smaller than 80 tokens is not relevant\n",
    "            \n",
    "            What makes text relevant: \n",
    "             - it does present findings, ideas, arguments,\n",
    "             - it describes something and contains semantic meaning, valuable ideas about a topic can infered from it\n",
    "        </Guidelines>\n",
    "    \"\"\"\n",
    "    \n",
    "    res = openai.chat.completions.create(\n",
    "        model=\"o3-mini\",\n",
    "        response_model=ChunkReview,\n",
    "        messages=[\n",
    "            {\"role\": \"assistant\", \"content\": prompt}\n",
    "        ]\n",
    "    )\n",
    "    \n",
    "    return ReviewedChunk(**chunk.model_dump(), relevant=res.relevant)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "ReviewedChunk(element_id='4d2733e4cdaa6f457e875d11f778b79d', text='Cytolysin linked to increased mortality', page_number=1, filename='ALD patent background + target paper.pdf', chunk_id=13, relevant=False)"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "review = chunk_review(chunks[12][0])\n",
    "\n",
    "review"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "import instructor\n",
    "from openai import AsyncOpenAI\n",
    "\n",
    "\n",
    "openai = instructor.from_openai(AsyncOpenAI())\n",
    "\n",
    "\n",
    "async def as_chunk_review(chunk: Chunk)-> ReviewedChunk:\n",
    "    print(chunk[0].text)\n",
    "    prompt  = f\"\"\"\n",
    "        the chunks will be stored.\n",
    "        You are a data annotator, your task is to review a corpus of data consisting of chunks from a source document.\n",
    "        Your perfomance is critical as it ensures that bad or irrelevant data doesn't flood the vector database where\n",
    "        the chunks will be stored\n",
    "        \n",
    "        here's the chunk text: <Text>{chunk[0].text}</Text>\n",
    "        \n",
    "        <Guidelines>\n",
    "            What makes text irrelevant:\n",
    "            - the text is too short and doesn't develop any idea or argument\n",
    "            - contains only numbers\n",
    "            - text that appears to be a caption for an image\n",
    "            - text that appears to only contain people's names\n",
    "            - text that is out of context\n",
    "            - as a rule of thumb text smaller than 80 tokens is not relevant\n",
    "            \n",
    "            What makes text relevant: \n",
    "             - it does present findings, ideas, arguments,\n",
    "             - it describes something and contains semantic meaning, valuable ideas about a topic can infered from it\n",
    "        </Guidelines>\n",
    "    \"\"\"\n",
    "    \n",
    "    res = await openai.chat.completions.create(\n",
    "        model=\"o3-mini\",\n",
    "        response_model=ChunkReview,\n",
    "        messages=[\n",
    "            {\"role\": \"assistant\", \"content\": prompt}\n",
    "        ]\n",
    "    )\n",
    "    \n",
    "    return ReviewedChunk(**chunk[0].model_dump(), relevant=res.relevant)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Article\n",
      "Bacteriophage targeting of gut bacterium attenuates alcoholic liver disease\n",
      "https://doi.org/10.1038/s41586-019-1742-x Received: 9 April 2019 Accepted: 2 October 2019 Published online: 13 November 2019\n",
      "\n",
      "https://doi.org/10.1038/s41586-019-1742-x\n",
      "Received: 9 April 2019\n",
      "\n",
      "Accepted: 2 October 2019\n",
      "\n",
      "Published online: 13 November 2019\n",
      "Yi Duan1,2,29, Cristina Llorente1,2,29, Sonja Lang1, Katharina Brandl3, Huikuan Chu1, Lu Jiang1,2, Richard C. White4, Thomas H. Clarke4, Kevin Nguyen4, Manolito Torralba5, Yan Shao6, Jinyuan Liu7, Adriana Hernandez-Morales8, Lauren Lessor9, Imran R. Rahman10, Yukiko Miyamoto1, Melissa Ly11, Bei Gao1, Weizhong Sun1, Roman Kiesel1, Felix Hutmacher1, Suhan Lee1, Meritxell Ventura-Cots12, Francisco Bosques-Padilla13, Elizabeth C. Verna14, Juan G. Abraldes15, Robert S. Brown Jr16, Victor Vargas17,18,\n",
      "Jose Altamirano17, Juan Caballería18,19, Debbie L. Shawcross20, Samuel B. Ho1,2, Alexandre Louvet21, Michael R. Lucey22, Philippe Mathurin21, Guadalupe Garcia-Tsao23,24, Ramon Bataller12, Xin M. Tu7, Lars Eckmann1, Wilfred A. van der Donk10,25,26, Ry Young8,9, Trevor D. Lawley6, Peter Stärkel27, David Pride1,11,28, Derrick E. Fouts4 & Bernd Schnabl1,2,28*\n",
      "Chronic liver disease due to alcohol-use disorder contributes markedly to the global burden of disease and mortality1–3. Alcoholic hepatitis is a severe and life-threatening form of alcohol-associated liver disease. The gut microbiota promotes ethanol- induced liver disease in mice4, but little is known about the microbial factors that are responsible for this process. Here we identify cytolysin—a two-subunit exotoxin that is secreted by Enterococcus faecalis5,6—as a cause of hepatocyte death\n",
      "and liver injury. Compared with non-alcoholic individuals or patients with alcohol-use disorder, patients with alcoholic hepatitis have increased faecal numbers of E. faecalis. The presence of cytolysin-positive (cytolytic) E. faecalis correlated with the severity of liver disease and with mortality in patients with alcoholic hepatitis. Using humanized mice that were colonized with bacteria from the faeces of patients with alcoholic hepatitis, we investigated the therapeutic effects of\n",
      "bacteriophages that target cytolytic E. faecalis. We found that these bacteriophages decrease cytolysin in the liver and abolish ethanol-induced liver disease in humanized mice. Our findings link cytolytic E. faecalis with more severe clinical outcomes and increased mortality in patients with alcoholic hepatitis. We show that bacteriophages can specifically target cytolytic E. faecalis, which provides a method for precisely editing the intestinal microbiota. A clinical trial with a larger cohort\n",
      "is required to validate the relevance of our findings in humans, and to test whether this therapeutic approach is effective for patients with alcoholic hepatitis.\n",
      "The most severe form of alcohol-related liver disease is alcoholic hepa- titis; mortality ranges from 20% to 40% at 1–6 months, and as many as 75% of patients die within 90 days of a diagnosis of severe alcoholic hepatitis7–9. Therapy with corticosteroids is only marginally effective9. Early liver transplantation is the only curative therapy, but is offered only at select centres and to a limited group of patients10.\n",
      "Alcohol-related liver disease can be transmitted via faecal micro- biota4. We investigated the microorganisms and microbial factors that are responsible for this transmissible phenotype and for progression of alcohol-related liver disease.\n",
      "Cytolysin linked to increased mortality\n",
      "We performed 16S ribosomal RNA (rRNA) gene sequencing to deter- mine whether chronic alcohol use and alcoholic hepatitis are associated\n",
      "A list of affiliations appears at the end of the paper.\n",
      "with an altered composition of the faecal microbiota. Differences in faecal microbiota composition were noted in patients with alcohol-use disorder and alcoholic hepatitis, compared to subjects without alcohol- use disorder (controls) (Fig. 1a, Extended Data Fig. 1a, b, Supplementary Tables 1, 2). One substantial difference that we observed was an increase in the proportion of Enterococcus spp. in patients with alcoholic hepa- titis: in these patients, 5.59% of faecal bacteria were Enterococcus\n",
      "spp. compared with almost none in controls (0.023%; for comparison, 0.004% of all reads were Enterococcus spp. in the Human Microbiome Project) or patients with alcohol-use disorder (0.024%). Faecal sam- ples from patients with alcoholic hepatitis had about 2,700-fold more E. faecalis than samples from controls, as measured by quantitative PCR (qPCR) (Extended Data Fig. 1c), which is consistent with the 16S rRNA sequencing results. About 80% of patients with alcoholic hepatitis are positive for\n",
      "E. faecalis in their faeces (Extended Data Fig. 1d).\n",
      "Nature | www.nature.com | 1\n",
      "\n",
      "1\n",
      "Article\n",
      "Controls 0.2 Alcohol-use disorder Alcoholic hepatitis ) % 5 7 . 7 1 ( 2 C P 0.1 0 −0.1 –0.2 –0.1 0 0.1 0.2 PC1 (29.25%)\n",
      "m\n",
      "\n",
      "b\n",
      "100 Cytolysin-negative Cytolysin-positive ) % ( s t c e j b u S 80 60 40 20 0 Controls AUD Alcoholic\n",
      "m\n",
      "\n",
      "=\n",
      ") % ( l a v i v r u S 100 75 50 25 0 0 30 Cytolysin-negative Cytolysin-positive 60 90 120 150 180 Time (d)\n",
      "d\n",
      "0.15 Cytolysin-negative Cytolysin-positive\n",
      "@\n",
      "\n",
      "@\n",
      "Fig. 1 | E. faecalis cytolysin is associated with mortality in patients with alcoholic hepatitis. a, 16S rRNA sequencing of faecal samples from controls (n = 14), patients with alcohol-use disorder (n = 43) or alcoholic hepatitis (n = 75). We use principal coordinate analysis (PCoA) based on Jaccard dissimilarity matrices to show β-diversity among groups at the genus level. The composition of faecal microbiota was significantly different between each group (P < 0.01). b, Percentage of subjects\n",
      "with faecal samples that were positive for both cylLL and cylLS DNA sequences (cytolysin-positive), in controls (n = 25), patients with alcohol-use disorder (n = 38) or alcoholic hepatitis (n = 82), assessed by qPCR. Statistically significant differences were detected between controls and patients with alcoholic hepatitis (P < 0.01), and between patients with alcohol- use disorder and patients with alcoholic hepatitis (P < 0.001). c, Kaplan–Meier curve of survival of patients with alcoholic\n",
      "hepatitis whose faecal samples were cytolysin-positive (n = 25) or cytolysin-negative (n = 54) (P < 0.0001). d, Core genome single-nucleotide polymorphism (SNP) tree of E. faecalis strains isolated from patients with alcoholic hepatitis (n = 93 strains, from 24 patients), showing phylogenetic diversity of cytolysin-positive (red) E. faecalis. Genomically identical isolates from the same patient were combined, and are shown as a single dot. Scale bar represents the nucleotide substitutions per\n",
      "SNP site. P values are determined by permutational multivariate analysis of variance (PERMANOVA) followed by false discovery rate (FDR) procedures (a), two-sided Fisher’s exact test followed by FDR procedures (b) or two-sided log- rank (Mantel–Cox) test (c). The exact group size (n) and P values for each comparison are listed in Supplementary Table 10.\n",
      "The colonization of mice with E. faecalis induces mild hepatic stea- tosis and exacerbates ethanol-induced liver disease11, by mechanisms that are unclear. Cytolysin is a bacterial exotoxin (or bacteriocin) that is produced by E. faecalis12, and which contains two post-translationally modified peptides (CylLL′′ and CylLS′′) in its bioactive form6. The two peptides are encoded by two separate genes: cylLL and cylLS, respec- tively12. Cytolysin has lytic activity against not only Gram-positive\n",
      "bacte- ria, but also eukaryotic cells13. We detected cylLL and cylLS genomic DNA (cytolysin-positive) in faecal samples from 30% of patients with alco- holic hepatitis; none of the faecal samples from controls and only one sample from a patient with alcohol-use disorder was cytolysin-positive, as detected by qPCR (Fig. 1b). Importantly, 89% of cytolysin-positive patients with alcoholic hepatitis died within 180 days of admission, compared to only 3.8% of cytolysin-negative patients (P < 0.0001)\n",
      "(Fig. 1c). Among the cytolysin-positive patients, 72.2% (13 out of 18) died owing to liver failure (including complications related to liver\n",
      "2 | Nature | www.nature.com\n",
      "failure, such as gastrointestinal bleeding) (Supplementary Table 2). Infection was not associated with 30-day, 90-day or 180-day mortality (P = 0.403, 0.234 or 0.098) in patients with alcoholic hepatitis.\n",
      "Our univariate logistic and Cox regression of laboratory and clinical parameters found an association between the detection of cytolysin- encoding genes in faeces and the international normalized ratio (INR), platelet count, the model for end-stage liver disease (MELD) score, the sodium MELD score, the age, serum bilirubin, INR and serum creatinine (ABIC) score and death (Supplementary Table 3). In the multivariate Cox analysis, detection of cytolysin-encoding genes in faeces was associated with\n",
      "90-day (P = 0.004) and with 180-day mortality (P = 0.001) (Supple- mentary Table 3), even after we adjusted for the geographical origin of the patient, antibiotic treatment, platelet count, and creatinine, bilirubin and INR as components of the MELD score. We found no multicollinear- ity between the detection of faecal cytolysin-encoding genes and these cofactors (variance inflation factor < 1.6), which indicates that cytolysin is an independent predictor of mortality in patients with alcoholic\n",
      "hepatitis. When we performed receiver-operating-characteristic curve analysis for 90-day mortality, cytolysin had an area under the curve of 0.81, which was superior to other widely used predictors for mortality in clinical practice (Extended Data Fig. 1e). On the basis of our findings, we propose that the detection of cytolysin may be a prognostic factor for more severe liver- related outcomes and increased risk of death, and a stronger predictor of mortality than MELD, ABIC and the\n",
      "discriminant function score.\n",
      "To determine phylogeny of E. faecalis in patients with alcoholic hepatitis, we performed targeted culturing from stool samples. Whole- genome sequencing of 93 E. faecalis isolates revealed a broad phylo- genetic diversity of cytolysin-positive E. faecalis from patients with alcoholic hepatitis (Fig. 1d), which indicates that cytolysin production is a variable trait among E. faecalis isolates and that cytolysin is car- ried in mobile genetic elements, which include both chromosomally encoded\n",
      "pathogenicity islands and plasmids14. Detection of any other antimicrobial resistance genes or virulence genes in E. faecalis isolates did not correlate with disease severity or mortality in patients with alcoholic hepatitis (Supplementary Table 4).\n",
      "The total amount of faecal E. faecalis, or faecal E. faecalis positivity, did not correlate with disease severity or mortality in patients with alcoholic hepatitis (Supplementary Tables 5, 6). Cytolysin-positive and cytolysin-negative patients with alcoholic hepatitis had similar amounts of faecal E. faecalis (Extended Data Fig. 1f). Although there were differences in the composition of the gut microbiota in patients with alcoholic hepatitis from different geographical regions (Extended Data\n",
      "Fig. 1g), the proportion of cytolysin-positive patients, total amount of faecal E. faecalis, faecal E. faecalis positivity (Extended Data Fig. 1h–j), treatment and clinical outcomes (30-day and 90-day mortality) did not differ significantly among the regions or centres (Supplemen- tary Table 7). In addition, cirrhosis was not associated with cytolysin positivity, the total amount of faecal E. faecalis or faecal E. faecalis positivity in patients with alcoholic hepatitis (Extended Data Fig. 1k–m,\n",
      "Supplementary Tables 4–6). These results confirm our findings that the presence of cytolysin-producing E. faecalis rather than the total amount or presence of E. faecalis per se determines the severity of alcoholic hepatitis and mortality.\n",
      "Cytolysin and ethanol-induced liver disease\n",
      "To determine whether cytolysin contributes to liver damage mediated by E. faecalis, we gavaged mice with a cytolytic E. faecalis strain (FA2- 2(pAM714)) or a non-cytolytic E. faecalis strain (FA2-2(pAM771))5; the mice were then placed on a chronic–binge ethanol diet15. Compared to mice gavaged with phosphate-buffered saline (PBS), mice fed with ethanol after they were gavaged with cytolytic E. faecalis developed more severe liver injury as indicated by a higher level of alanine amino-\n",
      "transferase (ALT) (Extended Data Fig. 2a) and increased hepatic stea- tosis (Extended Data Fig. 2b, c). Mice that were fed ethanol after they\n",
      ") 1 – l U ( T L A f o 400 300 200 100 0 * *** *** ** Control Ethanol\n",
      "\n",
      "b e d i r e c y l g i r t c i t a p e H ) r e v i l g r e p g m ( t n e t n o 50 40 30 20 10 0 **** **** *** * c\n",
      "c Control Ethanol Alcoholic hepatitis cytolysin-positive Alcoholic hepatitis cytolysin-positive Alcoholic hepatitis cytolysin-negative Alcoholic hepatitis cytolysin-negative no. 1 no. 2 no. 3 no. 4\n",
      "e\n",
      "Cxcl1 e g n a h c d l o F 8 6 4 2 ** ** ** ** 0 Control Ethanol\n",
      "f\n",
      "Col1a1\n",
      "20 15 10 5 ** *** * * 0\n",
      "h\n",
      "\n",
      "LDH\n",
      ") % 40 * * ** ** ( y t i c i x o 30 20 t o t 10 y C 0 0 0 0 200 0 400 200 200 400 400 0 0 0 200 0 400 200 200 400 400 CylLS″ (nM) CylLL″ (nM) – + Ethanol\n",
      "B\n",
      "Alcoholic hepatitis cytolysin-positive no. 1\n",
      "B\n",
      "Alcoholic hepatitis cytolysin-positive no. 2\n",
      "B\n",
      "Alcoholic hepatitis cytolysin-negative no. 3\n",
      "B\n",
      "Alcoholic hepatitis cytolysin-negative no. 4\n",
      "\n",
      "e g n 8 6 Il1b * * ** ** a h c d l o F 4 2 0 Control Ethanol\n",
      "g\n",
      ") % ( e c i M 100 80 60 40 20 **** **** **** **** 0 ND ND ND ND ND ND Control Ethanol\n",
      "Fig. 2 | Transplantation of faeces from cytolysin-positive patients with alcoholic hepatitis exacerbates ethanol-induced liver disease in gnotobiotic mice. a–g, C57BL/6 germ-free mice were colonized with faeces from two cytolysin-positive and two cytolysin-negative patients with alcoholic hepatitis, and subjected to the chronic–binge feeding model. a, Serum levels of ALT. b, Hepatic triglyceride content. c, Representative sections of liver stained with haematoxylin and eosin (H & E). d–f,\n",
      "Hepatic levels of mRNAs that encode Il1b, Cxcl1 and Col1a1. g, Proportions of mice that were positive for cytolysin in the liver, measured by qPCR for cylLS. h, Lactate dehydrogenase (LDH) assay to measure cytotoxicity of hepatocytes isolated from mice that were fed an oral isocaloric control diet (five groups, left) or chronic–binge ethanol diet (five\n",
      "groups, right), and incubated with vehicle, CylLS′′, CylLL′′ or both of the cytolysin subunits at the indicated concentrations without (–) or with (+) ethanol (25 mM) for 3 h. The survival of hepatocytes was determined in three independent experiments. Scale bar, 100 μm. Results are expressed as mean ± s.e.m. (a, b, d–f, h). P values are determined by one-way analysis of variance (ANOVA) with Tukey’s post hoc test (a, b, d–f), two-sided Fisher’s exact test followed by FDR procedures (g) or\n",
      "two-way ANOVA with Tukey’s post hoc test (h). All results were generated from at least three independent replicates. The exact group size (n) and P values for each comparison are listed in Supplementary Table 10. *P < 0.05, **P < 0.01, ***P < 0.001, ****P < 0.0001.\n",
      "were gavaged with cytolytic E. faecalis also had more liver inflamma- tion with higher expression levels of mRNAs that encode inflamma- tory cytokines and chemokines (Il1b, Cxcl1 and Cxcl2) (Extended Data Fig. 2d–f), compared with mice given PBS. Mice that were fed ethanol after they were gavaged with non-cytolytic E. faecalis had significantly less ethanol-induced liver injury, steatosis and inflammation (Extended Data Fig. 2a–f) and longer survival times (Extended Data Fig. 2g), as compared\n",
      "with mice that were fed ethanol after they were administered with cytolytic E. faecalis.\n",
      "To explore the mechanism of cytolysin-associated liver damage, we measured cytolysin in the liver. CylLS was significantly increased in the liver of mice given cytolytic E. faecalis but not in the liver of mice that were not given E. faecalis or of mice gavaged with non-cyto- lytic E. faecalis after chronic ethanol administration (Extended Data Fig. 2h). E. faecalis was detectable in the liver of mice given cytolytic and\n",
      "non-cytolytic E. faecalis and fed an ethanol diet, but not when mice were fed an isocaloric (control) diet (Extended Data Fig. 2i); this indicates that ethanol-induced changes in the gut barrier are necessary for the translocation of cytolytic E. faecalis from the intestine to the liver. The livers of ethanol-fed mice that were given cytolytic or non-cytolytic E. faecalis had positive E. faecalis cultures (Extended Data Fig. 2j). We observed an increased intestinal permeability in ethanol-fed\n",
      "mice com- pared with mice fed with an isocaloric diet, but this was independent of gavaging cytolytic or non-cytolytic E. faecalis after chronic ethanol administration (Extended Data Fig. 2k), indicating that cytolysin does not affect intestinal barrier function.\n",
      "Administration of cytolytic or non-cytolytic E. faecalis to mice did not significantly change the composition of the intestinal microbiota, as shown by 16S rRNA gene sequencing (Extended Data Fig. 2l). Cytolytic E. faecalis did not affect intestinal absorption or hepatic metabolism\n",
      "Nature | www.nature.com | 3\n",
      "\n",
      "3\n",
      "Article\n",
      "a Ef5.1 Ef5.2 Ef5.3 Ef5.4 Ef2.1 Ef2.2 Ef2.3\n",
      "b\n",
      "s l e v e l m u r e S ) 1 – l U ( T L A f o 400 300 200 100 0 Patient no. 5 Patient no. 2 * Patient no. 5 ** Patient no. 2\n",
      "\n",
      ") r e v i l g r e p g m ( t n e t n o c 50 40 30 20 10 0 Patient no. 5 Patient no. 2 * Patient no. 5 ** Patient no. 2 C. crescentus phages Cytolysin-positive E. faecalis phages C. crescentus phages Cytolysin-positive E. faecalis phages\n",
      "m\n",
      "\n",
      "m\n",
      "\n",
      "m\n",
      "\n",
      "=\n",
      "d Control Ethanol C. crescentus phages Cytolysin-positive E. faecalis phages C. crescentus phages Cytolysin-positive E. faecalis phages\n",
      "Patient no. 5\n",
      "\n",
      "Patient no. 2\n",
      "f\n",
      "\n",
      "e g n a h c d l o F\n",
      "12\n",
      "RN ov®O\n",
      "8\n",
      "\n",
      "6\n",
      "\n",
      "4\n",
      "\n",
      "2\n",
      "0\n",
      "Patient no. 5\n",
      "Cxcl1\n",
      "**\n",
      "Patient no. 2\n",
      "\n",
      "Patient no. 5\n",
      "**\n",
      "\n",
      "]\n",
      "Patient no. 2\n",
      "\n",
      "g Col1a1 e g n a h c d l o F 18 15 12 9 6 3 0 * * Patient no. 5 Patient no. 2 Patient no. 5 Patient no. 2\n",
      "Control\n",
      "Ethanol\n",
      "\n",
      "e g n a h c d l o F 12 10 8 6 4 2 0 Il1b * *** Patient no. 5 Patient no. 2 Patient no. 5 Patient no. 2 Control Ethanol\n",
      "h Liver cylLS ) % ( e c i M 100 80 60 40 20 0 ND ND ND ND ** ** Patient no. 5 Patient no. 2 Patient no. 5 Patient no. 2\n",
      "Fig. 3 | Phage therapy against cytolytic E. faecalis abolishes ethanol-induced liver disease in gnotobiotic mice. a, Transmission electron microscopy revealed that the phages we isolated were either siphophages (Ef5.1, Ef5.2, Ef5.3, Ef5.4 and Ef2.2) or myophages (Ef2.1 and Ef2.3). Scale bar, 50 nm. b–h, C57BL/6 germ-free mice were colonized with faeces from two cytolysin- positive patients with alcoholic hepatitis (faeces from one of these patients were also used in Fig. 2) and subjected to the\n",
      "chronic–binge feeding model, gavaged with control phages against C. crescentus (1010 plaque-forming units (PFUs)) or a cocktail of three or four different phages that target cytolytic E. faecalis (1010 PFUs), 1 day before an ethanol binge. b, Serum levels of ALT.\n",
      "c, Hepatic triglyceride content. d, Representative H & E-stained liver sections. Scale bar, 100 μm. e–g, Hepatic levels of mRNAs that encode Il1b, Cxcl1 and Col1a1. h, Proportions of mice that were positive for cytolysin in the liver, measured by qPCR for cylLS. Results are expressed as mean ± s.e.m. (b, c, e–g). P values are determined by two-way ANOVA with Tukey’s post hoc test (b, c, e–g) or two-sided Fisher’s exact test followed by FDR procedures (h). All results are generated from at least\n",
      "three independent replicates. The exact group size (n) and P values for each comparison are listed in Supplementary Table 10. *P < 0.05, **P < 0.01, *** P < 0.001.\n",
      "of ethanol, as indicated by serum levels of ethanol and hepatic levels of Adh1 and Cyp2e1 mRNAs (which encode the two primary enzymes that metabolize ethanol in the liver) (Extended Data Fig. 2m, n). These results indicate that E. faecalis that produce cytolysin promote ethanol- induced liver disease in mice.\n",
      "To extend our findings to humans, we colonized germ-free mice with faeces from cytolysin-positive and cytolysin-negative patients with alcoholic hepatitis (Supplementary Table 8). Consistent with our findings from mice colonized with cytolytic E. faecalis, gnotobi- otic C57BL/6 mice colonized with faeces from two cytolysin-positive patients developed more severe ethanol-induced liver injury, steatosis, inflammation and fibrosis than mice given faeces from two cytolysin- negative patients (Fig.\n",
      "2a–f, Extended Data Fig. 3a–d). Transplantation of faeces from cytolysin-positive patients reduced the survival time of the mice (Extended Data Fig. 3e) and increased translocation of\n",
      "4 | Nature | www.nature.com\n",
      "cytolytic E. faecalis to the liver after ethanol administration (Fig. 2g). The overall composition of the intestinal microbiota was not differ- ent between mice fed the control diet and colonized with faeces from cytolysin-positive or cytolysin-negative donors with alcoholic hepatitis, as shown by 16S rRNA gene sequencing. Mice transplanted with faeces from one of the cytolysin-positive patients with alcoholic hepatitis (patient no. 2) showed a microbiota that was significantly different from\n",
      "that of the other mouse groups after ethanol administration (Extended Data Fig. 3f). Non-cytolytic E. faecalis was not detected in stool samples from donors with cytolytic E. faecalis (Extended Data Fig. 3g). We did not observe differences in intestinal absorp- tion or hepatic metabolism of ethanol between mice colonized with faeces from cytolysin-positive versus cytolysin-negative patients (Extended Data Fig. 3h, i). Together, these results provide further evi- dence that cytolysin promotes\n",
      "ethanol-induced liver disease.\n",
      "a\n",
      "Ef6.1 Ef6.2 Ef6.3 Ef 6.4 Ef7.1 Ef7.2 Ef7.3 Ef 7.4\n",
      "b\n",
      "\n",
      "s l e v e l m u r e S\n",
      "400 300 200 100 0 Patient no. 6 Patient no. 7 Patient no. 6 Patient no. 7 Control Ethanol\n",
      "– l U ( T L A\n",
      "\n",
      "( T L A f\n",
      ") r e v i l g r e p g m ( t n e t n o c 50 40 30 20 10 0 Patient no. 6 Patient no. 7 Patient no. 6 Patient no. 7 Control Ethanol\n",
      "d\n",
      "Control Ethanol C. crescentus phages Cytolysin-negative E. faecalis phages C. crescentus phages Cytolysin-negative E. faecalis phages\n",
      "Patient no. 6\n",
      "\n",
      "Patient no. 7\n",
      "e g n a h c d l o F\n",
      "\n",
      "N w A O\n",
      "\n",
      "1\n",
      "0\n",
      "Patient no. 6\n",
      "Cxcl1\n",
      "Patient\n",
      "\n",
      "&6 - - Patient 6\n",
      "g Col1a1 h 6 4 2 0 Patient Patient Patient Patient no. 6 no. 7 no. 6 no. 7 Control Ethanol\n",
      "Control\n",
      "Ethanol\n",
      "m\n",
      "C. crescentus phages\n",
      "=\n",
      "Cytolysin-negative E. faecalis phages\n",
      "m\n",
      "C. crescentus phages\n",
      "m\n",
      "Cytolysin-negative E. faecalis phages\n",
      "e g n a h c d l o F 5 4 3 2 1 0 Il1b Patient no. 6 Patient no. 7 Patient no. 6 Patient no. 7 Control Ethanol\n",
      "\n",
      "s u c 1010 * * c o c o r e t n E l a c ) 1 – g U F C ( 109 108 107 106 e a F 105 Patient Patient Patient Patient no. 6 no. 7 no. 6 no. 7\n",
      "Fig. 4 | Phages that target non-cytolytic E. faecalis do not reduce ethanol- induced liver disease in gnotobiotic mice. a, Transmission electron microscopy revealed that the phages we isolated were either podophages (Ef6.2, Ef6.3, Ef7.2, Ef7.3 and Ef7.4) or siphophages (Ef6.1, Ef6.4 and Ef7.1). Scale bar, 50 nm. b–h, C57BL/6 germ-free mice were colonized with faeces from two cytolysin-negative patients with alcoholic hepatitis and subjected to the chronic–binge feeding model, gavaged with\n",
      "control phages against C. crescentus (1010 PFUs) or a cocktail of four different phages that target non-\n",
      "To determine the mechanism by which cytolysin increases liver dis- ease, we isolated hepatocytes from mice fed ethanol or control diets, and stimulated them with pure bioactive cytolysin peptides (CylLL′′ and CylLS′′)6. Incubation of the primary mouse hepatocytes with the two cytolysin subunits caused a dose-dependent increase in cell death compared to hepatocytes that were incubated with vehicle or with one subunit only (Fig. 2h). When we isolated hepatocytes from ethanol-fed mice and then\n",
      "incubated these hepatocytes with ethanol, we did not observe increased levels of cytolysin-induced cell death compared to hepatocytes isolated from mice on the control diet, which indicates that cytolysin-induced hepatocyte cell death was independent of ethanol. The cytotoxic effects of cytolysin are possibly mediated by pore forma- tion, resulting in cell lysis14.\n",
      "Bacteriophage treatment in liver disease\n",
      "To further demonstrate the potential causative role of cytolytic E. faecalis for the development of ethanol-induced steatohepatitis,\n",
      "cytolytic E. faecalis (1010 PFUs), 1 day before an ethanol binge. b, Serum levels of ALT. c, Hepatic triglyceride content. d, Representative H & E-stained liver sections. Scale bar, 100 μm. e–g, Hepatic levels of mRNAs that encode Il1b, Cxcl1 and Col1a1. h, Faecal colony-forming units (CFUs) of Enterococcus. Results are expressed as mean ± s.e.m. (b, c, e–h). P values are determined by two-way ANOVA with Tukey’s post hoc test (b, c, e–h). All results were generated from at least three\n",
      "independent replicates. The exact group size (n) and P values for each comparison are listed in Supplementary Table 10. *P < 0.05.\n",
      "we investigated the effects of treatment with bacteriophages (here- after, phages). Phages are ubiquitous in bacteria-rich environments, including the gut16. E. faecalis phages that are highly strain-specific can be isolated17, which potentially makes the direct editing of gut microbiota feasible. It has previously been shown that Atp4aSl/Sl mice, which lack gastric acid, have overgrowth of intestinal enterococci, which is associated with increased susceptibility to alcohol-induced\n",
      "steatohepatitis11. The gavaging of wild-type mice with an E. faecalis strain isolated from Atp4aSl/Sl mice led to increased ethanol-induced steatohepatitis11. We found that this same E. faecalis strain expressed cytolysin. We then isolated four distinct phages from sewage water. These phages lyse the cytolytic E. faecalis strain isolated from Atp4aSl/Sl mice. All four phages were podophages of the virulent Picovirinae group (Extended Data Fig. 4). Atp4aSl/Sl mice and their wild-type littermates\n",
      "were then placed on the chronic–binge ethanol diet and gavaged with the lytic phage cocktail. Phages directed against Caulobacter crescen- tus, a bacterium that is present in freshwater lakes and streams18 but that does not colonize humans or rodents19,20, were used as controls.\n",
      "Nature | www.nature.com | 5\n",
      "Article\n",
      "Compared to Atp4aSl/Sl mice gavaged with control phages or vehicle, Atp4aSl/Sl mice gavaged with phages that target cytolytic E. faecalis had less severe liver injury, steatosis and inflammation after chronic ethanol feeding (Extended Data Fig. 5a–f). Administration of E. faecalis phages significantly reduced levels of cytolysin in the liver (Extended Data Fig. 5g) as well as faecal amounts of Enterococcus (Extended Data Fig. 5h). Phage administration did not affect the overall composition of\n",
      "the faecal microbiome, intestinal absorption or hepatic metabolism of ethanol (Extended Data Fig. 5i–k).\n",
      "To develop a therapeutic approach to precisely edit the intestinal microbiota, we cultured cytolytic E. faecalis strains from the faecal samples of patients with alcoholic hepatitis. We then isolated lytic phages from sewage water against these cytolytic E. faecalis strains; these phages had siphophage or myophage morphology (Fig. 3a, Extended Data Fig. 6). Gnotobiotic mice were colonized with faeces from two cytolysin-positive patients with alcoholic hepatitis (Sup- plementary Table 8) and\n",
      "given three or four different—but patient- specific—lytic phages against cytolytic E. faecalis. The phages against cytolytic E. faecalis abolished ethanol-induced liver injury and steatosis, as shown by lower levels of ALT, lower percentages of hepatic cells positive for terminal deoxynucleotide transferase-mediated dUTP nick-end labelling, and lower levels of hepatic triglycerides and oil red O-staining (Fig. 3b–d, Extended Data Fig. 7a, b), as well as by decreased hepatic levels of Il1b,\n",
      "Cxcl1, Cxcl2, Col1a1 and Acta2 mRNAs, and reduced hepatic levels of cylLS, as compared with mice given control phages (against C. crescentus) (Fig. 3e–h, Extended Data Fig. 7c, d). Treatment with phages against cytolytic E. faecalis also reduced faecal amounts of Enterococcus (Extended Data Fig. 7e) without affecting the overall composition of the gut microbiota (Extended Data Fig. 7f). Intestinal absorption of ethanol and hepatic metabolism were similar in all groups (Extended Data Fig. 7g, h).\n",
      "To demonstrate that the effect of phage treatment occurs via the targeting of cytolysin-positive E. faecalis, rather than a reduction in cytolysin-negative E. faecalis, we colonized gnotobiotic mice with faeces from cytolysin-negative patients with alcoholic hepatitis (Sup- plementary Table 8). Phages against non-cytolytic E. faecalis from patients were isolated from sewage water; they had siphophage or podophage morphology (Fig. 4a, Extended Data Fig. 8). These phages did not reduce features of\n",
      "ethanol-induced liver disease compared with control phages (Fig. 4b–g, Extended Data Fig. 9a–h), despite the reduc- tion of faecal Enterococcus (Fig. 4h). Our findings indicate that treat- ment with lytic phages can selectively attenuate the ethanol-induced liver disease caused by cytolysin-positive E. faecalis in humanized mice.\n",
      "Discussion\n",
      "Phage-based therapies have predominantly been studied in patients with bacterial infections in the gastrointestinal tract21–23, urinary tract24,25 and other organ systems26–28. The results of these studies— although mixed in terms of efficacy—strongly suggest that phage treat- ment offers a safe alternative to antibiotics26,27. However, safety studies are required for complex populations (such as patients with alcoholic hepatitis), because phages can induce a strong immune reaction29. Further\n",
      "work is required to determine whether phages that target cyto- lytic E. faecalis might be used to treat patients with alcoholic hepatitis, a life-threatening disease that at present has no effective treatment. Eradication of this specific bacterial strain might produce better out- comes than current treatments, and environmental sources can be used to easily isolate phages that target cytolysin-positive E. faecalis. Here we provide an example of the efficacy of approaches based on phages in mice\n",
      "for a disease that is not considered a classic infectious disease. Our data also suggest that cytolysin may be used as a predictive biomarker of severe alcoholic hepatitis; an independent, prospective\n",
      "cohort is therefore needed to validate cytolysin as a biomarker, and to extend the phage findings in mice to human patients.\n",
      "Online content\n",
      "Any methods, additional references, Nature Research reporting sum- maries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author con- tributions and competing interests; and statements of data and code availability are available at https://doi.org/10.1038/s41586-019-1742-x.\n",
      "1. Lozano, R. et al. Global and regional mortality from 235 causes of death for 20 age groups in 1990 and 2010: a systematic analysis for the Global Burden of Disease Study 2010. Lancet 380, 2095–2128 (2012).\n",
      "2. Lee, B. P., Vittinghoff, E., Dodge, J. L., Cullaro, G. & Terrault, N. A. National trends and long- term outcomes of liver transplant for alcohol-associated liver disease in the United States. JAMA Intern. Med. 179, 340–348 (2019).\n",
      "\n",
      "3. Rehm, J. et al. Burden of disease associated with alcohol use disorders in the United States. Alcohol. Clin. Exp. Res. 38, 1068–1077 (2014).\n",
      "4. Llopis, M. et al. Intestinal microbiota contributes to individual susceptibility to alcoholic liver disease. Gut 65, 830–839 (2016).\n",
      "5. Ike, Y., Clewell, D. B., Segarra, R. A. & Gilmore, M. S. Genetic analysis of the pAD1 hemolysin/bacteriocin determinant in Enterococcus faecalis: Tn917 insertional mutagenesis and cloning. J. Bacteriol. 172, 155–163 (1990).\n",
      "6. Tang, W. & van der Donk, W. A. The sequence of the enterococcal cytolysin imparts unusual lanthionine stereochemistry. Nat. Chem. Biol. 9, 157–159 (2013).\n",
      "7. Maddrey, W. C. et al. Corticosteroid therapy of alcoholic hepatitis. Gastroenterology 75, 193–199 (1978).\n",
      "\n",
      "8. Dominguez, M. et al. A new scoring system for prognostic stratification of patients with alcoholic hepatitis. Am. J. Gastroenterol. 103, 2747–2756 (2008).\n",
      "\n",
      "9. Thursz, M. R. et al. Prednisolone or pentoxifylline for alcoholic hepatitis. N. Engl. J. Med. 372, 1619–1628 (2015).\n",
      "\n",
      "10. Mathurin, P. & Lucey, M. R. Management of alcoholic hepatitis. J. Hepatol. 56, S39–S45 (2012).\n",
      "11. Llorente, C. et al. Gastric acid suppression promotes alcoholic liver disease by inducing overgrowth of intestinal Enterococcus. Nat. Commun. 8, 837 (2017).\n",
      "12. Gilmore, M. S. et al. Genetic structure of the Enterococcus faecalis plasmid pAD1- encoded cytolytic toxin system and its relationship to lantibiotic determinants. J. Bacteriol. 176, 7335–7344 (1994).\n",
      "\n",
      "13. Cox, C. R., Coburn, P. S. & Gilmore, M. S. Enterococcal cytolysin: a novel two component peptide system that serves as a bacterial defense against eukaryotic and prokaryotic cells. Curr. Protein Pept. Sci. 6, 77–84 (2005).\n",
      "14. Van Tyne, D., Martin, M. J. & Gilmore, M. S. Structure, function, and biology of the Enterococcus faecalis cytolysin. Toxins 5, 895–911 (2013).\n",
      "15. Bertola, A., Mathews, S., Ki, S. H., Wang, H. & Gao, B. Mouse model of chronic and binge ethanol feeding (the NIAAA model). Nat. Protoc. 8, 627–637 (2013).\n",
      "16. Ogilvie, L. A. & Jones, B. V. The human gut virome: a multifaceted majority. Front. Microbiol. 6, 918 (2015).\n",
      "17. Chatterjee, A. et al. Bacteriophage resistance alters antibiotic-mediated intestinal expansion of enterococci. Infect. Immun. 87, e00085-19 (2019).\n",
      "\n",
      "18. Poindexter, J. S. Biological properties and classification of the Caulobacter group. Bacteriol. Rev. 28, 231–295 (1964).\n",
      "19. Shin, J. et al. Analysis of the mouse gut microbiome using full-length 16S rRNA amplicon sequencing. Sci. Rep. 6, 29681 (2016).\n",
      "20. Human Microbiome Project Consortium. Structure, function and diversity of the healthy human microbiome. Nature 486, 207–214 (2012).\n",
      "21. Marcuk, L. M. et al. Clinical studies of the use of bacteriophage in the treatment of cholera. Bull. World Health Organ. 45, 77–83 (1971).\n",
      "\n",
      "22. Sarker, S. A. et al. Oral phage therapy of acute bacterial diarrhea with two coliphage preparations: a randomized trial in children from Bangladesh. EBioMedicine 4, 124–137 (2016).\n",
      "23. Dalmasso, M., Hill, C. & Ross, R. P. Exploiting gut bacteriophages for human health. Trends Microbiol. 22, 399–405 (2014).\n",
      "\n",
      "24. Ujmajuridze, A. et al. Adapted bacteriophages for treating urinary tract infections. Front. Microbiol. 9, 1832 (2018).\n",
      "\n",
      "25. Khawaldeh, A. et al. Bacteriophage therapy for refractory Pseudomonas aeruginosa urinary tract infection. J. Med. Microbiol. 60, 1697–1700 (2011).\n",
      "26. Schooley, R. T. et al. Development and use of personalized bacteriophage-based therapeutic cocktails to treat a patient with a disseminated resistant Acinetobacter baumannii infection. Antimicrob. Agents Chemother. 61, e00954-17 (2017).\n",
      "\n",
      "27. Dedrick, R. M. et al. Engineered bacteriophages for treatment of a patient with a disseminated drug-resistant Mycobacterium abscessus. Nat. Med. 25, 730–733 (2019).\n",
      "28. Fish, R. et al. Compassionate use of bacteriophage therapy for foot ulcer treatment as an effective step for moving toward clinical trials. Methods Mol. Biol. 1693, 159–170 (2018).\n",
      "effective step for moving toward clinical trials. Methods Mol. Biol. 1693, 159–170 (2018).\n",
      "\n",
      "29. Górski, A. et al. Phages and immunomodulation. Future Microbiol. 12, 905–914 (2017).\n",
      "Publisher’s note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\n",
      "\n",
      "© The Author(s), under exclusive licence to Springer Nature Limited 2019\n",
      "6 | Nature | www.nature.com\n",
      "1Department of Medicine, University of California San Diego, La Jolla, CA, USA. 2Department of Medicine, VA San Diego Healthcare System, San Diego, CA, USA. 3Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California San Diego, La Jolla, CA, USA. 4J. Craig Venter Institute, Rockville, MD, USA. 5J. Craig Venter Institute, La Jolla, CA, USA. 6Host–Microbiota Interactions Laboratory, Wellcome Sanger Institute, Hinxton, UK. 7Division of Biostatistics and Bioinformatics,\n",
      "Department of Family Medicine and Public Health, University of California San Diego, La Jolla, CA, USA. 8Department of Biochemistry and Biophysics, Texas A & M University, College Station, TX, USA. 9Center for Phage Technology, Texas A & M AgriLife Research and Texas A & M University, College Station, TX, USA. 10Department of Biochemistry, University of Illinois at Urbana-Champaign, Urbana, IL, USA. 11Department of Pathology, University of California San Diego, La Jolla, CA, USA. 12Division of\n",
      "Gastroenterology, Hepatology and Nutrition, Department of Medicine, University of Pittsburgh Medical Center, Pittsburgh Liver Research Center, Pittsburgh, PA, USA. 13Hospital Universitario, Departamento de Gastroenterología, Universidad Autonoma de Nuevo Leon, Monterrey, Mexico. 14Division of Digestive and Liver Diseases, Department of Medicine, Columbia University College of Physicians and Surgeons, New York, NY, USA. 15Department of Medicine, University of Alberta, Edmonton, Alberta, Canada.\n",
      "16Division of Gastroenterology and Hepatology, Weill Cornell Medical College, New York, NY, USA. 17Liver Unit, Hospital Universitari Vall d’Hebron, Universitat Autònoma de Barcelona, Barcelona, Spain. 18Centro de Investigación en Red de Enfermedades Hepáticas y Digestivas (CIBEREHD), Barcelona, Spain. 19Liver Unit, Hospital Clinic, Barcelona, Spain. 20Liver Sciences, Department of Inflammation Biology, School of Infectious Diseases and Microbial Sciences, King’s College London, London, UK.\n",
      "21Service des Maladies de L’appareil Digestif et Unité INSERM, Hôpital Huriez, Lille, France. 22Division of Gastroenterology and Hepatology, Department of Medicine, University of Wisconsin School of Medicine and Public Health, Madison, WI, USA. 23Section of Digestive Diseases, Yale University School of Medicine, New Haven, CT, USA. 24Section of Digestive Diseases, VA Connecticut Healthcare System, West Haven, CT, USA. 25Department of Chemistry, University of Illinois at Urbana-Champaign, Urbana,\n",
      "IL, USA. 26Howard Hughes Medical Institute, University of Illinois at Urbana-Champaign, Urbana, IL, USA. 27St Luc University Hospital, Université Catholique de Louvain, Brussels, Belgium. 28Center for Innovative Phage Applications and Therapeutics, University of California San Diego, La Jolla, CA, USA. 29These authors contributed equally: Yi Duan, Cristina Llorente. *e-mail: beschnabl@ucsd.edu\n",
      "Nature | www.nature.com | 7\n",
      "\n",
      "7\n",
      "Methods\n",
      "\n",
      "Methods\n",
      "No statistical methods were used to predetermine sample size. The experiments were not randomized and investigators were not blinded to allocation during experiments and outcome assessment.\n",
      "Patient cohorts\n",
      "\n",
      "Patient cohorts have previously been described* 2, We evaluated\n",
      "26 subjects without alcohol-use disorder (controls; social drinkers consuming less than 20 g/day), 44 patients with alcohol-use disorder and 88 patients with alcoholic hepatitis. Patients with alcohol-use disorder fulfilling the DSM IV criteria33 of alcohol dependence and with active alcohol consumption (self-reported >60 g/day) presented with various stages of liver disease (21% had advanced F3/4 fibrosis based on fibrosis-4 index) (Supplementary Table 1). Patients with alcohol-use disorder\n",
      "were recruited from an alcohol withdrawal unit in San Diego and Brussels, where they followed a detoxification and rehabilitation programme. At admission to the hospital, a complete medication and medical history was taken, and a complete physical examination was performed, including collection of bio-specimens, basic demographic data (such as age, gender, weight and height) and self-reported daily alcohol consumption. Patients were actively drink- ing until the day of admission. Controls or\n",
      "patients with alcohol-use disorder did not take antibiotics or immunosuppressive medication during the two months preceding enrolment. Other exclusion criteria were diabetes, inflammatory bowel disease, known liver disease of any other aetiology, and clinically important cardiovascular, pulmonary or renal co-morbidities. Patients with alcoholic hepatitis were enrolled from the InTeam Consortium (ClinicalTrials.gov identifier number: NCT02075918) from centres in the USA, Mexico, UK, France and\n",
      "Spain. Inclusion criteria for this study were active alcohol abuse (>50 g/day for men and >40 g/day for women) in the past 3 months, aspartate aminotransferase (AST) > ALT and total bilirubin >3 mg/dl in the past 3 months, and a liver biopsy and/or clinical picture consistent with alco- holic hepatitis. Exclusion criteria were autoimmune liver disease (ANA >1/320), chronic viral hepatitis, hepatocellular carcinoma, complete portal vein thrombosis, extrahepatic terminal disease, pregnancy and a\n",
      "lack of signed informed consent. In all patients, the clinical picture was consistent with alcoholic hepatitis and in patients who underwent liver biopsy, the histology was consistent with the diagnosis of alco- holic hepatitis. Liver biopsies were only done if clinically indicated as part of routine clinical care for diagnostic purposes of alcoholic hepatitis. Bio-specimens were collected during their admission to the hospital. The median time of specimen collection was 4 days fol- lowing\n",
      "admission to the hospital (range 0–24, n = 82). For one patient who underwent liver transplantation, the transplantation date was considered as date of death. Patients were censored at the time point at which they were last seen alive. The baseline characteristics are shown in Supplementary Tables 1, 2. Faecal 16S rRNA sequencing, Enterococcus culture and qPCR were performed. The MELD score, ABIC score and discriminant function were calculated from all alcoholic hepatitis patients from whom\n",
      "respective laboratory values were avail- able. The protocol was approved by the Ethics Committee of Hôpital Huriez, Universidad Autonoma de Nuevo Leon, Hospital Universitari Vall d’Hebron, King’s College London, Yale University, University of North Carolina at Chapel Hill, Weill Cornell Medical College, Columbia University, University of Wisconsin, VA San Diego Healthcare System, University of California San Diego (UCSD) and Université Catholique de Louvain. Patients were enrolled after written\n",
      "informed consent was obtained from each patient.\n",
      "Mice\n",
      "\n",
      "C57BL/6 mice were purchased from Charles River and used in Fig. 2h and Extended Data Fig. 2. C57BL/6 germ-free mice were bred at UCSD and used in Figs. 2a–g, 3, 4, Extended Data Figs. 3, 7 and 9. Sublytic Atp4aSl/Sl\n",
      "8 | Nature | www.nature.com\n",
      "mice on a C57BL/6 background have previously been described11,34 and heterozygous mice were used for breeding; sublytic Atp4aSl/Sl littermate mice and their wild-type littermates were used in Extended Data Fig. 5.\n",
      "Female and male mice (age of 9–12 weeks) were placed on a chronic– binge ethanol diet (NIAAA model) as previously described15. Mice were fed with Lieber–DeCarli diet and the caloric intake from ethanol was 0% on days 1–5 and 36% from day 6 until the end of the study period. At day 16, mice were gavaged with a single dose of ethanol (5 g/kg body weight) in the early morning and killed 9 h later. Pair-fed control mice received a diet with an isocaloric substitution of dextrose.\n",
      "Stool samples from patients with alcoholic hepatitis (Fig. 1) were used for faecal transplantation in germ-free mice. Mice were gavaged with 100 μl of stool samples (1 g stool dissolved in 30 ml Luria–Bertani (LB) medium containing 15% glycerol under anaerobic conditions), starting at an age of 5–6 weeks and repeated 2 weeks later. Two weeks after the second gavage, mice were placed on the ethanol or control (isocaloric) diet.\n",
      "In studies of the effects of cytolysin, 5 × 108 CFUs of a cytolytic E. fae- calis strain (FA2-2(pAM714)), a non-cytolytic E. faecalis strain (FA2- 2(pAM771))5 (E. faecalis Δcytolysin) (kindly provided by M. S. Gilmore), or PBS (vehicle control) were fed to mice by gavage every third day, starting from day 6 through day 15 of ethanol feeding. Administration every third day was necessary, given that E. faecalis does not colonize mice11 (Extended Data Fig. 2o). To determine the effect of phage\n",
      "treat- ment, 1010 PFUs of E. faecalis phages (or C. crescentus phage phiCbK as control)35 were gavaged to the mice 1 day before the ethanol binge (at day 16). All animal studies were reviewed and approved by the Institu- tional Animal Care and Use Committee of UCSD.\n",
      "Bacteriophage isolation and amplification\n",
      "The E. faecalis strain from Atp4aSl/Sl mice faeces has previously been isolated11 and was used to isolate phages Efmus1, Efmus2, Efmus3 and Efmus4 (phages specific to the E. faecalis strain isolated from mouse faeces were named as Efmus with a number (Ef for E. faecalis, mus for mouse, digit for isolation order). E. faecalis strains from human stool samples were isolated using methods described below, and the cor- responding phages were named as Ef with patient number plus a digit (Ef for E.\n",
      "faecalis, last digit for isolation order). All E. faecalis strains were grown statically in brain–heart infusion (BHI) broth or on BHI agar at 37 °C. C. crescentus phage phiCbK was purified as previously described35.\n",
      "E. faecalis phages were isolated from untreated raw sewage water obtained from North City Water Reclamation Plant in San Diego. Fifty millilitres of raw sewage water was centrifuged at 8,000g for 1 min at room temperature to pellet large particles. The supernatant was passed through a 0.45-μm and then a 0.2-μm syringe filter (Whatman, PES membrane). One hundred microlitres of the clarified sewage was mixed with 100 μl overnight E. faecalis culture and then added to BHI broth top agar (0.5% agar)\n",
      "and poured over a BHI plate (1.5% agar). After overnight growth at 37 °C, the resulting plaques were recovered using a sterile pipette tip in 500 μl PBS. Phages were replaqued on E. faecalis three more times to ensure that the phages were clonal isolates.\n",
      "High-titre phage stocks were propagated by infecting 200 ml of expo- nentially growing E. faecalis at a multiplicity of infection of 0.1 in BHI broth containing 10 mM MgSO4. Lysis was allowed to proceed for up to six hours at 37 °C with shaking. The lysates were centrifuged at 10,000g for 20 min at room temperature to remove the remaining bacterial cells and debris. Supernatant was then vacuum-filtered through a 0.2-μm membrane filter and kept at 4 °C until use.\n",
      "Before mice were gavaged, 10–20 ml lysates were concentrated using Corning Spin-X UF Concentrators with 100,000-molecular weight cutoff (MWCO) to a volume of approximately 1 ml. Following concen- tration, the culture medium was replaced with PBS via diafiltration. The resulting lysate was further concentrated to a final volume of 0.5 ml and adjusted to the required PFUs.\n",
      "Whole-genome sequencing for phages\n",
      "For all phages except Efmus4, 10 ml of lysates were treated with 10 μg/ml each of DNase and RNase at 37 °C for 1 h and phages were precipitated by adding 1M NaCl and 10% (w/v) polyethylene glycol 8000 (PEG 8000) and incubated at 4 °C overnight. Precipitated phages were then pel- leted by centrifugation at 10,000g for 10 min at 4 °C and resuspended in 500 μl of resuspension buffer (5 mM MgSO4). Phage DNA was then extracted using Promega Wizard DNA Clean-up kit (Promega). Phage genomes were\n",
      "sequenced using a combination of Illumina and Oxford Nanopore Technologies (ONT) MinION platforms. Illumina sequencing libraries were prepared using the Nextera XT library kit with bead-based size selection before loading onto Illumina flow cells. Sequencing was performed with either Illumina MiSeq Reagent Kit v3 in 2 × 300-bp or NextSeq 500 Mid Output Kit in 2 × 150-bp paired-end formats. ONT MinION sequencing libraries were prepared using the Rapid Barcoding Kit (SQK-RBK004) and loaded onto\n",
      "MinION R9.4 flow cells. ONT reads were basecalled with Albacore v.2.3.4 (ONT). The sequence reads were demultiplexed and adapters trimmed from ONT reads using Porechop v.0.2.336. A hybrid Illumina-ONT de novo assembly was performed using the Unicycler v.0.4.7 pipeline37. Subsequently, Pilon v.1.2238 was used iteratively to polish the assemblies with Illumina reads until no addi- tional corrections could be made.\n",
      "For phage Efmus4, 109 PFUs of the phage was filtered sequentially using 0.45-μm and 0.2-μm filters (VWR) and purified on a caesium chloride (CsCl) density gradient39. One millilitre of the CsCl fraction was purified on Amicon YM-100 protein columns (Millipore) and treated with DNase I. DNA was isolated using a Qiagen UltraSens virus kit (Qia- gen), amplified using GenomiPhi V2 (GE Healthcare), and fragmented to 200 to 400 bp using a Bioruptor (Diagenode). Libraries were created using the Ion\n",
      "Plus fragment library kit and sequenced using a 316 Chip on an Ion Torrent Personal Genome Machine (Life Technologies). Reads were trimmed according to modified Phred scores of 0.5 using CLC Genomics Workbench 4.9 (Cambridge), and the remaining reads were assembled using CLC Genomics Workbench 4.9 based on 98% identity with a minimum of 50% read overlap39. Reads were assembled into a single contig of 18,186 bp (20,118 × coverage).\n",
      "Mapping of ONT reads to the hybrid assemblies was used to deter- mine the orientation and terminal ends of linear phage genomes, and reference genomes served as guides to orient circular phage genomes. Phage genome assemblies were annotated using the NCBI Prokaryotic Genome Annotation Pipeline (PGAP)40,41.\n",
      "Phage raw sequence reads and annotated genomes are available at NCBI under the following consecutive BioSample IDs (SAMN11089809– SAMN11089827). GenBank accession numbers include: Efmus1 (MK721195), Efmus2 (MK721197), Efmus3 (MK721185), Efmus4 (MK721193), Ef2.1 (MK693030), Ef2.2 (MK721189), Ef2.3 (MK721192), Ef5.1 (MK721199), Ef5.2 (MK721186), Ef5.3 (MK721200), Ef5.4 (MK721191), Ef6.1 (MK721187), Ef6.2 (MK721188), Ef6.3 (MK721196), Ef6.4 (MK721190), Ef7.1 (MK721194), Ef7.2 (MK721183), Ef7.3\n",
      "(MK721184) and Ef7.4 (MK721198).\n",
      "Genetic maps of phage genomes were generated by LinearDisplay. pl (https://github.com/JCVenterInstitute/LinearDisplay), a PERL script that uses Xfig (https://sourceforge.net/projects/mcj/) to render high- quality images. Preliminary annotation of genes was derived from the automated annotation and from Phage_Finder42, which uses curated hidden Markov models and databases of core phage gene to annotate core gene functions. Annotation was then manually reviewed to assign the final colours.\n",
      "Phage phylogenetic tree\n",
      "\n",
      "A phage whole-genome phylogeny tree was generated from a pairwise distance matrix calculated with the MASH program, which approxi- mates average nucleotide identity (ANI)43. First, a sketch file was created from all the 19 E. faecalis phage genomes isolated and sequenced in\n",
      "this study plus 54 Enterococcus phage genomes obtained from Gen- Bank, with 5,000 12-mers generated per genome (mash sketch -k 12 -s 5000). The sketch file was then compared to all the initial phage genome sequences to generate the ANI matrix using the mash distance command using default settings. The GGRaSP R package was used to calculate the UPMGA phylogeny from the ANI distance matrix, after redundant phage genomes (genomes ANI >99.985) were removed using the GGRaSP R package with a user\n",
      "defined cutoff of 0.015 (ggrasp. cluster (threshold = 0.015)). The resulting dendrogram was translated into newick format using the APE R package44, loaded into the iTOL tree viewer45, and annotated with taxonomic information and manually entered clade identification.\n",
      "Electron microscopy\n",
      "\n",
      "Phage morphology was examined by transmission electron microscopy of negatively stained grids, prepared using the valentine method46 with either 2% uranyl-acetate or 2% phosphotungstic acid, and examined at an acceleration voltage of 100 kV in the JEOL 1200 EX transmission electron microscope.\n",
      "Bacterial DNA extraction and 16S rRNA sequencing\n",
      "DNA from human stool samples, mouse liver sections or bacterial cul- ture was extracted as previously described11, and DNA from mouse faeces was extracted using QIAamp Fast DNA Stool kit (Qiagen). 16S rRNA PCR was completed using Illumina adaptor and barcode-ligated 16S primers targeting the V4 region of the 16S rRNA gene47,48. Amplicons were purified using the Qiaquick PCR purification kit (Qiagen) using manufacturer’s specifications. Purified amplicons were then quantified via TECAN assay\n",
      "(Tecan), normalized and pooled in preparation for 16S rRNA sequencing. Pooled library was quantified and checked for qual- ity using Agilent 2100 Bioanalyzer (Agilent Technologies). Library was sequenced on Illumina MiSeq (Illumina) using V2 reagent chemistry, 500 cycles, 2 × 250-bp format using manufacturer’s specifications. 16S sequence reads were processed and operational taxonomic units were determined using our MOTHUR-based 16S rDNA analysis workflow as previously described11,49. Raw 16S\n",
      "sequence reads can be found in the NCBI Sequence Read Archive (SRA) associated with Bioproject PRJNA525701.\n",
      "Real-time qPCR\n",
      "Bacterial genomic DNA was extracted from human stool samples and mouse liver11. RNA was extracted from mouse liver and cDNAs were generated11. Primer sequences for mouse genes were obtained from the NIH qPrimerDepot. Primer sequences for E. faecalis 16S rRNA gene, E. faecalis cylLS and cylLL genes have previously been described50,51. All primers used in this study are listed in Supplementary Table 9. Mouse gene expression and amplification of bacterial genes were determined with Sybr Green\n",
      "(Bio-Rad Laboratories) using ABI Ste- pOnePlus real-time PCR system. The qPCR value of mouse genes was normalized to 18S.\n",
      "E. faecalis isolation and whole-genome sequencing\n",
      "To isolate E. faecalis strains from human subjects, 50–300 mg of human stool was resuspended in 500 μl PBS, serial dilutions were made and 100 μl was placed on plates with selective medium, BBL Enterococcosel broth (Becton Dickinson). Enterococci colonies were identified by the production of dark brown or black colour, generated by hydrolysis of esculin to esculetin (which reacts with ferric ammonium citrate). Each Enterococcus colony was then picked, and qPCR was performed to identify E.\n",
      "faecalis, using specific primers against the E. faecalis 16S rRNA gene50. For each subject, between 1 and 6 E. faecalis colonies were analysed and bacterial genomic DNA was then extracted as described in ‘Bacterial DNA extraction and 16S rRNA sequencing’.\n",
      "DNA sequencing was performed on the Illumina HiSeq Ten X generat- ing paired-end reads (2 × 151 bp). Bacterial genomes were assembled\n",
      "Article\n",
      "and annotated using the previously described pipeline52. Antimicrobial resistance and virulence genes including cytolysin (cyl) genes carried by E. faecalis isolates were identified by comparing individual genome assemblies against the CARD and VFDB databases, respectively, using abricate v0.8.10 (https://github.com/tseemann/abricate)53,54.\n",
      "For the phylogeny of E. faecalis, the genome assemblies of the study isolates were annotated with Prokka55, and a pangenome estimated using Roary52. A 95% identity cutoff was used, and core genes were defined as those in 99% of isolates. A maximum likelihood tree of the SNPs in the core genes was created using RAxML56 and 100 bootstraps. The resulting tree was visualized using iTOL45. Genome sequence data of E. faecalis strains isolated in this study have been deposited in the European\n",
      "Nucleotide Archive (ENA) under the accession number PRJEB25007. Sequence reads are available at ENA under run acces- sion identifiers ERR3200171–ERR3200263.\n",
      "E. faecalis culture\n",
      "\n",
      "All E. faecalis strains were grown statically in BHI broth or on BHI agar plate at 37 °C. Fifty micrograms per millilitre erythromycin was added when cytolytic and non-cytolytic E. faecalis strains were grown (Extended Data Fig. 2).\n",
      "Determination of levels of faecal Enterococcus\n",
      "To determine levels of faecal enterococci in mice, 10–30 mg of mouse faeces was resuspended into 500 μl PBS and serial dilutions were made. Five microlitres of each dilution from each sample was spotted onto a plate with a selective medium, BBL enterococcosel broth (Becton Dickinson) and the plates were then incubated at 37 °C overnight. For Extended Data Fig. 2o, agar plates contained 50 μg/ml erythromycin. Enterococci colonies were identified by the production of a dark brown or black colour.\n",
      "Colony numbers of each sample were then counted, and CFUs were calculated.\n",
      "Cytolysin expression and purification\n",
      "To purify bioactive CylLL′′ and CylLS′′, an Escherichia coli heterologous expression system was used. In brief, either 6×His–CylLL or 6×His–CylLS were co-expressed with CylM (the enzyme that performs dehydration and cyclization reactions on cytolysin) in E. coli to yield fully dehydrated and cyclized full-length peptides. The His tag and leader peptide were then removed using recombinant CylA (27–412), the soluble domain of the native peptidase used in cytolysin maturation, to yield bioactive\n",
      "CylLL′′ or CylLS′′. The resulting core peptides were further purified by reversed-phase high-performance liquid chromatography (HPLC).\n",
      "The cylLL and cylLS genes were previously cloned into the MCSI of a pRSFDuet-1 backbone vector that contained the cylM gene in MCSII6. The cylA (27–412) gene was previously cloned into MCSI of a pRSFDuet-1 backbone vector57. E. coli BL21 Star (DE3) cells (50 μl) were transformed with 100 ng of either the cylLL_cylM:pRSFDuet, cylLS_cylM:pRSFDuet or cylA (27–412):pRSFDuet plasmids via KCM chemical transformation. The cells were plated on LB agar plates supplemented with kanamycin (50 μg/ml) and\n",
      "grown at 37 °C overnight. One colony was picked to inoculate 15 ml of LB broth supplemented with kanamycin overnight at 37 °C. The culture was used to inoculate 1.5 l of terrific broth supple- mented with kanamycin. Cultures were grown with shaking at 37 °C to an optical density at 600 nm (OD600) of 0.8. The temperature of the incu- bator was lowered to 18 °C and expression was induced with the addi- tion of 0.3 mM final concentration of isopropyl β-d-thiogalactoside. The cultures were allowed\n",
      "to incubate at 18 °C for 18 h. The cells were collected by centrifugation at 5,000g for 12 min. The cell paste was collected and frozen at -70 °C.\n",
      "For the purification of the protease CylA (27–412), the cell paste was thawed and resuspended in 50 ml LanP buffer (20 mM HEPES, 1 M NaCl, pH 7.5). The cell suspension was lysed by homogenization. The lysate was clarified by centrifugation at 13,000g for 45 min and filtered through a 0.45-μm centrifugal filter (Thermo Scientific). The clarified\n",
      "lysate was applied to a pre-equilibrated HisTrap HP 5 ml column (GE Healthcare) through a peristaltic pump. The loaded column was con- nected to an ÄKTA pure 25 M system. The protein was eluted by a linear gradient of LanP buffer to Elution Buffer (20 mM HEPES, 1 M NaCl, 500 mM imidazole, 10% glycerol, pH 7.5) over 30 min. The purest fractions, as determined by 4–20% SDS–PAGE, were combined, concentrated to 1 mg/ml by Amicon Ultra Centrifugal Filters (30 kDa MWCO), and buffer exchanged into\n",
      "storage buffer (20mM HEPES, 300 mM KCl, 10% glycerol, pH 7.5) by PD-10 desalting column (GE Healthcare). Protein concentration was determined by absorbance at 280 nm.\n",
      "For the purification of CylLL′′ and CylLS′′ peptides, the cell paste was thawed and resuspended in 50 ml of LanA Buffer B1 (6 M guanidine HCl, 20 mM NaH2PO4, 500 mM NaCl, 0.5 mM imidazole, pH 7.5). The cell suspension was lysed via sonication (2-s pulse on, 5-s pulse off, 7 min total pulse on time). The cell lysate was clarified by centrifugation at 13,000g for 45 min. The clarified cell lysate was filtered through a 0.45-μm centrifugal filter and applied via gravity flow to a pre-equili-\n",
      "brated, 2 ml bed volume of His60 Ni Superflow Resin (Clonetech). After the lysate had been applied, the resin was washed with 15 ml of LanA Buffer B2 (4 M guanidine HCl, 20 mM NaH2PO4, 500 mM NaCl, 30 mM imidazole, pH 7.5). The resin was washed again with 15 ml of LanA Wash Buffer (20 mM NaH2PO4, 500 mM NaCl, 30 mM imidazole, pH 7.5) to remove the guanidine HCl. The peptides were eluted with 10 ml of LanA elution buffer (20 mM NaH2PO4, 500 mM NaCl, 500 mM imidazole, pH 7.5). A 0.02 mg/ml final\n",
      "concentration of CylA (27–412) was added to the elution fraction and allowed to incubate at room temperature overnight to remove the leader peptide.\n",
      "The digestion was quenched by adding 2% (v/v) final concentration of trifluoroacetic acid. The solution was centrifuged at 4,500g for 10 min and filtered through a 0.45-μm syringe filter (Thermo Scientific). The core peptides were purified by semi-preparative reverse-phase HPLC using a Phenomenex Jupiter Proteo column (10 mm × 250 mm, 4 μm, 90 Å) connected to an Agilent 1260 Infinity II liquid chromatography system. The peptides were separated using a linear gradient of 3% (v/v) solvent B\n",
      "(acetonitrile + 0.1% trifluoroacetic acid) in solvent A (water + 0.1% trif- luoroacetic acid). The fractions were spotted on a matrix-assisted laser desorption/ionization (MALDI) target plate by mixing 1 μl of sample with 1 μl of a 25 mg/ml solution of Super-DHB (Sigma) in 80% acetonitrile/water + 0.1% trifluoroacetic acid. The fractions were analysed by MALDI–time of flight (TOF) mass spectrometry on a Bruker UltrafleXtreme MALDI–TOF/ TOF operating in positive ionization, reflector mode.\n",
      "Primary mouse hepatocytes\n",
      "Hepatocytes were isolated from C57BL/6 female mice fed the chronic– binge ethanol diet (NIAAA model)15. Livers were perfused in situ with 0.5 mM EGTA containing calcium-free salt solution and then perfused with a solution containing 0.02% (w/v) collagenase D (Roche Applied Science). Livers were then carefully minced and filtered using a 70-μm nylon cell strainer. Hepatocytes were centrifuged at 50g for 1 min after 3 washes. Hepatocyte viability was assessed by Trypan Blue (Thermo Fisher\n",
      "Scientific). Hepatocytes (1.5 × 105) were seeded on 12-well plates coated with rat collagen type I in DMEM-F12 (Thermo Fisher Scientific) with insulin–transferrin–selenium (1% v/v) (Thermo Fisher Scientific) and 40 ng/ml dexamethasone (MP Biomedicals) containing 10% (v/v) fetal bovine serum (FBS) (Gemini Bio-Products) and antibiotics. After 4 h, the culture was washed with DMEM-F12 medium and changed to the same complemented medium without FBS58. Then 16 h later, hepato- cytes were cultured with\n",
      "0 or 25 mM ethanol and stimulated with 0, 200 or 400 nM CylLS′′ and/or CylLL′′ in the same culture medium without FBS. After 3 h stimulation, hepatocyte cytotoxicity was assessed using Pierce LDH cytotoxicity detection kit (Thermo Fisher Scientific).\n",
      "Biochemical analysis\n",
      "Serum levels of ALT were determined using Infinity ALT kit (Thermo Scientific). Hepatic triglyceride levels were measured using Triglyceride\n",
      "Liquid Reagents kit (Pointe Scientific). Levels of serum lipopolysac- charide and faecal albumin were determined by enzyme-linked immu- nosorbent kits (Lifeome Biolabs and Bethyl Labs, respectively). Serum levels of ethanol were measured using ethanol assay kit (BioVision).\n",
      "Staining procedures\n",
      "Formalin-fixed tissue samples were embedded in paraffin and stained with H & E. To determine lipid accumulation, liver sections were embed- ded in OCT compound. Eight-micrometre frozen sections were then cut and stained with Oil Red O (Sigma-Aldrich). Representative images from each group of mice are shown in each figure. The terminal deoxynucleo- tide transferase-mediated dUTP nick-end labelling (TUNEL) assay was performed using an in situ cell death detection kit (Sigma-Aldrich). We randomly\n",
      "selected five high-power fields for counting TUNEL-positive cells and normalized numbers to total cells.\n",
      "Statistical analysis\n",
      "Results are expressed as mean ± s.e.m. (except when stated otherwise). Univariate and multivariate Cox regression analysis was used to detect associations of cytolysin with overall mortality. The multivariate model was adjusted for geographical origin of the patients, antibiotic treat- ment, platelet count, and creatinine, bilirubin and INR as components of the MELD score. Univariate logistic regression analysis of laboratory and clinical parameters associated with the detection of cytolysin and\n",
      "E. faecalis was performed. Univariate linear regression analysis of labo- ratory and clinical parameters associated with the log-transformed total amount of faecal E. faecalis measured with qPCR was performed. To associate log-transformed total E. faecalis and E. faecalis positiv- ity with mortality, univariate Cox regression was used. P values from univariate and multivariate Cox regression, univariate logistic regres- sion and univariate linear regression were determined by Wald test.\n",
      "Multicollinearity was examined using the variance inflation factor. Kaplan–Meier curves were used to compare survival between cytolysin- positive and cytolysin-negative patients with alcoholic hepatitis. Faecal E. faecalis, bacterial diversity and richness from controls and patients were compared using Kruskal–Wallis test with Dunn’s post hoc test. Region- and/or centre-specific clinical characteristics of patients with alcoholic hepatitis were compared with Kruskal–Wallis test for continu- ous\n",
      "and Fisher’s exact test for categorical variables. Faecal E. faecalis in patients with alcoholic hepatitis with or without cytolysin, and with or without cirrhosis, were compared with Mann–Whitney-Wilcoxon rank-sum test. Faecal E. faecalis in patients with alcoholic hepatitis from different region and/or centres were compared with the Kruskal– Wallis test. The percentage of subjects with faecal samples that were positive for E. faecalis and cytolysin was compared using Fisher’s exact test,\n",
      "followed by FDR procedures for multiple group comparisons. Jaccard dissimilarity matrices were used for PCoA, and P values were determined by PERMANOVA followed by FDR procedures to correct for multiple comparisons.\n",
      "For mouse and cell culture studies, the significance of multiple groups was evaluated using one-way or two-way ANOVA with Tukey’s post hoc test. Fisher’s exact test was used in the analysis of liver E. faecalis and cytolysin with FDR correction for multiple comparisons. Kaplan–Meier curves were used to compare survival between experi- mental mouse groups. PCoA based on Jaccard dissimilarity matrices was performed between experimental mouse groups and the P values were determined by PERMANOVA\n",
      "followed by FDR procedures to cor- rect for multiple comparisons.\n",
      "Exact P values for all comparisons, together with group size for each group, were listed in Supplementary Table 10. Statistical analyses were performed using R statistical software, R v.3.5.1 (R Foundation for Sta- tistical Computing) and GraphPad Prism v.6.01. A value of P < 0.05 was considered to be statistically significant (adjusted for multiple comparisons when performing multiple tests).\n",
      "Reporting summary\n",
      "Further information on research design is available in the Nature Research Reporting Summary linked to this paper.\n",
      "Data availability\n",
      "Raw 16S sequence reads can be found in the NCBI SRA associated with Bioproject PRJNA525701. Phage raw sequence reads and annotated genomes are available at NCBI under the following consecutive BioSa- mple identifiers (SAMN11089809–SAMN11089827). Genome sequence data of E. faecalis strains isolated in this study were registered at the ENA under study PRJEB25007.\n",
      "Code availability\n",
      "The PERL script for making the genetic maps of phage genomes can be found at https://github.com/JCVenterInstitute/LinearDisplay.\n",
      "30. Brandl, K. et al. Dysregulation of serum bile acids and FGF19 in alcoholic hepatitis. J. Hepatol. 69, 396–405 (2018).\n",
      "\n",
      "31. Gao, B. et al. Serum and fecal oxylipins in patients with alcohol-related liver disease. Dig. Dis. Sci. 64, 1878–1892 (2019).\n",
      "\n",
      "32. Lang, S. et al. Intestinal fungal dysbiosis and systemic immune response to fungi in patients with alcoholic hepatitis. Hepatology (2019).\n",
      "33. Ball, S. A., Tennen, H., Poling, J. C., Kranzler, H. R. & Rounsaville, B. J. Personality, temperament, and character dimensions and the DSM-IV personality disorders in substance abusers. J. Abnorm. Psychol. 106, 545–553 (1997).\n",
      "34. Krieg, L. et al. Mutation of the gastric hydrogen-potassium ATPase alpha subunit causes iron-deficiency anemia in mice. Blood 118, 6418–6425 (2011).\n",
      "35. Gill, J. J. et al. The Caulobacter crescentus phage phiCbK: genomics of a canonical phage. BMC Genomics 13, 542 (2012).\n",
      "\n",
      "36. Wick, R. R., Judd, L. M., Gorrie, C. L. & Holt, K. E. Completing bacterial genome assemblies with multiplex MinION sequencing. Microb. Genom. 3, e000132 (2017).\n",
      "\n",
      "37. Wick, R. R., Judd, L. M., Gorrie, C. L. & Holt, K. E. Unicycler: resolving bacterial genome assemblies from short and long sequencing reads. PLOS Comput. Biol. 13, e1005595 (2017).\n",
      "38. Walker, B. J. et al. Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement. PLoS ONE 9, e112963 (2014).\n",
      "39. Santiago-Rodriguez, T. M. et al. Transcriptome analysis of bacteriophage communities in periodontal health and disease. BMC Genomics 16, 549 (2015).\n",
      "\n",
      "40. Haft, D. H. et al. RefSeq: an update on prokaryotic genome annotation and curation. Nucleic Acids Res. 46, D851–D860 (2018).\n",
      "\n",
      "41. Tatusova, T. et al. NCBI prokaryotic genome annotation pipeline. Nucleic Acids Res. 44, 6614–6624 (2016).\n",
      "42. Fouts, D. E. Phage_Finder: automated identification and classification of prophage regions in complete bacterial genome sequences. Nucleic Acids Res. 34, 5839–5851 (2006).\n",
      "43. Ondov, B. D. et al. Mash: fast genome and metagenome distance estimation using MinHash. Genome Biol. 17, 132 (2016).\n",
      "44. Paradis, E. & Schliep, K. ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R. Bioinformatics 35, 526–528 (2019).\n",
      "\n",
      "45. Letunic, I. & Bork, P. Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees. Nucleic Acids Res. 44, W242–W245 (2016).\n",
      "46. Valentine, R. C., Shapiro, B. M. & Stadtman, E. R. Regulation of glutamine synthetase. XII. Electron microscopy of the enzyme from Escherichia coli. Biochemistry 7, 2143–2152 (1968).\n",
      "47. Haas, B. J. et al. Chimeric 16S rRNA sequence formation and detection in Sanger and 454-pyrosequenced PCR amplicons. Genome Res. 21, 494–504 (2011).\n",
      "\n",
      "48. Caporaso, J. G. et al. Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample. Proc. Natl Acad. Sci. USA 108 (Suppl 1), 4516–4522 (2011).\n",
      "49. Chen, P. et al. Supplementation of saturated long-chain fatty acids maintains intestinal eubiosis and reduces ethanol-induced liver injury in mice. Gastroenterology 148, 203–214.e16 (2015).\n",
      "50. Ryu, H. et al. Development of quantitative PCR assays targeting the 16S rRNA genes of Enterococcus spp. and their application to the identification of enterococcus species in environmental samples. Appl. Environ. Microbiol. 79, 196–204 (2013).\n",
      "51. Haas, W., Shepard, B. D. & Gilmore, M. S. Two-component regulator of Enterococcus faecalis cytolysin responds to quorum-sensing autoinduction. Nature 415, 84–87 (2002).\n",
      "52. Page, A. J. et al. Robust high-throughput prokaryote de novo assembly and improvement pipeline for Illumina data. Microb. Genom. 2, e000083 (2016).\n",
      "53. Chen, L. et al. VFDB: a reference database for bacterial virulence factors. Nucleic Acids Res. 33, D325–D328 (2005).\n",
      "\n",
      "54. Jia, B. et al. CARD 2017: expansion and model-centric curation of the comprehensive antibiotic resistance database. Nucleic Acids Res. 45, D566–D573 (2017)\n",
      "55. Seemann, T. Prokka: rapid prokaryotic genome annotation. Bioinformatics 30, 2068–2069 (2014).\n",
      "\n",
      "56. Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 30, 1312–1313 (2014).\n",
      "Article\n",
      "57. Tang, W., Bobeica, S. C., Wang, L. & van der Donk, W. A. CylA is a sequence-specific protease involved in toxin biosynthesis. J. Ind. Microbiol. Biotechnol. 46, 537–549 (2019).\n",
      "58. Iwaisako, K. et al. Protection from liver fibrosis by a peroxisome proliferator-activated receptor δ agonist. Proc. Natl Acad. Sci. USA 109, E1369–E1376 (2012).\n",
      "59. Clarke, T. H., Brinkac, L. M., Sutton, G. & Fouts, D. E. GGRaSP: a R-package for selecting representative genomes using Gaussian mixture models. Bioinformatics 34, 3032–3034 (2018).\n",
      "Acknowledgements S. Lang was supported by a DFG fellowship (LA 4286/1-1), C.L. was supported by an AASLD Pinnacle Research Award in Liver Disease and a pilot project award from Southern California Research Center for Alcoholic Liver and Pancreatic Disease (ALPD) and Cirrhosis (P50 AA011999), Y.S. was supported by a Wellcome Trust PhD studentship and I.R.R. was supported by National Institute of General Medical Sciences (NIGMS)–NIH Chemistry–Biology Interface Training Grant (T32-GM070421). This\n",
      "study was supported in part by a Biocodex Microbiota Foundation Grant, NIH grants R01 AA24726, U01 AA026939, by Award Number BX004594 from the Biomedical Laboratory Research & Development Service of the VA Office of Research and Development (to B.S.), the Wellcome Trust (WT098051) (to T.D.L.), NIH grant U01AA021908 (to R.B.) and services provided by P30 DK120515 and P50 AA011999.\n",
      "Author contributions Y.D. was responsible for acquisition, analysis and interpretation of data, and drafting of the manuscript; C.L. was responsible for study concept and design, acquisition,\n",
      "analysis and interpretation of data and key preliminary experiments; S. Lang, K.B., J.L. and X.M.T. provided assistance with statistical analysis; H.C., L.J., B.G., W.S., R.K., F.H. and S. Lee provided assistance with data acquisition; R.C.W., T.H.C., K.N., M.T. and D.E.F. were responsible for 16S rRNA sequencing, phage genome sequencing and data analysis; Y.S. and T.D.L. were responsible for bacterial genome sequencing and data analysis; A.H.-M., L.L. and R.Y. provided assistance with phage\n",
      "studies and were responsible for electron microscopy data; R.Y. provided critical revision of the manuscript; I.R.R. and W.A.v.d.D. were responsible for cytolysin expression and purification; Y.M. and L.E. provided assistance with the design and conduct of the gnotobiotic mouse studies; M.L. and D.P. provided assistance with phage isolation; M.V.-C., F.B.-P., E.C.V., J.G.A., R.S.B. Jr, V.V., J.A., J.C., D.L.S., S.B.H., A.L., M.R.L., P.M., G.G.-T., R.B. and P.S. were responsible for collection of\n",
      "human samples; D.E.F. and B.S. were responsible for the study concept and design, and editing the manuscript; B.S. was responsible for study supervision.\n",
      "Competing interests B.S. consults for the Ferring Research Institute; however, there is no competing interest with regard to this study. All other authors declare no competing interests.\n",
      "Additional information\n",
      "Supplementary information is available for this paper at https://doi.org/10.1038/s41586-019- 1742-x.\n",
      "Correspondence and requests for materials should be addressed to B.S.\n",
      "Reprints and permissions information is available at http://www.nature.com/reprints.\n",
      "[ - 00 Bacterial Genera Bacteroides Unclassified Lachnospiraceae Unclassified Ruminococcaceae 80 Veelllonella Akkermansia 70 Prevotella Blautia 60 Rosebuna Escherichia/Shigella Streptococcus 50 Unclassified Clostndiales 40 Dialister Faecalibacterium Clostndium_XiVa 30 Ruminococcus Enterococcus 20 Megasphaera Mogamonas Ruminococcus2 Relative sequence abundance (%) 10 Bifidobacterium | | = B Q=) = = - B: B ... [remaining 309 | Alcoholic Controls AUD hepatitis\n",
      "o Jede X % % % = N Shannon diversity Controls AUD Alcoholic hepatitis\n",
      "Simpson diversity\n",
      "5\n",
      "o\n",
      "\n",
      "o o\n",
      "(=] »\n",
      "o N\n",
      "\n",
      "o\n",
      "Controls\n",
      "k x\n",
      "AUD\n",
      "%\n",
      "\n",
      "%\n",
      "Alcoholic hepatitis\n",
      "Chao richness\n",
      "— de kK —\n",
      "10,000\n",
      "& 8 oo =34\n",
      "\n",
      "H» , =34 00\n",
      "w &\n",
      "2,000\n",
      "\n",
      "1,000\n",
      "Controls\n",
      "AUD\n",
      "Ll ]\n",
      "Alcoholic hepatitis\n",
      "1.0 0.8 -] » -Cytolysin, AUC=0.81 Sensitivity -MELD, AUC=0.70 0.2 -ABIC, AUC=0.66 -DF, AUC=0.63 0.0 0.0 0.2 0.4 0.6 0.8 1.0 1 - Specificity\n",
      "Enterococcus faecalis ek e K 150,000 * % 100,000 50,000 \" - 300 300 Relative abundance 100 100 . Controls AUD Alcoholic hepatitis\n",
      "\n",
      "@@ E. faecalis negative [ E. faecalis positive 100 80 60 40 20 Percent subjects (%) Controls AUD Alcoholic hepatitis\n",
      "@@\n",
      "[\n",
      "0.50 France Mexico 0.25 Spain 0.00 UK { USA East PC2 (16.74%) USA Midwest 0.25 L 0000 USA West -0.4 -0.2 0.4 PC1 A%)\n",
      "\n",
      "@@ E. faecalis negative 3 E. faecalis positive 100 80 60 40 20 Percent subjects (%) P & « ¥\n",
      "@@\n",
      "3\n",
      "mm Cytolysin negative Bl Cytolysin positive 100 80 60 40 20 Percent subjects (%) Cirrhosis Non- cirrhosis\n",
      "mm\n",
      "Bl\n",
      "Extended Data Fig. 1 | See next page for caption.\n",
      "Enterococcus faecalis 150,000 100,000 - 50,000 - - —— — Relative abundance 1,000 0007 | | Cytolysin Cytolysin negative positive\n",
      "\n",
      "Enterococcus faecalis 150,000 100,000 8 < g Relative abundance _ I - s & & Sl & R & o\n",
      "\n",
      "@@ E. faecalis negative 3 E. faecalis positive 100 80 60 40 20 Percent subjects (%) Cirrhosis Non- cirrhosis\n",
      "@@\n",
      "3\n",
      "Em Cytolysin negative Cytolysin positive s [ o =2 o o Percent subjects (%) S 2 & & W & X\n",
      "Em\n",
      "Enterococcus faecalis\n",
      "80,000\n",
      "Relative abundance\n",
      "60,000\n",
      "\n",
      "40,000\n",
      "\n",
      "20,000\n",
      ")\n",
      "Cirrhosis\n",
      "\n",
      "Non- cirrhosis\n",
      "Article\n",
      "Extended Data Fig. 1 | Intestinal dysbiosis in patients with alcoholic hepatitis. a, 16S rRNA sequencing of faecal samples from controls (n = 14), patients with alcohol-use disorder (n = 43), or alcoholic hepatitis (n = 75). The graph demonstrates the relative abundance of sequence reads in each genus. b, Bacterial diversity (Shannon index and Simpson index) and richness (Chao richness) was calculated in controls (n = 14), patients with alcohol-use disorder (n = 43) or alcoholic hepatitis (n =\n",
      "75). c, E. faecalis in faecal samples from controls (n = 25), patients with alcohol-use disorder (n = 38) or alcoholic hepatitis (n = 82), assessed by qPCR. d, Percentage of faecal samples positive for E. faecalis in controls (n = 25), patients with alcohol-use disorder (n = 38) or alcoholic hepatitis (n = 82), assessed by qPCR. E. faecalis was detected in faeces from 80% of patients with alcoholic hepatitis, versus 36% of controls (P < 0.001). There was also a significant difference between\n",
      "patients with alcohol-use disorder and patients with alcoholic hepatitis (P < 0.01). e, Receiver operating characteristic curves and area under the curve (AUC) for the comparison of 90-day mortality and cytolysin positivity (red; n = 57), MELD score (blue; n = 56), ABIC score (yellow; n = 57) and discriminant function (green; n = 42) in patients with alcoholic hepatitis. f, E. faecalis in faecal samples from patients with alcoholic hepatitis whose faecal samples were cytolysin- positive (n = 25)\n",
      "or cytolysin-negative (n = 54), assessed by qPCR (P = 0.8174). g, 16S rRNA sequencing of faecal samples from patients with alcoholic hepatitis from different centres (France, n = 9; Mexico, n = 6; Spain, n = 5; UK, n = 11; USA (east), n = 16; USA (Midwest), n = 12; USA (west), n = 16 patients). We used PCoA based on Jaccard dissimilarity matrices to show β-diversity among groups at the genus level. The composition of faecal microbiota was significantly different between patients from different\n",
      "regions (P < 0.01). h, Percentage of faecal samples that were positive for cylLL and cylLS DNA sequences (cytolysin- positive), in patients with alcoholic hepatitis from different centres (France,\n",
      "n = 16; Mexico, n = 6; Spain, n = 6; UK, n = 10; USA (east), n = 16; USA (Midwest), n = 13; USA (west), n = 15 patients), assessed by qPCR (P = 0.6094). i, E. faecalis in faecal samples from patients with alcoholic hepatitis from different centres, assessed by qPCR (P = 0.5648). j, Percentage of faecal samples that were positive for E. faecalis in patients with alcoholic hepatitis from different centres (France, n = 16; Mexico, n = 6; Spain, n = 6; UK, n = 10; USA (east), n = 16; USA (Midwest),\n",
      "n = 13; USA (west), n = 15 patients), assessed by qPCR (P = 0.0529). k, Percentage of subjects with faecal samples that were positive for cylLL and cylLS DNA sequences (cytolysin-positive), in patients with alcoholic hepatitis and with (n = 30) or without (n = 18) cirrhosis, assessed by qPCR (P = 0.3431). l, E. faecalis in faecal samples from patients with alcoholic hepatitis and with (n = 30) or without (n = 18) cirrhosis, assessed by qPCR (P = 0.5736). m, Percentage of faecal samples that were\n",
      "positive for E. faecalis in patients with alcoholic hepatitis and with (n = 30) or without (n = 18) cirrhosis, assessed by qPCR (P = 0.2878). Results are expressed as mean ± s.e.m. (c, f, i, l). For the box and whisker plots in b, the box extends from the 25th to 75th percentiles, and the centre line represents the median; for all three groups, the bottom whiskers show the minimum values; for the control group (black), the top whisker shows the maximum value; for the other two groups, the top\n",
      "whiskers represent the 75th percentile plus 1.5× the inter-quartile distance (the distance between the 25th and 75th percentiles); all values greater than this are plotted as individual dots. P values were determined by Kruskal– Wallis test (i) with Dunn’s post hoc test (b, c), two-sided Fisher’s exact test (h, j, k, m) followed by FDR procedures (d), two-sided Mann–Whitney Wilcoxon rank-sum test (f, l) or PERMANOVA (g). The exact group size (n) and P values for each comparison are listed in\n",
      "Supplementary Table 10. *P < 0.05, **P < 0.01, ***P < 0.001, ****P < 0.0001.\n",
      "Serum ALT 400 k 300 200 100 o1 A o Control Ethanol\n",
      "\n",
      "b u Fold change oo H Fod Control Ethanol\n",
      "«\n",
      "« ~ E. faecalis Acytolysin —— E. faecalis 100 920 80 70 Log-rank test 60 (Mantel-Cox) Percent survival (%) P<0.05 50 10 Days\n",
      "~\n",
      "\n",
      "——\n",
      "Fecal albumin\n",
      "\n",
      "Fecal albumin 400 30 - l ~ 200 10 [ [a] il Control Ethanol\n",
      "Serum ethanol 1,000 [T} 800 ] S 600 g —— 400 200 Ethanol\n",
      "Extended Data Fig. 2 | See next page for caption.\n",
      ") Control 3 Y Ethanol PBS E. faecalis E. faecalis Acytolysin\n",
      "\n",
      "Hepatic triglycerides PBS 40 W E. faecalis u B E. faecalis Acytolysin 30 L 20 10 oo [ =] i i Control Ethanol\n",
      "Cxcl1 — Fold change oo e 18 [ o Control Ethanol\n",
      "\n",
      "Liver cyLg 100 k k k, 80 60 40 Percent mice (%) 20 nd nd nd Control Ethanol\n",
      "\n",
      "Serum LPS 60 k —— 40 [N (ng/ml) 20 Qo l 2 [on] Control Ethanol\n",
      "\n",
      "Adht & S . P oo 1] 5 Fold change i S Control Ethanol\n",
      "\n",
      "Cxcl2 x 10 L] Fold change [al Control Ethanol\n",
      "Liver E. faecalis 100 © = = S = Percent mice (%) = nd Control Ethanol\n",
      "\n",
      "Liver Enterococcus\n",
      "1,500\n",
      "L]\n",
      "1,000\n",
      "\n",
      "500\n",
      "Ethanol\n",
      "\n",
      "0.50 PBS E. faecalis Control 0.25 ' E. faecalis Acytolysin PBS B E. faecalis Ethanol 0.00 ‘& B E. faecalis Acytolysin -0.25: -0.4 0.4 PC1 .)\n",
      "\n",
      "Cyp2e1 25 2.0 - o - = Fold change 1F 0.0 Control Ethanol\n",
      "Fecal Enterococcus 1010 108 1 104 102 24 48 72 Time (hrs)\n",
      "Article\n",
      "Extended Data Fig. 2 | Cytolytic E. faecalis causes the progression of ethanol-induced liver disease in mice. a–n, C57BL/6 mice were fed oral isocaloric (control) or chronic–binge ethanol diets and gavaged with vehicle (PBS), a cytolytic E. faecalis strain (FA2-2(pAM714)) (denoted E. faecalis) (5 × 108 CFUs) or a non-cytolytic E. faecalis strain (FA2-2(pAM771))5 (denoted E. faecalis Δcytolysin) (5 × 108 CFUs) every third day. a, Serum levels of ALT. b, Hepatic triglyceride content. c,\n",
      "Representative oil red O-stained liver sections. d–f, Hepatic levels of mRNAs. g, Kaplan–Meier curve of survival of mice on chronic–binge ethanol diets (day 0 denotes the start of ethanol feeding). Mice gavaged with PBS all survived, and are not included in the figure. A higher proportion of mice (n = 15) gavaged with non-cytolytic E. faecalis survived than did mice (n = 25) gavaged with cytolytic E. faecalis. h, Proportions of mice that were positive for cytolysin in the liver, measured by qPCR\n",
      "for cylLS (the gene that encodes cytolysin subunit CylLS′′). i, Proportions of mice that were positive for E. faecalis in the liver, measured by qPCR. About 80% of mice colonized with cytolytic E. faecalis, as well as those colonized with non- cytolytic E. faecalis, were positive for E. faecalis in their livers. j, Liver CFUs of Enterococcus in mice on a chronic–binge ethanol diet. k, Paracellular intestinal permeability was evaluated by measuring faecal albumin content and serum levels of\n",
      "lipopolysaccharide (LPS) by enzyme-linked immunosorbent assays. l, Faecal samples were collected and 16S rRNA genes were sequenced. PCoA based on Jaccard dissimilarity matrices showed no significant differences among mice gavaged with PBS, cytolytic or non-cytolytic E. faecalis following\n",
      "feeding with the control and ethanol diets. Compared to mice fed with a control diet, mice fed with an ethanol diet had significantly different faecal microbiomes after gavaging with E. faecalis (P < 0.05). m, n, Serum levels of ethanol and hepatic levels of Adh1 and Cyp2e1 mRNAs did not differ significantly among mice gavaged with PBS, cytolytic or non-cytolytic E. faecalis after ethanol feeding. o, Mice were gavaged with cytolytic or non- cytolytic E. faecalis strains (carrying the\n",
      "erythromycin resistance gene; 5 × 108 CFUs) at time 0, and faeces were collected 0, 8, 24, 48 and 72 h later. Faecal CFUs of Enterococcus were determined by culturing faecal samples on BBL enterococcosel broth agar plate with 50 μg ml-1 erythromycin. At time 0 and 72 h, five out of five and four out of five mice, respectively, had no detectable erythromycin-resistant Enterococcus in their faeces. These points are not shown on the graph, but have been included in the calculation of mean ± s.e.m.\n",
      "Scale bar, 100 μm. Results are expressed as mean ± s.e.m. (a, b, d–f, j, k, m–o). P values among groups of mice fed with the control or ethanol diet were determined by one-way ANOVA with Tukey’s post hoc test (a, b, d–f, j, k, m, n), two-sided log-rank (Mantel–Cox) test (g), two-sided Fisher’s exact test followed by FDR procedures (h, i) or PERMANOVA followed by FDR procedures (l). All results were generated from at least three independent replicates. The exact group size (n) and P values for\n",
      "each comparison are listed in Supplementary Table 10. P values between mice fed with a control diet and mice fed with an ethanol diet were determined by two-way ANOVA (k). *P < 0.05, **P < 0.01, ***P < 0.001, ****P < 0.0001.\n",
      "TUNEL AH cytolysin positive #1 r AH cytolysin positive #2 AH cytolysin negative #3 [N AH cytolysin negative #4 Percent positive (%) i il l 1l Control Ethanol\n",
      "\n",
      "R Control . 3 /3% ¥ Ethanol F R o Ve, AH cytolysin AH cytolysin AH cytolysin AH cytolysin positive #1 positive #2 negative #3 negative #4\n",
      "\n",
      "Cxcl2 Fold change - o oo 3 i Control Ethanol\n",
      "\n",
      "Acta2 10 Fold change Control Ethanol\n",
      "AH Cytolysin negative AH Cytolysin positive \" 0 90; 80; 70 Log-rank test 60; (Mantel-Cox) Percent survival (%) P<0.05 50 2 10 Days\n",
      "\n",
      "AH cytolysin positive #1 AH cytolysin positive #2 Control | AH cytolysin negative #3 ] AH cytolysin negative #4 H PC2 (24.13%) & 05 '..\n",
      "|\n",
      "\n",
      "]\n",
      "0.4- AH cytolysin pos ive #1 AH cytolysin posi #2 I Ethanol AH cytolysin negative #3 AH cytolysin negative #4 g [mmm|m i PC2 (22.82%) 1 A S | | y/ y 0. 02 0. Be (51%)\n",
      "\n",
      "Cytolysin positive E. faecalis a 8 3 2 8 8 Percent colonies (%) nd nd o Patient Patient Patient Patient #1 # #3 # AH ¢y lysin AH cytolysin po: negative\n",
      "Serum ethanol 1,000 800 600 400 - 200 Ethanol\n",
      "Extended Data Fig. 3 | Transplantation of cytolysin-positive faeces increases ethanol-induced liver disease in gnotobiotic mice. a–f, h, i, C57BL/6 germ- free mice were colonized with faeces from two cytolysin-positive and two cytolysin-negative patients with alcoholic hepatitis, and then fed isocaloric (control) or chronic–binge ethanol diets. a, Percentage of TUNEL-positive hepatic cells. b, Representative oil red O-stained liver sections. c, d, Hepatic levels of mRNAs that encode the\n",
      "inflammatory cytokine Cxcl2 and Acta2 (a marker of activated hepatic stellate cells). e, Kaplan–Meier curve of survival of mice on chronic–binge ethanol diets (day 0 denotes the start of ethanol feeding), gavaged with faeces from cytolysin-positive (n = 48 mice) or cytolysin-negative (n = 32 mice) patients with alcoholic hepatitis. f, Faecal samples were collected and 16S rRNA genes were sequenced. The graph shows PCoA of faecal microbiomes. No significant difference was observed between mice\n",
      "colonized with faeces from cytolysin-positive or cytolysin-negative donors with alcoholic hepatitis, following the control diet. Mice transplanted with faeces from a cytolysin-positive patient with alcoholic hepatitis (patient\n",
      "Ad1 Cyp2et 2.5 2.0 2.0 - 1.5 o 15 e o - % . 1.0{ H - 1.0 H Fold change Fold change 0.5: = W 0.5: 0.0 il 0.0 Control Ethanol Control Ethanol\n",
      "no. 2) showed a microbiota that was significantly different to that of the other mouse groups following ethanol administration (P < 0.01). g, Percentage of cytolysin-positive E. faecalis in four patients with alcoholic hepatitis. Stool samples from the four patients were placed on plates with selective medium, and Enterococcus colonies were identified by the production of a dark brown or black colour. Enterococcus colonies were confirmed to be E. faecalis by qPCR. The cytolysin status of each E.\n",
      "faecalis colony was determined by qPCR. h, Serum levels of ethanol were comparable among colonized mice after ethanol feeding. i, Hepatic levels of Adh1 and Cyp2e1 mRNAs did not differ significantly among colonized mice on control or ethanol diets. Scale bar, 100 μm. Results are expressed as mean ± s.e.m. (a, c, d, h, i). P values were determined by one-way ANOVA with Tukey’s post hoc test (a, c, d, h, i), two- sided log-rank (Mantel–Cox) test (e) or PERMANOVA followed by FDR procedures (f). All\n",
      "results were generated from at least three independent replicates. The exact group size (n) and P values for each comparison are listed in Supplementary Table 10. *P < 0.05, **P < 0.01, ***P < 0.001.\n",
      "Article\n",
      "a Head s v % | . 2 | % o . Tail Siphophage Myophage Podophage (long, flexible, (contractile tail) (short, noncontractile tail) noncontractile tail) S5 or ¥ @ ¥ % Efmus1 Efmus2 Efmus3 Efm us4 Efmus1 (17,927 bp) y b Efmus2 (18,366 bp) Efmus3 (18,286 bp) T SN Efmus4 (18,186 bp) D ol | encapsidation/terminase [ tail fiber DNA polymerase | [l endolysin/lysin | I hypothetical protein - major capsid [ tail components [ DNA binding intron/intein [ holin Il ortal\n",
      "Head\n",
      "\n",
      "Tail\n",
      "major capsid\n",
      "[\n",
      "\n",
      "[\n",
      "\n",
      "[\n",
      "Il\n",
      "ortal\n",
      "Extended Data Fig. 4 | Isolation and amplification of phages against cytolytic E. faecalis isolated from mice. a, BHI agar plate showing phage plaque morphology. The phage cocktail (100 μl) (102–103 PFUs) was mixed with overnight-grown E. faecalis culture (100 μl) and then added to BHI broth top agar (0.5% agar) and poured over a BHI plate (1.5% agar). After overnight growth at 37 °C, images were captured on an Epson Perfection 4990 Photo scanner. b, Simplified illustration of the morphologies\n",
      "of different phages. Siphophages have long, flexible noncontractile tails (left); myophages have contractile tails\n",
      "(middle); and podophages have short noncontractile tails (right). c, Transmission electron microscopy revealed that phages we isolated were all podophages (Efmus1, Efmus2, Efmus3 and Efmus4). d, Genetic map of phage genomes. The linear maps are based on nucleotide sequences of the phage genomes and predicted open reading frames. The name and length (in bp) of each genome are indicated to the left of each phage map. Protein-coding sequences are coloured on the basis of functional role categories.\n",
      "Scale bar, 50 nm. All results were generated from at least three independent replicates.\n",
      "Serum ALT 500 — 400 ~300 200 - 100 ll WT II WT Atpas Control Ethanol\n",
      "\n",
      "Hepatic triglycerides PBS C. crescentus phages 60 E. faecalis phages PBS C. crescentus phages EEEE E. faecalis phages '. i WT WT I/I Control Ethanol\n",
      "\n",
      "S =y 5 Control \" | ey 3 - 3 Ethanol > R - 2 e PBS C. crescentus phages E. faecalis phages PBS C. crescentus phages E. faecalis phages WT Atp4ass\n",
      "\n",
      "Cxcl1 15 es — e, 10 .- \"~ Fold change - o M | i WT WT /I Control Ethanol\n",
      "Cxcl2 pes 3 — i - o - =3 Fold change ] . H - _ B L & WT I/ WT Control Ethanol\n",
      "\n",
      "Liver cylLg 100 — s i 8 3 ] Percent mice (%) 8 2 nd o 3 /I WT / Control Ethanol\n",
      "\n",
      "b S — e - o - o Fold change 5 SUs WT Atpda WT A Control Ethanol\n",
      "\n",
      "Fecal Enterococcus 1010 10° 2108 108 10% il | 104 WT / WT / Control Ethanol\n",
      "\n",
      "PBS a C. crescentus phages o8 E. faecalis phages o o & > Control PBS (10.08%) ] C. crescentus phages Atpdassi PC2 E. faecalis phages -1.0 4 17.1) 0\n",
      "Serum ethanol\n",
      "Adh1\n",
      "800\n",
      "\n",
      "600\n",
      "\n",
      "400\n",
      "20\n",
      "o S il i\n",
      "Fold change\n",
      "H\n",
      "I I\n",
      "\n",
      "|\n",
      "\n",
      ":\n",
      "WT\n",
      "II\n",
      "WT\n",
      "\n",
      "WT\n",
      "/s\n",
      "Ethanol\n",
      "Control\n",
      "Ethanol\n",
      "Extended Data Fig. 5 | See next page for caption.\n",
      "2 PBS C. crescentus phages WT 5 E. faecalis phages Ethanol PBS g (17.92%) C. crescentus phages Atpdasus PC2 E. faecalis phages B 0.4 0 5 PC1 'z.\n",
      "\n",
      "Cyp2e1 S = - S P = 1.0 B Fold change 0.5 il I i 0.0 WT wT I I/I Control Ethanol\n",
      "Article\n",
      "Extended Data Fig. 5 | Phages reduce translocation of cytolysin to the liver and reduce ethanol-induced liver disease in Atp4aSl/Sl mice. a–k, Wild-type (WT) and Atp4aSl/Sl littermates were fed oral isocaloric (control) or chronic– binge ethanol diets, and gavaged with vehicle (PBS), control phages against C. crescentus (1010 PFUs) or a cocktail of four different phages that target cytolytic E. faecalis (1010 PFUs), 1 day before an ethanol binge. a, Serum levels of ALT. b, Hepatic triglyceride\n",
      "content. c, Representative oil red O-stained liver sections. d–f, Hepatic levels of mRNAs. g, Proportions of mice that were positive for cytolysin in the liver, measured by qPCR for cylLS. h, Faecal CFUs of Enterococcus. i, Faecal samples were collected and 16S rRNA genes were sequenced. PCoA based on Jaccard dissimilarity matrices found no significant\n",
      "difference in faecal microbiota among mice given PBS, control phage or phages that target cytolytic E. faecalis in each group. j, k, Serum levels of ethanol and hepatic levels of Adh1 and Cyp2e1 mRNAs did not differ significantly among colonized mice after ethanol feeding. Scale bar, 100 μm. Results are expressed as mean ± s.e.m. (a, b, d–f, h, j, k). P values were determined by two-way ANOVA with Tukey’s post hoc test (a, b, d–f, h, j, k), two-sided Fisher’s exact test followed by FDR\n",
      "procedures (g) or PERMANOVA followed by FDR procedures (i). All results were generated from at least three independent replicates. The exact group size (n) and P values for each comparison are listed in Supplementary Table 10. *P < 0.05, **P < 0.01, ***P < 0.001, ****P < 0.0001.\n",
      "<. 2. - 8s °s - Ef5 Ef2\n",
      "Ef2.1\n",
      "Ef2.3\n",
      "Ef5.1 F)- ) ) o l ) ) o (41,141 bp) Ef5.2 (41,418 bp) ) b o s i D Ef5.3 F)- b ) o ol e ) ) (39,115 bp) Ef5.4 (41,141 bp) W) e om Ef2.1 (140,938 bp) PP o ) ) s o ) ) o D DO O OO 0 0 ¢ Ef2.2 ( W W (58,400 bp) L Ef2.3 i oo ) ) o (147,289 bp) ) ) m) m) o m m Capsid structural components Tail and baseplate and assembly assembly [ encapsidation/terminase [ tail fibers - DNA polymerase - endolysin/lysin - hypothetical protein (] major capsid - tail component [ DNA binding I holin intron/intein I rortal\n",
      "[ tail measure DNA recombination | regulatory [ protease (scaffold protein) I salvage of [ transcription factor nucleosides and Il other nucleotides | other known functions ] DNA restriction/ modification e tRNA || nucleotide degradation\n",
      "Extended Data Fig. 6 | Isolation and amplification of phages against cytolytic E. faecalis strains isolated from patients with alcoholic hepatitis. a, BHI agar plates showing phage plaque morphology. b, Transmission electron microscopy graphs of myophages Ef2.1 and Ef2.3, stained with phosphotungstic acid showing contracted tails. c, Genetic map of phage genomes. The linear maps are based on nucleotide sequences of the phage\n",
      "genomes and predicted open reading frames. The name and length (in bp) of each genome are indicated to the left of each phage map. Protein-coding sequences are coloured on the basis of functional role categories. Sequences that encode tRNA genes are indicated by a cloverleaf structure. Scale bar, 50 nm. All results were generated from at least three independent replicates.\n",
      "Article\n",
      "\n",
      "a\n",
      "TUNEL C. crescentus phages Cytolysin-positive E. faecalis phages K k C. crescentus phages EE Cytolysin-positive E. faecalis phages Percent positive (%) Patient Patient Patient Patient #5 #2 #5 #2 Control Ethanol\n",
      "\n",
      "o e Control e A o ] Ethanol C. crescentus phages Cytolysin-positive C. crescentus phages ylin- ive E. faecalis phages E. faecalis phages Patient #5 Patient #2\n",
      "Patient #2\n",
      "\n",
      "Cxcl2 [T \" Fold change o H al Patient Patient Patient Patient #5 #2 #5 #2 Control Ethanol\n",
      "Acta2 10 I Fold change y B o H ¥ Patient Patient Patient Patient #5 #2 #5 #2 Control Ethanol\n",
      "\n",
      "Fecal Enterococcus 1010 10° 2108 O 107 108 108 | Patient Patient Patient Patient #5 #2 #5 #2 Control Ethanol\n",
      "\n",
      "Control C. crescentus phages 5 Cytolysin-positive E. faecalis phages | Patient #5 =) o C. crescentus phages Cytolysin-positive E. faecalis phages I Patient #2 =] o PC2 (7.13%) -0.5 -1.0 -0.5 c (80.98%) 0\n",
      "Ethanol C. crescentus phages 0.25 Cytolysin-positive E. faecalis phages Patient #5 C. crescentus phages Patient #2 LILILIL Cytolysin-positive E. faecalis phages PC2 (16.3%) -0.25 0.25 1.)\n",
      "\n",
      "Serum ethanol 800 60 = 400 20 Patient Patient #5 #2 Ethanol\n",
      "\n",
      "Adh1 2.0 - o oo o - ] Fold change 0.5 . 0.0 Patient Patient Patient Patient #5 #2 #5 #2 Control Ethanol\n",
      "Extended Data Fig. 7 | Phages that target cytolytic E. faecalis reduce ethanol-induced liver disease in gnotobiotic mice. a–h, C57BL/6 germ-free mice were colonized with faeces from two cytolysin-positive patients with alcoholic hepatitis (faeces from one patient were also used in Fig. 2). The mice were then fed oral isocaloric (control) or chronic–binge ethanol diets, and gavaged with control phages against C. crescentus (1010 PFUs) or a cocktail of 3 or 4 different phages that target cytolytic\n",
      "E. faecalis (1010 PFUs), one day before an ethanol binge. a, Percentage of TUNEL-positive hepatic cells. b, Representative oil red O-stained liver sections. c, d, Hepatic levels of mRNAs that encode the inflammatory cytokine Cxcl2, and Acta2 (a marker of activated hepatic stellate cells). e, Faecal CFUs of Enterococcus. f, Faecal samples were\n",
      "Cyp2e1 s = o Oo T - = o Fold change 0.5 | 0.0 Patient Patient Patient Patient #5 #2 #5 #2 Control Ethanol\n",
      "collected and 16S rRNA genes were sequenced. PCoA based on Jaccard dissimilarity matrices shows no significant differences in the faecal microbiota of mice gavaged with control phage and phages that target cytolytic E. faecalis in each group. g, h, Serum levels of ethanol and hepatic levels of Adh1 and Cyp2e1 mRNAs did not differ significantly among colonized mice after ethanol feeding. Scale bar, 100 μm. Results are expressed as mean ± s.e.m. (a, c–e, g, h). P values were determined by two-way\n",
      "ANOVA with Tukey’s post hoc test (a, c–e, g, h) or PERMANOVA followed by FDR procedures (f). All results were generated from at least three independent replicates. The exact group size (n) and P values for each comparison are listed in Supplementary Table 10. *P < 0.05, ***P < 0.001.\n",
      "%, H % % 84 S X %3 s y 3 3 g H £ S & X3 2 & F o %, 8% , o o Yo, s, 5 2. & e, & B, By ® N ) S <1 G %, A 2. g5eC f54 PhiM1E, fal Ef6 VB-EfaM-£f23 vB-EfaSEf PhiEF17H. PMBT2 B-EfaM-Ef2.1 -vB-EfaS-Ef5.2 iEF24C P2 Si r , EfaP_igg KA s A S S, 103 & 253 o o, 62 ¥ & S, S, - kN N & %, G [ Siphoviridae P ne 5 % & § N N g3 & N % Podoviridae k) Ef7 B Myoviridae Tree scale: 1\n",
      "Ef6.1 ) b e o 4 (40,429 bp) Ef6.2 (17,966 bp) ) o Ef6.3 ) o (18,136 bp) Ef6.4 s ) o (41,133 bp) Ef7.1 D ) | (58,018 bp) - Ef7.2 1 ) o s (18,737 bp) Ef7.3 oo (18,818 bp) Ef7.4 (18,415 bp) C apsid structural components Other N and assembly || encapsidation/terminase [ tail fibers - DNA polymerase [ endolysin/ysin | [ hypothetical protein - tail component I intron/intein major capsid [ DNA binding [ holin I portal [ tail measure [ regulatory DNA recombination [l protease (scaffold protein) I\n",
      "salvage of RNA l other nucleosides and nucleotides [ DNA restriction/ modification (| nucleotide degradation\n",
      "Extended Data Fig. 8 | See next page for caption.\n",
      "Article\n",
      "Extended Data Fig. 8 | Isolation and amplification of phages against non- cytolytic E. faecalis strains isolated from patients with alcoholic hepatitis. a, BHI agar plates showing phage plaque morphology. b, Genetic map of phage genomes. The linear maps are based on nucleotide sequences of the phage genomes and predicted open reading frames. The name and length (in bp) of each genome are indicated to the left of each phage map. Protein-coding sequences are coloured on the basis of functional\n",
      "role categories. Sequences that encode tRNA genes are indicated by a cloverleaf structure. c, Phylogenetic tree of Enterococcus phages. A whole-genome average nucleotide distance tree was constructed for 73 available Enterococcus phage genomes: 54 of these were\n",
      "from GenBank (denoted by black letters) and 19 were from this study (4 phages against cytolysin-positive E. faecalis isolated from mice (shown in blue letters); 7 phages against cytolysin-positive E. faecalis isolated from patients with alcoholic hepatitis (shown in pink letters); and 8 phages against cytolysin- negative E. faecalis isolated from patients with alcoholic hepatitis (shown in green letters)) with Mash43 using a sketch size of s = 5000 and a k-mer size of k = 12 and GGRaSP59\n",
      "(Methods). Coloured branches denote specific phage genera or subfamily: Sap6virus, P68virus and Spounavirinae. The scale bar represents per cent average nucleotide divergence. All results were generated from at least three independent replicates.\n",
      "TUNEL C. crescentus phages Cytolysin-negative E. faecalis phages C. crescentus phages L B | Cytolysin-negative E. faecalis phages Percent positive (%) | Patient Patient Patient Patient #6 #7 #6 #7 Control Ethanol\n",
      "|\n",
      "o o Control 2 . “ . L ‘ ok - -4 & l Ethanol 3 4 C. crescentus phages lyioiv C. crescentus phages lyin- ve E. faecalis phages E. faecalis phages Patient #6 Patient #7\n",
      "\n",
      "Cxcl2 L] L oo Fold change 0 ] i Patient Patient Patient Patient #6 #7 #6 #7 Control Ethanol\n",
      "\n",
      "Acta2 Ll Fold change T l ] g i Patient Patient Patient Patient #6 H#7 #6 H7 Control Ethanol\n",
      "\n",
      "Liver cylLg 100 =3 =3 o o B o Percent mice (%) =3 nd nd nd nd nd nd nd nd [ Patient Patient Patient Patient #6 #7 #6 #7 Control Ethanol\n",
      "Control\n",
      "Ethanol\n",
      "Control 1.0 C. crescentus phages g Cytolysin-positive E. faecalis phages | Patient #6 2 C. crescentus phages Cytolysin-positive E. faecalis phages | Patient #7 g ] PC2 (19.54%) -0.5 -1.0 -1.0 1.0 PC1\n",
      "\n",
      "Ethanol . - C. crescentus phages Cytolysin-positive E. faecalis phages Patient #6 0. C. crescentus phages u|| Cytolysin-positive E. faecalis phages Patient #7 0.0 PC2 (15.2%) -0.3 -0.2 0.2 0.50 (.)\n",
      "\n",
      "Serum ethanol 1,000 800 (1] 600 I (mg/dl) 400 200 Patient Patient #6 #7 Ethanol\n",
      "Adh1 25 20 - LT - Fold change & S Patient Patient Patient Patient #6 #7 #6 #7 Control Ethanol\n",
      "Extended Data Fig. 9 | Phages that target non-cytolytic E. faecalis do not reduce ethanol-induced liver disease in gnotobiotic mice. a–h, C57BL/6 germ-free mice were colonized with faeces from two cytolysin-negative patients with alcoholic hepatitis. Transplanted gnotobiotic mice were fed oral isocaloric (control) or chronic–binge ethanol diets and gavaged with control phages against C. crescentus (1010 PFUs) or a cocktail of four different phages targeting non-cytolytic E. faecalis (1010 PFUs),\n",
      "1 day before an ethanol binge. a, Percentage of TUNEL-positive hepatic cells. b, Representative oil red O-stained liver sections. c, d, Hepatic levels of mRNAs that encode the inflammatory cytokine Cxcl2, and Acta2 (a marker of activated hepatic stellate cells). e, Proportions of mice that were positive for cytolysin in the liver, measured by qPCR for cylLS. f, Faecal samples were collected and 16S rRNA\n",
      "Cyp2e1 25 2.0 O - e - o Fold change = o Patient Patient Patient Patient #6 #7 #6 #7 Control Ethanol\n",
      "genes were sequenced. PCoA based on Jaccard dissimilarity matrices found no significant difference in faecal microbiota among mice gavaged with control phages and phages that target cytolytic E. faecalis in each group. g, h, Serum levels of ethanol and hepatic levels of Adh1 and Cyp2e1 mRNAs did not differ significantly among colonized mice after ethanol feeding. Scale bar, 100 μm. Results are expressed as mean ± s.e.m. (a, c, d, g, h). P values were determined by two-way ANOVA with Tukey’s post\n",
      "hoc test (a, c, d, g, h), two-sided Fisher’s exact test followed by FDR procedures (e) or PERMANOVA followed by FDR procedures (f). All results were generated from at least three independent replicates. The exact group size (n) and P values for each comparison are listed in Supplementary Table 10.\n",
      "natureresearch\n",
      "Reporting Summary\n",
      "Nature Research wishes to improve the reproducibility of the work that we publish. This form provides structure for consistency and transparency in reporting. For further information on Nature Research policies, see Authors & Referees and the Editorial Policy Checklist.\n",
      "Statistics\n",
      "For all statistical analyses, confirm that the following items are present in the figure legend, table legend, main text, or Methods section. n/a Confirmed The exact sample size (n) for each experimental group/condition, given as a discrete number and unit of measurement A statement on whether measurements were taken from distinct samples or whether the same sample was measured repeatedly The statistical test(s) used AND whether they are one- or two-sided Only common tests should be described\n",
      "solely by name; describe more complex techniques in the Methods section. A description of all covariates tested A description of any assumptions or corrections, such as tests of normality and adjustment for multiple comparisons A full description of the statistical parameters including central tendency (e.g. means) or other basic estimates (e.g. regression coefficient) AND variation (e.g. standard deviation) or associated estimates of uncertainty (e.g. confidence intervals) For null hypothesis\n",
      "testing, the test statistic (e.g. F, t, r) with confidence intervals, effect sizes, degrees of freedom and P value noted Give P values as exact values whenever suitable. For Bayesian analysis, information on the choice of priors and Markov chain Monte Carlo settings For hierarchical and complex designs, identification of the appropriate level for tests and full reporting of outcomes Estimates of effect sizes (e.g. Cohen's d, Pearson's r), indicating how they were calculated\n",
      "Our web collection on statistics for biologists contains articles on many of the points above.\n",
      "Software and code\n",
      "\n",
      "Policy information about availability of computer code\n",
      "Data collection All biochemical assays were measured using SoftMax Pro 7.0.3; gPCRs were run with StepOnePlus real-time PCR system; Liver histological pictures were taken with DP Controller and DP Manager (Olympus); Phage electronic microscopy pictures were taken using Maxim DL5; Plates were scanned using EPSON 4990 Photo; All pictures were viewed using Image\n",
      "Data analysis\n",
      "Bacteriophage sequencing and phage tree: Albacore v2.3.4 (ONT), Porechop v0.2.3, Unicycler v0.4.7 pipeline, Pilon v1.22, CLC Genomics Workbench 4.9, NCBI Prokaryotic Genome Annotation Pipeline, in-house PERL script using Xfig, Phage_Finder, MASH program, GGRaSP and APE R-package, iTOL tree viewer 16S sequencing: MOTHUR-based 16S rDNA analysis workflow E. faecalis genome sequencing and tree: abricate v0.8.10 , Prokka, Roary, RAXML, iTOL Statistical analyses: R\n",
      "statistical software 3.5.1, GraphPad Prism v6.01\n",
      "For manuscripts utilizing custom algorithms or software that are central to the research but not yet described in published literature, software must be made available to editors/reviewers. We strongly encourage code deposition in a community repository (e.g. GitHub). See the Nature Research guidelines for submitting code & software for further information.\n",
      "Data\n",
      "\n",
      "Policy information about availability of data\n",
      "All manuscripts must include a data availability statement. This statement should provide the following information, where applicable:\n",
      "\n",
      "- Accession codes, unique identifiers, or web links for publicly available datasets\n",
      "- A list of figures that have associated raw data\n",
      "- A description of any restrictions on data availability\n",
      "Raw 16S sequence reads can be found in the NCBI SRA associated with Bioproject PRJNA525701. Bacteriophage raw sequence reads and annotated genomes are\n",
      "Corresponding author(s): Bernd Schnabl\n",
      "Last updated by author(s): Sep 5, 2019\n",
      "n a t u r e r e s e a r c h | r e p o r t i n g s u m m a r y O c t o b e r 2 0 1 8 1\n",
      "n a t u r e r e s e a r c h | r e p o r t i n g s u m m a r y O c t o b e r 2 0 1 8 2\n",
      "available at NCBI under the following consecutive BioSample IDs (SAMN11089809 – SAMN11089827). Genome sequence data of E. faecalis strains isolated in this study were registered at ENA under Study PRJEB25007.\n",
      "Field-specific reporting\n",
      "Please select the one below that is the best fit for your research. If you are not sure, read the appropriate sections before making your selection. Life sciences Behavioural & social sciences Ecological, evolutionary & environmental sciences For a reference copy of the document with all sections, see nature.com/documents/nr-reporting-summary-flat.pdf\n",
      "Life sciences study design\n",
      "All studies must disclose on these points even when the disclosure is negative.\n",
      "Sample size No power analyses or other calculations were used to predetermine sample sizes. Sample sizes were chosen based on prior literature using similar experimental paradigms (Nat Commun. 2017;8:2137; Gut. 2019;68:1504-1515) Data exclusions No data were excluded Replication In vivo experiments: more than two technical replicates (from different cohorts, on different dates), as well as biological replicates were performed to ensure data reproducibility; In vitro experiments: three\n",
      "independent experiments and also replicates were performed on different dates to ensure data reproducibility. All replications were successful. Randomization Mice of similar age and weight were randomly assigned to experimental and control groups. Blinding The investigators were not blinded during cell and animal experiment assays.\n",
      "Reporting for specific materials, systems and methods\n",
      "\n",
      "We require information from authors about some types of materials, experimental systems and methods used in many studies. Here, indicate whether each material, system or method listed is relevant to your study. If you are not sure if a list item applies to your research, read the appropriate section before selecting a response.\n",
      "Methods n/a Involved in the study Antibodies ChIP-seq Eukaryotic cell lines Flow cytometry Palaeontology MRI-based neuroimaging Animals and other organisms Human research participants Clinical data\n",
      "Animals and other organisms\n",
      "Policy information about studies involving animals; ARRIVE guidelines recommended for reporting animal research Laboratory animals Female and male C57BL/6 mice (age, 9-12 weeks) (strain: wild type, Atpdasl/sl) Wild animals No wild animals were involved in the study. Field-collected samples No field-collected samples were involved in the study\n",
      "Ethics oversight All animal studies were reviewed and approved by the Institutional Animal Care and Use Committee of the University of California, San Diego.\n",
      "Note that full information on the approval of the study protocol must also be provided in the manuscript.\n",
      "Human research participants\n",
      "\n",
      "Policy information about studies involving human research participants\n",
      "Population characteristics\n",
      "Alcoholic hepatitis patients were from multiple centers from United States, Mexico and Europe, with the age ranged from 30 to\n",
      "Population characteristics 75. Alcohol use disorder patients and non-alcoholic controls were from United States and Europe, with the age ranged from 27 to 74. Both genders were included in all populations. Detailed descriptions in Methods, Extended Data Tables 1 and 2\n",
      "Recruitment\n",
      "Patients with alcohol use disorder fulfilling the DSM IV criteria (J Abnorm Psychol. 1997;106:545-553) were recruited from an alcohol withdrawal unit in San Diego, USA and Brussels, Belgium where they followed a detoxification and rehabilitation program. Alcoholic hepatitis patients were enrolled from the InTeam Consortium (ClinicalTrials.gov identifier number: NCT02075918) from centers in the USA, Mexico, United Kingdom, France and Spain. Detailed inclusion and\n",
      "exclusion criteria are listed in Methods\n",
      "Ethics oversight\n",
      "The protocol was approved by the Ethics Committee of Hopital Huriez (Lille, France), Universidad Autonoma de Nuevo Leon (Monterrey, México), Hospital Universitari Vall d'Hebron (Barcelona, Spain), King's College London (London, UK), Yale University (New Haven, USA), University of North Carolina at Chapel Hill (Chapel Hill, USA), Weill Cornell Medical College (New York, USA), Columbia University (New York, USA), University of Wisconsin (Madison, USA), VA San Diego\n",
      "Healthcare System (San Diego, USA) and Université Catholique de Louvain (Brussels, Belgium).\n",
      "Note that full information on the approval of the study protocol must also be provided in the manuscript.\n",
      "n a t u r e r e s e a r c h | r e p o r t i n g s u m m a r y O c t o b e r 2 0 1 8 3\n",
      "RES EARCH\n",
      "MICROBIOME\n",
      "Lactose drives Enterococcus expansion to promote graft-versus-host disease\n",
      "C. K. Stein-Thoeringer1,2,3, K. B. Nichols1,2, A. Lazrak1,2, M. D. Docampo1,2, A. E. Slingerland1,2, J. B. Slingerland1,2, A. G. Clurman4, G. Armijo1,2, A. L. C. Gomes1,2, Y. Shono1,2, A. Staffas1,2, M. Burgos da Silva1,2, S. M. Devlin5, K. A. Markey1,2,4, D. Bajic6, R. Pinedo7, A. Tsakmaklis8,9, E. R. Littmann1,10, A. Pastore1, Y. Taur11, S. Monette12, M. E. Arcila13, A. J. Pickard14, M. Maloy4, R. J. Wright1, L. A. Amoretti1, E. Fontana1, D. Pham15, M. A. Jamal15, D. Weber16, A. D. Sung17, D.\n",
      "Hashimoto18, C. Scheid8, J. B. Xavier19, J. A. Messina20, K. Romero21, M. Lew17, A. Bush17, L. Bohannon17, K. Hayasaka22, Y. Hasegawa18, M. J. G. T. Vehreschild8,9,23, J. R. Cross14, D. M. Ponce2,4, M. A. Perales2,4, S. A. Giralt2,4, R. R. Jenq15, T. Teshima18,22, E. Holler16, N. J. Chao17, E. G. Pamer1,2,10, J. U. Peled1,2,4*†, M. R. M. van den Brink1,2,4*†\n",
      "Disruption of intestinal microbial communities appears to underlie many human illnesses, but the mechanisms that promote this dysbiosis and its adverse consequences are poorly understood. In patients who received allogeneic hematopoietic cell transplantation (allo-HCT), we describe a high incidence of enterococcal expansion, which was associated with graft-versus-host disease (GVHD) and mortality. We found that Enterococcus also expands in the mouse gastrointestinal tract after allo-HCT and\n",
      "exacerbates disease severity in gnotobiotic models. Enterococcus growth is dependent on the disaccharide lactose, and dietary lactose depletion attenuates Enterococcus outgrowth and reduces the severity of GVHD in mice. Allo-HCT patients carrying lactose-nonabsorber genotypes showed compromised clearance of postantibiotic Enterococcus domination. We report lactose as a common nutrient that drives expansion of a commensal bacterium that exacerbates an intestinal and systemic inflammatory disease.\n",
      "community of mostly anaerobic bacteria, and a hallmark of microbial imbalance (dysbiosis) observed in many disease states involves the expansion of facul- tative anaerobic bacteria (1). Enterococci are facultative anaerobes that colonize the intes- tines of almost every species, from insects to mammals (2), and make up a very small proportion (<0.1%) of the gut microbiota in healthy humans (3). However, enterococci are also pathogens; the species Enterococcus faecium and Enterococcus faecalis\n",
      "are impor- tant causes of multidrug-resistant infections in patients (4). In single-center studies, E. faecium has been observed to dominate the fecal microbiota of immunocompromised patients after allogeneic hematopoietic cell transplantation (allo-HCT), a curative-intent therapy for hematological malignancies (5–7). Moreover, fecal domination with vancomycin- resistant enterococci increases the risk of\n",
      "bloodstream infection in allo-HCT patients (5, 8). Patients with severe graft-versus-host disease (GVHD) after allo-HCT have poor outcomes with only ~30% long-term survival (9). Gut microbiota perturbations caused by broad-spectrum antibiotics and a reduction in microbial diversity are associated with in- creased transplant-related mortality and lethal GVHD in humans and mice (10–13). Besides causing infections, experimental studies in gno- tobiotic mice have revealed that enterococci play an\n",
      "important role in colitis (14) by stimulat- ing antigen-presenting cells and CD4+RORg+ T cell infiltration, causing intestinal inflam- mation (15). In this study, we investigated the role of enterococci in the development of acute GVHD, both in allo-HCT patients and preclinical allo-HCT mouse models.\n",
      "We used 16S ribosomal RNA (rRNA) gene sequencing to study the fecal microbiota of 1325 adult allo-HCT recipients at four HCT centers: Memorial Sloan Kettering Cancer\n",
      "Center (MSKCC) (United States), Duke Uni- versity (United States), Hokkaido University (Japan), and University Hospital Regensburg (Germany). Patient characteristics are shown in table S1. We observed high abundance of enterococci soon after transplantation in samples from all four transplant centers (Fig. 1B and fig. S1B). We defined Enterococcus domination as relative genus abundance ≥0.3 (≥30%) in any fecal sample, following a thresh- old we have used previously (5) (materials and methods and\n",
      "fig. S2C). The incidence of domination rose comparably across cen- ters, with up to 65% of patients exhibiting a domination event after allo-HCT (Fig. 1A). E. faecium was the dominant species in both the MSKCC and the multicenter-validation cohort (Duke, Hokkaido, and Regensburg) (Fig. 1B, fig. S1, and table S2), where 40.1% of MSKCC patients (441 of 1101 patients) and 46.0% of multicenter-validation patients (103 of 224 patients) met criteria for domination at any time point between day −20 and\n",
      "day +80 relative to the date of allo-HCT, in which cells are infused on day 0.\n",
      "early posttransplant period (day 0 to +21) was associated with significantly reduced overall survival and increased GVHD-related mortality in both the MSKCC and multicenter- validation cohort, as well as an increased risk of moderate-to-severe acute GVHD in the MSKCC cohort (Fig. 1, C and D, fig. S2, A and B, and table S3). The risk of relapse or disease progression was not associated with enterococcal domination in either cohort. The association of domination by genus Enterococcus with clinical\n",
      "outcomes in the MSKCC cohort remained significant in a multivariate analysis adjusted for graft source, disease, conditioning intensity, gender, and age (table S4). In a subset of MSKCC patients, the vanA operon was found in 152 (37.4%) of 406 patients that had samples available for analysis, indicating the presence of vancomycin-resistant enterococci (VRE) (fig. S2E). Notably, expansions of several different taxa were detected in fecal samples in this study, but the Enterococcus genus was the\n",
      "one most commonly observed dominating the microbiota in all four transplant centers (fig. S3 and tables S5 and S6).\n",
      "1Department of Immunology, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 2Weill Cornell Medical College, New York, NY, USA. 3German Cancer Research Center (DKFZ), Heidelberg, Germany. 4Adult Bone Marrow Transplantation Service, Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 5Epidemiology and Biostatistics, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 6Department of Internal Medicine II, Technical University\n",
      "of Munich, Munich, Germany. 7Gnotobiotic Facility, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 8Department I of Internal Medicine, Center for Integrated Oncology Aachen Bonn Cologne Duesseldorf, University of Cologne, Cologne, Germany. 9German Center for Infection Research, Partner site Bonn-Cologne, Cologne, Germany. 10Department of Medicine, Section of Infectious Medicine and Global Health, University of Chicago, Chicago, IL, USA. 11Infectious Disease Service, Department of\n",
      "Medicine, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 12Laboratory of Comparative Pathology, Memorial Sloan Kettering Cancer Center, The Rockefeller University, Weill Cornell Medicine, New York, NY, USA. 13Diagnostic Molecular Pathology Laboratory, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 14Donald B. and Catherine C. Marron Cancer Metabolism Center, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 15Department of Genomic Medicine, The University of\n",
      "Texas MD Anderson Cancer Center, Houston, TX, USA. 16Internal Medicine III, University Clinic Regensburg, Regensburg, Germany. 17Division of Hematologic Malignancies and Cellular Therapy, Department of Medicine, Duke University Medical Center, Durham, NC, USA. 18Department of Hematology, Hokkaido University, Faculty of Medicine, Sapporo, Japan. 19Computational and Systems Biology Program, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 20Division of Infectious Diseases, Department of\n",
      "Medicine, Duke University, Durham, NC, USA. 21Office of Clinical Research, Duke University School of Medicine, Durham, NC, USA. 22Division of Laboratory and Transfusion Medicine, Hokkaido University Hospital, Sapporo, Japan. 23Department of Internal Medicine, Infectious Diseases, Goethe University Frankfurt, University Hospital Frankfurt, Frankfurt am Main, Germany.\n",
      "*These authors contributed equally to this work.\n",
      "\n",
      "†Corresponding author. Email: peledj@mskcc.org (J.U.P.); vandenbm@mskcc.org (M.R.M.v.d.B.)\n",
      "\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019) 29 November 2019\n",
      "29 November 2019\n",
      "1 of 7\n",
      "\n",
      "D o w n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "RES EARCH | R E P O R T\n",
      "To further investigate these clinical obser- vations, we examined the fecal microbiota of mice early after transplantation using well- established mouse models of allo-HCT. In a major histocompatibility complex (MHC)– matched, minor-antigen-mismatched allo-HCT model [C57BL/6-to-129S1/Sv transplant (C57BL/ 6→129S1/Sv)], we performed 16S rRNA gene sequencing of fecal samples and found that E. faecalis dominated the fecal microbiota at posttransplant day +8 in mice who re- ceived T cell–replete\n",
      "grafts and developed lethal, acute GVHD (Fig. 2A and fig. S4A). In contrast to the patients who had pro- longed antibiotic exposures, this expansion of E. faecalis was independent of antibiotic administration and dependent upon GVHD, as it was not observed in control recipients of T cell–depleted allografts in which GVHD did\n",
      "not develop. This posttransplant expansion of enterococci was consistently found in two additional lethal GVHD models: C57BL/6→ BALB/c mice (MHC-disparate model after ir- radiation conditioning) (Fig. 2B) and LP/ J→C57BL/6 mice [MHC-matched, minor- antigen–mismatched after busulfan and cyclophosphamide conditioning (16)] (Fig. 2C). The expansion of enterococci in murine allo- HCT recipients with GVHD was accompanied by an increase in Enterococcus colony–forming units recovered from mesenteric\n",
      "lymph nodes, consistent with increased bacterial translo- cation (Fig. 2B).\n",
      "Although we observed E. faecium domina- tion in patients and a transient expansion of E. faecalis in GVHD mice, we hypothesized that both members of this genus might be associated with GVHD. Of note, E. faecium\n",
      "only recently became recognized as a major human pathogen; before the 1990s it was E. faecalis that caused >90% of clinical in- fections (17). Because E. faecalis expands in mice with GVHD and is the major Enterococcus species in laboratory mice, we next investigated whether E. faecalis contributes to GVHD. We colonized germ-free C57BL/6 mice with a com- munity of six bacterial strains (Akkermansia muciniphila, Lactobacillus johnsonii, Blautia producta, Bacteroides sartorii, Clostridium bolteae,\n",
      "and Parabacteroides diastonis; see materials and methods) (10, 18, 19) 21 days prior to allo-HCT (LP/J→gnotobiotic C57BL/6). One group of mice was cocolonized on day −21 with E. faecalis OG1RF, which remained detectable in mouse feces on days 0 and +7 (Fig. 2D, right panel, and fig. S4E). GVHD was exacerbated in E. faecalis–harboring mice\n",
      "A\n",
      "Incidence of domination\n",
      "0.75 n f o o i e t a c 0.50 n n i e m d o i c d n i s u e c v c i t o a 0.25 c l o u m r e t u n c E 0 20 0 -20 day relative to HCT\n",
      "\n",
      "Prevalence of domination MSKCC 0.6 Duke s e Hokkaido l p Regensburg m a s d 0.4 e t f o a n n i m o i t o c d a r s 0.2 f u c c o c o r e t n E 0 -20 20 0 day relative to HCT\n",
      "«\n",
      "\n",
      "=\n",
      "\n",
      "«\n",
      "B\n",
      "E. faecium other E. spp. 1.00 E. faecium 0.60 E. cecorum M S E. durans 0.30 K C E. mundtii C 0.10 e c E. rivorum n a d E. moraviensis 0.01 n u E. gallinarum b a v 1.00 a E. lactis e l v i i d t E. malodoratus a a 0.60 l t e i r o E. ureasiticus n 0.30 c E. faecalis o h o 0.10 r t 0.01 20 60 80 −20 0 60 40 20 40 0 −20 80 day relative to HCT\n",
      "®\n",
      "\n",
      "®\n",
      "\n",
      "®\n",
      "%\n",
      "Fig. 1. Enterococcus domination occurs globally and increases risk of GVHD and mortality after allo-HCT. Fecal microbiota were profiled using 16S rRNA gene sequencing of 1325 adult allo-HCT recipients. The patients attended one of four HCT centers in different countries: MSKCC (United States), Duke University (United States), Hokkaido University (Japan), and University Hospital Regensburg (Germany). (A) (Left) Cumulative incidence of patients who experienced at least one instance of genus\n",
      "Enterococcus domination of the gut microbiota [domination defined as a genus relative abundance of ≥0.3 (on a unitless scale from 0 to 1) over the course of allo-HCT (day −20 to +24 relative to HCT; using 7-day sliding windows) at different transplant centers]. (Right) Fraction of fecal specimens with enterococcal domination of the gut microbiota. (B) Relative abundance of different Enterococcus spp. in the microbiota of allo-HCT patients from the MSKCC and multicenter-validation cohort over the\n",
      "course of HCT, determined by 16S rRNA gene sequencing of fecal samples. Each\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019)\n",
      "\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019) 29 November 2019\n",
      "\n",
      "29 November 2019\n",
      "spp.\n",
      "C\n",
      "\n",
      "D\n",
      "Overall survival 1.0 0.8 g n i v i v 0.6 r u s n o i t 0.4 r o p o Enterococcus ≥ 0.3 (n=149) r p < 0.3 (n=389) Enterococcus 0.2 HR, 1.97; p < 0.001 0 15 10 5 0 20 months after HCT\n",
      "\n",
      "1.0 0.8 g n i v i 0.6 v r u s n o i 0.4 t r o p Enterococcus ≥ 0.3 (n=54) o r p Enterococcus < 0.3 (n=113) 0.2 HR, 1.95; p = 0.03 0 0 5 15 20 10 months after HCT\n",
      "GVHD-related mortality\n",
      "Enterococcus ≥ 0.3 (n=148)\n",
      "—\n",
      "Enterococcus < 0.3 (n=389)\n",
      "---\n",
      "0.2\n",
      "\n",
      "HR, 2.04; p = 0.01\n",
      "e c n e d i c n i e v i t a l u m u c\n",
      "\n",
      "4\n",
      "0.15\n",
      "\n",
      "0.1\n",
      "4\n",
      "0.05\n",
      "\n",
      "0\n",
      "o1\n",
      "10\n",
      "\n",
      "5\n",
      "\n",
      "15\n",
      "\n",
      "20\n",
      "months after HCT\n",
      "GVHD-related mortality Enterococcus ≥ 0.3 (n=53) < 0.3 (n=111) Enterococcus 0.2 HR, 5.8; p = 0.002 e c n e 0.15 d i c n i e 0.1 v i t a l u m 0.05 u c 0 20 5 15 10 0\n",
      "point represents a fecal sample, and color indicates the different Enterococcus spp.; the red dotted line indicates the threshold for domination set at a relative abundance ≥0.3. (C) Overall survival (left) and cumulative incidence of GVHD- related mortality (right) in the T cell replete graft recipients in the MSKCC patient cohort (see table S3), stratified into nondominated and Enterococcus- dominated groups (domination is defined as the relative genus abundance ≥0.3 in at least one sample\n",
      "between day 0 and +21). n, number of individuals. (D) Overall survival (left) and cumulative incidence of GVHD-related mortality (right) in Enterococcus-dominated (at genus level) versus nondominated allo-HCT patients in the combined multicenter-validation cohort (table S3). Clinical outcomes in (C) and (D) were analyzed using the R packages survival and cmprsk. Wald values of P <0.05 signify higher risks (HR, hazard ratios) of mortality among patients with Enterococcus domination as compared\n",
      "with those without domination.\n",
      "2 of 7\n",
      "D\n",
      "o\n",
      "n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "RES EARCH | R E P O R T\n",
      "A\n",
      "C57BL/6 = 12951/Sv Survival Enterococcus Akkermansia 100 0.5 M Bacteroides M Blautia x50 rel. abundance 0 Pl i Clostridium 0 10 20 30 0 10 20 30 0O 10 20 30 0 10 20 30 0 10 20 30 [ Lactobacillus 1.0 Oscillospira A p=0013 W Roseburia 0 20 40 60 80 100 days after HCT 0.5 — BM (n=5) — BM+T (n=5) rel. abundance 0 0 10 20 30 10 20 30 O 10 20 30 0 10 20 30 0 10 days after HCT C57BL/6 — BALB/c 16S rRNA feces o before HCT Enterococcus feces Enterococcus mLN Survival e BM -~ 4x10* e BM 10°% e BM+T 8 ®\n",
      "BM+T 100 10\" ° 1 3x10* — BM (n=20) 102 107 2x10* — BM+T ® 50 102 10¢ g Enterococcus el.abundance 10+ A 19 o CFUs/g stool CFUs/mLN 1x10* 10° ° ) 10° o p<0.001 10 -1 4 7 10 10 7 10 0 5 10 15 20 days after HCT days after HCT days after HCT days after HCT c LP/J C57BL/6 16S rRNA feces Survival 14 days rel. to HCT o before bu/cy-cond 100 @ before HCT VVVYVYY v 1074 e BM ® BM+T — BM (n=9) AAA 1074 ¥ 50 g — BM+T (n=10) V busulfan g 'V cyclophosphamide Enterococcus rel. abundance 10° o 8 HCT p<0.01 10\n",
      "-7 -1 10 0 20 40 60 80 100 days after HCT days after HCT D LP/ - C57BL/6 Survival 16S rRNA feces -7 days rel. to HCT B 0 m -21 100 EF] (n=4) - day 0 day 7 v \\AAAA v > o AAA BM [+ EF] (n=5) BM+T [- EF] (n=10) Se 'V colonization V busulfan 50 | BM+T [+ EF] (n=12) o e -1+ EF g WV cyclophosphamide WV HCT p<0.001 Enterococcus rel. abundance 0 0 20 40 60 80 HCT: - BM BM+T days after HCT\n",
      "(n=32)\n",
      "Fig. 2. Enterococcus dominates mouse gut microbiota after HCT and can exacerbate GVHD. (A) (Left) High-density sampling and 16S rRNA gene sequencing of fecal microbiota from 129S1/Sv mice (1 box = 1 mouse) receiving bone marrow (BM; upper row) or T cell–replete bone marrow [BM+T (2 × 106 T cells); lower row]. (Right) BM+T transplanted mice develop lethal GVHD as shown by survival analysis. rel., relative. (B) (Left) Relative abundance of the genus Enterococcus in BALB/c host mice transplanted\n",
      "with C57BL/6 BM or BM+T (1 × 106 T cells) at different time points relative to HCT. (Middle) Colony-forming units (CFUs) of enterococci in fecal samples and in mesenteric lymph nodes (mLN). Scatter plot data show means ± SEM. (Right) Survival of BALB/c recipient mice after HCT [BM versus BM+T (1 × 106 T cells)]. (C) (Left) Schematic showing HCT of LP/J BM versus BM+T (4 × 106 T cells) into C57BL/6 mice after chemotherapy conditioning. (Middle) Relative abundance of the genus Enterococcus in the\n",
      "feces of\n",
      "transplanted mice at different time points relative to HCT. (Right) Comparison of overall survival between BM and BM+T mice. bu/cy-cond., busulfan/ cyclophosphamide conditioning. (D) (Left) Schematic showing colonization of germ-free C57BL/6 mice with a 6-strain minimal microbiota with (+EF) or without (-EF) E. faecalis OG1RF (2 × 107 CFUs per mouse); after 14 days, colonized mice received chemotherapy conditioning with busulfan and cyclophosphamide and, subsequently, an HCT of LP/J BM versus\n",
      "BM+T (4 × 106 T cells). (Middle) Comparison of overall survival. (Right) Relative abundances of E. faecalis spiked to the minimal microbiota in the EF+ group with samples collected at the day of HCT (day 0) and 7 days later (n = 4 to 11 mice per group; P = 0.09, paired testing of relative abundances of enterococci of day 0 versus BM+T day +7). Scatter plot data are presented as means ± SEM **P < 0.01, ***P < 0.001 (independent t test for BM versus BM+T); survival data were statistically analyzed\n",
      "using Mantel-Cox log-rank test.\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019)\n",
      "\n",
      "29 November 2019\n",
      "3 of 7\n",
      "\n",
      "o w n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "RES EARCH | R E P O R T\n",
      "A allo HCTin AML patients == pre HCT/ non-dom. (i == post HCT/ domin. { . . . Pyrimidine déoxyriboncl. salvage [ Superpath. alanine biosyn. ‘Guarlosine ribancl. denovo biosyn. Adenosine riboncl. denovo biosyn. [ Pyuvate feri. to acetate actate N Lactose and galctose degradation [N Calvin-Banson-Bassham cycld GDP-mannose biosyn. Superpath. Lysine, Lhreon, Lmeth. Glycolysis IV P-diacylglycerol biosyn. | CDP-diacylglycerol biosyn. Il Superpath. aromatic AA biosyn. N -2denosyl-L-methionine cycte | I\n",
      "Chorismate biosyn. 2ch degradataion V i hoshote | 3 E B 0 1 2 3 LDA score (log 10)\n",
      "A allo HCTin AML patients pre HCT (diverse) == pre HCT/ non-dom. (i M Enterococcus == post HCT/ domin. { . . . Pyrimidine déoxyriboncl. salvage [ B Akkermansia Superpath. alanine biosyn. W Blautia ‘Guarlosine ribancl. denovo biosyn. Adenosine riboncl. denovo biosyn. [ W Clostridium Pyuvate feri. to acetate actate N Lactose and galctose degradation [N Dorea Calvin-Banson-Bassham cycld Eubacterium GDP-mannose biosyn. Superpath. Lysine, Lhreon, Lmeth. biosyn. ost HCT Glycolysis IV (E. F. domin.) I\n",
      "Lactobacillus P-diacylglycerol biosyn. | Faecalibacterium CDP-diacylglycerol biosyn. Il Superpath. aromatic AA biosyn. 1| Oscillospira N -2denosyl-L-methionine cycte | I Chorismate biosyn. Ruminococcus 2ch degradataion V i hoshote | [ Streptococcus 3 E B 0 1 2 3 W Veillonella LDA score (log 10) I Other\n",
      "pre HCT (diverse) M Enterococcus B Akkermansia W Blautia W Clostridium Dorea Eubacterium Lmeth. biosyn. ost HCT (E. F. domin.) I Lactobacillus Faecalibacterium 1| Oscillospira | Ruminococcus | [ Streptococcus W Veillonella I Other\n",
      "B C57BL/6 — 12951/Sv\n",
      "M\n",
      "\n",
      "B\n",
      "W\n",
      "\n",
      "W\n",
      "I\n",
      "\n",
      "1|\n",
      "day 7 after HCT = BM (n=5) == BM+T (n=5) Lactose and galactose degradation — Thiamine formation from pyrithiamine Pyrimidin deoxyriboncl. salvage Lysine iosynthesis hamnose degradation | ! s-aminoiidazole ribdncl. biosyn supepath 5+ aminoimid. fiboncl.biosyn. Inogine- Inosine- phosph biosyn. | — Peptifloglycanbigsynthesis || hosph biosyn. I I scrine and ycne biosyn. | S-adenosyL-methionne cycle | -1 0 i 2 LDA score (log 10)\n",
      "=\n",
      "\n",
      "==\n",
      "\n",
      "[\n",
      "W\n",
      "I\n",
      "D\n",
      "invitro Enterococcus growth\n",
      "E. faecalis - ° & Incubation in 0D600 nm regular BHI 02462810 hours < Regular BHI -+ Lactase pretreatment\n",
      "15 0.0 —— = 024681 hours\n",
      "E. faecium - Regular BHI -+ Lactase pretreatment 15 - - ° 10 Incubation in 0D600 nm o 0D600 nm 05 regular BHI | [= 0.0 0246 810 024 8 10 hours hours\n",
      "\n",
      "- Regular BHI -+ Lactase pretreatment 15 - 10 0D600 nm 05 0.0 024 8 10 hours\n",
      "\n",
      "- ° Incubation in 0D600 nm o regular BHI | [= 0246 810 hours\n",
      "F Clostridum vs. Enterococcus (genera) in allo-HCT patients Kendall tau =-0.21; p<0.001 1 3 o = 3 2 B o Clostridium 3 2 rel. abundance - 0 107 107 1 Enterococcus rel. abundance\n",
      "Butyrate in C57BL/6 — BALB/c\n",
      "\n",
      "cecal content . 100- * BM BM+T 50- B butyrate pm/mg\n",
      "or in BHI broth pretreated with lactase. (Right) E. faecalis or E. faecium incubated in lactase-pretreated BHI were put into regular BHI broth after 8 hours to assess growth dynamics in regular BHI (gray symbols). Data are results of four experiments combined; values represent means ± SEM. (E) (Left) Fecal butyrate concentrations (means ± SEM) from pre- and posttransplant fecal samples from AML patients from MSKCC, who received allo-HCT and were selected based on a highly diverse pre-HCT\n",
      "microbiota and a posttransplant E. faecium domination [by 16S rRNA gene sequencing; 6 (out of 8) patients are presented in (A)]. (Right) Correlation of butyrate concentrations with relative abundances of the genus Enterococcus (n = 139 patients; 8 patients from left panel), and 131 allo-HCT patients from a dataset published by Haak et al. (28); statistical analysis was performed using Kendall’s tau rank correlation coefficient. (F) Stool samples were collected at the time of engraftment (~24\n",
      "days after allo-HCT). Data show Kendall’s tau rank correlation of relative abundances of the genera Clostridium and Enterococcus from the dataset of Haak et al. (G) Butyrate concentration (mean ± SEM) in cecal contents of BALB/c mice transplanted with C57BL/6 BM or BM+T (1 × 106 T cells) at day +7 after HCT. Statistical analysis: *P < 0.05 [paired t test (E) or independent t test (F)].\n",
      "4 of 7\n",
      "==\n",
      "\n",
      "==\n",
      "C\n",
      "Metagenomic metabolic pathways Enterococcus isolates\n",
      "\n",
      "Amino sugar and nucleotide Pyrimidine metabolism sugar metabolism Purine metabolism Lactose and ‘metabolism alactose AminoacytRNA meta Glycolysis/ gluconeogenesis Pyruvate metabolism Arginine proline metab. Methane metabolism Cysteine methionine metab. Glycine serine threonine metab. Other pathways Oxidative phosphorilation E. faecalis (BALB/c mice, GVHD) Pentose phosphate pathway Fructose mannose metabolism\n",
      "E. faecium (human isolate)\n",
      "Amino sugar and nucleotide sugar metabolism Pyrimidine metabolism Glycolysis/ Purine metabolism gluconeogenesis Sucrose metabolism Methane metabolism Fructose mannose metabolism l Pentose pathway Pyruvate metabolism AminoacyHRNA metab. Other pathways Pentose glucuronate interconversions Glycine serine threonine metabolism\n",
      "E Butyrate in allo-HCT patients\n",
      "feces 15 —_— © preHCT postHCT 10 butyrate mM\n",
      "©\n",
      "Kendall tau =-0.15; p = 0.008 20 0o butyrate mM oo 3 - orts o 0 107 10 _I- Enterococcus rel. abundance\n",
      "Fig. 3. Metagenomic and metabolomic analyses of Enterococcus-dominated fecal specimens in human HCT patients and mice. (A) (Left) Differential abundances of shotgun-sequenced and HUMAnN2-annotated bacterial metabolic pathways between paired pre- and post-HCT fecal samples from MSKCC patients who received allo-HCT for acute myeloid leukemia (AML) analyzed by linear discriminant analysis (LDA) coupled with effect size measurements (LEfSe). pre HCT, day −8 to −1 before allo-HCT; post HCT, day +3 to\n",
      "+25 after allo-HCT. (Right) Pie chart showing mean relative abundances of bacterial genera (analyzed by MetaPhlAn2) found in patient fecal samples pre- and post-HCT; data are aggregated across all patients. non-dom., nondominated; domin., dominated. (B) LEfSe analysis of bacterial metabolic pathway abundances in HCT day +7 fecal samples of 129S1/SV mice transplanted with C57BL/6 BM versus BM+T (2 × 106 T cells) (see Fig. 2A). (C) Pie charts with metabolic pathway abundances determined by\n",
      "whole-genome sequencing of E. faecalis (isolated from feces of a BALB/c GVHD mouse, day +7 after HCT; upper panel) and E. faecium (human isolate; ATCC #700221; lower panel); only pathways with an abundance ≥2% are shown in both panels. (D) (Left) In vitro growth of E. faecalis (mouse GVHD isolate; upper panel) and E. faecium (ATCC #700221; lower panel) in nontreated BHI broth\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019)\n",
      "\n",
      "29 November 2019\n",
      "D o w n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "RES EARCH | R E P O R T\n",
      "vated and proliferating CD4+ T cells (fig. S4D; CD4+CD25+; CD4+Ki67+), and an increased number and percentage of CD4+RORg+ T helper 17 (TH17) cells in colon lamina propria (fig. S4D). Posttransplant administration of E. faecalis\n",
      "Survival 100 += ' - BM [ctr] (n=10) ' - BM [LF] (n=10) — BM+T [ctr] (n=20) 50 — BM+T [LF] (n=20) survival [%] p<0.05 %020 40 60 80 100 days after HCT\n",
      "' -\n",
      "\n",
      "' -\n",
      "\n",
      "—\n",
      "Survival 100 BM [ctr] (n=8) BM [LF] (n=10) BM+T [ctr] (n=19) 50 BM+T [LF] (n=13) survival [%] p<0.01 0 20 40 60 80 100 days after HCT\n",
      "day-7to 0 1.0 0.5 Enterococcus rel. abundance day 25 1.0 N 05 Enterococcus rel. abundance TT+ / T/C\n",
      "T/C\n",
      "OG1RF to conventionally housed, T cell–replete bone marrow (BM+T)–transplanted BALB/c mice also aggravated GVHD (fig. S5A). These findings indicate that E. faecalis can aggravate GVHD severity.\n",
      "16S rRNA feces\n",
      "\n",
      "16S rRNA feces o BM [ctr] © BMI[LF] ¥ 107 © BM+T [ctr] © BM+T[LF] 102 Enterococcus rel. abundance 10° days r HCT\n",
      "o\n",
      "©\n",
      "\n",
      "©\n",
      "\n",
      "©\n",
      "165 rRNA feces 107! o BM [ctr] o BM[LF] 102 © BM+T [ctr] 10, . © BM+T [LF] Enterococcus 10 2, .1 rel. abundance 10 14 days after HCT\n",
      "o\n",
      "\n",
      "o\n",
      "Acute GVHD (grade 2-4) 06 — — T/T+T 04 0.2 cumulative incidence HR, 1.08; p=0.62 20 40 60 80 100 days after HCT\n",
      "—\n",
      "SEM; *P < 0.05 (independent t test). (C) (Left) Relative abundance (log10) of Enterococcus (genus) by days relative to the day of antibiotic cessation (broad- spectrum antibiotics for neutropenic fever: intravenous piperacillin-tazobactam, intravenous imipenem-cilastatin, or intravenous meropenem). Box plot inserts display the median relative abundances of the genus Enterococcus of time binned in the indicated day ranges relative to antibiotic cessation; whiskers represent maximum and minimum.\n",
      "Statistical analysis of box plot data: *P < 0.05 (Wilcoxon rank test). (D) Cumulative incidence of acute GVHD grade 2 to 4 in rs4988235 SNP-genotyped MSKCC patients (T cell–depleted grafts excluded; graft source: BM/PBSC unmodified = 213 patients; cord blood = 102 patients; C/C = 175, T/C+T/T = 140). The cumulative incidence of grade 2 to 4 acute GVHD was compared between genotype groups using the R package cmprsk.\n",
      "5 of 7\n",
      "(Fig. 2D and fig. S4B). Serum interferon-g con- centrations were significantly elevated in E. faecalis–colonized mice (fig. S4C), and we observed a significantly increased num- ber of donor T cells, an increase of acti-\n",
      "A\n",
      "C57BL/6 — BALB/c days rel. to HCT -7 0 14 v W HCT ctr vs. LF chow\n",
      "W\n",
      "B LP/J = C57BL/6\n",
      "days rel. to HCT -7 -3 14 YVVYV V AAA W busulfan W HT W cyclophosphamide ctrvs. LF chow\n",
      "W\n",
      "\n",
      "W\n",
      "C Allo HCT patients: Enterococcus abundance by lactase genotype\n",
      "\n",
      "0 o { - 153 -1 ’ -2 Enterococcus log10 rel. abundance -4 -20 -10 0 10 20 days relative to antibiotic termination T/T+T/C ~ C/C lactose absorber non-absorber\n",
      "\n",
      "day-7to 0 1.0 0.5 Enterococcus rel. abundance day 25 1.0 N 05 Enterococcus rel. abundance TT+ / T/C\n",
      "~\n",
      "Fig. 4. Lactose-free diet reduces experimental GVHD and lactase genotypes associated with microbiota dynamics after allo-HCT in humans. (A) (Left) Schematic showing that BALB/c recipient mice received C57BL/6 BM or BM+T (5 × 105 T cells) and were fed control chow (ctr) versus lactose-free chow (LF) from day −7 to +14 relative to transplant. Comparison of survival between BM and BM+T mice (middle) and relative abundance of the genus Enterococcus in mouse feces (right) are shown. Scatter plot data\n",
      "presented as means ± SEM; *P < 0.05 (independent t test). (B) (Left) Schematic showing HCT of LP/J BM versus BM+T (4 × 106 T cells) into C57BL/6 mice after chemotherapy conditioning. Comparison of survival between BM and BM+T mice (middle) and relative abundance of the genus Enterococcus at different time points relative to HCT (right) are shown. Scatter plot data presented as means ±\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019)\n",
      "\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019) 29 November 2019\n",
      "o w n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "RES EARCH | R E P O R T\n",
      "We next considered whether posttransplant defects of mucosal defense mechanisms facil- itate enterococcal expansion. Immunoglobulin A (IgA) coating of intestinal bacteria can be protective in colitis and is important for maintaining mucosal integrity (20). How- ever, we did not observe members of the genus Enterococcus to be enriched in either IgA- negative or IgA-positive fecal fractions, even though total fecal IgA was significantly re- duced in allo-HCT recipients with GVHD (fig. S5, B to D).\n",
      "Reduction of IgA by transplant- ing IgA-deficient bone marrow (BM) from activation-induced cytidine deaminase knock- out mice did not further increase enterococcal expansion (fig. S5E). Intestinal antimicrobial peptides of the Reg3 family can suppress the growth of VRE (21) and are reported to play a major role in GVHD (22). Accordingly, we found that both Reg3B/G transcripts and interleukin-22 protein, which regulates Reg3 expression (23), were reduced in the ileum of GVHD mice (fig. S5F).\n",
      "Next, we analyzed microbiota-intrinsic fac- tors and used shotgun metagenomic sequenc- ing to characterize the metabolic potential of the Enterococcus-dominated fecal microbiota. Pre- and posttransplant fecal samples from MSKCC patients who received allo-HCT for acute myeloid leukemia were selected for sequencing on the basis of having a highly diverse pre-HCT microbiota and posttrans- plant E. faecium domination (by 16S rRNA gene sequencing). We focused on microbial metabolic pathways that\n",
      "specifically charac- terize domination by comparing them with the highly diverse pretransplant microbiota from the same patients. Pathways involved in DNA synthesis and, notably, in lactose and galactose degradation were enriched in the E. faecium–dominated, posttransplant micro- biota. In contrast, amino acid synthesis and starch-degradation pathways were more prev- alent in pretransplant specimens (Fig. 3A). The lactose-and-galactose degradation path- way was also significantly enriched in the\n",
      "posttransplant E. faecalis–dominated samples of mice with GVHD (Fig. 3B). Comparison of whole-genome sequencing from isolates of E. faecium (from a human allo-HCT patient) and of E. faecalis (from a mouse with GVHD) revealed that genes encoding lactose and galactose metabolism accounted for ~3% of their genomes (Fig. 3C). In silico analysis of these Enterococcus genomes and publicly available genomes of other members of the gnotobiotic six-strain consortium revealed that enterococci are\n",
      "specifically enriched in enzymes of the tagatose-type galactose path- way for galactose-to-glucose degradation (24) (materials and methods and fig. S6, A and B). Enterococcal growth depends on lactose in vitro, as both E. faecalis and E. faecium strains cultured in brain-heart\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019)\n",
      "infusion (BHI) broth depleted of lactose (by lactase; fig. S7A) did not grow (Fig. 3D). Growth was reinstated upon transfer to re- gular BHI, excluding antibacterial effects of lactase treatment (Fig. 3D). Enterococcal expansion after allo-HCT was accompanied by a loss of Clostridium spp. in the microbiota of allo-HCT patients (Fig. 3, A and F) and of mice with GVHD (fig. S8, A to C, and table S7). This may be important for allo-HCT patients, as high abundances of clostridia are associated with\n",
      "better survival and low- er incidence of GVHD (12, 25). Commen- sal clostridia are known to produce large amounts of butyrate (26), which mitigates lethal GVHD in mice through protecting energy homeostasis of enterocytes (27). We observed that posttransplant enterococcal domination and a loss of clostridia were accompanied by a significant reduction in fecal butyrate in both allo-HCT patients and mice with GVHD (Fig. 3, E and G) (28). A loss of this key metabolite may contribute to the poor\n",
      "outcomes in Enterococcus-dominated patients and mice.\n",
      "Given that the optimal growth of enterococci depends on lactose availability in vitro, we investigated whether enterococcal expansion can be mitigated by feeding mice lactose-free chow (fig. S7B and table S8). In the C57BL/ 6→BALB/c model, the absence of dietary lactose significantly reduced posttransplant Enterococcus bloom and mitigated GVHD (Fig. 4A and fig. S9B). Flow cytometric anal- ysis of donor T cells on day +14 revealed a reduction in the percentage of activated and proliferating CD4+\n",
      "T cells (CD4+CD69+; CD4+ Ki67+) as well as a reduction in the percent- age of CD4+Tbet+ (TH1) T cells (fig. S9). The effect of a lactose-free diet on enterococcal outgrowth and GVHD was replicated in the LP/J→C57BL/6 mouse model (Fig. 4B and table S9 for changes in non-enterococcal taxa). Intestinal mucosal damage by irradiation or allo-reactive T cells may affect the expres- sion of lactase, the enzyme found on small- intestine enterocytes that facilitates lactose absorption through\n",
      "disaccharide cleavage. Duodenal lactase transcript abundance pro- gressively declined in BM+T recipients over the course of transplantation (fig. S9C), which may induce a lactose-intolerant–like state in mice, allowing nondigested lactose to reach the lower intestinal tract and serve as a car-\n",
      "Next, we explored whether enterococci ex- pansion is associated with lactose tolerance in human allo-HCT patients by genotyping 602 patients from the MSKCC cohort with available pretransplant germline DNA sam- ples for the gene polymorphism rs4988235 (-13910*T). This single-nucleotide polymorphism (SNP) regulates lactase expression and pre- dicts lactose absorption and/or tolerance (C/T\n",
      "29 November 2019\n",
      "or T/T alleles) and malabsorption (C/C alleles) in the upper gut (29). Although abundance of the genus Enterococcus increased comparably during exposure to broad-spectrum antibiotics in both lactose absorbers and malabsorbers, enterococcal domination was significantly prolonged in malabsorbers after cessation of antibiotics (Fig. 4D and fig. S10). This finding suggests that the maintenance of enterococ- cal domination and microbiota recovery after broad-spectrum antibiotic exposure is signif-\n",
      "icantly modulated by the luminal availability of lactose as a growth substrate.\n",
      "significant risk factor for the development of acute GVHD and for increased overall and GVHD-related mortality after allo-HCT. Our findings extend previous reports from smaller single-center analyses that posttransplant VRE bacteremia and fecal domination are associated with worse outcomes after allo-HCT (7, 8, 30). In gnotobiotic mouse models, enterococci exacerbate GVHD, consistent with previous reports of aggravated colitis in models of in- flammatory bowel disease (14) or systemic autoimmune\n",
      "responses (31). We previously identified Blautia abundance (a genus within class Clostridia) as a predictor of protection from lethal GVHD (12), whereas here we describe Enterococcus domination as a risk factor for GVHD. These two findings are noteworthy in light of our recent observa- tion that a B. producta strain can inhibit VRE growth via the production of a lantibiotic pro- tein (32). We identified a microbiota-intrinsic mechanism that is dependent on lactose uti- lization and favors the\n",
      "expansion of enterococci. This process may be triggered through a loss of lactase produced by enterocytes damaged by conditioning or allo-reactive T cells. We validated this concept experimentally, by showing that depletion of lactose in vitro and in vivo inhibited enterococcal expan- sion and mitigated GVHD, and clinically, by showing that patients harboring a lactose- malabsorption allele experienced prolonged Enterococcus domination after antibiotic ex- posure. These observations in mice and\n",
      "allo- HCT patients provide proof-of-concept for a novel, non–antibiotic-based therapeutic strategy, such as a lactose-free diet, to attenu- ate the outgrowth of pathobionts like enter- ococci and possibly improve clinical outcomes by modulating dietary sources of nutrients for\n",
      "REFERENCES AND NOTES\n",
      "1. Y. Litvak, M. X. Byndloss, A. J. Bäumler, Science 362, eaat9076 (2018).\n",
      "\n",
      "2. M. S. Gilmore, F. Lebreton, W. van Schaik, Curr. Opin. Microbiol. 16, 10–16 (2013).\n",
      "\n",
      "3. S. Schloissnig et al., Nature 493, 45–50 (2013).\n",
      "\n",
      "4. F. Lebreton et al., Cell 169, 849–861.e13 (2017).\n",
      "\n",
      "5. Y. Taur et al., Clin. Infect. Dis. 55, 905–914 (2012).\n",
      "\n",
      "6. C. Ubeda et al., J. Clin. Invest. 120, 4332–4341 (2010).\n",
      "\n",
      "7. E. Holler et al., Biol. Blood Marrow Transplant. 20, 640–645 (2014).\n",
      "6 of 7\n",
      "\n",
      "o w n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "RES EARCH | R E P O R T\n",
      "8. C. D. Ford et al., Biol. Blood Marrow Transplant. 23, 340–346 (2017).\n",
      "\n",
      "9. H. J. Khoury et al., Haematologica 102 , 958–966 (2017).\n",
      "\n",
      "10. Y. Shono et al., Sci. Transl. Med. 8, 339ra71 (2016).\n",
      "\n",
      "11. Y. Taur et al., Blood 124, 1174–1182 (2014).\n",
      "\n",
      "12. R. R. Jenq et al., Biol. Blood Marrow Transplant. 21, 1373–1383 (2015).\n",
      "\n",
      "13. Y. Taur et al., Sci. Transl. Med. 10, eaap9489 (2018).\n",
      "\n",
      "14. N. Steck et al., Gastroenterology 141, 959–971 (2011).\n",
      "\n",
      "15. N. Geva-Zatorsky et al., Cell 168, 928–943.e11 (2017).\n",
      "16. K. Riesner, M. Kalupa, Y. Shi, S. Elezkurtaj, O. Penack, Bone Marrow Transplant. 51, 410–417 (2016).\n",
      "17. K. Dubin, E. G. Pamer, Microbiol. Spectr. doi:10.1128/ microbiolspec.BAD-0014-2016 (2014).\n",
      "\n",
      "18. R. R. Jenq et al., J. Exp. Med. 209, 903–911 (2012).\n",
      "\n",
      "19. S. Caballero et al., Cell Host Microbe 21, 592–602.e4 (2017).\n",
      "\n",
      "20. N. W. Palm et al., Cell 158, 1000–1010 (2014).\n",
      "\n",
      "21. K. Brandl et al., Nature 455, 804–807 (2008).\n",
      "\n",
      "22. D. Zhao et al., J. Clin. Invest. 128, 4970–4979 (2018).\n",
      "\n",
      "23. C. A. Lindemans et al., Nature 528, 560–564 (2015).\n",
      "24. K. H. Schleifer, A. Hartinger, F. Götz, FEMS Microbiol. Lett. 3, 9–11 (1978)\n",
      "\n",
      "25. D. Weber et al., Biol. Blood Marrow Transplant. 23, 845–852 (2017).\n",
      "\n",
      "26. P. A. Gill, M. C. van Zelm, J. G. Muir, P. R. Gibson, Aliment. Pharmacol. Ther. 48 , 15–34 (2018).\n",
      "\n",
      "27. N. D. Mathewson et al., Nat. Immunol. 17, 505–513 (2016).\n",
      "\n",
      "28. B. W. Haak et al., Blood 131, 2978–2986 (2018).\n",
      "29. C. J. Ingram, C. A. Mulcare, Y. Itan, M. G. Thomas,\n",
      "D. M. Swallow, Hum. Genet. 124, 579–591 (2009).\n",
      "\n",
      "30. J. Vydra et al., Clin. Infect. Dis. 55, 764–770 (2012).\n",
      "\n",
      "31. S. Manfredo Vieira et al., Science 359, 1156–1161 (2018).\n",
      "\n",
      "32. S. G. Kim et al., Nature 572, 665–669 (2019).\n",
      "ACKN OW LEDG MEN TS\n",
      "We thank G. M. Dunny and J. Willett (Department of Microbiology, University of Minnesota) as well as R. Zbasnik and V. Schlegel (Department of Food Science, University of Nebraska) for helpful discussion and for providing materials for revision of the manuscript. Funding: This work was supported by a German Research Foundation (DFG) scholarship to\n",
      "Bone Marrow Transplantation to C.K.S.-T.; partially supported by the DFG research consortium TR221 “GvH/GvL” (project B13) to E.H.; by NCI awards R01-CA228358 (M.R.M.v.d.B.), R01-CA228308 (M.R.M.v.d.B.), MSKCC Cancer Center Core Grant P30 CA008748, and Project 4 of P01-CA023766 (M.R.M.v.d.B.); NHLBI award R01-HL125571 (M.R.M.v.d.B.), R01-HL123340 (M.R.M.v.d.B.), and K08-HL143189 (J.U.P.); NIA National Institute of Aging award Project 2 of P01-AG052359 (M.R.M.v.d.B.); NIAID award U01 AI124275\n",
      "(M.R.M.v.d.B.); R01 AI032135 (E.G.P.); AI095706 (E.G.P.); U01 AI124275 (E.G.P. and J.B.X.); Tri-Institutional Stem Cell Initiative award 2016-013 (M.R.M.v.d.B.); the Lymphoma Foundation (M.R.M.v.d.B. and N.K.); the Susan and Peter Solomon Divisional Genomics Program (M.R.M.v.d.B.); the Parker Institute for Cancer Immunotherapy at Memorial Sloan Kettering Cancer Center (M.R.M.v.d.B. and J.U.P.); the Sawiris Foundation (J.U.P.); the Society of Memorial Sloan Kettering Cancer Center (J.U.P.); MSKCC\n",
      "Cancer Systems Immunology Pilot Grant (J.U.P.); Empire Clinical Research Investigator Program (J.U.P.); Seres Therapeutics (M.R.M.v.d.B., J.U.P., J.B.S., A.L.C.G., A.G.C., A.E.S., and A.D.S.); Japan Society for the Promotion of Science KAKENHI (17H04206 to T.T. and 17K09945 to D.H.); the Center of Innovation Program from Japan Science and Technology Agency (T.T.); Mochida Memorial Foundation for Medical and Pharmaceutical Research (D.H.); R56 AI137269-01 (J.B.X.); Conquer Cancer Foundation Young\n",
      "Investigator Award/Gilead Sciences (N.K.); NIH KL2 TR001115-03 (NCATS CTSA to A.D.S.); NIA 2P30AG028716-11 (Claude D. Pepper Older Americans Independence Center to A.D.S.); NCI R01CA203950-01 (to N.J.C., A.D.S., L.B., M.L, and A.B.); NIH 1R01HL124112-01A (A.D.S. and R.R.J.); and NIA R21AG066388- 01 (A.D.S. and N.J.C.). Author contributions: C.K.S.-T., E.G.P., J.U.P, and M.R.M.v.d.B. conceptualized the project; C.K.S.-T., A.L.C.G., A.S., S.D., A.P., J.R.C., Y.T., R.R.J., J.B.X., E.R.L., and\n",
      "J.U.P. were involved in data curation and formal analyses; C.K.S.-T., K.B.N., A.L., M.D.D., A.E.S., J.B.S., A.G.C., G.A., Y.S., M.B.d.S., K.A.M., D.B., R.P., A.T., S.M., M.E.A., A.J.P., M.M., R.J.W., L.A.A., E.F., D.P., M.A.J., D.W., A.D.S., D.H., C.S., J.A.M., K.R., M.L., A.B., L.B., K.H., and Y.H. conducted\n",
      "and M.R.M.v.d.B.\n",
      "\n",
      "the\n",
      "project and provided validation; and C.K.S.-T., J.U.P., A.L.C.G., and M.R.M.v.d.B. prepared data presentation and wrote the manuscript with contributions from all other authors. Competing interests: M.R.M.v.d.B has received research support from Seres Therapeutics; has consulted, received honorarium from, or participated in advisory boards for Seres Therapeutics, Flagship Ventures, Novartis, Evelo, Jazz Pharmaceuticals, Therakos, Amgen, Magenta Therapeutics, Merck & Co, Inc., Acute Leukemia\n",
      "Forum (ALF), and DKMS Medical Council (Board); and has IP Licensing with Seres Therapeutics and Juno Therapeutics and stock options from Smart Immune. J.U.P. reports research funding, intellectual property fees, and travel reimbursement from Seres Therapeutics and consulting fees from Davolterra. A.D.S. received research support from Seres Therapeutics, Merck, and Novartis. E.G.P. has received speaker honoraria from Bristol-Myer Squibb, Celgene, Seres Therapeutics, MedImmune, Novartis, and\n",
      "Ferring Pharmaceuticals; is an inventor on patent application number WPO2015179437A1, entitled “Methods and compositions for reducing Clostridium difficile infection,” and patent number WPO2017091753A1, entitled “Methods and compositions for reducing vancomycin-resistant Enterococci infection or colonization”; and holds patents that receive royalties from Seres Therapeutics Inc. Other authors have no competing interests. Data and materials availability: All data are available in the manuscript\n",
      "or the supplementary materials. Sequencing data have been deposited into Sequence Read Archive under Bioproject number PRJNA545312.\n",
      "SUPPLEMENTARY MATERIALS\n",
      "\n",
      "science.sciencemag.org/content/366/6469/1143/suppl/DC1 Materials and Methods\n",
      "Table S1 to S10\n",
      "\n",
      "Fig S1 to S10\n",
      "References (33–53)\n",
      "View/request a protocol for this paper from Bio-protocol.\n",
      "\n",
      "18 March 2019; accepted 22 October 2019 10.1126/science.aax3760\n",
      "D o w n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019) 29 November 2019\n",
      "7 of 7\n",
      "Science\n",
      "Lactose drives Enterococcus expansion to promote graft-versus-host disease\n",
      "C. K. Stein-Thoeringer, K. B. Nichols, A. Lazrak, M. D. Docampo, A. E. Slingerland, J. B. Slingerland, A. G. Clurman, G. Armijo, A. L. C. Gomes, Y. Shono, A. Staffas, M. Burgos da Silva, S. M. Devlin, K. A. Markey, D. Bajic, R. Pinedo, A. Tsakmaklis, E. R. Littmann, A. Pastore, Y. Taur, S. Monette, M. E. Arcila, A. J. Pickard, M. Maloy, R. J. Wright, L. A. Amoretti, E. Fontana, D. Pham, M. A. Jamal, D. Weber, A. D. Sung, D. Hashimoto, C. Scheid, J. B. Xavier, J. A. Messina, K. Romero, Lew, A.\n",
      "Bush, L. Bohannon, K. Hayasaka, Y. Hasegawa, M. J. G. T. Vehreschild, J. R. Cross, D. M. Ponce, M. A. Perales, S. A. Giralt, R. R. Jenq, T. Teshima, E. Holler, N. J. Chao, E. G. Pamer, J. U. Peled and M. R. M. van den Brink M.\n",
      "Science 366 (6469), 1143-1149. DOI: 10.1126/science.aax3760\n",
      "Lactose can fuel GVHD\n",
      "Allogeneic hematopoietic cell transplantation (allo-HCT) is used to treat certain hematopoietic malignancies, but patients have a risk of developing graft-versus-host disease (GVHD). Stein-Thoeringer et al. performed a large-scale analysis of more than 1300 patients treated with allo-HCT across four clinical centers (see the Perspective by Zitvogel and Kroemer). High levels of bacteria from the Enterococcus genus were associated with greater incidence of GVHD and mortality. Lactose appears to\n",
      "provide a substrate for Enterococcus growth, and patients with a lactose-malabsorption genotype had a greater abundance of Enterococcus. A lactose-free diet limited Enterococcus growth, reduced the severity of GVHD, and improved survival in gnotobiotic mouse models.\n",
      "Science , this issue p. 1143; see also p. 1077\n",
      "\n",
      "ARTICLE TOOLS http://science.sciencemag.org/content/366/6469/1143 SUPPLEMENTARY MATERIALS http://science.sciencemag.org/content/suppl/2019/11/25/366.6469.1143.DC1 RELATED CONTENT http://science.sciencemag.org/content/sci/366/6469/1077.full REFERENCES This article cites 52 articles, 15 of which you can access for free http://science.sciencemag.org/content/366/6469/1143#BIBL PERMISSIONS http://www.sciencemag.org/help/reprints-and-permissions\n",
      "o w n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "Use of this article is subject to the Terms of Service\n",
      "Science (print ISSN 0036-8075; online ISSN 1095-9203) is published by the American Association for the Advancement of Science, 1200 New York Avenue NW, Washington, DC 20005. The title Science is a registered trademark of AAAS.\n",
      "Copyright © 2019 The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works\n",
      "CSV file saved as GVHD-ALD-papers.csv\n"
     ]
    }
   ],
   "source": [
    "import nest_asyncio\n",
    "nest_asyncio.apply()\n",
    "\n",
    "import asyncio\n",
    "import pandas as pd\n",
    "from typing import List\n",
    "\n",
    "semaphore = asyncio.Semaphore(20)\n",
    "\n",
    "async def limited_chunk_review(chunk):\n",
    "    async with semaphore:\n",
    "        return await as_chunk_review(chunk)\n",
    "    \n",
    "async def process_all_chunks(chunks):\n",
    "    tasks = [limited_chunk_review(chunk) for chunk in chunks]\n",
    "    reviewed_chunks = await asyncio.gather(*tasks)\n",
    "    return reviewed_chunks\n",
    "\n",
    "async def main():\n",
    "    reviewed_chunks = await process_all_chunks(chunks)\n",
    "\n",
    "    reviewed_dicts = [rc.model_dump() for rc in reviewed_chunks]\n",
    "\n",
    "    df = pd.DataFrame(reviewed_dicts)\n",
    "    df.to_csv(\"GVHD-ALD-papers.csv\", index=False)\n",
    "    print(\"CSV file saved as GVHD-ALD-papers.csv\")\n",
    "    \n",
    "\n",
    "if __name__ == \"__main__\":\n",
    "    asyncio.run(main())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "relevant\n",
      "False    547\n",
      "True     204\n",
      "Name: count, dtype: int64\n",
      "relevant\n",
      "False    140.307130\n",
      "True     407.897059\n",
      "Name: text_length, dtype: float64\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAABv0AAAJOCAYAAACUQctNAAAAOnRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjEwLjAsIGh0dHBzOi8vbWF0cGxvdGxpYi5vcmcvlHJYcgAAAAlwSFlzAAAPYQAAD2EBqD+naQAAry1JREFUeJzs3Qd8VFX2wPEzk95DD10QBGmK6ArqWgBBwY6urgXcv5XFhnVxbYCCsvbuKqKuBXtZQGlSpCmCSJMOhh5Kes/M+3/OxclmQgIJJHnzZn7fz+ftZN578+bMJOu73HPvuS7LsiwBAAAAAAAAAAAA4FhuuwMAAAAAAAAAAAAAcHRI+gEAAAAAAAAAAAAOR9IPAAAAAAAAAAAAcDiSfgAAAAAAAAAAAIDDkfQDAAAAAAAAAAAAHI6kHwAAAAAAAAAAAOBwJP0AAAAAAAAAAAAAhyPpBwAAAAAAAAAAADgcST8AAAAAAAAAAADA4Uj6AQhqJSUlcv/990vLli3F7XbLJZdcUqfvf/3118sxxxxTp+8JAACcee/esmWLuFwuefrpp8Xp/vOf/0jHjh0lIiJCkpOT6/S933nnHfM96vcJAAAQymbPnm3aRZ999tkRvf6xxx4zr9+7d684DX1yCFUk/YAQsHHjRrnlllukbdu2Eh0dLYmJiXL66afLCy+8IPn5+RIIXn31VdNBU9Pefvtt+de//iWXX365vPvuuzJ8+PBKzz377LNNQ8a3xcTESLdu3eT5558Xr9db47EBAIC6TwT5Nm0THXfccXLbbbfJ7t27a/39ly1bJtdee60ZiBQVFSX169eXvn37yoQJE8Tj8YgdpkyZYjpyatqaNWtMJ8uxxx4rb775pvz73/8+bEeSb9MkoXbO3HHHHZKRkVHjsQEAEAj9H3rPO/XUU+0OJWCUbw9Utmm/TU358MMPTX9PVWn75IILLpBAVd3PE2jokwNqTngNXgtAAJo8ebJcccUVpnNp8ODB0qVLFykqKpJ58+bJfffdJ6tWrTpkR0xdNnobNmxoOohq0vfffy/NmzeX5557rkrnt2jRQsaOHWt+1lFM2mjSROGePXvkiSeeqNHYAABA3Rs1apS0adNGCgoKTHvotddeM8mvlStXSmxsrElS1XTHwltvvSW33nqrNGnSRK677jpp3769ZGdny8yZM+WGG26QnTt3yoMPPih1TT/3K6+8UuOJPx1Rrt+hDjBr165dlV6jv4f4+HjJzc0138tLL70kS5cuNb8jAACCyQcffGASSD/99JNs2LChyvfKYHbZZZf5fQ85OTkydOhQufTSS80xH21L1RTt79H231133SXBIBg+D31yQM0g6QcEsc2bN8tVV10lrVu3Nsmvpk2blh4bNmyYaVxqUjCYpaWlVaukVFJSkhmF76MddFqaSjuetJMwLCysliIFAAB14fzzz5eTTz7Z/HzjjTdKgwYN5Nlnn5Wvv/5a/vrXv5qZZjVp0aJFpj3Rq1cvk2RLSEgoPaadMj///LPpoKlLmliLi4ur1faXqk4bTKsy6AAwpRUqtA378ccfmw7RP/3pT7UWKwAAdd1Ps2DBAvniiy/M/U4TgI8++midxqADc3QwuFY9CBQ6o0s3H034aNJP95Xto0Fwo08OqBmU9wSC2Lhx48zoqPHjx/sl/Hx0FNWdd97pt/7d6NGjTSkmnRmoI8901HlhYaHf63SafUUjwvX8sjP1fGW05s+fL3fffbc0atTIdDDpSC0dpVP2dTrjcM6cOVUu2aCdVffcc09piawOHTqY9W8sy/JbE2fWrFnm2r7r6sjz6tBG8CmnnGJG4/s6sHzef/996dGjhyk5oCW6tHNq69atVWpga3mCzp07m+vrSDVt7Kenp5eeoyUjtBxrRbTT0NdZqbQsWO/evaVx48bmu+jUqZMZLV9ZKQodMa+dZ/re+h7vvffeQedqOS0dTaWv0WvqaCudKVq2hrv+Xeg/TvTvSM/R34Wun1j+7wUAgECm91BfJ1xla39U5d5dmZEjR5o2iHbqlU34+eg9vaJKB1qJwdcm07bI4sWL/Y4vX77cvM5Xvj0lJUX+7//+T/bt21dhuazVq1fL1VdfLfXq1ZMzzjjDvFZn+amypZSqUp1BvweNq1mzZmYgWdkynPrd+Tovte1XWbvxcP785z+Xlqkv68cff5TzzjvPdArpzMyzzjrLtDWr4ttvvzXX1fao/i4GDhxo2ok+2pbUeH///feDXjtixAiJjIws/Z3/8MMPpppGq1atSttB2nYqXzpfv2edwbh9+3aztrT+rN/Lvffee1BZV9/syK5du5rfqZ6nn1UTwzXRBgUA2E/bA3ov1nuQDnjR5z7FxcXmv+t/+9vfDnpdVlaWuTfo/aO6/ybXe5uWM9f38t3Dv/vuu9J732mnnWYGQel9Re8vFa39pvc3Lb2tA3T0HnrRRReZe1tF93ndr20SbS/pe+l76tIrNVVCXL83/Z70+9B21DfffFN6XPtt9P6pfUq+/iGlg971/n/llVea53pcB8HrPd/XBqqptd+qcp/W99dKXNo+O+ecc0ybRqtUaT9eeRqjft8av/b7aHtj6tSpfn1cVfk82s7Q2XLav6PfXZ8+fcz3UlXaH/SXv/zFLBmkfy/an6iVM3y0TXbCCSdU+Frts+vfv79UF31y9MnhyDDTDwhi//3vf80NRBtwVaGj3XXdO21AaUJNO1V0Wv1vv/0mX3755RHHcfvtt5tGrd6MNBmnN1dtcOrobaXP9RztBPnnP/952JIN2nDTBo8m9LQk1oknnmgaPFquVBuXWspTG3n/+c9/TINGE5++8gDHH398teP3JRDLjlbX6z788MOmwaPfmyYxdeTRmWeeKb/88sshR7ZrY0ITotqQ10azdjK+/PLL5nXaaaUzDLQhqjd07eDTBo6PNuB0xoCuU+ijjQltrOh3Eh4ebn7vf//7301DRjviytIGnf5+9XsbMmSIaXhrZ5Q2lPQaSr8v7RDT37s21E866STTsNCG9LZt20wjX6+t76eNlZtvvtl8rytWrDDf/bp16+Srr76q9vcMAIAdfEkl7bw4mnt3RfLy8kypSm0faHKoqrSUkXZu6PtqG0Q7gLS01aZNm0rfa/r06ea5xqQJP1/Jdn3UtkL5BJ4mqLSs6JgxY0xbqnv37rJjxw5zHW0zVYV26mkSU9ci1NH3a9euNe0Qba/4vgdt12nnhbYdfSU7y47cr077S2kb0kcrV+hMTW23aLvS7XaXdrRoEu5QMwL1M2rbRzucnnrqKfO70fg0Aaq/R+1U0XaddpZ88sknpl1Zlu7r169faTyffvqpuYZ+D/q3ozMStS2obSU9VpYm9/R9de0m7VydMWOGPPPMMyapq6/30faZ/p3pZ9T2pQ7I08+lv09f59LRtEEBAPbTxJve03UgiVYY8N1H9d/9eh/VQdI6C/CNN94w5/jov7G1M18TG6q6/ybXe6jey7QvRv9N70sI6WATvc4111xjZv9NnDjRtBkmTZpkEpM+2m+gr9cy5T179jSDtsse99F1kvW4L9GofTM66EbvcZq4PJrSk9rGOf30001y7B//+IdJgmlMOqjm888/N9+dJl70O9XPoPdHbbfpd6Xxa7JSBy8p7XvKzMw0923fcjDaZjla1blPa5JJB/fo34Oer8nWBx54wAz+0baAb8C7tnO0FLwm2bTNp+1E7Q8rqyqf58knnzRtJ00c67navtTfu/b9VYXGqH832r+mbZMXX3zRfAZf0kr/Nm666SZTwUITmj76961/kw899NARfaf0ydEnhyNgAQhKmZmZOqTJuvjii6t0/rJly8z5N954o9/+e++91+z//vvvS/fp80cfffSga7Ru3doaMmRI6fMJEyaYc/v27Wt5vd7S/cOHD7fCwsKsjIyM0n2dO3e2zjrrrCrF+tVXX5nrPv744377L7/8csvlclkbNmwo3afX1GtXhZ7bsWNHa8+ePWZbs2aNdd9995n3GjhwYOl5W7ZsMfE/8cQTfq9fsWKFFR4e7rdfvw/9Xnx++OEHc70PPvjA77Xfffed3379/UVFRVn33HOP33njxo0zn/H3338v3ZeXl3fQZ+nfv7/Vtm1bv30ah77H3LlzS/elpaUd9D6PPPKIOe+LL7446Lq+3+N//vMfy+12m89T1uuvv25eO3/+/INeCwCAnXztkhkzZpj7/NatW62JEydaDRo0sGJiYqxt27Yd1b27Ir/++qs5584776xSjJs3bzbna0z79+8v3f/111+b/f/9738Pef//6KOPDrrXa5tN9/31r3896Pxhw4aZY1WhbYbIyEirX79+lsfjKd3/8ssvm2u8/fbbB72nfs+H4zt37dq15nxtZ+m19HfSqFEjKzc3t7QN0r59e9PGKduu1O+hTZs21rnnnnvQ71q/T5WdnW0lJydbN910k99779q1y0pKSvLb36tXL6tHjx5+5/3000/meu+9957f+5Y3duzYg9pp+vekrx01apTfud27d/d7H21r63l33HHHQdf1fd7qtEEBAIHn559/Nv+tnz59eul/31u0aOHXTpg6depB93w1YMAAv3/jV+ff5Ppcz121atVBMZW/nxUVFVldunSxevfuXbpvyZIl5hp33XWX37nXX3/9Qf1DN9xwg9W0aVNr7969fudeddVV5p5b0f2zItomKH/tPn36WF27drUKCgpK9+l3eNppp5k2Qlna7omNjbXWrVtn/etf/zLX0r6ksrSfp2yb73D03LJ9Q+VV5z6t/U/l2xaFhYVWSkqKNWjQoNJ9zzzzzEGx5+fnm74r3T9r1qzDfh49R889/vjjzXv4vPDCC2a/xleVttpFF13kt//vf/+72a/tXaV9fNHR0dYDDzzgd562beLi4qycnJxDvg99cgfQJ4eaQHlPIEjpCCpVURmpiugaM0rLcJalM/7U0az9p6NOyo421xErOuK5otJJVY1V63jriJzysWp7VkeRHU2pCB2JppvWDdfROzp6RkcB+eioOx1VoyOKdLSNb9MRVzqCvvyIq7J05LeWozr33HP9XqujenQUlu+1Wi5BR3bpqLWyJSl0dqSOmis7W0BLGfjoaC29npZV0NH/+rwsLTPgK5el9HNqmQU910dHyGlJBh0lV57v96ifQ0cS6XdU9nP4SqQd6jsAAMBOOktN739aAkdHy+v9V2el6ajxo7l310R7zEdHF5ed4ea7d5e9X5e9/2tpJY1J2whq6dKlB11T10Q5Gjo7TWcA6Ah9HSXuoyO6td1ytOtEa3tEfy86glxHNWupIm3TabkrtWzZMlm/fr0pUaolTH2/Bx0Br+Wp5s6da9pnFdHZjFomSWdUlP0dantSZ9+V/R3qd79kyRK/sqLa/tKySRdffHGF37/GoNfT6hrabtOR4of7/vV3Wr79pe2sitZ18rW/jqYNCgAIjFl+WtVIyzn6/vuu9x2dXecr+az/ptaZPL7KSEpnU+m9zFea8kj+Ta59BNofUF7Z+5m+j/Yh6D2qbFvCVwpUZy+VpRWbytJ7oN7PLrzwQvNz2bh0xrteu6I2SlXs37/fzFbUe6BWQ/BdV9sEem1tI2jlJx+duaXtN51VpTPCdBZa2ft4bajufVrbkWXXr9OZnVq1oGz7QL97baNqv5SPloXU9ld16cy2srNHK2pfHkr5WWu+37+vP1G/b/2OP/roo9J+LP271r9lnY1ZlfWk6ZOjTw41g/KeQJDSG5TSxlBVaAJOO3C0g6UsvWnqtPgjTdCp8uWsfJ1YVVkHp7JYdQ2Z8h1ovtKdRxOrdjS9+eabpgGhnT1aMkDLBJRd4Fobk3rT18ZERSor8eV7rd70teRERcrWKNcGvU7JX7hwoelE0ni0E0rLZpWl5Qe0g0jP0zJTZel7aYPGp6LSYvr7KPu70PcZNGhQpZ/B9zm01IA2UA73OQAACCS6jt1xxx1nyu9ox5v+Q7tsEuto7t1H2x6rTttJO7+01KZ2FJaPoXwHg2rTpo0cDV/7Sr+vsrTzSMvJH037y9fBod+Xtru0XJSWWirbiaK/B6WlkCqjn7tssrT8a30dIZX9npSWA9NBcNqpo2tba5tPO1a046fseampqfLII4+YUkvl27Tlv3/f+nyHa39p+1bXpKnM0bRBAQD20uSH3rM14edbR1jp4BMt+azlwLWMtLZP9N/jWsJRy3nqoBNNcuh6f2WTftX9N3ll7QAt4/n444+bwTVl1wIrO3Db119U/hrl+4/0Hq6DbLTcuG5ViauqtCyi3gM1gadbZdf2DeLS+6m2J/S+ru09/bm2Vfc+reu0lS/Hru0DXbe57Hev5cDLn1f+u6+Lvrnyn0vj0r8LX0l2pSUxtQ2l5cm11KYOGtOSr5p0rQr65OiTQ80g6QcEKe2U0I4DraVdHeUbEtXhG5lWno6irkjZ0TKBQkce6QwAH60Xr/WztdPH10jUxod+Tzr6vKLPdqg68PpabVyUXay7rLI3bB0dp6PbdWSRNjD0URtU2mgt2xjQ0e06uufZZ581sxa0801HWmkt7/Ij3mvqd6HX1Trz+p4V0TgAAAhEOoLatz5aVVTn3l2edsho552usVEdVblf6+jmBQsWmLXndH1jbX9orLo2TEUz3som0AKRdgzpzAZfG0jbGbrOjHauaPvH95l0xLd+3opU1gbzvVbX9dMBbeXp78hH2886AlvbXdr+03VbNMGn6wCWbfPqCHFNvOraO9oO0zakzjDQdVmq2v6qrqNpgwIA7KWz1HRdNk386VaetjM06ae0EoGu6af/vdcZUnpP0nuNzv450n+TV9QO0MSMzqLSe7Cudde0aVOTMNH1cjXpWF2++5/OXqtskM6RrPNb9tq6Hp3O7KtI+UTY1KlTzaMmVHQttNpe97a69+m67iur6ferqP9QfzeaZH3//ffN35U+aturbD/bodAnVzX0yeFwSPoBQeyCCy4wo6t0tEmvXr0OeW7r1q3NTUNHi/hmzCkdkaMjtfR42VEouq8sLfekDdgjVZ1ko8aio4V01HzZ2X5aBsB3vKZog1QbrNrg1saljsrR0Ux6Q9ZRbjpToDr0tRq7NlwO1/mmjR39Herocr2R62gp7YTSzigfXSBYR+PpKPOyI4aOZiq/xni4ZLGe8+uvv5rGzdEkigEACHTVuXeXpx0FOrtMO/q2bt1aY/8A184rnRGgM/10tln5GW211f5Sa9euNTP7yrYBdcZCVTtzqkI7a3TEtJah0g4W7fzU34NvYFt138v3Wu3kqcprdWS3ljDTz6rtL/09asePjyZx161bJ++++64Z0e6jpdeOlMaonZOaSKxstt/RtEEBAPbSJIPeh7TiQHk6k09Ljb/++uumraHJEk3A6T3ojDPOMO2If/7znzX+b3KdZa8zqPT+ozMKfTTpV1F/kd7vy86u0tl35RMm2kejg2Nqsl2gfG0PTUpW5dpaFvOtt96S+++/33z3moT88ccf/Qb61HRfRm3cp/W7X716tblu2XjLf/eqtvtmtJ1ZdranxqB/Fzo7r2xSS0uxazlOHTClM+W0FOmRDoCiT67yc+iTw6Gwph8QxLRxozepG2+80STvytMRKS+88IL5ecCAAeax/DR136iRgQMH+t1cdN2UsjS5WNlMv6rQOMsnEiujsep7aY32snQUjd7stPxSTX+PWkrD911cdtllpsGiHW3lR+Poc60pXxkdla+xjx49+qBjJSUlB30H2um0Y8cO01jVG3rZch7K13AqG4eWDyjfSK8OLSOg76X/6CjP9z76OXQ0u5ZdKC8/P9+sbQMAQDCo7r27PE1e6f1Tyxrl5OQcdFxnsmnyqDoquv9X1I47HN/aKlVpg2kHm45c1lHWZd93/Pjxpu1Rtq1YE3SWn5a98s2w07VWtA369NNPV/g9aumnyuioc00WjhkzxrTpDvdabQvpd6xr0mhHj3b4lF2HpqLvX3/2tauPhL6nXkPbl+X53udo2qAAAPvov5E1saf3E11jrvx22223mUHNmjhQOptI92tCQWepa3ujfF9ATfybXO8p2odSti9HSzVqoqYs38w6nQ1Y1ksvvXTQ9fR+psnEipIWh7pXH44mTM8++2yT/KlowHnZa2u7RvvBtLqD3vu1P0XXEtSfy9J7e0Ul0Y9Ubdyn9bvX37Pvb8O3lnNFv/ea/jzllU9Y+37/5fvgtM2rA9RuueUW02Yru27hkaBP7n/ok0NVMdMPCGLaMaIlGfSmpLP3dCRyly5dzIhsLQelnRhagkhpmQgd+aTJO73J6aKzP/30k+mE0nISvoWmlTaebr31VnMj0tJGejPSkWG+kkxHQjtyXnvtNVNLXksyaIOusnVXdKS1xqMj3bRBqrFPmzZNvv76a7nrrrtKR3PXFF1oVxONepPX2vF6fY1zxIgR5v31+9HRbDrqTW/KN998sxmBVBH9XrXhM3bsWFMzX8t36Eg1HTGlvw/tLNLGvY++r15br+drQJelr9cOOP1OfA0qvenr93ekMy+1TNhnn31mShb83//9n/nd6KhzbWTqyEP9vrURpyPv9e9ARzDpKCltOOlsS92vfw/VKZ0GAECgqu69uzwtB6SdJDpzTEv/6D1UR8lr597s2bPN/VXbFdWhCSydBTBu3DjTCaLr12hbqOwaQVWh93h1xx13mE4lbWvorLqK6Oh9bftoB4uWENVyYDoTTjsATznllKPu0ClPv+M777zTtEt0tL6+p7bFtGOpc+fOZhagfm7t8NC2iH4n2jlaET2m7Uz97rVElH5G/TxatnPy5MmmHVN2MJm2o7StqZ1L+nsq38Gjv0dtD2r7TN9fr68dnEe6XrXS99P4NKmqf1u+Mq1aek2PaYfw0bRBAQD20Xu93k/03lmRnj17mvuSzkjz3XP0UZMqOnhIy/iVrcikauLf5DpgR+91es/R2Vm6Dpi2WbRPpuy6ctpe0L4IHVykCRWNd86cOWbWuyo70+jJJ5808ehahTrDS/tTtD9Bk246w0p/PlIam8581O9Dr62z/3SAu1a30vKd2jeltP2gcer7adtGP5/2Y+k99OKLLy4tk6qfS2dv6Vq+2pbRSgNlZ/ZXRGe3VdRu6969u/k+a/o+rW1QbaP89a9/NZ9LZ4Dq34lvjbuy3/2RfJ7q0M+hf8P6fep3rqU79e+mbNlZ33ehfY/aTta/W217HQ365OiTwxGwAAS9devWWTfddJN1zDHHWJGRkVZCQoJ1+umnWy+99JJVUFBQel5xcbE1cuRIq02bNlZERITVsmVLa8SIEX7nKI/HYz3wwANWw4YNrdjYWKt///7Whg0brNatW1tDhgwpPW/ChAk6BMVavHix3+tnzZpl9uujz65du6yBAwea2PTYWWeddcjPlJ2dbQ0fPtxq1qyZibV9+/bWv/71L8vr9fqdp9fp3Llzlb6nQ507e/ZsE9ejjz5auu/zzz+3zjjjDCsuLs5sHTt2tIYNG2atXbu29Bz9PvR7Ke/f//631aNHDysmJsZ85q5du1r333+/tWPHjoPOveaaa8x79+3bt8LYvvnmG6tbt25WdHS0+R0/9dRT1ttvv21es3nz5tLzNA79jiv63OW/73379lm33Xab1bx5c/M306JFC/NZ9u7dW3pOUVGReS/9zqKioqx69eqZz6R/Q5mZmRXGCgCAXSprl5RXE/fuiixZssS6+uqrS9suet/s06eP9e6775q2ldL7tsaobZryyrdDtm3bZl166aVWcnKylZSUZF1xxRUmlvLn6c+6b8+ePQdds6SkxLr99tutRo0aWS6Xy5x3OC+//LJp8+hnaNKkiTV06FArPT3d75xDvWd5hzpX2xP62cq2U3755Rfrsssusxo0aGDaH/q7+stf/mLNnDnzoN912XaQ0rantlv1mtpuOvbYY63rr7/e+vnnnw967zfffNNcQ3/X+fn5Bx1fvXq1aZvFx8ebNrG2tX/99VfzGn3/sn9P2k6s7HOX/33o716/X21/6e/l/PPPN387ZVWlDQoACBwXXnihue/k5uZWeo7ej/Te6vs3t/ZtaJ+M3isef/zxCl9T1X+T6zX0PlGR8ePHm/4Ufb3eT/QeVtE9SmPXa9SvX9/c+y655BJz39HznnzySb9zd+/ebc7V+PUzpaSkmDaPtqWqStsF5ds0auPGjdbgwYPNNfXa2mdxwQUXWJ999pk5/vXXX5vXPfPMM36vy8rKMm2GE044wXxvKicnx7TNtC2lr6mo/VeWHtfzKtpuuOGGat2nK+t/qqgdumnTJtOXo21QbRvcc8895j30fRctWlR6XmWfx9cH9+mnn/pd19fuLNtuqYjv70HbPpdffrlpG+nfmvYZVdRGUuPGjTOvGTNmjFVV9Mn9D31yOFou/Z8jSRYCAAAAAAAAAEKPzpTSWV0640vLcqPu6KzL4cOHmxmOWvkg0OiMOY1PZ+KVXesOQN0g6QcAAAAAAAAAqJCuExYTE+O3T5eL0TUHNbHTsmVL22ILte9e1/TTZKuWc/SVWA0kmmrQEpQNGjQwpScB1D3W9AMAAAAAAAAAVEjXEV6yZIlZZzY8PFy+/fZbs+n6aST8atdll11mZsudeOKJkpmZaWZW6tpturZfIMnNzTXrzmmib8WKFfL111/bHRIQspjpBwAAAAAAAACo0PTp02XkyJGyevVqycnJMUmo6667Tv75z3+aJCBqt5TnW2+9ZWZU6uy+Tp06yf333y9XXnmlBBKNr02bNpKcnCx///vf5YknnrA7JCBkkfQDAAAAAAAAAAAAHM5tdwAAAAAAAAAAAAAAjg5JPwAAAAAAAAAAAMDhKLosIl6vV3bs2CEJCQnicrnsDgcAAAQgrYienZ0tzZo1E7ebcVMAAACoGfRLAQCAmuqXIuknYhpWLVu2tDsMAADgAFu3bpUWLVrYHQYAAACCBP1SAACgpvqlSPqJmJFUvi8rMTHR7nAAAEAAysrKMp0xvnYDAAAAUBPolwIAADXVL0XST6S0dII2rGhcAQCAQ6HkEgAAAGoS/VIAAKCm+qVYkAYAAAAAAAAAAABwOJJ+AAAAAAAAAAAAgMOR9AMAAAAAAAAAAAAcjqQfAAAAAAAAAAAA4HAk/QAAAAAAAAAAAACHI+kHAAAAAAAAAAAAOBxJPwAAAAAAAAAAAMDhSPoBAAAAAAAAAAAADkfSDwAAAAAAAAAAAHA4kn4AAAAAAAAAAACAw5H0AwAAAAAAAAAAABwu3O4AAAAAAACBwWtZUuCxJL/E+8dj2Z//91jktcRr6fki1h+vO/AoYlkiTYsypPfCz8Xlcou4XAc29x8/h4WJREaJREWLKyq69GeJijrwPMr3PFpceiw6WiQmTlz6OgAAAABApUj6AQAAAECQyyn2Snqhx2yZRR6TzPMl8PI9lhT88Vjo0dTd0YsvLBTZtcMkAg+l6u/mEomLF1dikkhikrgS/nhMTBZJSDzwqM81SQgAAAAAIYqkHwAAAAAEgbxir+z/I7Gn2/9+PjAzz9kskdxssXKzRXZuqzxZqDME/0gGuho0FmnURFyNGotLH2Pj6zZkAAAAAKhjJP0AAAAAwCF0Rt7+cgk93881NUvP0QoLRPYUiLVnt1gb1/ofi40TV8Mm4mroSwYe+NmVVM+uaAEAAACgRpH0AwAAAIAA5LEs2Z1XIjtyS2SHeSyWjCKv3WE5V16uWKmbzOYnMupAIjClubibtxJXi1YHkoK6HiEAAAAAOAhJPwAAAAAIALrWnknw5RabJJ8m/EqYvFf7igrF2rFVZMdW8Sxd9L9EYPOW4mre2iQBXS1aiysuwe5IAQAAAOCQSPoBAAAAQB0r9lqy08zgKy6dyZdTzCy+gEoEbt5gtlLJ9Q8kAE0isLW4UpqLK5x/UgMAAAAIHPwLBQAAAABqWX6JVzZnFcvW3GLZnlsse/M9QorPYTL2i6XbymUHnoeHi6tVG3G17SDuYzuIK6WZ3RECAAAACHEk/QAAAACgFqTll8jGzCLZmFUk23NLhEqdQaakRKxN683mnTFJJD5BXG2PE3fb48R17HHiik+0O0IAAAAAIYakHwAAAADUUMnO37OLTZJPk31ZlOsMLTnZYi1fIp7lS0TEJdIk5Y8EYAdxtW4rrvAIuyMEAAAAEORI+qHUk7/stTsEBIh/dG9odwgAAACOkFnkKZ3Npwm/EqbzwbBEdu8U7+6dIgvnHCgFqom/jl3FfXxXccUl2B0gAABAwKO/Gj70V1cdST8AAAAAqCKvZZlSnb7ZfHsKPHaHBKeUAt24zmzeKV+Kq3UbcXU64UACkDKgAAAAAGoIST8AAAAAOATLsmRzdrGs2l9okn0FHqbz4ShYXrG2bDSb99svxdXKlwDsJq4EEoAAAAAAjhxJPwAAAACowL6CElmxr1BWpRdKNuvzoTZYlli/bzKb97uvxNVSE4DdxN1JE4BJdkcHAAAAwGFI+gEAAADAHwpKvLI6vVBW7C+UnXkldoeDUEsApm4ym/e7r8XVsrW4TzhFXF27iysyyu7oAAAAADiA2+4AAAAAAMDudfo2ZBbJV5uz5KWV+2XatlwSfrCZJdbWLeKZ9KmUPDNSSr75WLzbfrc7KCCoPfnkk+JyueSuu+4q3VdQUCDDhg2TBg0aSHx8vAwaNEh2797t97rU1FQZOHCgxMbGSuPGjeW+++6TkhLuIQAAwB7M9AMAAAAQkvbkl5gZfav2F0huCev0IUAVFYr1y0/i0a1xiri7nyruE04WV0ys3ZEBQWPx4sXyxhtvSLdu3fz2Dx8+XCZPniyffvqpJCUlyW233SaXXXaZzJ8/3xz3eDwm4ZeSkiILFiyQnTt3yuDBgyUiIkLGjBlj06cBAAChjJl+AAAAAEJGfolXft6TL++syZDxazLkp7R8En5wjrRd4p369YHZf5//R7yb1oll8fcLHI2cnBy55ppr5M0335R69eqV7s/MzJTx48fLs88+K71795YePXrIhAkTTHJv0aJF5pxp06bJ6tWr5f3335cTTzxRzj//fBk9erS88sorUlRUZOOnAgAAoYqkHwAAAICgtz232JTvfHnlfpmxLVd25VN6DQ7mKRFr5TLx/OcNKXlprHh+mCFWdpbdUQGOpOU7dbZe3759/fYvWbJEiouL/fZ37NhRWrVqJQsXLjTP9bFr167SpEmT0nP69+8vWVlZsmrVqjr8FAAAAAdQ3hMAAABAUNIZUBuyiuTH3fmyLZckH4JU+j7xfv+teGdPFVenE8Td6yxxN2tpd1SAI0ycOFGWLl1qynuWt2vXLomMjJTk5GS//Zrg02O+c8om/HzHfccqU1hYaDYfTRICAADUBJJ+AAAAAIJKideSVfsLTenOfYUeu8MB6obXK9bKX8Sz8hfxtmoj7p5niatjZ3G5KPADVGTr1q1y5513yvTp0yU6OrpO33vs2LEycuTIOn1PAAAQGmj9AwAAAAgKBSVeWbArT15btV++3ZpDwg8hy0rdLJ5P3pGSl54Uz0/zxCpmbTGgPC3fmZaWJieddJKEh4ebbc6cOfLiiy+an3XGnq7Ll5GR4fe63bt3S0pKivlZH/V5+eO+Y5UZMWKEWTPQt2kCEgAAoCYw0w8AAACAo2UWeWRxWr4s31coRV7L7nCAwCr9+e2X4p0zTdx/OsNsrphYu6MCAkKfPn1kxYoVfvv+9re/mXX7HnjgAWnZsqVERETIzJkzZdCgQeb42rVrJTU1VXr16mWe6+MTTzxhkoeNGzc2+3TmYGJionTq1KnS946KijIbAABATSPpBwAAAMCRduWVmBKea9ILxWt3MEAgy8s1a/55588S90mnmnX/XEn17I4KsFVCQoJ06dLFb19cXJw0aNCgdP8NN9wgd999t9SvX98k8m6//XaT6OvZs6c53q9fP5Pcu+6662TcuHFmHb+HHnpIhg0bRlIPAADYgqQfAAAAAEfZlFUkP+7Ol99ziu0OBXCW4iLx/viDeBfPF1e3kyXsrHPFlVzf7qiAgPXcc8+J2+02M/0KCwulf//+8uqrr5YeDwsLk0mTJsnQoUNNMlCThkOGDJFRo0bZGjcAAAhdJP0AAAAAOMK6jEL5YWee7ClgrT7gqHi9Yi37SUpWLBH3ST3FfWZfccUn2h0VYLvZs2f7PY+OjpZXXnnFbJVp3bq1TJkypQ6iAwAAODySfgAAAAAC2vbcYpm1PVe25ZbYHQoQXDweM+vPu2yxuE85Xdxn9GbNPwAAAMDBSPoBAAAACEj7Czwye0eurMsssjsUIPjLfi6YJd4lC816f+6eZ4orKtruqAAAAABUE0k/AAAAAAElt9gr83flybK9BeK1OxgglBQWiHf2VPH+NE/cp58j7j+dIa7wCLujAgAAAFBFJP0AAAAABIRiryU/peXLj7vzpchr2R0OELrycsU7fZJ4F80V95/PFfdJp4orLMzuqAAAAAAcBkk/AAAAALbyWpYs31co83blSU4xc/uAgJGdJd4pn4t34WwJ63+xuDt0tjsiAAAAAIdA0g8AAACAbdZnFsqcHXmyt8BjdygAKpO+TzwT3xZvu44Sdt4l4mrQyO6IAAAAAFSApB8AAACAOrczt1i+35ErW3NK7A4FQBVZG9ZIyWv/EnfPM8V95rniioyyOyQAAAAAZZD0AwAAAFBnsoo8Mmt7rvyWUWR3KACOhMcj3vmzxLt8qYSde4G4u55kd0QAAAAA/kDSDwAAAECdrNv3854CmbczT4q8lt3hADha2Zni+eID8S5ZKGHnXyquJs3sjggAAAAIeST9AAAAANSqXXkl8m1qtuzOZ90+INhYv2+SkjeeE/fJvcR9znniiom1OyQAAAAgZJH0AwAAAFArijyWzN2ZK0v2FAhz+4AgZnnFu3i+eFctk7C+A8Xd/VS7IwIAAABCEkk/AAAAADVufWahTN+aK1nFXrtDAVBX8nLF880nB5J/F/5FXEn17I4IAAAACCluuwMAAAAAEDxyi73y5eYs+XxTNgk/IERZG9dJyav/Es/PC+0OBQAAAAgpzPQDAAAAUCNW7y+U6dtyJN9DMU8g5BUVinfyZ2L99uuBWX/J9e2OCAAAAAh6zPQDAAAAcFRyir3y+aYs+eb3bBJ+APxYm9ZLyWtPi+fnBWJZ/PcBAAAAqE3M9AMAAABwxFbsK5CZ23OlgGQfgEPO+vtcrNXLJewiZv0BAAAAtYWZfgAAAACqLbvYI59uzJTJqTkk/ABUibX5j1l/i5n1BwAAANQGZvoBAAAAqJaNmUUyKTVb8kvotAdwBLP+pnx+YK2/i68SV1I9uyMCAAAAggYz/QAAAABUideyZPb2XPl0UxYJPwBHxdq8QUreeFa8a1fZHQoAAAAQNEj6AQAAADis7CKPfLg+Uxal5dsdCoBgkZ8nnolvi2fq12J5PHZHAwAAADge5T0BAAAAHNKmrCL57++U8wRQO7yL5oq1dYuEXX6duJLr2x0OAAAA4Fi2zvR77LHHxOVy+W0dO3YsPV5QUCDDhg2TBg0aSHx8vAwaNEh2797td43U1FQZOHCgxMbGSuPGjeW+++6TkpISGz4NAAAAEITlPHfkyicbKecJoHZZ21MPlPv8bbndoQAAAACOZftMv86dO8uMGTNKn4eH/y+k4cOHy+TJk+XTTz+VpKQkue222+Syyy6T+fPnm+Mej8ck/FJSUmTBggWyc+dOGTx4sERERMiYMWNs+TwAAABAsJTz/HpLtmzLZUAdgDpSkC+eT94V65TTxd3vInGV6R8AAAAAcHi2t6A1yadJu/IyMzNl/Pjx8uGHH0rv3r3NvgkTJsjxxx8vixYtkp49e8q0adNk9erVJmnYpEkTOfHEE2X06NHywAMPmFmEkZGRNnwiAAAAwNko5wnATt7F88W77XcJ13Kf9RvaHQ4AAADgGLaW91Tr16+XZs2aSdu2beWaa64x5TrVkiVLpLi4WPr27Vt6rpb+bNWqlSxcuNA818euXbuahJ9P//79JSsrS1atWmXDpwEAAACci3KeAALGzm0Hyn2u/MXuSAAAAADHsHWm36mnnirvvPOOdOjQwZTmHDlypPz5z3+WlStXyq5du8xMveTkZL/XaIJPjyl9LJvw8x33HatMYWGh2Xw0SQgAAACEMsp5Agg4RYXi+fx9sbZuEXf/i8Xltn3cMgAAABDQbE36nX/++aU/d+vWzSQBW7duLZ988onExMTU2vuOHTvWJBgBAAAAHCjnOen3bMljdh+AAOT9aZ5Ye9Mk7IrB4oquvb4CAAAAwOkCapiczuo77rjjZMOGDWadv6KiIsnIyPA7Z/fu3aVrAOqjPi9/3HesMiNGjDBrBvq2rVu31srnAQAAAALdot15ppwnCT8AgczatE5K3npBrH177A4FAAAACFgBlfTLycmRjRs3StOmTaVHjx4SEREhM2fOLD2+du1as+Zfr169zHN9XLFihaSlpZWeM336dElMTJROnTpV+j5RUVHmnLIbAAAAEGrr932XmiOzd+TZHQoAVM2+PSbx5920zu5IAAAAgIBka9Lv3nvvlTlz5siWLVtkwYIFcumll0pYWJj89a9/laSkJLnhhhvk7rvvllmzZsmSJUvkb3/7m0n09ezZ07y+X79+Jrl33XXXya+//ipTp06Vhx56SIYNG2YSewAAAAAOVuSx5LNNWbJsX4HdoQBA9RTki+eDN8WzeL7dkQAAAAABx9Y1/bZt22YSfPv27ZNGjRrJGWecIYsWLTI/q+eee07cbrcMGjRICgsLpX///vLqq6+Wvl4ThJMmTZKhQ4eaZGBcXJwMGTJERo0aZeOnAgAAAAJXdrFHPtuYJbvzPXaHAgBHxusV75QvRNJ2ifv8S8TlDrM7IgAAACAg2Jr0mzhx4iGPR0dHyyuvvGK2yrRu3VqmTJlSC9EBAAAAwSUtv8Qk/LKKvXaHAgBHzfvzArPGX9gVg8UVE2t3OAAAAIDtAmpNPwAAAAC1Y3NWkXywLpOEH4CgYm1eb9b5s/am2R0KAAAAYDuSfgAAAECQW76vQD7dmCWFXsvuUACg5u3fKyXjXxTvlo12RwIAAADYiqQfAAAAEMTm7siVKak5wvw+AEGtIF88H/xbvGtXSqByuVyH3B577DG7QwQAAIDD2bqmHwAAAIDa4fFaJtm3Kr3Q7lAAoG6UlIjnk3dFLrpS3CecLIFm586dpT9//PHH8sgjj8jatWtL98XHx5f+bFmWeDweCQ+n2wYAAABVx0w/AAAAIMgUlHjl441ZJPwAhB6vVzxfTRTPorkSaFJSUkq3pKQkM7vP93zNmjWSkJAg3377rfTo0UOioqJk3rx5cv3118sll1zid5277rpLzj777NLnXq9Xxo4dK23atJGYmBg54YQT5LPPPrPhEwIAAMBuDBkDAAAAgkhGoUc+3ZQl+wo8docCADaxxDv1a5G8XAnrfb44yT/+8Q95+umnpW3btlKvXr0qvUYTfu+//768/vrr0r59e5k7d65ce+210qhRIznrrLNqPWYAAAAEDpJ+AAAAQJDYk18iEzdkSm6JZXcoAGA77w8zRPLzxD3gUnG5nFHoaNSoUXLuuedW+fzCwkIZM2aMzJgxQ3r16mX2acJQZwm+8cYbJP0AAABCDEk/AAAAIEgSfh9tyJQ8En4AUMr78wKxCvIk7JKrxRUWJoHu5JOrtxbhhg0bJC8v76BEYVFRkXTv3r2GowMAAECgI+kHAAAAONxeEn4AUClr5TLxFBRI2F+GiCsiUgJZXFyc33O32y2W5f/f9uLi4tKfc3JyzOPkyZOlefPmfufpuoAAAAAILST9AAAAAAcj4QcAh2dtWCOe/7whYX+9QVwxseIUui7fypUr/fYtW7ZMIiIizM+dOnUyyb3U1FRKeQIAAECcUdQeAAAAwEH2FhxI+LGGHwAcnrV1i0n8WQX54hS9e/eWn3/+Wd577z1Zv369PProo35JwISEBLn33ntl+PDh8u6778rGjRtl6dKl8tJLL5nnAAAACC0k/QAAAACnJvzWk/ADgOqwdm4TzwdvilVUKE7Qv39/efjhh+X++++XU045RbKzs2Xw4MF+54wePdqcM3bsWDn++OPlvPPOM+U+27RpY1vcAAAAsIfLKl8cPgRlZWVJUlKSZGZmSmJiooSqJ3/Za3cICBD/6N7Q7hAAIODQXkAg2VdQIh+S8EMAa1e4Ty6e+m+7wwAq5WrVVsKuvSng1/hDaKCdCQAVo78aPvRXS5XbC8z0AwAAAByEhB8AHD0rdZN4PnpbrJJiu0MBAAAAagxJPwAAAMAhSPgBQM2xNq8Xz8fviOUpsTsUAAAAoEaQ9AMAAAAcgIQfANQ8a8Ma8Xz6nlgej92hAAAAAEeNpB8AAADggITfR+uzSPgBQC2w1q4Szxfvi+X12h0K6thrr70m3bp1M+vi6NarVy/59ttvS4+fffbZ4nK5/LZbb73V7xqpqakycOBAiY2NlcaNG8t9990nJSXMHgUAAPYIt+l9AQAAAFTB/gKPSfjllNAZDQC1xVq9XDxhH0rYpVeLy8X46FDRokULefLJJ6V9+/ZiWZa8++67cvHFF8svv/winTt3NufcdNNNMmrUqNLXaHLPx+PxmIRfSkqKLFiwQHbu3CmDBw+WiIgIGTNmjC2fCQAAhDaSfgAAAECAyizyyIcbMkn4AUAdsFb8Ip6wcAm76EozowvB78ILL/R7/sQTT5jZf4sWLSpN+mmST5N6FZk2bZqsXr1aZsyYIU2aNJETTzxRRo8eLQ888IA89thjEhkZWSefAwAAwIfhawAAAEAAKvB45dONWZJTTMIPAOqKtWyxeKd+bXcYsIHO2ps4caLk5uaaMp8+H3zwgTRs2FC6dOkiI0aMkLy8vNJjCxculK5du5qEn0///v0lKytLVq1aVeefAQAAgJl+AAAAQIDxWJZ8uSlb9hZ47A4FAEKO98cfRBKTJOy0c+wOBXVgxYoVJslXUFAg8fHx8uWXX0qnTp3Msauvvlpat24tzZo1k+XLl5sZfGvXrpUvvvjCHN+1a5dfwk/5nuuxyhQWFprNR5OEAAAANYGkHwAAABBgvkvNkd9ziu0OAwBClnf6ZHElJIm760l2h4Ja1qFDB1m2bJlkZmbKZ599JkOGDJE5c+aYxN/NN99cep7O6GvatKn06dNHNm7cKMcee+wRv+fYsWNl5MiRNfQJAAAA/ofyngAAAEAAWbArT1bs/9/ofwCAHSzxfD1RvJs32B0Iapmuu9euXTvp0aOHScadcMIJ8sILL1R47qmnnmoeN2w48Heha/3t3r3b7xzf88rWAVRaJlSTjL5t69atNfiJAABAKCPpBwAAAASI1fsLZe7O/60VBACwkccjno8niLV7h92RoA55vV6/0ptl6YxApTP+lJYF1fKgaWlppedMnz5dEhMTS0uEViQqKsqcU3YDAACoCZT3BAAAAALA1pximZyabXcYAICyCguk5MO3JPzGO025TwQXnXF3/vnnS6tWrSQ7O1s+/PBDmT17tkydOtWU8NTnAwYMkAYNGpg1/YYPHy5nnnmmdOvWzby+X79+Jrl33XXXybhx48w6fg899JAMGzbMJPYAAADqGjP9AAAAAJvtL/DI55uyxGPZHQkA4CBZmVLy0dtiFVF6OdjoDL3Bgwebdf10rb7FixebhN+5555ryn7OmDHDJPY6duwo99xzjwwaNEj++9//lr4+LCxMJk2aZB511t+1115rrjdq1ChbPxcAAAhdzPQDAAAAbJRX4pVPNmZKARk/AAhcO7eJ54sPJOzK68XlYvx0sBg/fnylx1q2bClz5sw57DVat24tU6ZMqeHIAAAAjgwtVQAAAMAmJV7LzPDLKPLaHQoA4DCstavEO+1/s7wAAACAQEPSDwAAALCBZVky6fds2Z5bYncoAIAq8i6aK57FC+wOAwAAAKgQST8AAADABnN25MmajCK7wwAAVJP3u6/Em7rZ7jAAAACAg5D0AwAAAOrYsr0Fsigt3+4wAABHwusRz6fviZWTZXckAAAAgB+SfgAAAEAd2ppTLNO25tgdBgDgaORkieez/4jl9dgdCQAAAFCKpB8AAABQR/JKvPLNlmzx2h0IAOCoWb9vEu/0SXaHAQAAAJQi6QcAAADUAcuyZNLv2ZJdTMoPAIKFd9Fc8a78xe4wAAAAAIOkHwAAAFAHfkzLl01ZxXaHAQCoYZ5vPhFrzy67wwAAAABI+gEAAAC1bVtOsczdkWd3GACA2lBcJCUfvyNWYYHdkQAAACDEkfQDAAAAalE+6/gBQPDbt0c8X31kSjkDAAAAdiHpBwAAANSiyb/nSBbr+AFA0LPWrBTv/O/tDgMAAAAhjKQfAAAAUEt+SsuXDVlFdocBAKgj3u+/E++mdXaHAQAAgBBF0g8AAACoBTtyi2X2jly7wwAA1CXLe6DMZz7ruAIAAKDukfQDAAAAalhBiVe+0nX8WNoJAEJPdpZ4Jn1qdxQAAAAIQST9AAAAgBo2OTVHsopYxw8AQpW1erl4f/3Z7jAAAAAQYkj6AQAAADVocVq+rM9kHT8ACHWeb78UK2O/3WEAAAAghJD0AwAAAGrIzjzW8QMA/KGwQDxffiSWxcxvAAAA1A2SfgAAAEANKPR45evN2eJhHT8AwB+s1E3inT/b7jAAAAAQIkj6AQAAADVg1vY8yWAdPwBAOd7Z34m1a7vdYQAAACAEkPQDAAAAjlJqdrEs21dgdxgAgEDk8UjJFx+IVVJsdyQAAAAIciT9AAAAgKNQ4rXku605docBAAhke3aLd/pku6MAAABAkCPpBwAAAByF+bvyZH+hx+4wAAABzvvTPPFuXGt3GAAAAAhiJP0AAACAI5SWXyI/7s63OwwAgCNY4vnmY7GKCu0OBAAAAEGKpB8AAABwBLyWJVNSc8RrdyAAAOfIyhTv99/ZHQUAAACCFEk/AAAA4AgsTsuXXXkldocBAHBgmU9r5za7wwAAAEAQIukHAAAAVFNGoUfm7cqzOwwAgBNZXvFM+kwsi7niAAAAqFkk/QAAAIBq+m5rjhTTVwsAOELWjq3i/Wm+3WEAAAAgyJD0AwAAAKphxb4C2ZJdbHcYAACH8876TqysTLvDAAAAQBAh6QcAAABUUW6xV2Zuz7U7DABAMCgsEM93X9kdBQAAAIIIST8AAACgimZsy5ECj2V3GACAIGH9tly861bbHQYAAACCBEk/AAAAoAo2ZBbJbxlFdocBAAgynilfiFXM/QUAAABHj6QfAAAAcBiFHq9M25pjdxgAgGCUmS7eWVPtjgIAAABBgKQfAAAAcBgLduVLVrHX7jAAAEHK++NcsXbtsDsMAAAAOBxJPwAAAOAQMos8smRPvt1hAACCmdcrnqlf2x0FAAAAHI6kHwAAAHAIP+zMkxLL7igAAMHO2rJBvOtW2x0GAAAAHIykHwAAAFCJ3Xklsmp/od1hAABChGf6JLG8lJMGAADAkSHpBwAAAFRi9o5cYZIfAKDO7N0t1i8/2h0FAAAAHIqkHwAAAFCBLdlFsjm72O4wAAAhxjN7qlhFzDIHAABA9ZH0AwAAAMqxLEtmbc+1OwwAQCjKyRbvgtl2RwEAAAAHIukHAAAAlLM6vVB253vsDgMAEKI06WdlZ9kdBgAAAByGpB8AAABQhsdrydydeXaHAQAIZcVF4pn1nd1RAAAAwGFI+gEAAABlLNlbIJlFXrvDAACEOGvZYrHSdtkdBgAAAByEpB8AAADwhwKPVxbuYpYfACAAWF7xzJhkdxQAAABwEJJ+AAAAwB8W7c6XfI9ldxgAABjW+t/Eu2md3WEAAADAIUj6AQAAACKSVeSRn9Py7Q4DAAA/Xtb2AwAAQBWR9AMAAABE5IedeVLCJD8AQICxtv3ObD8AAABUCUk/AAAAhLw9+SWycn+h3WEAAFAh7w8z7A4BAAAADkDSDwAAACFPZ/kxyQ8AEKisLRvFm7rZ7jAAAAAQ4Ej6AQAAIKTtKyiRdZlFdocBAMAhMduv5r322mvSrVs3SUxMNFuvXr3k22+/LT1eUFAgw4YNkwYNGkh8fLwMGjRIdu/e7XeN1NRUGThwoMTGxkrjxo3lvvvuk5KSEhs+DQAAAEk/AAAAhLhFu/PtDgEAgMOyNqwRa8dWu8MIKi1atJAnn3xSlixZIj///LP07t1bLr74Ylm1apU5Pnz4cPnvf/8rn376qcyZM0d27Nghl112WenrPR6PSfgVFRXJggUL5N1335V33nlHHnnkERs/FQAACGUBk/TTRpbL5ZK77rqrdB8jqgAAAFCbsoo8sjqdtfwAAM7gYbZfjbrwwgtlwIAB0r59eznuuOPkiSeeMP1PixYtkszMTBk/frw8++yzJhnYo0cPmTBhgknu6XE1bdo0Wb16tbz//vty4oknyvnnny+jR4+WV155xSQCAQAAQjLpt3jxYnnjjTdMSYWyGFEFAACA2rQ4LV88LOYHAHAIa80qsdJ22h1GUNI+pokTJ0pubq4p86mz/4qLi6Vv376l53Ts2FFatWolCxcuNM/1sWvXrtKkSZPSc/r37y9ZWVmlswUrUlhYaM4puwEAAARF0i8nJ0euueYaefPNN6VevXql+xlRBQAAgNqUX+KVX/cxyw8A4CQWs/1q2IoVK8zsvqioKLn11lvlyy+/lE6dOsmuXbskMjJSkpOT/c7XBJ8eU/pYNuHnO+47VpmxY8dKUlJS6dayZcta+WwAACD02J700/KdOluv7MgpxYgqAAAA1KYlewqkyMs0PwCAs1irfhVr3x67wwgaHTp0kGXLlsmPP/4oQ4cOlSFDhpgB5rVpxIgRZrC7b9u6lbUaAQBAzQgXG2nZhKVLl5rynuXV9oiqkSNH1tCnAAAAgNMUey1Zsiff7jAA4Ig89cNP8tVv62Xt3v0SEx4uPVs2kzHn/lk6NKxvjm9Jz5TjXhhf4Ws/vOICubzzcQftL/Z45JHv58t36zfL5vRMSYqKkt5tW8kTff8szRLjzTmFJSVyyzfT5b9rNkqT+Fh5aWAf6XNs69JrPDN/sWzNzJbnB/Sutc8OzfpZ4pk3U8IvvsruSIKC9j21a9fO/KxVprSP6oUXXpArr7zSVJHKyMjw65vavXu3pKSkmJ/18aeffvK7nh73HauMzirUDQAAIGhm+ukopjvvvFM++OADiY6OrtP3ZkQVAABAaFuxr0DyWcwPgEP9sGWrDD3lRPnhxr/KlMGXS4nXKwP/87nkFhWb4y2TEiT1nlv8tkfO7iXxkRFyXrtjKrxmXnGJLNuZJg+e2VN+vOVa+eTKC2XdvnS57KOvS895a8kKWbpjt8y98Sq5sUc3Gfz5FLGsA/8t1UTh+CUrZFTv0+voWwht1vIlYmWm2x1GUPJ6vaZClCYAIyIiZObMmaXH1q5dK6mpqWbNP6WPWh40LS2t9Jzp06dLYmKiKREKAAAQMjP9tHynNopOOukkv0WT586dKy+//LJMnTqVEVUAAACocdpB/fOeArvDAIAjNum6QX7P37qkvzT/1+smIffnY1pImNstKQlxfud8vWaDmeEXHxVZ4TWToqPk28GX++17YUBvOe3NDyU1I0taJSfKmj375YIOx0rnxg2lbb0k+cf0ubI3L18axcXK7ZNmmNmGidH8W7tOeL3i/XmhhPUZYHckjqaDws8//3yzlEx2drZ8+OGHMnv2bNMnpWvt3XDDDXL33XdL/fr1TSLv9ttvN4m+nj17mtf369fPJPeuu+46GTdunKk69dBDD5mlbOh3AgAAITXTr0+fPmY0lNZN920nn3yyXHPNNaU/M6IKAAAANW1jVrHsL/TYHQYA1JjMgkLzWC+m4io6mgz8ddce+Vv3rtW+rktEkv9I5HVLaSQLUrdLfnGxTNvwuzSNj5OGsTHy4fLfJCo8XC45vn0NfBpUlfeXH8XylNgdhqNpf9LgwYPNun7aT6WlPTXhd+6555rjzz33nFxwwQUyaNAgOfPMM80A8y+++KL09WFhYTJp0iTzqH1U1157rbneqFGjbPxUAAAglNk20y8hIUG6dOnity8uLk4aNGhQup8RVQAAAKhpi9NYyw9A8PB6Lbn3u9lyWstm0qVJwwrPmbB0pXRsWF96tWpW5esWFJfIgzN+kCu7diydvXd9986yYvceOeGVd6WBJvuuuEDS8wtk1KwFMv36v8gjM+fLpyvXSNv6yfLvi/tJ88SEGvucqEBujlirfhVXtx52R+JY48dXvPaljy5H88orr5itMq1bt5YpU6bUQnQAAAAOSvpVhY6ocrvdZkSV1lPv37+/vPrqqweNqBo6dKhJBmrScMiQIYyoAgAAQIXS8kvk95wDa14BQDC4Y8pMWZW2T2b935UVHtdZeRNXrJEHzzq1ytcs9njkr59OEl2u7+WBfUr3R4SFyYtlnqsbv5oqw07tbtYD/GbNBvl56GB5ev5iGf7tLPnkyouO4pOhKryL54ubpB8AAAACMemnddPLYkQVAAAAatLPzPIDEETunDxTpqzbJDP/dqW0SKp4Vt3nq9dLXnGxXHtCp2ol/FIzs2TakCsOuUbf7M2psjptr7xx0bnyj2lz5bz2bSQuMsKsHfjaT8uO+HOh6qxtv4u1c5u4mrawOxQAAACE8pp+AAAAQF3KLfbKqvQD614BgJNZlmUSfl+v2SBTh1whbeolVXruO0tXygUdjpVGcbFVTvht2Jch3w2+3JTwPFT5zzsmfy+vXHiuhLnd4rEsKfF6/7iOVzxe6wg/HarL89M8u0MAAABAgCDpBwAAgJDwy94C8dAHDSAIaLLtw+Vr5L1BAyQhMlJ2ZeeaTUt5lrVhX7r88Ps2+b+TulR4nS4vTZCvfltfmvC78pNJsnTHbnl30ACTtPNdt6jEc9Brn5i7SM5v30a6N21snvdq2cxca/muPWaW32nVWD8QR8dauUys/Dy7wwAAAEAACKjyngAAAEBtzYpZvr/A7jAAoEa88fOv5rHvO5/67X/r4v4yuHvn0ufv/LJKWiQmyLnHHlPhddbtS5esgiLz8/asHJm0dqP5+ZTX/+N33vQhV8hZbVqWPl+5e698vmqdLL71utJ9gzodJ3O3bJPeEz6W4xrUMwlJ1JGSYvEu/VHCTj/H7kgAAABgM5elPSAhLisrS5KSkiQzM1MSExMlVD35y167Q0CA+Ef3hnaHAAABh/aCs/2eXSQfbciyOwwgZLQr3CcXT/233WEAoSO5voTfMUJcLgo6ORHtTACoGP3V8KG/WqrcXqA1CAAAgKC3cj9r+QEAgljGfrHWr7E7CgAAANiMpB8AAACCWpHHkrUZB8rXAQAQrLyL59sdAgAAAGxG0g8AAABBbV1moRR5Q76iPQAgyFkb1oqVmW53GAAAALARST8AAAAEtRX7KO0JAAgFlniXL7U7CAAAANiIpB8AAACCVlaRR1Jziu0OAwCAOuFdscTuEAAAAGAjkn4AAAAIWiv3FwqFPQEAIWPPbrF2brM7CgAAANiEpB8AAACCOukHAEAo8a6gxCcAAECoIukHAACAoLQ9t1j2F3rsDgMAgDrlXfmLWJbX7jAAAABgA5J+AAAACErM8gMAhKTsLLE2b7A7CgAAANiApB8AAACCTonXkt/SSfoBAEKTd/kSu0MAAACADUj6AQAAIOhsyCySAo9ldxgAANjC+m2FWMXFdocBAACAOkbSDwAAAEFnxf4Cu0MAAMA+RYVirV1pdxQAAACoYyT9AAAAEFRyi72yOYvZDQCA0OZdsdTuEAAAAFDHSPoBAAAgqKxOLxSv3UEAAGAza8NasfJy7A4DAAAAdYikHwAAAILKusxCu0MAAMB+Xo9Z2w8AAAChg6QfAAAAgkZBiVe255TYHQYAAAHBu3aV3SEAAACgDpH0AwAAQNDYnF1MaU8AAP5gbVovVhEz4AEAAEIFST8AAAAEjY1ZRXaHAABA4PCUmLX9AAAAEBpI+gEAACAoWJYlm0j6AQDgx7t2pd0hAAAAoI6Q9AMAAEBQ2JVXInkllt1hAAAQUKz1v4nl9dgdBgAAAOpAeF28CQAAAFDbNjDLDwCAg+XniZW6RVzHHCvBIDc3V5588kmZOXOmpKWlidfrv5rvpk2bbIsNAADAbiT9AAAAEBQ2ZRXbHQIAAAE720+CJOl34403ypw5c+S6666Tpk2bisvlsjskAACAgEHSDwAAAI6XW+yVnXkldocBAEBA8m74TcLOvUCCwbfffiuTJ0+W008/3e5QAAAAAg5r+gEAAMDxNlHaEwCAyqXtEiszXYJBvXr1pH79+naHAQAAEJBI+gEAAMDxNpL0AwDgkLxa4jMIjB49Wh555BHJy8uzOxQAAICAQ3lPAAAAOJrXsmRzNuv5AQBw2HX9Tj5NnKh79+5+a/dt2LBBmjRpIsccc4xERET4nbt06VIbIgQAAAgMJP0AAADgaNtySqTQY9kdBgAAAc3avEEsj0dcYWHiNJdccondIQAAADgCST8AAAA4GqU9AQCoguIisXZuE1eL1uI0jz76qN0hAAAAOAJr+gEAAMDRSPoBAFA11u8bxenatm0r+/btO2h/RkaGOQYAABDKSPoBAADAsTKLPLK3wGN3GAAAOIL1+yZxui1btojHc/C9v7CwULZt22ZLTAAAAIGC8p4AAABwrN+zi+0OAQAAx7C2bhHL8orL5bwx4N98803pz1OnTpWkpKTS55oEnDlzprRp08am6AAAAAIDST8AAAA41o7cErtDAADAOQryRXbvEklpJk5zySWXmEeXyyVDhgzxOxYRESHHHHOMPPPMMzZFBwAAEBhI+gEAAMCxduQx0w8AgOrw/r5RwhyY9PN6veZRZ/MtXrxYGjZsaHdIAAAAAcd59RwAAAAAESn2WrInn/X8AACoDivV2ev6bd68mYQfAABAJZjpBwAAAEfalVcilt1BAADgMNbvm8XJXnzxxQr3a9nP6OhoadeunZx55pkSFhZW57EBAADYjaQfAAAAHGlHLqU9AQCottxssfamiathY3Gi5557Tvbs2SN5eXlSr149sy89PV1iY2MlPj5e0tLSpG3btjJr1ixp2bKl3eECAADUKcp7AgAAwJF25JXYHQIAAI5k/e7cEp9jxoyRU045RdavXy/79u0z27p16+TUU0+VF154QVJTUyUlJUWGDx9ud6gAAAB1jpl+AAAAcKSduST9AAA4Et7UTeLu0VOc6KGHHpLPP/9cjj322NJ9WtLz6aeflkGDBsmmTZtk3Lhx5mcAAIBQw0w/AAAAOE5OsVeyir12hwEAgCM5eabfzp07paTk4IE/um/Xrl3m52bNmkl2drYN0QEAANiLpB8AAAAcZzvr+QEAcOQy08XKyxUnOuecc+SWW26RX375pXSf/jx06FDp3bu3eb5ixQpp06aNjVECAADYg6QfAAAAHIfSngAAHB1r13ZxovHjx0v9+vWlR48eEhUVZbaTTz7Z7NNjKj4+Xp555hm7QwUAAKhzrOkHAAAAx9mRR9IPAICjYe3aIdL2OHGalJQUmT59uqxZs0bWrVtn9nXo0MFsZWcDAgAAhCJm+gEAAMBRLMuSXST9AAA4KtbuHeJkHTt2lIsuushsZRN+1TF27Fg55ZRTJCEhQRo3biyXXHKJrF271u+cs88+W1wul9926623+p2TmpoqAwcOlNjYWHOd++67r8J1BwEAAGobM/0AAADgKHsKPFLktewOAwAA58/0cyCPxyPvvPOOzJw5U9LS0sTr9fod//7776t8rTlz5siwYcNM4k+TdA8++KD069dPVq9eLXFxcaXn3XTTTTJq1KjS55rcKxuPJvx0BuKCBQtk586dMnjwYImIiJAxY8Yc9ecFAACoDpJ+AAAAcBTW8wMAoAbsTRPLUyKuMGd1Dd15550m6aeJti5dupiZd0fqu+++83uu19WZekuWLJEzzzzTL8mnSb2KTJs2zSQJZ8yYIU2aNJETTzxRRo8eLQ888IA89thjEhkZecTxAQAAVJezWnYAAAAIeTvyiu0OAQAA5/N6RNJ2iTRtIU4yceJE+eSTT2TAgAE1fu3MzEzzWL9+fb/9H3zwgbz//vsm8XfhhRfKww8/XDrbb+HChdK1a1eT8PPp37+/DB06VFatWiXdu3c/6H0KCwvN5pOVlVXjnwUAAIQmkn4AAABwlB3M9AMAoMZKfLoclvTTmXPt2rWr8etqmdC77rpLTj/9dDOD0Ofqq6+W1q1bS7NmzWT58uVmBp+u+/fFF1+Y47t27fJL+Cnfcz1W2VqCI0eOrPHPAAAAQNIPAAAAjmFZluwv9NgdBgAAQcHa7bx1/e655x554YUX5OWXXz6q0p7l6dp+K1eulHnz5vntv/nmm0t/1hl9TZs2lT59+sjGjRvl2GOPPaL3GjFihNx9991+M/1atmx5FNEDAAAcQNIPAAAAjpFV7BWPZXcUAAAEz0w/p9Gk3KxZs+Tbb7+Vzp07S0REhN9x3wy86rjttttk0qRJMnfuXGnR4tAzH0899VTzuGHDBpP005KfP/30k985u3fvNo+VrQMYFRVlNgAAgJpG0g8AAACOkc4sPwAAQnqmX3Jyslx66aU1VkHg9ttvly+//FJmz54tbdq0Oexrli1bZh51xp/q1auXPPHEE5KWliaNGzc2+6ZPny6JiYnSqVOnGokTAACgqkj6AQAAwDFI+gEAUIMK8sXK2C+u5PriFBMmTKjRkp4ffvihfP3115KQkFC6Bl9SUpLExMSYEp56fMCAAdKgQQOzpt/w4cPlzDPPlG7duplz+/XrZ5J71113nYwbN85c46GHHjLXZjYfAACoa+46f0cAAADgCKUXeu0OAQCAoGLtOVCK0klKSkpkxowZ8sYbb0h2drbZt2PHDsnJyanWdV577TXJzMyUs88+28zc820ff/yxOR4ZGWneRxN7HTt2NOsJDho0SP773/+WXiMsLMyUBtVHnfV37bXXyuDBg2XUqFE1/KkBAAAOj5l+AAAAcAxm+gEAULOs9H3iJL///rucd955kpqaKoWFhXLuueeaWXpPPfWUef76669Xq7znobRs2VLmzJlz2Ou0bt1apkyZUuX3BQAAqC3M9AMAAIBjkPQDAKCGOSzpd+edd8rJJ58s6enppgSnj67zN3PmTFtjAwAAsBsz/QAAAOAIOho/g6QfAAA1Stf0c5IffvhBFixYYEpvlnXMMcfI9u3bbYsLAAAgEDDTDwAAAI6QXeyVkkNX4QIAAEFe3tPr9YrHc/AgoG3btpkynwAAAKGMpB8AAAAcgdKeAADUgnRnzfTr16+fPP/886XPXS6X5OTkyKOPPioDBgywNTYAAAC7Ud4TAAAAjpBe6LU7BAAAgk9RoVh5OeKKjRcneOaZZ6R///7SqVMnKSgokKuvvlrWr18vDRs2lI8++sju8AAAAGxF0g8AAACOwEw/AABqh5W+3zFJvxYtWsivv/4qH3/8sXnUWX433HCDXHPNNRITE2N3eAAAALYi6QcAAABHIOkHAEAt0XX9mrcSpwgPDzdJPt18Nm3aJLfeeqtMmzbN1tgAAADsxJp+AAAAcASSfgAA1N5MP6fLzs6WmTNn2h0GAACArUj6AQAAIOBZliUZRST9AACoDZbO9AMAAIDjkfQDAABAwMsp9kqx1+4oAAAIUhnOn+kHAAAAkn4AAABwgIwiMn4AANQWi6QfAABAUAi3OwAAAADgcPJKSPoBAFBrsrMk0HXv3l1cLlelx/Py8uo0HgAAgEBE0g8AAAABr8Bj2R0CAADBq6RYrOIicUVESqC65JJL7A4BAAAg4JH0AwAAQMArYKYfAAC1Ky9XJClwk36PPvqo3SEAAAAEPNb0AwAAQMDLZ6YfAAC1n/QDAACAo5H0AwAAQMArKCHpBwBAbbJI+gEAADgeST8AAAAEvHwP5T0BAKhVJP0AAABCM+nXtm1b2bdv30H7MzIyzDEAAACgJjHTDwCA2sVMPwAAgBBN+m3ZskU8Hs9B+wsLC2X79u01ERcAAABQqoCZfgAA1C6HJP3ee+890/9UXlFRkTkGAAAQysKrc/I333xT+vPUqVMlKSmp9LkmAWfOnCnHHHNMzUYIAACAkJfvYaYfAAC1Kj9PnOBvf/ubnHfeedK4cWO//dnZ2ebY4MGDbYsNAADAUTP9LrnkErO5XC4ZMmRI6XPdrrrqKpk+fbo888wzVb7ea6+9Jt26dZPExESz9erVS7799tvS4wUFBTJs2DBp0KCBxMfHy6BBg2T37t1+10hNTZWBAwdKbGysafDdd999UlJSUp2PBQAAgABHeU8AAGqXU8p7WpZl+qXK27Ztm9/gdAAAgFBUrZl+Xu+Bskpt2rSRxYsXS8OGDY/qzVu0aCFPPvmktG/f3jTa3n33Xbn44ovll19+kc6dO8vw4cNl8uTJ8umnn5qG22233SaXXXaZzJ8/v3R2oSb8UlJSZMGCBbJz504zoisiIkLGjBlzVLEBAAAgMHgtS4q8JP0AAKhVAZ706969u0n26danTx8JD/9fl5b2D23evNnMAAQAAAhl1Ur6+WhDqiZceOGFfs+feOIJM/tv0aJFJiE4fvx4+fDDD6V3797m+IQJE+T44483x3v27CnTpk2T1atXy4wZM6RJkyZy4oknyujRo+WBBx6Qxx57TCIjI2skTgAAANiHWX4AANS+QJ/pp1Wm1LJly6R///6mIpSP9v/ocjNaIQoAACCUHVHST+n6fbqlpaWVzgD0efvtt6t9PR2VpTP6cnNzTZnPJUuWSHFxsfTt27f0nI4dO0qrVq1k4cKFJumnj127djUJPx9t+A0dOlRWrVplRoFVRBd8Lrvoc1ZWVrXjBQAAQN3I9/i3NQEAQC0oyJdA9uijj5pHTe5deeWVEh0dbXdIAAAAzl7Tz2fkyJHSr18/k/Tbu3evpKen+23VsWLFCjM6KyoqSm699Vb58ssvpVOnTrJr1y4zUis5OdnvfE3w6TGlj2UTfr7jvmOVGTt2rCkX6ttatmxZrZgBAABQdwo8zPQDAKDWlRSLE+hg8MoSfm+88UadxwMAAOD4mX6vv/66vPPOO3LdddcddQAdOnQwpRkyMzPls88+kyFDhsicOXOkNo0YMULuvvtuv5l+JP4AAAACUz7lPQEAqH0ejziBrtt3xx13yJgxYyQiIsLs0wHpf/vb32TevHlyyy232B0iAACAs2b6FRUVyWmnnVYjAehsvnbt2kmPHj3MDLwTTjhBXnjhBUlJSTHvk5GR4Xf+7t27zTGlj/q8/HHfscrorMLExES/DQAAAIGpgPKeAADUPock/WbNmmWqRJ1yyimyevVqmTx5snTp0sUM6NZB5QAAAKHsiJJ+N954o3z44Yc1H42IWR9Q19vTJKCO2NISoj5r166V1NRUs+af0kctD6rrCvpMnz7dJPG0RCgAAACcr5DyngAA1L6SEnECHYSuyT1N9J100kly6aWXyvDhw2X27NnSunVru8MDAABwXnnPgoIC+fe//y0zZsyQbt26lZZT8Hn22WerXGbz/PPPN/XYs7OzTSJRG2lTp041a+3dcMMNpgxn/fr1TSLv9ttvN4m+nj17mtfruoKa3NMyo+PGjTPr+D300EMybNgwM5sPAAAAzmeR8wMAoPZZXrEsr7hcRzQ+vE6tW7dOfv75Z2nRooXs2LHDDBLPy8uTuLg4u0MDAABwXtJv+fLlcuKJJ5qfV65c6XfM5XJV+To6Q2/w4MGyc+dOk+TTBKIm/M4991xz/LnnnhO32y2DBg0ys//69+8vr776aunrw8LCZNKkSTJ06FCTDNTGna4JOGrUqCP5WAAAAAhEVW9eAgCAoy3xGR7YSb8nn3xSHn30Ubn55pvlX//6l2zYsMEMBtc+pffff7+0OhQAAEAoCj/S+uk1Yfz48Yc8Hh0dLa+88orZKqOlG6ZMmVIj8QAAAAAAAIR20s+/mlOgeeGFF+Srr74ylaOUlvn86aef5MEHH5Szzz7bDBoHAAAIVUeU9AMAAADqChP9AACow3X9Any1lBUrVkjDhg399umyMzrr74ILLrAtLgAAAMcm/c4555xDlvH8/vvvjyYmAAAAAAAA2DHTL8Bpwi8jI0M+++wz2bhxo9x3331Sv359Wbp0qbRr187u8AAAAJyX9POt5+dTXFwsy5YtM+v76Zp6AAAAAAAAcBhPiQS65cuXS9++fSUpKUm2bNkiN910k0n6ffHFF5Kamirvvfee3SECAAA4K+n33HPPVbj/sccek5ycnKONCQAAAAAAAHXNATP9hg8fLtdff72MGzdOEhISSvcPGDBArr76altjAwAAsJu7Ji927bXXyttvv12TlwQAAECIY00/wHkywqIl4/iTpLBFG7ESkvh/MuAUDpjp9/PPP8stt9xy0P7mzZvLrl27bIkJAADA0TP9KrNw4UKJjo6uyUsCAAAAABxmb3icjG/fv/R5lLdEWhRnSUphpjTMz5DkvAyJy94vUVn7xZ2Z7ojZRQACQ1RUlGRlZR20f926ddKoUSNbYgIAAHB00u+yyy7ze25ZluzcudOMtnr44YdrKjYAAAAAQBAodIfLxqj6ZpNE/2Muy5KmJTnSrDBTGhVkSP28DInPSZforP0SlrlfXIUFdoUNhJ6wGh0bXqN0vb4WLVrIRRddJKNGjZJPPvnE7He5XObYAw88IIMGDbI7TAAAAFsdUWtOF0suy+12S4cOHUyjq1+/fjUVGwAAACAuqgICQc1yuWRHRILZJL7FQcfrefKlRVGWNNaEYH6GJOWkS0x2ukRoQjDn4Nk+AI5CeOAm/dq0aWMGnD/zzDNy+eWXS+PGjSU/P1/OOussU9azV69e8sQTT9gdJgAAgK2OqDU3YcKEmo8EAAAAAIBy0sNiJD0mRiSmiUg9/2Mx3uLSsqEN8g6UDY0tWzbU67UrbMCZAjjpp1WmfAPRp0+fLvPmzZPly5dLTk6OnHTSSdK3b1+7QwQAALDdUbXmlixZIr/99pv5uXPnztK9e/eaigsAAAAAgEPKd0fI+qgGZitfNtRteU3Z0OZaNjQ/Q+r9UTY0ylc2tKjQrrCBwBUeIYFMS3n6nHHGGWYDAADAUSb90tLS5KqrrpLZs2dLcnKy2ZeRkSHnnHOOTJw4kYWTAQAAAAC28rrcsj0i0WwS3/Kg4w1K8qR5cZY0zj9QNjQxJ93MEgzP2C+uvBxbYgZsF8Br+qmHH35YYmNjD3nOs88+W2fxAAAABJojas3dfvvtkp2dLatWrZLjjz/e7Fu9erUMGTJE7rjjDvnoo49qOk4AAACEKJewqB+AmrcvPNZsEpNy0LE4LRtalClNCjOloa4jmKsJwXSJzNwv7qwMEYuyoQhS4WESyFasWCGRkZFVmgkIAAAQio4o6ffdd9/JjBkzShN+qlOnTvLKK69Iv379ajI+AAAAAADqVK47QtZGNzSbJPkfC7c80qwkW5oWZEqjgkxTNjQue79EZ6VLWOY+keJiu8IGjo47TFwutwSyL7/8Uho3bmx3GAAAAAHriJJ+Xq9XIiIOrvOu+/QYAAAAUFMYtA8gkJS4wiQ1ItlsknDw8cbFudKsOFMaF2RK/bwMSchJlxgtG6rrCObl2hEyUDXhgV3ak1l8AAAAh3dELbrevXvLnXfeacp4NmvWzOzbvn27DB8+XPr06XMklwQAAAAqFBVGJx8A50iLiDObxDYTqe9/LN5TJC2KMyWlMFMa5GVIoq9saJavbKhlV9hAwCf9LP7/AQAAcFhH1KJ7+eWX5aKLLpJjjjlGWrY8sCD61q1bpUuXLvL+++8fySUBAACACsWQ9AMQJHLCImVNWCNZE93ooLKhEZZHmhdnS9PCDGmYnynJeRkSn7NfojL3S1jmfpGSErvCRqgIC+yk34QJEyQpqdz/cQAAAODniFp0muhbunSpWddvzZo1Zp+u79e3b98juRwAAABQqZjwwF5fCABqQrErTLZEJpvtoLKhliVNPLnSrChTGuf/UTY0N12is/ZLeMY+cRXk2xQ1gkqAz/QbMmSI3SEAAAAEvGq16L7//nu57bbbZNGiRZKYmCjnnnuu2VRmZqZ07txZXn/9dfnzn/9cW/ECAAAgxMSEM9MPQIhzuWR3eLzZJLa5SAP/w0meAmlenCUp+RlSP1/LhmZIbPZ+icjcL+7sLM0a2hU5nCTAk361YezYsfLFF1+YAe0xMTFy2mmnyVNPPSUdOnQoPaegoEDuuecemThxohQWFkr//v3l1VdflSZNmpSek5qaKkOHDpVZs2ZJfHy8SVDqtcND8DsFAAD2qlbr4/nnn5ebbrrJJPzK0xILt9xyizz77LMk/QAAAFBjYsKY6QcAh5IZFm221dGNRer5H4vylkjzkmxJKciQRvkZkqRlQ/9YR9CUDfV47AobAcYVHSOhZs6cOTJs2DA55ZRTpKSkRB588EHp16+frF69WuLi4sw5w4cPl8mTJ8unn35q+r50MPxll10m8+fPN8c9Ho8MHDhQUlJSZMGCBbJz504ZPHiwREREyJgxY2z+hAAAINRUK+n366+/mhFPldGG0dNPP10TcQEAAABGuNslEW6RYq/dkQCA8xS6w2VTZD2zSbnxuy7LkpSSnANlQwsypF5uhiTkpEt0drqEadnQwgK7woYd4srXlQ1+3333nd/zd955Rxo3bixLliyRM88801S1Gj9+vHz44YfSu3fv0rUFdYkbrYLVs2dPmTZtmkkS6hI4OvvvxBNPlNGjR8sDDzwgjz32mERGRtr06QAAQCiqVtJv9+7dZqRSpRcLD5c9e/bURFwAAACA32y/Yi9ZPwCoSZbLJTsjEswmcS0OKhuaXJIvLYqzpEmBriOY7lc21JWTTdnQIOOKPTCzzQkyMjLks88+k40bN8p9990n9evXl6VLl5qkW/PmzY/4uprkU3o9pcm/4uJi6du3b+k5HTt2lFatWsnChQtN0k8fu3bt6lfuU0uAarnPVatWSffu3Q96Hy0TqptPVpaW4QUAAKjjpJ82nFauXCnt2rWr8Pjy5culadOmNRAWAAAA8D/R4S7JKrY7CgAILRnhMWZbGdOkgrKhxdKyOFtSCjOlYX66JOVmSFxOukTpOoKZ6SJeyoY6Tly8OIH2PWkSTkttbtmyxSxDo0k6XZtP19Z77733jui6Xq9X7rrrLjn99NOlS5cuZt+uXbvMTL3k5GS/czXBp8d855RN+PmO+45VRNf7Gzly5BHFCQAAUGNJvwEDBsjDDz8s5513nkRHR/sdy8/Pl0cffVQuuOCC6lwSAAAAqOK6fnQgA0CgKHRHyIao+maTxDZ+x9yWV5pq2dDCTGlUkCn18tIlXsuGZv1RNrTofzOcEEAcMtPv7rvvluuvv17GjRsnCQkJfn1WV1999RFfV9f204Hu8+bNk9o2YsQI8znKzvRr2bJlrb8vAAAIftVK+j300ENm5NRxxx1nFi7u0KGD2b9mzRp55ZVXzOLF//znP2srVgAAAISomHCX3SEAAKrI63LL9ohEs0n8wYmM+p48aV6oZUMzpH5+hiTmpEtMdrpEZO4TV26OLTFDxOWQmX6LFy+WN954o8LqVJXNrDsc7eOaNGmSzJ07V1q0aFG6PyUlRYqKikw50bKz/XT5Gz3mO+enn37yu54e9x2rSFRUlNkAAABsTfppeYIFCxaYuuQ6KsmyDtTvd7lcpl65Jv7KlzQAAAAAjlZMuM70AwAEg/1hsbI/NlZWxB6cEInzFkuzokxpWpgpDfIzDpQNzd4vkVn7xZ2VoTUYbYk5JDgk6afJsorWwFu3bp00atSoWtfSfq3bb79dvvzyS5k9e7a0aeM/a7VHjx4SEREhM2fOlEGDBpl9a9euNWVEe/XqZZ7r4xNPPCFpaWnSuHFjs2/69OmSmJgonTp1OopPCgAAUMtJP9W6dWuZMmWKpKeny4YNG0wDqX379lKvXrkC/wAAAEANiQljph8AhIJcd4Ssj25oNknyPxbm9UqzkmyTFGyUnyHJeRmmbGhU1n4Jz9gnUszir0fDFeuMpN9FF10ko0aNkk8++aR0ILom4R544IHSxFx1Snp++OGH8vXXX5tSob6ZgrpeYExMjHm84YYbTClOXTdQE3maJNREX8+ePc25/fr1M8m96667zpQc1WtopSy9NrP5AABAwCf9fDTJd8opp9RsNAAAAEAFopnpBwAhz+N2y9bIJLNJfKuDjjcsyZUWRVnSuCBD6uVlSGJuusRkpUu4lg3Ny7UlZkeJc8aafs8884xcfvnlZlZdfn6+nHXWWSbR5ptxVx2vvfaaeTz77LP99k+YMMGsG6iee+45cbvdJqFYWFhoKl29+uqrpeeGhYWZ0qBaFUtjiIuLkyFDhpjEJAAAgGOSfgAAAEBdYaYfAOBw9obHmU1im4rU9z8W5ymSFsWZkqJlQ/MOlA2NzdkvkZlaNjRTxKJsqMQ6I+mns++0fOa8efNk+fLlkpOTIyeddJL07du32tfyLVtzKNHR0WY5G90OVxULAADAbiT9AAAAEPBY0w8AcDRywyJlbVgjWRvd6KCyoeGWR5oXZ5t1BMuXDQ3L2C9SEgJlQ6OixRXmrC6iM844w2wAAAD4H2e16AAAABCSYsKZ6QcAqB0lrjD5PTLZbJLQ2v+gZUljT540L8yQxgWZUj8vQxJy0yVa1xHM3C+u/DwJCnHOWM9PvfjiixXu17X9dFZeu3bt5MwzzzRlNwEAAEINST8AAAAEvFhm+gEA7OBySVp4nNkkrrlIA//DCZ5CaVmUJU0KM6RB/oF1BGOz0iUia7+4s7Vs6OHLRwYCl4OSfrrG3p49eyQvL0/q1atn9qWnp0tsbKzEx8dLWlqatG3bVmbNmiUtW7a0O1wAAIA6RdIPAAAAAS8xwi06188ZXacAgFCRHRYlq2MamU2S/Y9FekukRXG2pBRmSMOCTEnOzZC4nP0SlblfwjLTRTwlEjCSDyTPnGDMmDHy73//W9566y059thjzb4NGzbILbfcIjfffLOcfvrpctVVV8nw4cPls88+sztcAACAOkXSDwAAAAEvzO2ShAi3ZBV77Q4FAIAqKXKHy6aoemaTRP9jLsuSlJJcaVaUKY0KMqR+bobE56ZLjK4jqGVDC/LrNFZXcrkpjAHsoYceks8//7w04ae0pOfTTz8tgwYNkk2bNsm4cePMzwAAAKGGpB8AAAAcITkqjKQfACAoWC6X7IyIN1tFZUOTPAXSQsuGFvyvbGhMdrpEakIwO6vG5767HDTTb+fOnVJScvAsSd23a9cu83OzZs0kOzvbhugAAADsRdIPAAAAjpAc6ZZUu4MAAKAOZIZFS2ZMtKyKaSxSLh8X5S3+o2xopjTMz5DkvAyJy94vUbqOoCkb6qn+GybXF6c455xzTClPLe/ZvXt3s++XX36RoUOHSu/evc3zFStWSJs2bWyOFAAAoO6R9AMAAIBjZvoBABDqCt0RsjGqvtkqKhvatCRHmhX+UTY0L0Pic9Il2lc2tLCgwmu6HJT0Gz9+vFx33XXSo0cPiYiIKJ3l16dPH3NMxcfHyzPPPGNzpAAAAHWPpB8AAAAcgaQfAACHLxu6IyLBbBLf4qDj9Tz50qIwU5oUZkr9vHRJzMmQmJx0CU9yTnnPlJQUmT59uqxZs0bWrVtn9nXo0MFsZWcDAgAAhCKSfgAAAHBMeU8AAHDk0sNiJD02RlbEppSWDU2KdMvQMOcNrOnYsaPZAAAA8D8k/QAAAOAI9ZjpBwBAjavvwPvrtm3b5JtvvpHU1FQpKiryO/bss8/aFhcAAIDdSPoBAADAEWLC3RId5pICj2V3KAAABI360c5K+s2cOVMuuugiadu2rSnx2aVLF9myZYtYliUnnXSS3eEBAADYihpJAAAAcIwGDuuYBAAg0Dltpt+IESPk3nvvlRUrVkh0dLR8/vnnsnXrVjnrrLPkiiuusDs8AAAAW5H0AwAAgGM0cFjHJAAAgc5pSb/ffvtNBg8ebH4ODw+X/Px8iY+Pl1GjRslTTz1ld3gAAAC2IukHAAAAx2CmHwAAoV3eMy4urnQdv6ZNm8rGjRtLj+3du9fGyAAAAOzHmn4AAABwjAbRNF8BAKgp4S6RxAhnjQfv2bOnzJs3T44//ngZMGCA3HPPPabU5xdffGGOAQAAhDJ6TQAAAOAYzPQDAKDm1IsKE5fLJU7y7LPPSk5Ojvl55MiR5uePP/5Y2rdvb44BAACEMpJ+AAAAcIzkSLeZlVBi2R0JAADO19Bhg2k8Ho9s27ZNunXrVlrq8/XXX7c7LAAAgIDhrBoOAAAACGk6G0FnJQAAgKOXEuusseBhYWHSr18/SU9PtzsUAACAgETSDwAAAI7itFkJAAAEqmZxEeI0Xbp0kU2bNtkdBgAAQEAi6QcAAABHaerADkoAAAKNy4Ez/dTjjz8u9957r0yaNEl27twpWVlZfhsAAEAoc17rDgAAACGteRxNWAAAamLmfIRbU3/OMmDAAPN40UUXmbLfPpZlmee67h8AAECooscEAAAAjtIkJlzCXCIey+5IAABwrmYOHUQza9Ysu0MAAAAIWM5s4QEAACBkhbtdJvG3I6/E7lAAAHCsprHOLJd91lln2R0CAABAwGJNPwAAADiOU2cnAAAQKJo6cD0/nx9++EGuvfZaOe2002T79u1m33/+8x+ZN2+e3aEBAADYiqQfAAAAHKd5nDNnJwAAEAgi3CKNYsLEiT7//HPp37+/xMTEyNKlS6WwsNDsz8zMlDFjxtgdHgAAgK1I+gEAAMBxmOkHAMCR0zLZbpdLnOjxxx+X119/Xd58802JiPjfIKDTTz/dJAEBAABCGUk/AAAAOE5SZJjE6zQFAAAQUqU9165dK2eeeeZB+5OSkiQjI8OWmAAAAAIFPSUAAABwpGYO7rAEAMBOTR1cJjslJUU2bNhw0H5dz69t27a2xAQAABAoSPoBAADAkZpT4hMAgJAbOHPTTTfJnXfeKT/++KO4XC7ZsWOHfPDBB3LvvffK0KFD7Q4PAADAVs5t5QEAACCkNXPwLAUAAOwSE+6S5Kgwcap//OMf4vV6pU+fPpKXl2dKfUZFRZmk3+233253eAAAALYi6QcAAABHSokNF7dLxGvZHQkAAM7h5PX8lM7u++c//yn33XefKfOZk5MjnTp1kvj4eLtDAwAAsB3lPQEAAOBIEW6XNI5xdsclAAB1zelJv/fff9/M8IuMjDTJvj/96U8k/AAAAP5A0g8AAACO5eQ1iQAAsEOzWGeXxx4+fLg0btxYrr76apkyZYp4PB67QwIAAAgYJP0AAADgWM3jSPoBAFCdTqDm8c6+d+7cuVMmTpxoynz+5S9/kaZNm8qwYcNkwYIFdocGAABgO5J+AAAAcKzmcc6erQAAQF3ShF90mLO7gsLDw+WCCy6QDz74QNLS0uS5556TLVu2yDnnnCPHHnus3eEBAADYytnDuwAAABDSkqPCJDHCLVnFXrtDAQAg4B2bGCnBJDY2Vvr37y/p6eny+++/y2+//WZ3SAAAALZy9vAuAAAAhLxjk4KrAxMAgNrSNkiSfnl5eWam34ABA6R58+by/PPPy6WXXiqrVq2yOzQAAABbMdMPAAAAjp+18MveArvDAAAgoCVEuKVxjPO7ga666iqZNGmSmeWna/o9/PDD0qtXL7vDAgAACAjOb+0BAAAgpB2TECERbhEqfAIAULm2icGxDm5YWJh88sknpqyn/lzWypUrpUuXLrbFBgAAYDeSfgAAAHC0cLdLWsVHyMasYrtDAQAgYAVLaU8t61lWdna2fPTRR/LWW2/JkiVLxOPx2BYbAACA3VjTDwAAAI7XjnX9AAColNt1YGZ8MJk7d64MGTJEmjZtKk8//bT07t1bFi1aZHdYAAAAtmKmHwAAAByvXWKkTJVcu8MAACAgtYiLkKgw54/73rVrl7zzzjsyfvx4ycrKMmv6FRYWyldffSWdOnWyOzwAAADbOb/FBwAAgJCXEBkmjWP81/UBAAAHHBsE6/ldeOGF0qFDB1m+fLk8//zzsmPHDnnppZfsDgsAACCgMNMPAAAAQTPbLy0/3+4wAAAIOMGwnt+3334rd9xxhwwdOlTat29vdzgAAAABiZl+AAAACAqs6wcAwMESI9zSKMb5Y77nzZsn2dnZ0qNHDzn11FPl5Zdflr1799odFgAAQEAh6QcAAICg0DQ2XGLDXXaHAQBAQAmGWX6qZ8+e8uabb8rOnTvllltukYkTJ0qzZs3E6/XK9OnTTUIQAAAg1JH0AwAAQFBwuVxybJB0bAIAUFPaBsF6fmXFxcXJ//3f/5mZfytWrJB77rlHnnzySWncuLFcdNFFdocHAABgK5J+AAAACBrHUuITAIBSYS6RYxKC997YoUMHGTdunGzbtk0++uijar9+7ty5cuGFF5oZgzp46KuvvvI7fv3115v9ZbfzzjvP75z9+/fLNddcI4mJiZKcnCw33HCD5OTkHPVnAwAAcFzSb+zYsXLKKadIQkKCGZF1ySWXyNq1a/3OKSgokGHDhkmDBg0kPj5eBg0aJLt37/Y7JzU1VQYOHCixsbHmOvfdd5+UlJTU8acBAACA3dokRJgOTgAAINIiLkIiQ+DGGBYWZvqUvvnmm2q9Ljc3V0444QR55ZVXKj1Hk3xaUtS3lU8uasJv1apVpsTopEmTTCLx5ptvPuLPAgAAcDRsXcl5zpw5JqGniT9N0j344IPSr18/Wb16tSnXoIYPHy6TJ0+WTz/9VJKSkuS2226Tyy67TObPn2+Oezwek/BLSUmRBQsWmAbY4MGDJSIiQsaMGWPnxwMAxyseeY/dISBARDz6jN0hAFUSFeaWlvERsiW72O5QAACwXTtmwB/S+eefb7ZDiYqKMn1OFfntt9/ku+++k8WLF8vJJ59s9r300ksyYMAAefrpp80MQgAAgJCZ6acNIy2V0LlzZzOy6p133jGz9pYsWWKOZ2Zmyvjx4+XZZ5+V3r17S48ePWTChAkmubdo0SJzzrRp00yS8P3335cTTzzRNNZGjx5tRmkVFRXZ+fEAAABgg3as6wcAgOj8vk71ouwOw/Fmz55tqkppKdGhQ4fKvn37So8tXLjQlPT0JfxU3759xe12y48//ljpNQsLCyUrK8tvAwAACLo1/TTJp+rXr28eNflXXFxsGkw+HTt2lFatWpmGldLHrl27SpMmTUrP6d+/v2kwaXmFitC4AgAACF7MagAAQKRNYoTERQRUt4/jaGnP9957T2bOnClPPfWUqVilg8216pTatWuXSQiWFR4ebvq19NihlrvRala+rWXLlrX+WQAAQGgImNaf1+uVu+66S04//XTp0qWL2acNpMjISDNqqixN8PkaT/pYNuHnO+47VhEaVwAAAMErOSpMWsTZWsUeAADbdakfbXcIjnfVVVfJRRddZAab65qBumaflvLU2X9HY8SIEWbgu2/bunVrjcUMAABCW8Ak/XRtv5UrV8rEiRNr/b1oXAEAAAS3rg3o6AQAhK4ot0vaM/O9xrVt21YaNmwoGzZsMM91rb+0tDS/c0pKSmT//v2VrgPoWycwMTHRbwMAAAiapN9tt91mRkvNmjVLWrRoUbpfG0i6Ll9GRobf+bt37y5tPOmjPi9/3HesIjSuAAAAgtvxyVFCRTMAQKjqkBwpEW5d1Q81adu2bWZNv6ZNm5rnvXr1Mn1WujyNz/fff2+qWZ166qk2RgoAAEKVrV0hlmWZhN+XX35pGkVt2rTxO96jRw+JiIgwtdN91q5dK6mpqaZhpfRxxYoVfiOrpk+fbhJ5nTp1qsNPAwAAgEARGeaSDslRdocBAIAtKO1ZNTk5ObJs2TKzqc2bN5uftd9Jj913332yaNEi2bJli+mbuvjii6Vdu3bSv39/c/7xxx9v1v276aab5KeffpL58+ebfi4tC9qsWTObPx0AAAhF4XaX9Pzwww/l66+/loSEhNI1+HSdvZiYGPN4ww03yN13320WQdZE3u23324SfT179jTn9uvXzyT3rrvuOhk3bpy5xkMPPWSurTP6AAAAEJq61o+SlfsL7Q4DAIA6lRjplpbxrG1bFT///LOcc845pc+1/0kNGTJEXnvtNVm+fLm8++67ZjafJvG0D2r06NF+/U0ffPCBSfT16dNH3G63DBo0SF588UVbPg8AAICtrUBtQKmzzz7bb/+ECRPk+uuvNz8/99xzpY2mwsJCM5rq1VdfLT03LCzMlAYdOnSoSQbGxcWZxtmoUaPq+NMAAAAgkLSKj5CkSLdkFnntDgUAgDrTpV6UuFyU9qwK7Y/SKlSVmTp16mGvoYPUdUA7AACAhHrS71ANK5/o6Gh55ZVXzFaZ1q1by5QpU2o4OgAAADiZdnh2qR8l83fl2x0KAAB1htKeAAAAocvWNf0AAACA2tSVjk8AQAhpGhsu9aPD7A4DAAAANiHpBwAAgKCVHBVmynwCABAKdIY7AAAAQhdJPwAAAAS1rnSAAgBCgNsl0qke9zwAAIBQZuuafgAAAEBt65AcJdO35UqR9/DrSQMA4FTHJkZKTDhju+FcxSPvsTsEBIiIR5+xOwQAcCxagwAAAAhqkWEu6ZAcaXcYAADUKkp7AgAAgJl+AAAACHpdG0TLiv2FdocRUp4aeJJk7Nx60P6eV/xNzv37CJnx+lOyftFsydi1XeLqNZBOZ58v/YaOkOiExEqvWZiXI9+9OFpWz/5W8jLTpX6zVnLaX2+SUy+/vvScSc88LEv/O1EiY2Kl/+0PS/cBl5ceWzH9a1k66RMZ8sIHtfCJAcA+0WEuaZfIABcAAIBQR9IPAAAAQa9VfIQkR7olo8hrdyghY9j708TyeEqf7964RsYPvVy6nnuxZO3ZZbYBd42Uxm2Pk4yd2+TLMfdK9p5dcs2/JlR6zcnPPCIbF/8gVz7+mtRr1lLWL5wtXz95vyQ0SpFOZ50nv82ZKr9+97n836ufyt7UTfL5yDvluF7nmKRiQXaWTH1ljNz42ud19A0AQN3pXD9KwnRRPwAAAIQ0ynsCAAAgZGb7oe7E12soCQ2blG6/zZ0m9VscI216nCYp7Y6Xa59+R44/q780aNlGjv3Tn6X/sAfNOZ6Skkqvmbp8sZx04VXS9uTTpV6zVvKnQYMlpX1n2bZyqTmetnmdtO1xurTodKKceN5lEh2XIPt3pJpj374wUnpe/jdJbtqizr4DAKgLmuo7uVGM3WEAAAAgAJD0AwAAQEjoWj+Kxq9NSoqLZNm3n8nJF18tLlfFM1EKcrJMki4svPJiJK26nSK/zflOMtN2imVZsnHxPNmbulHa9zzbHG96XGfZtnqZ5GdlyPbVv0pxYb40bNlGtvyySLavWW5KgQJAsGmfFCn1osLsDgMAAAABgPKeAAAACAmJkWHSsV6UrE5nbb+6tnrWFCnIzpQeF/21wuO56fvk+zeflVMuu+6Q17nogbHyxeN3y5PndRN3eLi4XG657OFnzexBddxpvaX7gCvk5WvPlYjoGLli5MsSERMrX429X6547CVZ9OkEWfjxWxKXXF8ufehZaXJsx1r5vABQl/7UmFl+AAAAOICkHwAAAELGqY1jSPrZ4OevPpDjTusjiY1SDjpWkJMt79x5tVnbr+8t9x/yOgsmviVbVyyRwc+9b8p0bl66UL5+8gFz3XannmXO6Xvr/WbzmfHGv6Tdn840ScJZ45+VOz+ZK2vmTpNPHh4mt384sxY+LQDUnWax4dIiPsLuMAAAABAgqHAEAACAkNEkNlyOSaBztC6l79gqG36aK6dceu1Bxwpzc2TCbVdKVGy8XPvMuxIWUfnvprggX6a9/IQMvHuUWQtQS3medtWN0q3fJTL3vVcqfE3a5vWybMqncu7f/yGblsyXNif1MmsNdut3sexYs9y8PwA4GbP8AAAAUBZJPwAAAISUnnSQ1qkl33wk8fUbSoczzj1oht/4v19hEn2Dn/uPRERFH/I6npIS8ZQUi8vt/08YtzvMrO9Xnu776ol7ZODdo01S0fJ4zTV811Jer6cGPiEA2CMp0i3HJUfaHQYAAAACCEk/AAAAhJRjEiOlSUyY3WGEBK/Xa5J+J11wpYSFh/sl/N7++xVSnJ8ngx55XgpzsyV7726zeT3/S8Q9e1kvWfX9ZPNzdHyCWbvv2+dHyqaf58v+7b+bay+d/Il0PmfAQe+9+Mv3Ja5eAzMrULU+8U+ycfEPkrr8Z5n3/uvSuG0HiUlIqpPvAQBqw8mNYsTtctkdBgAAAAIIa/oBAAAg5JzaJFa+2ZJtdxhBb8OPcyRj1zbpcfE1fvu1tObWlUvMz09f/Ce/Y/dPWiL1mrUyP+/ZskEKcrJKj/117L9l6kuPy8f/vFXysjKkXtMW0m/Yg3Lq5df7XSN7X5rMGv+cDH1nSum+ll1Okj9f+3ezfqCW+Lxi1Mu18pkBoC5EhbnkhAaHniENAACA0EPSDwAAACGnY3KkzIl0S2aR1+5Qgtpxvc6RsUv3HLS/7cmnV7i/vPLnJDRsIpePfOmwr0to0FgemLz0oP19br7XbADgdCc2iJbIMGb5AQAAwB/lPQEAABBytBzaKaztBwBwILdLS3syyw8AAAAHI+kHAACAkKRl0WKYJQEAcJjjk6MkIZK1aQEAAHAwkn4AAAAISRFul5zETAkAgMP8iZnqAAAAqARJPwAAAISsHo1iJJzJfgAAh2gdHyFNYsPtDgMAAAABiqQfAAAAQlZsuFu6NWC2HwDAGZjlBwAAgEMh6QcAAAAJ9Q5UJvsBAAJdg+gwaZsYYXcYAAAACGAk/QAAABDSkqPCpENypN1hAABwSGekxIrLxTAVAAAAVI6kHwAAAEJezyaxdocAAEClUmLDpSMDVAAAAHAYJP0AAAAQ8rQztX0SnakAgMB0djNm+QEAAODwSPoBAAAAInJOszgaxwCAgNMmIUKOSWBgCgAAAA6Pfg0AAABAROpHh8mJDaPtDgMAAD9nNYuzOwQAAAA4BEk/AAAA4A+np8RKpJvyaQCAwNCpXpQpQQ0AAABUBUk/AAAA4A9xEW45tUmM3WEAACA6BuXMprF2hwEAAAAHIekHAAAAlPGnxjESH0EzGQBgrxMbREtyVJjdYQAAAMBB6M0AAAAAyohwu+TPzKwAANhIS01ryWkAAACgOkj6AQAAAOV0qx8ljaKZXQEAsMcpjaNNyWkAAACgOmhBAgAAAOW4XC45u1mc3WEAAEJQbLhLTm3MLD8AAABUH0k/AAAAoALHJkVK6/gIu8MAAISY01JiJTLMZXcYAAAAcCCSfgAAAEAlzmnObD8AQN1JjnRL94bRdocBAAAAhyLpBwAAAFQiJTZcOteLsjsMAECIOLNpnIS5mOUHAACAI0PSDwAAADiEM5vFSjj9rwCAWtYkJkyOrxdpdxgAAABwMJJ+AAAAwCEkRYZJj0YxdocBAAhiOrakX8t4cTHLDwAAAEeBpB8AAABwGL1SYiSG6X4AgFqi6/g1j4uwOwwAAAA4HEk/AAAA4DCiw9zSp3mc3WEAAIJQQoRbzmoWa3cYAAAACAIk/QAAAIAq6FI/WtomMAsDAFCz+rWMk6gwumcAAABw9GhVAgAAAFXUv1W8RLop8wkAqBnHJUVK+6Qou8MAAABAkCDpBwAAAFRRUmQYJdgAADUiKswl57akdDQAAABqDkk/AAAAoBpOahgtLeLC7Q4DAOBwZzWNlYSIMLvDAAAAQBAh6QcAAABUg8vlkvNbxUsYVT4BAEdIB490bxhtdxgAAAAIMiT9AAAAgGpqEB0up6VQ5hMAUH06aOS8lvFmEAkAAABQk0j6AQAAAEegZ5MYaRRNWTYAQPWc2iRGGsZQJjoQzJ07Vy688EJp1qyZScJ+9dVXfscty5JHHnlEmjZtKjExMdK3b19Zv3693zn79++Xa665RhITEyU5OVluuOEGycnJqeNPAgAAcABJPwAAAOAIhLlcMqB1vDBPAwBQVfWjwuS0JswUDxS5ublywgknyCuvvFLh8XHjxsmLL74or7/+uvz4448SFxcn/fv3l4KCgtJzNOG3atUqmT59ukyaNMkkEm+++eY6/BQAAAD/w9AyAAAA4Ag1jY2QUxrHyE9p+XaHAgBwgPNaxUu4m+EigeL88883W0V0lt/zzz8vDz30kFx88cVm33vvvSdNmjQxMwKvuuoq+e233+S7776TxYsXy8knn2zOeemll2TAgAHy9NNPmxmEAAAAdYmZfgAAAMBR+HPTWEmOpFkNADi0ExpESav4CLvDQBVt3rxZdu3aZUp6+iQlJcmpp54qCxcuNM/1UUt6+hJ+Ss93u91mZiAAAEBdo3cCAAAAOAoRbpec3yre7jAAAAEsLtwl5zSLszsMVIMm/JTO7CtLn/uO6WPjxo39joeHh0v9+vVLz6lIYWGhZGVl+W0AAAA1gaQfAAAAcJRaJ0SaGRwAAFSkf8t4iQ6nCwYHjB071swa9G0tW7a0OyQAABAkaHECAAAANeCc5nESH0HzGgDgr0ejaDkumYEhTpOSkmIed+/e7bdfn/uO6WNaWprf8ZKSEtm/f3/pORUZMWKEZGZmlm5bt26tlc8AAABCD70SAAAAQA2IDnPLwFbx4rI7EABAwGgSE0ZZT4dq06aNSdzNnDmzdJ+W4dS1+nr16mWe62NGRoYsWbKk9Jzvv/9evF6vWfuvMlFRUZKYmOi3AQAA1ITwGrkKAAAAAGmTGCm9UmJkwa58u0MBANgs0u2SS9okSrib4SCBKicnRzZs2FD6fPPmzbJs2TKzJl+rVq3krrvukscff1zat29vkoAPP/ywNGvWTC655BJz/vHHHy/nnXee3HTTTfL6669LcXGx3HbbbXLVVVeZ8wAAAOoaST8AAID/b+9OgOSqzvthv909S88+2vcNCSE2CRBYkCCzWAREIBbGbIYADpvjYMrowxBStliMLeOKsYEC5MRGGAJhcSohwTHfZ6BQ/mwGiT9g9tUGBxCLQKNtRjOa+epeMYMEEiBZ0u2eeZ7KrZ7b93bPaZy5On1+97wHtqDpQ2vjj8s74tXl7Vk3BYAMHTyqLvpVF7JuBp9g4cKFccABB/Tsz549O3086aST4rrrrotzzz03VqxYEaeffno6o2/fffeNO++8M4rFYs9rbrzxxjTo+8IXvhD5fD6OPPLIuOKKKzL5PAAAQj8AANiCcrlc/NXYhpj/7HuxoqMr6+YAkIHJ/atj5/4fBkOUpv333z+6uro+8d/0iy++ON02JpkVeNNNN22lFgIAbBpr+gEAwBZWX5mPw8c2WN8PoA8aWCzEQaPqs24GAAB9kNAPAAC2grENVfHnQ2uzbgYA21BFLuKLYxui0jp+AABkQOgHAABbyZ8PrYmxDZVZNwOAbWTGyPoYVGMlFQAAsiH0AwCArSRZC+jwMQ1RX6HbDdDbTWquit0GWscPAIDsGH0AAICtqK4yH39lfT+AXq25Kh8zR1vHDwCAbAn9AABgKxvdUBnTh1nfD6A3yn+wjl91wRALAADZ0iMFAIBtYJ8hNTHO+n4Avc7+w+tiWJ3rOwAA2RP6AQDAtlrfb2xDNFTqggP0FuMbK2OvQdbxAwCgNBhxAACAbaS2wvp+AL1F/+pCHD6mIb2pAwAASoHQDwAAtqFR9ZWx33Dr+wGUs2IhF1/erjGKFYZVAAAoHXqnAACwje09pDZ27leddTMA2MyBlFnjGqJ/sZB1UwAAYD1CPwAAyMCho+tjZF1F1s0AYBMdNKouxjZUZd0MAAD4GKEfAABkoJDPxZHbNUa/al1ygHKxx8Bi7D6wJutmAADABhlhAACAjNRU5NeuCVXIZd0UAD7FuIbKmDGyLutmAADARgn9AAAgQwOKFXHEuIbIy/0AStaAYiG+mF6rXawBAChdQj8AAMjYmIaqOHhUfdbNAGAD6ipycfT4ZFa2IRQAAEqbHisAAJSAKQOK8WdDrBMFUEqq8rk4anxTNFUVsm4KAAB8KqEfAACUiM8Pr4td+1dn3QwAPhgw+eLYhhhaW5F1UwAA4DMR+gEAQAmZObo+tmuszLoZAH3eX4yqj/FNVVk3AwAAyiP0+5//+Z84/PDDY/jw4ZHL5eI//uM/1jve1dUVc+bMiWHDhkVNTU3MmDEjXnjhhfXOWbJkSRx//PHR2NgYzc3Nccopp8Ty5cu38ScBAIAtI5/LxayxjTHMzBKAzCTllncbWMy6GQAAUD6h34oVK2LKlClx1VVXbfD4D3/4w7jiiiti3rx58dvf/jbq6uri4IMPjtbW1p5zksDvqaeeit/85jdxxx13pEHi6aefvg0/BQAAbFlVhVwctV1j9KtWmANgW9u5X3VabhkAAMpNprcPz5w5M902JJnl95Of/CS+/e1vxxe/+MX0ueuvvz6GDBmSzgg89thj45lnnok777wzHnnkkdhzzz3Tc6688so49NBD4x//8R/TGYQAAFCOaivzccz4prj++fdjZUdX1s0B6BMmNlXFoWPqs24GAABslpK9dfiVV16JN998My3p2a2pqSmmTZsWDz74YLqfPCYlPbsDv0Ryfj6fT2cGAgBAOWuuLsTR45uiOp/LuikAvd72TVXxxXENUci55gIAUJ5KNvRLAr9EMrNvXcl+97HkcfDgwesdr6ioiP79+/ecsyFtbW3R0tKy3gYAAKVoaG1FHDOhMaoLBqEBtpbxjZUxa6zADwCA8layod/WNHfu3HTWYPc2atSorJsEAAAbNbyuMo4ZL/gD2FqB35fGNUbBrGoAAMpcyYZ+Q4cOTR8XL1683vPJfvex5PGtt95a73hHR0csWbKk55wNOf/882Pp0qU922uvvbZVPgMAAKXruuuuS0vFl1Pwd6zgD2CL2q6hMo4Q+AEA0EuUbOg3bty4NLi7++67e55LynAma/Xts88+6X7y+P7778eiRYt6zrnnnnuis7MzXftvY6qrq6OxsXG9DQCA8nTyySdHLpf72Pbiiy9GbzOsrjKOm9AURcEfwJ9sXENlfGm7xqgQ+AEA0EtUZPnLly9fvt5gzCuvvBKPPfZYuibf6NGj45vf/GZccsklsf3226ch4He+850YPnx4zJo1Kz1/xx13jEMOOSROO+20mDdvXrS3t8eZZ54Zxx57bHoeAAB9Q9InnD9//nrPDRo0KHrrGn/HTmiKm19cGq1rurJuDkBZGlMv8AMAoPfJdKbfwoULY/fdd0+3xOzZs9Of58yZk+6fe+658Y1vfCNOP/302GuvvdKQ8M4774xisdjzHjfeeGNMmjQpvvCFL8Shhx4a++67b/zTP/1TZp8JAIBtL6nkkFSJWHe7/PLLY9ddd426urp0Deevf/3raX9yYx5//PE44IADoqGhIa0EMXXq1LS/2u2+++6L6dOnR01NTfp+Z511VqxYsSKyDP7M+APYdKPrK+PL4xujUuAHAEAvk+lMv/333z+6ujZ+d3JSluniiy9Ot41JZgXedNNNW6mFAACUq3w+H1dccUVaMeLll19OQ7/kprKrr756g+cff/zx6Q1o11xzTRQKhbQCRWVlZXrspZdeSmcTJlUorr322nj77bfTChPJ9tEZhtsy+Dvugxl/q8z4A/hMRtVXxFECPwAAeqlMQz8AANgS7rjjjqivr+/ZnzlzZtx22209+2PHjk0Du6997WsbDf1effXV+Na3vpVWkUgkJea7zZ07Nw0Fk/Lz3ceSQHG//fZLQ8J1K1FsS0OS4G/7pvjXJPjrEPwBfJKRdRVx1HZNAj8AAHotoR8AAGUvKcuZhG/dkpKed911VxrWPfvss9HS0hIdHR3R2toaK1eujNra2o+9R1Jq/tRTT40bbrghZsyYEUcddVSMHz++p/TnE088kZaW75ZUrOjs7EzXpU7Wms7K4JoPZ/ytFPwBbDTwO3p8U1QpiwwAQC+W6Zp+AACwJSQh34QJE3q2tra2OOyww2Ly5Mnxb//2b7Fo0aK46qqr0nNXr169wfe48MIL46mnnoq//Mu/jHvuuSd22mmn+Pd///f0WLIW4BlnnJGW/OzekiDwhRde6AkGs9Qd/NVWGMwG+KgRyQy/8Y0CPwAAej0z/QAA6HWSkC+ZhfejH/0oXdsvceutt37q6yZOnJhuZ599dhx33HHpen1HHHFE7LHHHvH000+ngWKpGvRB8JeU+jTjD2CtcQ2VMWtcQ1QX3PMMAEDvp9cLAECvk4Rz7e3tceWVV8bLL7+cluycN2/eRs9ftWpVnHnmmXHvvffGH/7wh7j//vvjkUce6Snbed5558UDDzyQnpPM8ktm+N1+++3pfqkFf1+Z0BT1lbr5AJP7V6cz/AR+AAD0FXq+AAD0OlOmTInLLrssLr300thll13StfiS9f02plAoxLvvvhsnnnhiOtPv6KOPjpkzZ8ZFF12UHk/KhC5YsCCef/75mD59euy+++4xZ86cGD58eJSagTUVceLEphhULGTdFIDM7Du0Ng4d0xD5nJKeAAD0Hbmurq4+X/unpaUlmpqaYunSpdHY2Bh91Q/+7ztZN4ES8fe7D8y6CZSI9ov+n6ybQImovOBH0dfpL1Bu2tZ0xu2vLIuXl7Vn3RSAbSafi5g5qj52HVDMuinwmelnruX7J918/6Sb8Wq6Ga+Oz9xfMNMPAAB6oaSc3ZfHN8buAw18A31DdT4XR2/XKPADAKDPqsi6AQAAwNaRlLU7eFR9NFfl497XV0afL/EB9FoNlfl0/b7BNYY5AADou/SGAQCgl5s2pDaaqwtxxx+WRXtn1q0B2LKSNUyTwK+xylqmAAD0bUI/AADoA3Zoro7Gynz88uWWWNFhzh/QO4xtqIwjxjWkJY0BAKCv0ysGAIA+YlhdZZy4Q3MMLJoNA5S/XftXpzP8BH4AALCWnjEAAPQhTVWFOGFiUzo7BqBc/fnQmvjLMQ1RyOWybgoAAJQMoR8AAPQxxUI+jh7fGFMGVGfdFIBNHsQ4dHR9TB9Wl3VTAACg5FjTDwAA+qB8LhczRzdEv+pC3Pv6yqybA/Cpaity8VdjG2JsQ1XWTQEAgJIk9AMAgD5s7yG10VxViF+9uizaO7NuDcCGjayriC+ObYiGKmuSAgDAxgj9AACgj5vUrzoGFgvxH79fFu+0rsm6OQDrmTa4JvYbXpvOUAYAADbOmn4AAEAMrKmIk3ZojsnW+QNKRLGQiy9v1xgHjKgT+AEAwGcg9AMAAFKV+VwcOrohDh9TH1V5A+xAdobVVsRXJzXHhCbr95GdCy+8MHK53HrbpEmTeo63trbG3/3d38WAAQOivr4+jjzyyFi8eHGmbQYA+jahHwAAsJ6d+xfj5B2aY3CNtbOAbW/qoGKcsH1TNFm/jxKw8847xxtvvNGz3XfffT3Hzj777Piv//qvuO2222LBggXx+uuvx5e+9KVM2wsA9G3W9AMAAD6mf7EQJ05sjnv+d0U8+k5r1s0B+oDqfC5mjqmPSc3KDFM6KioqYujQoR97funSpfHzn/88brrppjjwwAPT5+bPnx877rhjPPTQQ7H33ntn0FoAoK8z0w8AANiginwu/mJUfcwa1xDVBeU+ga1nSE0hTp7ULPCj5LzwwgsxfPjw2G677eL444+PV199NX1+0aJF0d7eHjNmzOg5Nyn9OXr06HjwwQczbDEA0JeZ6QcAAHyiZBB+aE1F3P77ZfHGyo6smwP0MrsNKMaMkXXpjQZQSqZNmxbXXXdd7LDDDmlpz4suuiimT58eTz75ZLz55ptRVVUVzc3N671myJAh6bFP0tbWlm7dWlpattpnAAD6FqEfAADwqZqrC3HCxKZY8PrKePitVVk3B+gFqvK5OHhUXbqOKJSimTNn9vw8efLkNAQcM2ZM3HrrrVFTU7PZ7zt37tw0QAQA2NKU9wQAAD6TQi4XB46oiy9v1xg1yn0Cf4LBNYU4aYcmgR9lJZnVN3HixHjxxRfTdf5Wr14d77///nrnLF68eINrAK7r/PPPT9cE7N5ee+21rdxyAKCvEPoBAACbZEJTVXx1UnOMrFM4BNg0yf0Cnx9WGyfv0BwDiq4hlJfly5fHSy+9FMOGDYupU6dGZWVl3H333T3Hn3vuuXTNv3322ecT36e6ujoaGxvX2wAAtgQ9bAAAYJM1VhXi+O2bYtE7rfE/r6+M1Z1dWTcJKHEj6iri0NH1wj7KxjnnnBOHH354WtLz9ddfjwsuuCAKhUIcd9xx0dTUFKecckrMnj07+vfvnwZ33/jGN9LAb++998666QBAH6WnDQAAbJZcLhd7DqqJiU1V8f++tjxeamnPuklACarMR+w3rC6mDiqm1w0oF3/84x/TgO/dd9+NQYMGxb777hsPPfRQ+nPixz/+ceTz+TjyyCOjra0tDj744Lj66quzbjYA0IcJ/QAAgD951t9R45vimffa4q4/Lo8VHWb9AWuNbaiMQ0bVR3N1IeumwCa7+eabP/F4sViMq666Kt0AAEqB0A8AANgiduxXHeMaKuOe11fEE++2Zd0cIEPVhVx8YURdTB5QzLopAADQZwj9AACALaZYkY9DRzfEzv2q487Xlsd7bZ1ZNwnYxpKSv38xqj7qk7qeAADANiP0AwAAtrgxDVVxyqR+cf+bK+O3i1eF6A96v7qKXBw0sj4m9avOuikAANAnCf0AAICtoiKfi/2G16VlP3/96vJ4Y2VH1k0CtpJkdu+MkXVRU2F2HwAAZEXoBwAAbFWDayrixIlNsfDt1vg/b6yM1Z1dWTcJ2EIaK/Nx8Kj6GN9UlXVTAACgzxP6AQAAW10ul4u9BtfExOaq+P9eWx4vtbRn3STgT1CVz8XeQ2rSv+vKfC7r5gAAAEI/AABgW2qqKsRR45vixaWrY8HrK+Lt1jVZNwnYBEnxzt0GFuPPh9ZGXaVSngAAUEqEfgAAwDY3oakqxjdWxu+WtMV9b6yMlvbOrJsEfIa/2wOG18aAoqEEAAAoRXrqAABAZiU/Jw8oxk79qmPh26viwcWrom2N9f6g1AyrrYgDRtTF6PrKrJsCAAB8AqEfAACQqYp0bbDa2G1AMR5YvCoefXtVdMj+IHONVfnYb1htGswnIT0AAFDahH4AAEBJKFbk48ARdTF1UDH+zxsr46klbSH7g22vupCLPxtSE1MH1aShPAAAUB6EfgAAQElpqirEYWMa4nODa2LB6yvipZb2rJsEfUKS7+0+sBj7Dq2Nmop81s0BAAA2kdAPAAAoSYNrKuKo8U3xh2Wr497XV8YbKzuybhL0Wjs0V8X+w+uiX3Uh66YAAACbSegHAACUtDENVXHSDlXx7HttseCNFfFeW2fWTYJeIfdB2DdtSE0Mq63MujkAAMCfSOgHAACUhUn9qmNic1U8/V5bPPzWqnhr1ZqsmwRlqSIXsUv/Yhr2mdkHAAC9h9APAAAoG/lcLg0rku2VltVp+PfKMmv+wWdRXcjFHgOLseegmqirtGYfAAD0NkI/AACgLI1rrEq3t1Z1pOHfM++1xZqurFsFpaehMh97Da6J3QYUo6qQFPUEAAB6I6EfAABQ1gbXVMRhYxpiv+G1seit1vi/77ZGm/QPYmCxEJ8bXBM796+OQk7YBwAAvZ3QDwAA6BUaKgux/4i6+LOhtfH4u62x8O1VsXR1Z9bNgm1uZF1Ful7fhMaqyAn7AACgzxD6AQAAvUpSvjApZTh1UDGee3/tun9vrOzIulmw1U1oqoq9B9fEyPrKrJsCAABkQOgHAAD0SvlcLnbsV51ury5vj4cXr4oXW1Zn3SzYomorcrFzv+rYbWAxBhR9xQcAgL7MNwIAAKDXG11fmW7vta2J3y1pjaeWtCn9SdnKR8T4pqrYtX91OrsvCbgBAACEfgAAQJ/Rr7oQnx9WF9OH1sZryzviySWt8ez7q2N1Z1fWTYNPNahYiF0HFNOZfXWVSfQHAADwIaEfAADQ5+RyuRjdUJluB43qiuffb4snl7TF75e1h/iPUlJdyMVO/apjcv/qGFZnrT4AAGDjhH4AAECfVpnPxc79i+m2rH1NPL1kbQD4duuarJtGH5UU6xzbUJnO6pvYVBUVeeU7AQCATyf0AwAA+EBDZSGmDalNtzdXri3/+fR7bbGyw/w/tr5+1fnYtX8xdulfHY1VhaybAwAAlBmhHwAAwAYMra2IobX1ceCIunipZXU6++/FpatjjfyPLaixKh/bN1XFpObqGFWvfCcAALD5hH4AAACfIJ/LxfZN1enWtqYzXm5pT8O/JAhslQCyGQYVCzGxuSr9/6kkXAYAANgSfLsAAAD4jKoL+dixX3W6dXZ1xR9XdKQBYLItabMGIBuWrMg3sr4iDfmSNfqaq5XuBAAAtjyhHwAAwGbOABxdX5luSQnQJa1r4oWlbfHKsvZ4bXm7MqB9XF1FLrZrrEq3cQ2VUazIZ90kAACglxP6AQAAbAH9i4WYVqyNaUMi2ju70uDv5ZbVaQj4bqtZgH1hNt+IuoqeoG9ITSFyueRZAACAbUPoBwAAsIVV5j+c5ZVoWb0mDf9eaVmdhoErOkwDLHeV+UjX4xtRWxnD6yrSGZ9m8wEAAFkS+gEAAGxljVWFmDIg2Yo9IeDrKzrijZUd8frK9nhzZUe0d2bdSj5JU1U+RtRVprP5ksfBNYW0xCsAAECpEPoBAABkEAIm26R+1el+Z1dXvNO6Jt5YsTYETALBZN98wGxU5D6YxbdOyFeXTO0DAAAoYUI/AACAjCUzxgbXVKTblFg7GzBZFzCZAfj6iva1MwJXdESL6YBbdRZfUqYzCfmS/x0KZvEBAABlRugHAABQousCjqqvTLduK9o70wDw3daOeH91Z7zXtibdWlZ3mhX4KYqFXPSvLkS/6kL0LxbSn7v3qwoCPgAAoPwJ/QAAAMpEUmJyQlNVuq1rTVdXLG3rjPdXrw0Bu7ckGHy/bU2s6eo7ZTnXDfXSnz/Yr61QnhMAAOjdhH4AAABlLilFmQZdxcLHjnV1dcWy9rWzAt9v++Bx9ZpY1dEVrWs6o3VN8tgVbSWcDCbL6dUU8lFTkYuainwa4CUz95LHuspc9KsqRL9iIRor85FTlhMAAOijhH4AAAC9WBKCNVYV0m1Mw8bPS8LB7gAwDQM7NvJz8tjRFW2dXZHWFM2l/5dua3/fhz9/uL/2mY+ek88lZTc/DPNqCh88JvuFJNxbu1+RnAgAAMAnEvoBAACQhoNrw7dk7+MzBgEAAChtFjUAAAAAAACAMif0AwAAAAAAgDIn9AMAAAAAAIAyJ/QDAAAAAACAMif0AwAAAAAAgDIn9AMAAAAAAIAyJ/QDAAAAAACAMif0AwAAAAAAgDLXa0K/q666KsaOHRvFYjGmTZsWDz/8cNZNAgAAAKAPMC4FAJSCXhH63XLLLTF79uy44IIL4tFHH40pU6bEwQcfHG+99VbWTQMAAACgFzMuBQCUil4R+l122WVx2mmnxVe/+tXYaaedYt68eVFbWxvXXntt1k0DAAAAoBczLgUAlIqyD/1Wr14dixYtihkzZvQ8l8/n0/0HH3ww07YBAAAA0HsZlwIASklFlLl33nkn1qxZE0OGDFnv+WT/2Wef3eBr2tra0q3b0qVL08eWlpboy1qXL8u6CZSIlpaqrJtAiWhv/fBaSd9W2cf/jVy3n9DV1ZV1UwAAKBHGpbYc3z/p5vsn3YxX0814dXzmcamyD/02x9y5c+Oiiy762POjRo3KpD1Qaj7+1wH0eT+4KusWlIxly5ZFU1NT1s0AAKBMGZeCT+H7J/ARxqs/+7hU2Yd+AwcOjEKhEIsXL17v+WR/6NChG3zN+eefny6w3K2zszOWLFkSAwYMiFwut9XbTGmn5Ukn+7XXXovGxsasmwOUANcFuiV3UiUdq+HDh2fdFAAASoRxKbYk3z+Bj3JdYFPHpco+9KuqqoqpU6fG3XffHbNmzerpLCX7Z5555gZfU11dnW7ram5u3ibtpTwkF1AXUWBdrgskzPADAGBdxqXYGnz/BD7KdYHPOi5V9qFfIrk76qSTToo999wzPve5z8VPfvKTWLFiRXz1q1/NumkAAAAA9GLGpQCAUtErQr9jjjkm3n777ZgzZ068+eabsdtuu8Wdd975sUWUAQAAAGBLMi4FAJSKXhH6JZKSCRsrmwCfVVJe44ILLvhYmQ2g73JdAAAAPo1xKbYE3z+Bj3JdYFPlupLV/wAAAAAAAICylc+6AQAAAAAAAMCfRugHAAAAAAAAZU7oBx+47rrrorm5OetmAAAAANDHGJcCYEsQ+tHrnHzyyZHL5T62vfjii1k3DcjQhq4L624XXnhh1k0EAACgzBmXAjbEuBTbSsU2+02wDR1yyCExf/789Z4bNGhQZu0BsvfGG2/0/HzLLbfEnDlz4rnnnut5rr6+vufnrq6uWLNmTVRU+GcSAACATWNcCvgo41JsK2b60StVV1fH0KFD19suv/zy2HXXXaOuri5GjRoVX//612P58uUbfY/HH388DjjggGhoaIjGxsaYOnVqLFy4sOf4fffdF9OnT4+ampr0/c4666xYsWLFNvqEwKZa93rQ1NSU3kXVvf/ss8+mf+u//vWv07/15BqS/I0nd2jOmjVrvff55je/Gfvvv3/PfmdnZ8ydOzfGjRuXXg+mTJkSv/zlLzP4hAAAAJQC41LARxmXYlsR+tFn5PP5uOKKK+Kpp56KX/ziF3HPPffEueeeu9Hzjz/++Bg5cmQ88sgjsWjRovj7v//7qKysTI+99NJL6V1bRx55ZDzxxBPp3RnJhfjMM8/chp8I2NKSv/Mf/OAH8cwzz8TkyZM/02uSjtX1118f8+bNS68vZ599dpxwwgmxYMGCrd5eAAAAyoNxKeDTGJdiSzA/lF7pjjvuWG9K9MyZM+O2227r2R87dmxccskl8bWvfS2uvvrqDb7Hq6++Gt/61rdi0qRJ6f7222+/3sU06Xwld1Z0H0s6bvvtt19cc801USwWt+KnA7aWiy++OA466KDPfH5bW1t8//vfj7vuuiv22Wef9Lntttsu/bL105/+NL0mAAAA0LcYlwI2h3EptgShH71SUv4g6eR0S0onJBe/pFOUTJduaWmJjo6OaG1tjZUrV0Ztbe3H3mP27Nlx6qmnxg033BAzZsyIo446KsaPH99TYiG5k+rGG29cr9ZyMp36lVdeiR133HEbfVJgS9pzzz036fxkIfbkGvLRDtnq1atj991338KtAwAAoBwYlwI2h3EptgShH71S0pmaMGFCz/7vf//7OOyww+Jv//Zv43vf+170798/vePhlFNOSS+CG+pcXXjhhfGVr3wlfvWrX6X1lC+44IK4+eab44gjjkhrrp9xxhlpvfSPGj169Fb/fMDWu3Z8tPxK8sVpXe3t7T0/d6+/kFwnRowYsd55Sf11AAAA+h7jUsDmMC7FliD0o09Iap8ndzv96Ec/Si+WiVtvvfVTXzdx4sR0S2ohH3fccTF//vy0c7XHHnvE008/vV4HDuh9Bg0aFE8++eR6zz322GM96yjstNNOaScqKbuiZAIAAAAbYlwK2BzGpdgca/+VgV4u6QQld0FceeWV8fLLL6elEZLFTTdm1apV6eLH9957b/zhD3+I+++/P104ubs8wnnnnRcPPPBAek5yoX3hhRfi9ttvt2Ay9DIHHnhgLFy4MF0QOfk7T+6sXLez1dDQEOecc076BSxZiD1ZTP3RRx9NrzXJPgAAABiXAjaHcSk2h9CPPmHKlClx2WWXxaWXXhq77LJLWvM8qaO+MYVCId5999048cQT0zuqjj766HTR5Ysuuig9Pnny5FiwYEE8//zzMX369LRG8pw5c2L48OHb8FMBW9vBBx8c3/nOd+Lcc8+NvfbaK5YtW5ZeF9b13e9+Nz0nuaYkX8AOOeSQtKzCuHHjMms3AAAApcO4FLA5jEuxOXJdHy0KCwAAAAAAAJQVM/0AAAAAAACgzAn9AAAAAAAAoMwJ/QAAAAAAAKDMCf0AAAAAAACgzAn9AAAAAAAAoMwJ/QAAAAAAAKDMCf0AAAAAAACgzAn9AAAAAAAAoMwJ/YBe5d57741cLhfvv/9+1k0BAAAAoA8xLgVkTegHlJSTTz457RwlW2VlZYwbNy7OPffcaG1tzbppAAAAAPRixqWAcleRdQMAPuqQQw6J+fPnR3t7eyxatChOOumktLN16aWXZt00AAAAAHox41JAOTPTDyg51dXVMXTo0Bg1alTMmjUrZsyYEb/5zW/SY52dnTF37tz0TquampqYMmVK/PKXv/zE97vvvvti+vTp6fnJe5511lmxYsWK9Ng//MM/xLRp0z72muR9L7744vTnRx55JA466KAYOHBgNDU1xX777RePPvroeucnnb+f/exnccQRR0RtbW1sv/328Z//+Z/rnfPUU0/FYYcdFo2NjdHQ0JC26aWXXuo5nrx+xx13jGKxGJMmTYqrr776T/ivCAAAAMCmMi5lXArKmdAPKGlPPvlkPPDAA1FVVZXuJx2r66+/PubNm5d2Vs4+++w44YQTYsGCBRt8fdJ5Se7QOvLII+OJJ56IW265Je1snXnmmenx448/Ph5++OH1OjnJ+ybnfuUrX0n3ly1blt7VlbzuoYceSjtOhx56aPr8ui666KI4+uij09cmx5P3XrJkSXrsf//3f+Pzn/982nG855570jvF/uZv/iY6OjrS4zfeeGPMmTMnvve978UzzzwT3//+9+M73/lO/OIXv9hK/2UBAAAA+CTGpYxLQbnJdXV1dWXdCIB1a6f/y7/8S3pXUdLxaGtri3w+H7feemt6N1L//v3jrrvuin322afnNaeeemqsXLkybrrppnTB5AMOOCDee++9aG5uTo8VCoX46U9/2nN+0klK7opK7qpKfs9uu+2Wdr6Szkz3XVZJByjpSG1IcldX8t7J70va1H1H1be//e347ne/m+4n711fXx+//vWv085d8p4333xzPPfcc2lN+I+aMGFC+trjjjuu57lLLrkk/vu//zvtXAIAAACwdRmXMi4F5c6afkDJSTpH11xzTdpB+fGPfxwVFRVp5ye50ynpRCUlDda1evXq2H333Tf4Xo8//nh6h1Nyx1K35F6HpIP0yiuvpGULkjufrr322rRzlRz713/915g9e3bP+YsXL047TknH7a233oo1a9ak7Xj11VfX+12TJ0/u+bmuri4tl5Ccn3jsscfSsgkb6lglnzO5o+uUU06J0047ref5pHOZlG0AAAAAYNswLrWWcSkoT0I/oOQkHZPkDqNE0ulJ6pj//Oc/j1122SV97le/+lWMGDFivdck5Qk2ZPny5XHGGWek9dI/avTo0eljchfTeeedl9ZDX7VqVbz22mtxzDHH9JyXlFB499134/LLL48xY8akvyu5oyvp1K3rox2n5C6rpBOXSOq2b0zSxsQ///M/f6yOe3I3GAAAAADbhnGpDxmXgvIj9ANKWlJCISlBkNzh9Pzzz6cdm+ROpqQMwmexxx57xNNPP93TWduQkSNHpu+X3HWVdK6SO7YGDx7cc/z+++9PFy9O6qEnks7XO++8s0mfI7nbKqmD3t7e/rFO2JAhQ2L48OHx8ssvp3d3AQAAAJA941JAucln3QCAT3PUUUf11D8/55xz0kWSk45KUnoguQvqyiuv3OjCwsmdUknt8WSB5KSUwQsvvBC33357z4LJ3ZJOTVLb/LbbbvtYBydZIPmGG25IFzL+7W9/mx7/pDukNiT5fS0tLXHsscfGwoUL03Yk75nUUu9ebDlZDPqKK65IO5G/+93vYv78+XHZZZdt8n8vAAAAALYM41JAORH6ASUvqZ2edE5++MMfxvnnn5/WOE86Iknd82Qx4qSswrhx4zZ6J9OCBQvSDktSuzypsT5nzpz0DqZ1ffnLX05LJSQ10WfNmrXesaSEQ7IAc3J31l//9V+nJRnWvePqsxgwYEC6CHNSMiG5e2vq1Klp2YTuu6uShZ1/9rOfpR2qXXfdNT3nuuuu2+jnAgAAAGDrMy4FlJNcV7I6KAAAAAAAAFC2zPQDAAAAAACAMif0AwAAAAAAgDIn9AMAAAAAAIAyJ/QDAAAAAACAMif0AwAAAAAAgDIn9AMAAAAAAIAyJ/QDAAAAAACAMif0AwAAAAAAgDIn9AMAAAAAAIAyJ/QDAAAAAACAMif0AwAAAAAAgDIn9AMAAAAAAIAob/8/RE2yGP5eVQEAAAAASUVORK5CYII=",
      "text/plain": [
       "<Figure size 1800x600 with 3 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import matplotlib.pyplot as plt\n",
    "\n",
    "df = pd.read_csv(rf\"C:\\Users\\vtorr\\Work\\Projects\\aipatent\\evals\\GVHD-ALD-papers.csv\")\n",
    "\n",
    "relevance = df[\"relevant\"].value_counts()\n",
    "print(relevance)\n",
    "\n",
    "\n",
    "df[\"text_length\"] = df[\"text\"].str.len()\n",
    "\n",
    "\n",
    "avg_text_length = df.groupby(\"relevant\")[\"text_length\"].mean()\n",
    "print(avg_text_length)\n",
    "\n",
    "\n",
    "fig, axs = plt.subplots(1, 3, figsize=(18, 6))\n",
    "\n",
    "\n",
    "relevance.plot(kind='bar', color=['skyblue', 'salmon'], ax=axs[0])\n",
    "axs[0].set_xlabel(\"Relevance\")\n",
    "axs[0].set_ylabel(\"Count\")\n",
    "axs[0].set_title(\"Count of Relevance\")\n",
    "axs[0].set_xticklabels(relevance.index, rotation=0)\n",
    "\n",
    "\n",
    "axs[1].pie(\n",
    "    relevance, \n",
    "    labels=relevance.index, \n",
    "    autopct='%1.1f%%', \n",
    "    colors=['skyblue', 'salmon'],\n",
    "    startangle=90\n",
    ")\n",
    "axs[1].set_title(\"Pie Chart of Relevance\")\n",
    "axs[1].axis('equal')\n",
    "\n",
    "avg_text_length.plot(kind='bar', color=['salmon', 'skyblue'], ax=axs[2])\n",
    "axs[2].set_xlabel(\"Relevance\")\n",
    "axs[2].set_ylabel(\"Average Text Length\")\n",
    "axs[2].set_title(\"Average Text Length by Relevance\")\n",
    "axs[2].set_xticklabels(avg_text_length.index, rotation=0)\n",
    "\n",
    "plt.tight_layout()\n",
    "plt.show()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Filtered CSV saved as GVHD-ALD-papers-not-false.csv\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Read the original CSV\n",
    "df = pd.read_csv(r\"C:\\Users\\vtorr\\Work\\Projects\\aipatent\\evals\\GVHD-ALD-papers.csv\")\n",
    "\n",
    "# Filter rows where 'relevant' is not False (i.e. True or any value other than False)\n",
    "filtered_df = df[df[\"relevant\"] != False]\n",
    "\n",
    "# Save the filtered DataFrame to a new CSV\n",
    "filtered_df.to_csv(r\"C:\\Users\\vtorr\\Work\\Projects\\aipatent\\evals\\GVHD-ALD-papers-not-false.csv\", index=False)\n",
    "\n",
    "print(\"Filtered CSV saved as GVHD-ALD-papers-not-false.csv\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
=======
{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Dataframe Shape: (751, 5) \n",
      "\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>element_id</th>\n",
       "      <th>text</th>\n",
       "      <th>page_number</th>\n",
       "      <th>filename</th>\n",
       "      <th>chunk_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>38ce4f336a6096a4e0eb4efc385c6884</td>\n",
       "      <td>admission to the hospital (range 0–24, n = 82)...</td>\n",
       "      <td>8</td>\n",
       "      <td>ALD patent background + target paper.pdf</td>\n",
       "      <td>201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>d816fd6f4542d1b8eb4f77190745d144</td>\n",
       "      <td>respective laboratory values were avail- able....</td>\n",
       "      <td>8</td>\n",
       "      <td>ALD patent background + target paper.pdf</td>\n",
       "      <td>202</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>202</th>\n",
       "      <td>077ff414cee1eb0193dbc2ca6bd1badb</td>\n",
       "      <td>informed consent was obtained from each patient.</td>\n",
       "      <td>8</td>\n",
       "      <td>ALD patent background + target paper.pdf</td>\n",
       "      <td>203</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>203</th>\n",
       "      <td>8c2cd087d7482ec1cc860453e9de31fb</td>\n",
       "      <td>Mice\\n\\nC57BL/6 mice were purchased from Charl...</td>\n",
       "      <td>8</td>\n",
       "      <td>ALD patent background + target paper.pdf</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>43d130fb7cbac4d01b6a808303dff089</td>\n",
       "      <td>8 | Nature | www.nature.com</td>\n",
       "      <td>8</td>\n",
       "      <td>ALD patent background + target paper.pdf</td>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>595</th>\n",
       "      <td>828e3d7554e59588cca8ddf4d1db363a</td>\n",
       "      <td>GVHD-related mortality Enterococcus ≥ 0.3 (n=5...</td>\n",
       "      <td>2</td>\n",
       "      <td>GvHD-paper.pdf</td>\n",
       "      <td>596</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>596</th>\n",
       "      <td>9699d347d90b3ff6d6ed7bf145b4b245</td>\n",
       "      <td>point represents a fecal sample, and color ind...</td>\n",
       "      <td>2</td>\n",
       "      <td>GvHD-paper.pdf</td>\n",
       "      <td>597</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>597</th>\n",
       "      <td>10ac17b617ed6906ffae7f3f788f203e</td>\n",
       "      <td>between day 0 and +21). n, number of individua...</td>\n",
       "      <td>2</td>\n",
       "      <td>GvHD-paper.pdf</td>\n",
       "      <td>598</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>598</th>\n",
       "      <td>ba25ff6c46b5bba6c5e2795bfcda92d4</td>\n",
       "      <td>with those without domination.</td>\n",
       "      <td>2</td>\n",
       "      <td>GvHD-paper.pdf</td>\n",
       "      <td>599</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>599</th>\n",
       "      <td>f73b871ed5f104c18f8b1bc12bee8eaf</td>\n",
       "      <td>2 of 7</td>\n",
       "      <td>2</td>\n",
       "      <td>GvHD-paper.pdf</td>\n",
       "      <td>600</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>400 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                           element_id  \\\n",
       "200  38ce4f336a6096a4e0eb4efc385c6884   \n",
       "201  d816fd6f4542d1b8eb4f77190745d144   \n",
       "202  077ff414cee1eb0193dbc2ca6bd1badb   \n",
       "203  8c2cd087d7482ec1cc860453e9de31fb   \n",
       "204  43d130fb7cbac4d01b6a808303dff089   \n",
       "..                                ...   \n",
       "595  828e3d7554e59588cca8ddf4d1db363a   \n",
       "596  9699d347d90b3ff6d6ed7bf145b4b245   \n",
       "597  10ac17b617ed6906ffae7f3f788f203e   \n",
       "598  ba25ff6c46b5bba6c5e2795bfcda92d4   \n",
       "599  f73b871ed5f104c18f8b1bc12bee8eaf   \n",
       "\n",
       "                                                  text  page_number  \\\n",
       "200  admission to the hospital (range 0–24, n = 82)...            8   \n",
       "201  respective laboratory values were avail- able....            8   \n",
       "202   informed consent was obtained from each patient.            8   \n",
       "203  Mice\\n\\nC57BL/6 mice were purchased from Charl...            8   \n",
       "204                        8 | Nature | www.nature.com            8   \n",
       "..                                                 ...          ...   \n",
       "595  GVHD-related mortality Enterococcus ≥ 0.3 (n=5...            2   \n",
       "596  point represents a fecal sample, and color ind...            2   \n",
       "597  between day 0 and +21). n, number of individua...            2   \n",
       "598                     with those without domination.            2   \n",
       "599                                             2 of 7            2   \n",
       "\n",
       "                                     filename  chunk_id  \n",
       "200  ALD patent background + target paper.pdf       201  \n",
       "201  ALD patent background + target paper.pdf       202  \n",
       "202  ALD patent background + target paper.pdf       203  \n",
       "203  ALD patent background + target paper.pdf       204  \n",
       "204  ALD patent background + target paper.pdf       205  \n",
       "..                                        ...       ...  \n",
       "595                            GvHD-paper.pdf       596  \n",
       "596                            GvHD-paper.pdf       597  \n",
       "597                            GvHD-paper.pdf       598  \n",
       "598                            GvHD-paper.pdf       599  \n",
       "599                            GvHD-paper.pdf       600  \n",
       "\n",
       "[400 rows x 5 columns]"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import lancedb\n",
    "import pandas as pd\n",
    "\n",
    "df1 = pd.read_csv(rf\"C:\\Users\\vtorr\\Work\\Projects\\aipatent\\app\\data\\tmp\\ALD patent background + target paper.csv\")\n",
    "df2 = pd.read_csv(rf\"C:\\Users\\vtorr\\Work\\Projects\\aipatent\\app\\data\\tmp\\GvHD-paper.csv\")\n",
    "\n",
    "df = pd.concat([df1, df2], ignore_index=True)\n",
    "\n",
    "df['chunk_id'] = range(1, len(df) + 1)\n",
    "\n",
    "print(f\"Dataframe Shape: {df.shape} \\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pydantic import BaseModel\n",
    "\n",
    "class Chunk(BaseModel):\n",
    "    element_id: str\n",
    "    text: str\n",
    "    page_number: int\n",
    "    filename: str\n",
    "    chunk_id: int\n",
    "    \n",
    "class ChunkReview(BaseModel):\n",
    "    relevant: bool\n",
    "    \n",
    "class ReviewedChunk(BaseModel):\n",
    "    element_id: str\n",
    "    text: str\n",
    "    page_number: int\n",
    "    filename: str\n",
    "    chunk_id: int\n",
    "    relevant: bool\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[Chunk(element_id='38dc4ba7c2bf7ed7323042c35312ad5f', text='Article', page_number=1, filename='ALD patent background + target paper.pdf', chunk_id=1)]"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "chunks = []\n",
    "for _,row in df.iterrows():\n",
    "    chunk = [\n",
    "        Chunk(\n",
    "        element_id=row[\"element_id\"],\n",
    "        text=row[\"text\"],\n",
    "        page_number=row[\"page_number\"],\n",
    "        filename=row[\"filename\"],\n",
    "        chunk_id=row[\"chunk_id\"]    \n",
    "    )]\n",
    "    \n",
    "    chunks.append(chunk)\n",
    "\n",
    "chunks[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "import instructor\n",
    "from openai import OpenAI\n",
    "\n",
    "\n",
    "openai = instructor.from_openai(OpenAI())\n",
    "\n",
    "def chunk_review(chunk: Chunk)-> ReviewedChunk:\n",
    "    \n",
    "    prompt  = f\"\"\"\n",
    "        the chunks will be stored.\n",
    "        You are a data annotator, your task is to review a corpus of data consisting of chunks from a source document.\n",
    "        Your perfomance is critical as it ensures that bad or irrelevant data doesn't flood the vector database where\n",
    "        the chunks will be stored\n",
    "        \n",
    "        here's the chunk text: <Text>{chunk.text}</Text>\n",
    "        \n",
    "        <Guidelines>\n",
    "            What makes text irrelevant:\n",
    "            - the text is too short and doesn't develop any idea or argument\n",
    "            - contains only numbers\n",
    "            - text that appears to be a caption for an image\n",
    "            - text that appears to only contain people's names\n",
    "            - text that is out of context\n",
    "            - as a rule of thumb text smaller than 80 tokens is not relevant\n",
    "            \n",
    "            What makes text relevant: \n",
    "             - it does present findings, ideas, arguments,\n",
    "             - it describes something and contains semantic meaning, valuable ideas about a topic can infered from it\n",
    "        </Guidelines>\n",
    "    \"\"\"\n",
    "    \n",
    "    res = openai.chat.completions.create(\n",
    "        model=\"o3-mini\",\n",
    "        response_model=ChunkReview,\n",
    "        messages=[\n",
    "            {\"role\": \"assistant\", \"content\": prompt}\n",
    "        ]\n",
    "    )\n",
    "    \n",
    "    return ReviewedChunk(**chunk.model_dump(), relevant=res.relevant)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "ReviewedChunk(element_id='4d2733e4cdaa6f457e875d11f778b79d', text='Cytolysin linked to increased mortality', page_number=1, filename='ALD patent background + target paper.pdf', chunk_id=13, relevant=False)"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "review = chunk_review(chunks[12][0])\n",
    "\n",
    "review"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "import instructor\n",
    "from openai import AsyncOpenAI\n",
    "\n",
    "\n",
    "openai = instructor.from_openai(AsyncOpenAI())\n",
    "\n",
    "\n",
    "async def as_chunk_review(chunk: Chunk)-> ReviewedChunk:\n",
    "    print(chunk[0].text)\n",
    "    prompt  = f\"\"\"\n",
    "        the chunks will be stored.\n",
    "        You are a data annotator, your task is to review a corpus of data consisting of chunks from a source document.\n",
    "        Your perfomance is critical as it ensures that bad or irrelevant data doesn't flood the vector database where\n",
    "        the chunks will be stored\n",
    "        \n",
    "        here's the chunk text: <Text>{chunk[0].text}</Text>\n",
    "        \n",
    "        <Guidelines>\n",
    "            What makes text irrelevant:\n",
    "            - the text is too short and doesn't develop any idea or argument\n",
    "            - contains only numbers\n",
    "            - text that appears to be a caption for an image\n",
    "            - text that appears to only contain people's names\n",
    "            - text that is out of context\n",
    "            - as a rule of thumb text smaller than 80 tokens is not relevant\n",
    "            \n",
    "            What makes text relevant: \n",
    "             - it does present findings, ideas, arguments,\n",
    "             - it describes something and contains semantic meaning, valuable ideas about a topic can infered from it\n",
    "        </Guidelines>\n",
    "    \"\"\"\n",
    "    \n",
    "    res = await openai.chat.completions.create(\n",
    "        model=\"o3-mini\",\n",
    "        response_model=ChunkReview,\n",
    "        messages=[\n",
    "            {\"role\": \"assistant\", \"content\": prompt}\n",
    "        ]\n",
    "    )\n",
    "    \n",
    "    return ReviewedChunk(**chunk[0].model_dump(), relevant=res.relevant)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Article\n",
      "Bacteriophage targeting of gut bacterium attenuates alcoholic liver disease\n",
      "https://doi.org/10.1038/s41586-019-1742-x Received: 9 April 2019 Accepted: 2 October 2019 Published online: 13 November 2019\n",
      "\n",
      "https://doi.org/10.1038/s41586-019-1742-x\n",
      "Received: 9 April 2019\n",
      "\n",
      "Accepted: 2 October 2019\n",
      "\n",
      "Published online: 13 November 2019\n",
      "Yi Duan1,2,29, Cristina Llorente1,2,29, Sonja Lang1, Katharina Brandl3, Huikuan Chu1, Lu Jiang1,2, Richard C. White4, Thomas H. Clarke4, Kevin Nguyen4, Manolito Torralba5, Yan Shao6, Jinyuan Liu7, Adriana Hernandez-Morales8, Lauren Lessor9, Imran R. Rahman10, Yukiko Miyamoto1, Melissa Ly11, Bei Gao1, Weizhong Sun1, Roman Kiesel1, Felix Hutmacher1, Suhan Lee1, Meritxell Ventura-Cots12, Francisco Bosques-Padilla13, Elizabeth C. Verna14, Juan G. Abraldes15, Robert S. Brown Jr16, Victor Vargas17,18,\n",
      "Jose Altamirano17, Juan Caballería18,19, Debbie L. Shawcross20, Samuel B. Ho1,2, Alexandre Louvet21, Michael R. Lucey22, Philippe Mathurin21, Guadalupe Garcia-Tsao23,24, Ramon Bataller12, Xin M. Tu7, Lars Eckmann1, Wilfred A. van der Donk10,25,26, Ry Young8,9, Trevor D. Lawley6, Peter Stärkel27, David Pride1,11,28, Derrick E. Fouts4 & Bernd Schnabl1,2,28*\n",
      "Chronic liver disease due to alcohol-use disorder contributes markedly to the global burden of disease and mortality1–3. Alcoholic hepatitis is a severe and life-threatening form of alcohol-associated liver disease. The gut microbiota promotes ethanol- induced liver disease in mice4, but little is known about the microbial factors that are responsible for this process. Here we identify cytolysin—a two-subunit exotoxin that is secreted by Enterococcus faecalis5,6—as a cause of hepatocyte death\n",
      "and liver injury. Compared with non-alcoholic individuals or patients with alcohol-use disorder, patients with alcoholic hepatitis have increased faecal numbers of E. faecalis. The presence of cytolysin-positive (cytolytic) E. faecalis correlated with the severity of liver disease and with mortality in patients with alcoholic hepatitis. Using humanized mice that were colonized with bacteria from the faeces of patients with alcoholic hepatitis, we investigated the therapeutic effects of\n",
      "bacteriophages that target cytolytic E. faecalis. We found that these bacteriophages decrease cytolysin in the liver and abolish ethanol-induced liver disease in humanized mice. Our findings link cytolytic E. faecalis with more severe clinical outcomes and increased mortality in patients with alcoholic hepatitis. We show that bacteriophages can specifically target cytolytic E. faecalis, which provides a method for precisely editing the intestinal microbiota. A clinical trial with a larger cohort\n",
      "is required to validate the relevance of our findings in humans, and to test whether this therapeutic approach is effective for patients with alcoholic hepatitis.\n",
      "The most severe form of alcohol-related liver disease is alcoholic hepa- titis; mortality ranges from 20% to 40% at 1–6 months, and as many as 75% of patients die within 90 days of a diagnosis of severe alcoholic hepatitis7–9. Therapy with corticosteroids is only marginally effective9. Early liver transplantation is the only curative therapy, but is offered only at select centres and to a limited group of patients10.\n",
      "Alcohol-related liver disease can be transmitted via faecal micro- biota4. We investigated the microorganisms and microbial factors that are responsible for this transmissible phenotype and for progression of alcohol-related liver disease.\n",
      "Cytolysin linked to increased mortality\n",
      "We performed 16S ribosomal RNA (rRNA) gene sequencing to deter- mine whether chronic alcohol use and alcoholic hepatitis are associated\n",
      "A list of affiliations appears at the end of the paper.\n",
      "with an altered composition of the faecal microbiota. Differences in faecal microbiota composition were noted in patients with alcohol-use disorder and alcoholic hepatitis, compared to subjects without alcohol- use disorder (controls) (Fig. 1a, Extended Data Fig. 1a, b, Supplementary Tables 1, 2). One substantial difference that we observed was an increase in the proportion of Enterococcus spp. in patients with alcoholic hepa- titis: in these patients, 5.59% of faecal bacteria were Enterococcus\n",
      "spp. compared with almost none in controls (0.023%; for comparison, 0.004% of all reads were Enterococcus spp. in the Human Microbiome Project) or patients with alcohol-use disorder (0.024%). Faecal sam- ples from patients with alcoholic hepatitis had about 2,700-fold more E. faecalis than samples from controls, as measured by quantitative PCR (qPCR) (Extended Data Fig. 1c), which is consistent with the 16S rRNA sequencing results. About 80% of patients with alcoholic hepatitis are positive for\n",
      "E. faecalis in their faeces (Extended Data Fig. 1d).\n",
      "Nature | www.nature.com | 1\n",
      "\n",
      "1\n",
      "Article\n",
      "Controls 0.2 Alcohol-use disorder Alcoholic hepatitis ) % 5 7 . 7 1 ( 2 C P 0.1 0 −0.1 –0.2 –0.1 0 0.1 0.2 PC1 (29.25%)\n",
      "m\n",
      "\n",
      "b\n",
      "100 Cytolysin-negative Cytolysin-positive ) % ( s t c e j b u S 80 60 40 20 0 Controls AUD Alcoholic\n",
      "m\n",
      "\n",
      "=\n",
      ") % ( l a v i v r u S 100 75 50 25 0 0 30 Cytolysin-negative Cytolysin-positive 60 90 120 150 180 Time (d)\n",
      "d\n",
      "0.15 Cytolysin-negative Cytolysin-positive\n",
      "@\n",
      "\n",
      "@\n",
      "Fig. 1 | E. faecalis cytolysin is associated with mortality in patients with alcoholic hepatitis. a, 16S rRNA sequencing of faecal samples from controls (n = 14), patients with alcohol-use disorder (n = 43) or alcoholic hepatitis (n = 75). We use principal coordinate analysis (PCoA) based on Jaccard dissimilarity matrices to show β-diversity among groups at the genus level. The composition of faecal microbiota was significantly different between each group (P < 0.01). b, Percentage of subjects\n",
      "with faecal samples that were positive for both cylLL and cylLS DNA sequences (cytolysin-positive), in controls (n = 25), patients with alcohol-use disorder (n = 38) or alcoholic hepatitis (n = 82), assessed by qPCR. Statistically significant differences were detected between controls and patients with alcoholic hepatitis (P < 0.01), and between patients with alcohol- use disorder and patients with alcoholic hepatitis (P < 0.001). c, Kaplan–Meier curve of survival of patients with alcoholic\n",
      "hepatitis whose faecal samples were cytolysin-positive (n = 25) or cytolysin-negative (n = 54) (P < 0.0001). d, Core genome single-nucleotide polymorphism (SNP) tree of E. faecalis strains isolated from patients with alcoholic hepatitis (n = 93 strains, from 24 patients), showing phylogenetic diversity of cytolysin-positive (red) E. faecalis. Genomically identical isolates from the same patient were combined, and are shown as a single dot. Scale bar represents the nucleotide substitutions per\n",
      "SNP site. P values are determined by permutational multivariate analysis of variance (PERMANOVA) followed by false discovery rate (FDR) procedures (a), two-sided Fisher’s exact test followed by FDR procedures (b) or two-sided log- rank (Mantel–Cox) test (c). The exact group size (n) and P values for each comparison are listed in Supplementary Table 10.\n",
      "The colonization of mice with E. faecalis induces mild hepatic stea- tosis and exacerbates ethanol-induced liver disease11, by mechanisms that are unclear. Cytolysin is a bacterial exotoxin (or bacteriocin) that is produced by E. faecalis12, and which contains two post-translationally modified peptides (CylLL′′ and CylLS′′) in its bioactive form6. The two peptides are encoded by two separate genes: cylLL and cylLS, respec- tively12. Cytolysin has lytic activity against not only Gram-positive\n",
      "bacte- ria, but also eukaryotic cells13. We detected cylLL and cylLS genomic DNA (cytolysin-positive) in faecal samples from 30% of patients with alco- holic hepatitis; none of the faecal samples from controls and only one sample from a patient with alcohol-use disorder was cytolysin-positive, as detected by qPCR (Fig. 1b). Importantly, 89% of cytolysin-positive patients with alcoholic hepatitis died within 180 days of admission, compared to only 3.8% of cytolysin-negative patients (P < 0.0001)\n",
      "(Fig. 1c). Among the cytolysin-positive patients, 72.2% (13 out of 18) died owing to liver failure (including complications related to liver\n",
      "2 | Nature | www.nature.com\n",
      "failure, such as gastrointestinal bleeding) (Supplementary Table 2). Infection was not associated with 30-day, 90-day or 180-day mortality (P = 0.403, 0.234 or 0.098) in patients with alcoholic hepatitis.\n",
      "Our univariate logistic and Cox regression of laboratory and clinical parameters found an association between the detection of cytolysin- encoding genes in faeces and the international normalized ratio (INR), platelet count, the model for end-stage liver disease (MELD) score, the sodium MELD score, the age, serum bilirubin, INR and serum creatinine (ABIC) score and death (Supplementary Table 3). In the multivariate Cox analysis, detection of cytolysin-encoding genes in faeces was associated with\n",
      "90-day (P = 0.004) and with 180-day mortality (P = 0.001) (Supple- mentary Table 3), even after we adjusted for the geographical origin of the patient, antibiotic treatment, platelet count, and creatinine, bilirubin and INR as components of the MELD score. We found no multicollinear- ity between the detection of faecal cytolysin-encoding genes and these cofactors (variance inflation factor < 1.6), which indicates that cytolysin is an independent predictor of mortality in patients with alcoholic\n",
      "hepatitis. When we performed receiver-operating-characteristic curve analysis for 90-day mortality, cytolysin had an area under the curve of 0.81, which was superior to other widely used predictors for mortality in clinical practice (Extended Data Fig. 1e). On the basis of our findings, we propose that the detection of cytolysin may be a prognostic factor for more severe liver- related outcomes and increased risk of death, and a stronger predictor of mortality than MELD, ABIC and the\n",
      "discriminant function score.\n",
      "To determine phylogeny of E. faecalis in patients with alcoholic hepatitis, we performed targeted culturing from stool samples. Whole- genome sequencing of 93 E. faecalis isolates revealed a broad phylo- genetic diversity of cytolysin-positive E. faecalis from patients with alcoholic hepatitis (Fig. 1d), which indicates that cytolysin production is a variable trait among E. faecalis isolates and that cytolysin is car- ried in mobile genetic elements, which include both chromosomally encoded\n",
      "pathogenicity islands and plasmids14. Detection of any other antimicrobial resistance genes or virulence genes in E. faecalis isolates did not correlate with disease severity or mortality in patients with alcoholic hepatitis (Supplementary Table 4).\n",
      "The total amount of faecal E. faecalis, or faecal E. faecalis positivity, did not correlate with disease severity or mortality in patients with alcoholic hepatitis (Supplementary Tables 5, 6). Cytolysin-positive and cytolysin-negative patients with alcoholic hepatitis had similar amounts of faecal E. faecalis (Extended Data Fig. 1f). Although there were differences in the composition of the gut microbiota in patients with alcoholic hepatitis from different geographical regions (Extended Data\n",
      "Fig. 1g), the proportion of cytolysin-positive patients, total amount of faecal E. faecalis, faecal E. faecalis positivity (Extended Data Fig. 1h–j), treatment and clinical outcomes (30-day and 90-day mortality) did not differ significantly among the regions or centres (Supplemen- tary Table 7). In addition, cirrhosis was not associated with cytolysin positivity, the total amount of faecal E. faecalis or faecal E. faecalis positivity in patients with alcoholic hepatitis (Extended Data Fig. 1k–m,\n",
      "Supplementary Tables 4–6). These results confirm our findings that the presence of cytolysin-producing E. faecalis rather than the total amount or presence of E. faecalis per se determines the severity of alcoholic hepatitis and mortality.\n",
      "Cytolysin and ethanol-induced liver disease\n",
      "To determine whether cytolysin contributes to liver damage mediated by E. faecalis, we gavaged mice with a cytolytic E. faecalis strain (FA2- 2(pAM714)) or a non-cytolytic E. faecalis strain (FA2-2(pAM771))5; the mice were then placed on a chronic–binge ethanol diet15. Compared to mice gavaged with phosphate-buffered saline (PBS), mice fed with ethanol after they were gavaged with cytolytic E. faecalis developed more severe liver injury as indicated by a higher level of alanine amino-\n",
      "transferase (ALT) (Extended Data Fig. 2a) and increased hepatic stea- tosis (Extended Data Fig. 2b, c). Mice that were fed ethanol after they\n",
      ") 1 – l U ( T L A f o 400 300 200 100 0 * *** *** ** Control Ethanol\n",
      "\n",
      "b e d i r e c y l g i r t c i t a p e H ) r e v i l g r e p g m ( t n e t n o 50 40 30 20 10 0 **** **** *** * c\n",
      "c Control Ethanol Alcoholic hepatitis cytolysin-positive Alcoholic hepatitis cytolysin-positive Alcoholic hepatitis cytolysin-negative Alcoholic hepatitis cytolysin-negative no. 1 no. 2 no. 3 no. 4\n",
      "e\n",
      "Cxcl1 e g n a h c d l o F 8 6 4 2 ** ** ** ** 0 Control Ethanol\n",
      "f\n",
      "Col1a1\n",
      "20 15 10 5 ** *** * * 0\n",
      "h\n",
      "\n",
      "LDH\n",
      ") % 40 * * ** ** ( y t i c i x o 30 20 t o t 10 y C 0 0 0 0 200 0 400 200 200 400 400 0 0 0 200 0 400 200 200 400 400 CylLS″ (nM) CylLL″ (nM) – + Ethanol\n",
      "B\n",
      "Alcoholic hepatitis cytolysin-positive no. 1\n",
      "B\n",
      "Alcoholic hepatitis cytolysin-positive no. 2\n",
      "B\n",
      "Alcoholic hepatitis cytolysin-negative no. 3\n",
      "B\n",
      "Alcoholic hepatitis cytolysin-negative no. 4\n",
      "\n",
      "e g n 8 6 Il1b * * ** ** a h c d l o F 4 2 0 Control Ethanol\n",
      "g\n",
      ") % ( e c i M 100 80 60 40 20 **** **** **** **** 0 ND ND ND ND ND ND Control Ethanol\n",
      "Fig. 2 | Transplantation of faeces from cytolysin-positive patients with alcoholic hepatitis exacerbates ethanol-induced liver disease in gnotobiotic mice. a–g, C57BL/6 germ-free mice were colonized with faeces from two cytolysin-positive and two cytolysin-negative patients with alcoholic hepatitis, and subjected to the chronic–binge feeding model. a, Serum levels of ALT. b, Hepatic triglyceride content. c, Representative sections of liver stained with haematoxylin and eosin (H & E). d–f,\n",
      "Hepatic levels of mRNAs that encode Il1b, Cxcl1 and Col1a1. g, Proportions of mice that were positive for cytolysin in the liver, measured by qPCR for cylLS. h, Lactate dehydrogenase (LDH) assay to measure cytotoxicity of hepatocytes isolated from mice that were fed an oral isocaloric control diet (five groups, left) or chronic–binge ethanol diet (five\n",
      "groups, right), and incubated with vehicle, CylLS′′, CylLL′′ or both of the cytolysin subunits at the indicated concentrations without (–) or with (+) ethanol (25 mM) for 3 h. The survival of hepatocytes was determined in three independent experiments. Scale bar, 100 μm. Results are expressed as mean ± s.e.m. (a, b, d–f, h). P values are determined by one-way analysis of variance (ANOVA) with Tukey’s post hoc test (a, b, d–f), two-sided Fisher’s exact test followed by FDR procedures (g) or\n",
      "two-way ANOVA with Tukey’s post hoc test (h). All results were generated from at least three independent replicates. The exact group size (n) and P values for each comparison are listed in Supplementary Table 10. *P < 0.05, **P < 0.01, ***P < 0.001, ****P < 0.0001.\n",
      "were gavaged with cytolytic E. faecalis also had more liver inflamma- tion with higher expression levels of mRNAs that encode inflamma- tory cytokines and chemokines (Il1b, Cxcl1 and Cxcl2) (Extended Data Fig. 2d–f), compared with mice given PBS. Mice that were fed ethanol after they were gavaged with non-cytolytic E. faecalis had significantly less ethanol-induced liver injury, steatosis and inflammation (Extended Data Fig. 2a–f) and longer survival times (Extended Data Fig. 2g), as compared\n",
      "with mice that were fed ethanol after they were administered with cytolytic E. faecalis.\n",
      "To explore the mechanism of cytolysin-associated liver damage, we measured cytolysin in the liver. CylLS was significantly increased in the liver of mice given cytolytic E. faecalis but not in the liver of mice that were not given E. faecalis or of mice gavaged with non-cyto- lytic E. faecalis after chronic ethanol administration (Extended Data Fig. 2h). E. faecalis was detectable in the liver of mice given cytolytic and\n",
      "non-cytolytic E. faecalis and fed an ethanol diet, but not when mice were fed an isocaloric (control) diet (Extended Data Fig. 2i); this indicates that ethanol-induced changes in the gut barrier are necessary for the translocation of cytolytic E. faecalis from the intestine to the liver. The livers of ethanol-fed mice that were given cytolytic or non-cytolytic E. faecalis had positive E. faecalis cultures (Extended Data Fig. 2j). We observed an increased intestinal permeability in ethanol-fed\n",
      "mice com- pared with mice fed with an isocaloric diet, but this was independent of gavaging cytolytic or non-cytolytic E. faecalis after chronic ethanol administration (Extended Data Fig. 2k), indicating that cytolysin does not affect intestinal barrier function.\n",
      "Administration of cytolytic or non-cytolytic E. faecalis to mice did not significantly change the composition of the intestinal microbiota, as shown by 16S rRNA gene sequencing (Extended Data Fig. 2l). Cytolytic E. faecalis did not affect intestinal absorption or hepatic metabolism\n",
      "Nature | www.nature.com | 3\n",
      "\n",
      "3\n",
      "Article\n",
      "a Ef5.1 Ef5.2 Ef5.3 Ef5.4 Ef2.1 Ef2.2 Ef2.3\n",
      "b\n",
      "s l e v e l m u r e S ) 1 – l U ( T L A f o 400 300 200 100 0 Patient no. 5 Patient no. 2 * Patient no. 5 ** Patient no. 2\n",
      "\n",
      ") r e v i l g r e p g m ( t n e t n o c 50 40 30 20 10 0 Patient no. 5 Patient no. 2 * Patient no. 5 ** Patient no. 2 C. crescentus phages Cytolysin-positive E. faecalis phages C. crescentus phages Cytolysin-positive E. faecalis phages\n",
      "m\n",
      "\n",
      "m\n",
      "\n",
      "m\n",
      "\n",
      "=\n",
      "d Control Ethanol C. crescentus phages Cytolysin-positive E. faecalis phages C. crescentus phages Cytolysin-positive E. faecalis phages\n",
      "Patient no. 5\n",
      "\n",
      "Patient no. 2\n",
      "f\n",
      "\n",
      "e g n a h c d l o F\n",
      "12\n",
      "RN ov®O\n",
      "8\n",
      "\n",
      "6\n",
      "\n",
      "4\n",
      "\n",
      "2\n",
      "0\n",
      "Patient no. 5\n",
      "Cxcl1\n",
      "**\n",
      "Patient no. 2\n",
      "\n",
      "Patient no. 5\n",
      "**\n",
      "\n",
      "]\n",
      "Patient no. 2\n",
      "\n",
      "g Col1a1 e g n a h c d l o F 18 15 12 9 6 3 0 * * Patient no. 5 Patient no. 2 Patient no. 5 Patient no. 2\n",
      "Control\n",
      "Ethanol\n",
      "\n",
      "e g n a h c d l o F 12 10 8 6 4 2 0 Il1b * *** Patient no. 5 Patient no. 2 Patient no. 5 Patient no. 2 Control Ethanol\n",
      "h Liver cylLS ) % ( e c i M 100 80 60 40 20 0 ND ND ND ND ** ** Patient no. 5 Patient no. 2 Patient no. 5 Patient no. 2\n",
      "Fig. 3 | Phage therapy against cytolytic E. faecalis abolishes ethanol-induced liver disease in gnotobiotic mice. a, Transmission electron microscopy revealed that the phages we isolated were either siphophages (Ef5.1, Ef5.2, Ef5.3, Ef5.4 and Ef2.2) or myophages (Ef2.1 and Ef2.3). Scale bar, 50 nm. b–h, C57BL/6 germ-free mice were colonized with faeces from two cytolysin- positive patients with alcoholic hepatitis (faeces from one of these patients were also used in Fig. 2) and subjected to the\n",
      "chronic–binge feeding model, gavaged with control phages against C. crescentus (1010 plaque-forming units (PFUs)) or a cocktail of three or four different phages that target cytolytic E. faecalis (1010 PFUs), 1 day before an ethanol binge. b, Serum levels of ALT.\n",
      "c, Hepatic triglyceride content. d, Representative H & E-stained liver sections. Scale bar, 100 μm. e–g, Hepatic levels of mRNAs that encode Il1b, Cxcl1 and Col1a1. h, Proportions of mice that were positive for cytolysin in the liver, measured by qPCR for cylLS. Results are expressed as mean ± s.e.m. (b, c, e–g). P values are determined by two-way ANOVA with Tukey’s post hoc test (b, c, e–g) or two-sided Fisher’s exact test followed by FDR procedures (h). All results are generated from at least\n",
      "three independent replicates. The exact group size (n) and P values for each comparison are listed in Supplementary Table 10. *P < 0.05, **P < 0.01, *** P < 0.001.\n",
      "of ethanol, as indicated by serum levels of ethanol and hepatic levels of Adh1 and Cyp2e1 mRNAs (which encode the two primary enzymes that metabolize ethanol in the liver) (Extended Data Fig. 2m, n). These results indicate that E. faecalis that produce cytolysin promote ethanol- induced liver disease in mice.\n",
      "To extend our findings to humans, we colonized germ-free mice with faeces from cytolysin-positive and cytolysin-negative patients with alcoholic hepatitis (Supplementary Table 8). Consistent with our findings from mice colonized with cytolytic E. faecalis, gnotobi- otic C57BL/6 mice colonized with faeces from two cytolysin-positive patients developed more severe ethanol-induced liver injury, steatosis, inflammation and fibrosis than mice given faeces from two cytolysin- negative patients (Fig.\n",
      "2a–f, Extended Data Fig. 3a–d). Transplantation of faeces from cytolysin-positive patients reduced the survival time of the mice (Extended Data Fig. 3e) and increased translocation of\n",
      "4 | Nature | www.nature.com\n",
      "cytolytic E. faecalis to the liver after ethanol administration (Fig. 2g). The overall composition of the intestinal microbiota was not differ- ent between mice fed the control diet and colonized with faeces from cytolysin-positive or cytolysin-negative donors with alcoholic hepatitis, as shown by 16S rRNA gene sequencing. Mice transplanted with faeces from one of the cytolysin-positive patients with alcoholic hepatitis (patient no. 2) showed a microbiota that was significantly different from\n",
      "that of the other mouse groups after ethanol administration (Extended Data Fig. 3f). Non-cytolytic E. faecalis was not detected in stool samples from donors with cytolytic E. faecalis (Extended Data Fig. 3g). We did not observe differences in intestinal absorp- tion or hepatic metabolism of ethanol between mice colonized with faeces from cytolysin-positive versus cytolysin-negative patients (Extended Data Fig. 3h, i). Together, these results provide further evi- dence that cytolysin promotes\n",
      "ethanol-induced liver disease.\n",
      "a\n",
      "Ef6.1 Ef6.2 Ef6.3 Ef 6.4 Ef7.1 Ef7.2 Ef7.3 Ef 7.4\n",
      "b\n",
      "\n",
      "s l e v e l m u r e S\n",
      "400 300 200 100 0 Patient no. 6 Patient no. 7 Patient no. 6 Patient no. 7 Control Ethanol\n",
      "– l U ( T L A\n",
      "\n",
      "( T L A f\n",
      ") r e v i l g r e p g m ( t n e t n o c 50 40 30 20 10 0 Patient no. 6 Patient no. 7 Patient no. 6 Patient no. 7 Control Ethanol\n",
      "d\n",
      "Control Ethanol C. crescentus phages Cytolysin-negative E. faecalis phages C. crescentus phages Cytolysin-negative E. faecalis phages\n",
      "Patient no. 6\n",
      "\n",
      "Patient no. 7\n",
      "e g n a h c d l o F\n",
      "\n",
      "N w A O\n",
      "\n",
      "1\n",
      "0\n",
      "Patient no. 6\n",
      "Cxcl1\n",
      "Patient\n",
      "\n",
      "&6 - - Patient 6\n",
      "g Col1a1 h 6 4 2 0 Patient Patient Patient Patient no. 6 no. 7 no. 6 no. 7 Control Ethanol\n",
      "Control\n",
      "Ethanol\n",
      "m\n",
      "C. crescentus phages\n",
      "=\n",
      "Cytolysin-negative E. faecalis phages\n",
      "m\n",
      "C. crescentus phages\n",
      "m\n",
      "Cytolysin-negative E. faecalis phages\n",
      "e g n a h c d l o F 5 4 3 2 1 0 Il1b Patient no. 6 Patient no. 7 Patient no. 6 Patient no. 7 Control Ethanol\n",
      "\n",
      "s u c 1010 * * c o c o r e t n E l a c ) 1 – g U F C ( 109 108 107 106 e a F 105 Patient Patient Patient Patient no. 6 no. 7 no. 6 no. 7\n",
      "Fig. 4 | Phages that target non-cytolytic E. faecalis do not reduce ethanol- induced liver disease in gnotobiotic mice. a, Transmission electron microscopy revealed that the phages we isolated were either podophages (Ef6.2, Ef6.3, Ef7.2, Ef7.3 and Ef7.4) or siphophages (Ef6.1, Ef6.4 and Ef7.1). Scale bar, 50 nm. b–h, C57BL/6 germ-free mice were colonized with faeces from two cytolysin-negative patients with alcoholic hepatitis and subjected to the chronic–binge feeding model, gavaged with\n",
      "control phages against C. crescentus (1010 PFUs) or a cocktail of four different phages that target non-\n",
      "To determine the mechanism by which cytolysin increases liver dis- ease, we isolated hepatocytes from mice fed ethanol or control diets, and stimulated them with pure bioactive cytolysin peptides (CylLL′′ and CylLS′′)6. Incubation of the primary mouse hepatocytes with the two cytolysin subunits caused a dose-dependent increase in cell death compared to hepatocytes that were incubated with vehicle or with one subunit only (Fig. 2h). When we isolated hepatocytes from ethanol-fed mice and then\n",
      "incubated these hepatocytes with ethanol, we did not observe increased levels of cytolysin-induced cell death compared to hepatocytes isolated from mice on the control diet, which indicates that cytolysin-induced hepatocyte cell death was independent of ethanol. The cytotoxic effects of cytolysin are possibly mediated by pore forma- tion, resulting in cell lysis14.\n",
      "Bacteriophage treatment in liver disease\n",
      "To further demonstrate the potential causative role of cytolytic E. faecalis for the development of ethanol-induced steatohepatitis,\n",
      "cytolytic E. faecalis (1010 PFUs), 1 day before an ethanol binge. b, Serum levels of ALT. c, Hepatic triglyceride content. d, Representative H & E-stained liver sections. Scale bar, 100 μm. e–g, Hepatic levels of mRNAs that encode Il1b, Cxcl1 and Col1a1. h, Faecal colony-forming units (CFUs) of Enterococcus. Results are expressed as mean ± s.e.m. (b, c, e–h). P values are determined by two-way ANOVA with Tukey’s post hoc test (b, c, e–h). All results were generated from at least three\n",
      "independent replicates. The exact group size (n) and P values for each comparison are listed in Supplementary Table 10. *P < 0.05.\n",
      "we investigated the effects of treatment with bacteriophages (here- after, phages). Phages are ubiquitous in bacteria-rich environments, including the gut16. E. faecalis phages that are highly strain-specific can be isolated17, which potentially makes the direct editing of gut microbiota feasible. It has previously been shown that Atp4aSl/Sl mice, which lack gastric acid, have overgrowth of intestinal enterococci, which is associated with increased susceptibility to alcohol-induced\n",
      "steatohepatitis11. The gavaging of wild-type mice with an E. faecalis strain isolated from Atp4aSl/Sl mice led to increased ethanol-induced steatohepatitis11. We found that this same E. faecalis strain expressed cytolysin. We then isolated four distinct phages from sewage water. These phages lyse the cytolytic E. faecalis strain isolated from Atp4aSl/Sl mice. All four phages were podophages of the virulent Picovirinae group (Extended Data Fig. 4). Atp4aSl/Sl mice and their wild-type littermates\n",
      "were then placed on the chronic–binge ethanol diet and gavaged with the lytic phage cocktail. Phages directed against Caulobacter crescen- tus, a bacterium that is present in freshwater lakes and streams18 but that does not colonize humans or rodents19,20, were used as controls.\n",
      "Nature | www.nature.com | 5\n",
      "Article\n",
      "Compared to Atp4aSl/Sl mice gavaged with control phages or vehicle, Atp4aSl/Sl mice gavaged with phages that target cytolytic E. faecalis had less severe liver injury, steatosis and inflammation after chronic ethanol feeding (Extended Data Fig. 5a–f). Administration of E. faecalis phages significantly reduced levels of cytolysin in the liver (Extended Data Fig. 5g) as well as faecal amounts of Enterococcus (Extended Data Fig. 5h). Phage administration did not affect the overall composition of\n",
      "the faecal microbiome, intestinal absorption or hepatic metabolism of ethanol (Extended Data Fig. 5i–k).\n",
      "To develop a therapeutic approach to precisely edit the intestinal microbiota, we cultured cytolytic E. faecalis strains from the faecal samples of patients with alcoholic hepatitis. We then isolated lytic phages from sewage water against these cytolytic E. faecalis strains; these phages had siphophage or myophage morphology (Fig. 3a, Extended Data Fig. 6). Gnotobiotic mice were colonized with faeces from two cytolysin-positive patients with alcoholic hepatitis (Sup- plementary Table 8) and\n",
      "given three or four different—but patient- specific—lytic phages against cytolytic E. faecalis. The phages against cytolytic E. faecalis abolished ethanol-induced liver injury and steatosis, as shown by lower levels of ALT, lower percentages of hepatic cells positive for terminal deoxynucleotide transferase-mediated dUTP nick-end labelling, and lower levels of hepatic triglycerides and oil red O-staining (Fig. 3b–d, Extended Data Fig. 7a, b), as well as by decreased hepatic levels of Il1b,\n",
      "Cxcl1, Cxcl2, Col1a1 and Acta2 mRNAs, and reduced hepatic levels of cylLS, as compared with mice given control phages (against C. crescentus) (Fig. 3e–h, Extended Data Fig. 7c, d). Treatment with phages against cytolytic E. faecalis also reduced faecal amounts of Enterococcus (Extended Data Fig. 7e) without affecting the overall composition of the gut microbiota (Extended Data Fig. 7f). Intestinal absorption of ethanol and hepatic metabolism were similar in all groups (Extended Data Fig. 7g, h).\n",
      "To demonstrate that the effect of phage treatment occurs via the targeting of cytolysin-positive E. faecalis, rather than a reduction in cytolysin-negative E. faecalis, we colonized gnotobiotic mice with faeces from cytolysin-negative patients with alcoholic hepatitis (Sup- plementary Table 8). Phages against non-cytolytic E. faecalis from patients were isolated from sewage water; they had siphophage or podophage morphology (Fig. 4a, Extended Data Fig. 8). These phages did not reduce features of\n",
      "ethanol-induced liver disease compared with control phages (Fig. 4b–g, Extended Data Fig. 9a–h), despite the reduc- tion of faecal Enterococcus (Fig. 4h). Our findings indicate that treat- ment with lytic phages can selectively attenuate the ethanol-induced liver disease caused by cytolysin-positive E. faecalis in humanized mice.\n",
      "Discussion\n",
      "Phage-based therapies have predominantly been studied in patients with bacterial infections in the gastrointestinal tract21–23, urinary tract24,25 and other organ systems26–28. The results of these studies— although mixed in terms of efficacy—strongly suggest that phage treat- ment offers a safe alternative to antibiotics26,27. However, safety studies are required for complex populations (such as patients with alcoholic hepatitis), because phages can induce a strong immune reaction29. Further\n",
      "work is required to determine whether phages that target cyto- lytic E. faecalis might be used to treat patients with alcoholic hepatitis, a life-threatening disease that at present has no effective treatment. Eradication of this specific bacterial strain might produce better out- comes than current treatments, and environmental sources can be used to easily isolate phages that target cytolysin-positive E. faecalis. Here we provide an example of the efficacy of approaches based on phages in mice\n",
      "for a disease that is not considered a classic infectious disease. Our data also suggest that cytolysin may be used as a predictive biomarker of severe alcoholic hepatitis; an independent, prospective\n",
      "cohort is therefore needed to validate cytolysin as a biomarker, and to extend the phage findings in mice to human patients.\n",
      "Online content\n",
      "Any methods, additional references, Nature Research reporting sum- maries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author con- tributions and competing interests; and statements of data and code availability are available at https://doi.org/10.1038/s41586-019-1742-x.\n",
      "1. Lozano, R. et al. Global and regional mortality from 235 causes of death for 20 age groups in 1990 and 2010: a systematic analysis for the Global Burden of Disease Study 2010. Lancet 380, 2095–2128 (2012).\n",
      "2. Lee, B. P., Vittinghoff, E., Dodge, J. L., Cullaro, G. & Terrault, N. A. National trends and long- term outcomes of liver transplant for alcohol-associated liver disease in the United States. JAMA Intern. Med. 179, 340–348 (2019).\n",
      "\n",
      "3. Rehm, J. et al. Burden of disease associated with alcohol use disorders in the United States. Alcohol. Clin. Exp. Res. 38, 1068–1077 (2014).\n",
      "4. Llopis, M. et al. Intestinal microbiota contributes to individual susceptibility to alcoholic liver disease. Gut 65, 830–839 (2016).\n",
      "5. Ike, Y., Clewell, D. B., Segarra, R. A. & Gilmore, M. S. Genetic analysis of the pAD1 hemolysin/bacteriocin determinant in Enterococcus faecalis: Tn917 insertional mutagenesis and cloning. J. Bacteriol. 172, 155–163 (1990).\n",
      "6. Tang, W. & van der Donk, W. A. The sequence of the enterococcal cytolysin imparts unusual lanthionine stereochemistry. Nat. Chem. Biol. 9, 157–159 (2013).\n",
      "7. Maddrey, W. C. et al. Corticosteroid therapy of alcoholic hepatitis. Gastroenterology 75, 193–199 (1978).\n",
      "\n",
      "8. Dominguez, M. et al. A new scoring system for prognostic stratification of patients with alcoholic hepatitis. Am. J. Gastroenterol. 103, 2747–2756 (2008).\n",
      "\n",
      "9. Thursz, M. R. et al. Prednisolone or pentoxifylline for alcoholic hepatitis. N. Engl. J. Med. 372, 1619–1628 (2015).\n",
      "\n",
      "10. Mathurin, P. & Lucey, M. R. Management of alcoholic hepatitis. J. Hepatol. 56, S39–S45 (2012).\n",
      "11. Llorente, C. et al. Gastric acid suppression promotes alcoholic liver disease by inducing overgrowth of intestinal Enterococcus. Nat. Commun. 8, 837 (2017).\n",
      "12. Gilmore, M. S. et al. Genetic structure of the Enterococcus faecalis plasmid pAD1- encoded cytolytic toxin system and its relationship to lantibiotic determinants. J. Bacteriol. 176, 7335–7344 (1994).\n",
      "\n",
      "13. Cox, C. R., Coburn, P. S. & Gilmore, M. S. Enterococcal cytolysin: a novel two component peptide system that serves as a bacterial defense against eukaryotic and prokaryotic cells. Curr. Protein Pept. Sci. 6, 77–84 (2005).\n",
      "14. Van Tyne, D., Martin, M. J. & Gilmore, M. S. Structure, function, and biology of the Enterococcus faecalis cytolysin. Toxins 5, 895–911 (2013).\n",
      "15. Bertola, A., Mathews, S., Ki, S. H., Wang, H. & Gao, B. Mouse model of chronic and binge ethanol feeding (the NIAAA model). Nat. Protoc. 8, 627–637 (2013).\n",
      "16. Ogilvie, L. A. & Jones, B. V. The human gut virome: a multifaceted majority. Front. Microbiol. 6, 918 (2015).\n",
      "17. Chatterjee, A. et al. Bacteriophage resistance alters antibiotic-mediated intestinal expansion of enterococci. Infect. Immun. 87, e00085-19 (2019).\n",
      "\n",
      "18. Poindexter, J. S. Biological properties and classification of the Caulobacter group. Bacteriol. Rev. 28, 231–295 (1964).\n",
      "19. Shin, J. et al. Analysis of the mouse gut microbiome using full-length 16S rRNA amplicon sequencing. Sci. Rep. 6, 29681 (2016).\n",
      "20. Human Microbiome Project Consortium. Structure, function and diversity of the healthy human microbiome. Nature 486, 207–214 (2012).\n",
      "21. Marcuk, L. M. et al. Clinical studies of the use of bacteriophage in the treatment of cholera. Bull. World Health Organ. 45, 77–83 (1971).\n",
      "\n",
      "22. Sarker, S. A. et al. Oral phage therapy of acute bacterial diarrhea with two coliphage preparations: a randomized trial in children from Bangladesh. EBioMedicine 4, 124–137 (2016).\n",
      "23. Dalmasso, M., Hill, C. & Ross, R. P. Exploiting gut bacteriophages for human health. Trends Microbiol. 22, 399–405 (2014).\n",
      "\n",
      "24. Ujmajuridze, A. et al. Adapted bacteriophages for treating urinary tract infections. Front. Microbiol. 9, 1832 (2018).\n",
      "\n",
      "25. Khawaldeh, A. et al. Bacteriophage therapy for refractory Pseudomonas aeruginosa urinary tract infection. J. Med. Microbiol. 60, 1697–1700 (2011).\n",
      "26. Schooley, R. T. et al. Development and use of personalized bacteriophage-based therapeutic cocktails to treat a patient with a disseminated resistant Acinetobacter baumannii infection. Antimicrob. Agents Chemother. 61, e00954-17 (2017).\n",
      "\n",
      "27. Dedrick, R. M. et al. Engineered bacteriophages for treatment of a patient with a disseminated drug-resistant Mycobacterium abscessus. Nat. Med. 25, 730–733 (2019).\n",
      "28. Fish, R. et al. Compassionate use of bacteriophage therapy for foot ulcer treatment as an effective step for moving toward clinical trials. Methods Mol. Biol. 1693, 159–170 (2018).\n",
      "effective step for moving toward clinical trials. Methods Mol. Biol. 1693, 159–170 (2018).\n",
      "\n",
      "29. Górski, A. et al. Phages and immunomodulation. Future Microbiol. 12, 905–914 (2017).\n",
      "Publisher’s note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\n",
      "\n",
      "© The Author(s), under exclusive licence to Springer Nature Limited 2019\n",
      "6 | Nature | www.nature.com\n",
      "1Department of Medicine, University of California San Diego, La Jolla, CA, USA. 2Department of Medicine, VA San Diego Healthcare System, San Diego, CA, USA. 3Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California San Diego, La Jolla, CA, USA. 4J. Craig Venter Institute, Rockville, MD, USA. 5J. Craig Venter Institute, La Jolla, CA, USA. 6Host–Microbiota Interactions Laboratory, Wellcome Sanger Institute, Hinxton, UK. 7Division of Biostatistics and Bioinformatics,\n",
      "Department of Family Medicine and Public Health, University of California San Diego, La Jolla, CA, USA. 8Department of Biochemistry and Biophysics, Texas A & M University, College Station, TX, USA. 9Center for Phage Technology, Texas A & M AgriLife Research and Texas A & M University, College Station, TX, USA. 10Department of Biochemistry, University of Illinois at Urbana-Champaign, Urbana, IL, USA. 11Department of Pathology, University of California San Diego, La Jolla, CA, USA. 12Division of\n",
      "Gastroenterology, Hepatology and Nutrition, Department of Medicine, University of Pittsburgh Medical Center, Pittsburgh Liver Research Center, Pittsburgh, PA, USA. 13Hospital Universitario, Departamento de Gastroenterología, Universidad Autonoma de Nuevo Leon, Monterrey, Mexico. 14Division of Digestive and Liver Diseases, Department of Medicine, Columbia University College of Physicians and Surgeons, New York, NY, USA. 15Department of Medicine, University of Alberta, Edmonton, Alberta, Canada.\n",
      "16Division of Gastroenterology and Hepatology, Weill Cornell Medical College, New York, NY, USA. 17Liver Unit, Hospital Universitari Vall d’Hebron, Universitat Autònoma de Barcelona, Barcelona, Spain. 18Centro de Investigación en Red de Enfermedades Hepáticas y Digestivas (CIBEREHD), Barcelona, Spain. 19Liver Unit, Hospital Clinic, Barcelona, Spain. 20Liver Sciences, Department of Inflammation Biology, School of Infectious Diseases and Microbial Sciences, King’s College London, London, UK.\n",
      "21Service des Maladies de L’appareil Digestif et Unité INSERM, Hôpital Huriez, Lille, France. 22Division of Gastroenterology and Hepatology, Department of Medicine, University of Wisconsin School of Medicine and Public Health, Madison, WI, USA. 23Section of Digestive Diseases, Yale University School of Medicine, New Haven, CT, USA. 24Section of Digestive Diseases, VA Connecticut Healthcare System, West Haven, CT, USA. 25Department of Chemistry, University of Illinois at Urbana-Champaign, Urbana,\n",
      "IL, USA. 26Howard Hughes Medical Institute, University of Illinois at Urbana-Champaign, Urbana, IL, USA. 27St Luc University Hospital, Université Catholique de Louvain, Brussels, Belgium. 28Center for Innovative Phage Applications and Therapeutics, University of California San Diego, La Jolla, CA, USA. 29These authors contributed equally: Yi Duan, Cristina Llorente. *e-mail: beschnabl@ucsd.edu\n",
      "Nature | www.nature.com | 7\n",
      "\n",
      "7\n",
      "Methods\n",
      "\n",
      "Methods\n",
      "No statistical methods were used to predetermine sample size. The experiments were not randomized and investigators were not blinded to allocation during experiments and outcome assessment.\n",
      "Patient cohorts\n",
      "\n",
      "Patient cohorts have previously been described* 2, We evaluated\n",
      "26 subjects without alcohol-use disorder (controls; social drinkers consuming less than 20 g/day), 44 patients with alcohol-use disorder and 88 patients with alcoholic hepatitis. Patients with alcohol-use disorder fulfilling the DSM IV criteria33 of alcohol dependence and with active alcohol consumption (self-reported >60 g/day) presented with various stages of liver disease (21% had advanced F3/4 fibrosis based on fibrosis-4 index) (Supplementary Table 1). Patients with alcohol-use disorder\n",
      "were recruited from an alcohol withdrawal unit in San Diego and Brussels, where they followed a detoxification and rehabilitation programme. At admission to the hospital, a complete medication and medical history was taken, and a complete physical examination was performed, including collection of bio-specimens, basic demographic data (such as age, gender, weight and height) and self-reported daily alcohol consumption. Patients were actively drink- ing until the day of admission. Controls or\n",
      "patients with alcohol-use disorder did not take antibiotics or immunosuppressive medication during the two months preceding enrolment. Other exclusion criteria were diabetes, inflammatory bowel disease, known liver disease of any other aetiology, and clinically important cardiovascular, pulmonary or renal co-morbidities. Patients with alcoholic hepatitis were enrolled from the InTeam Consortium (ClinicalTrials.gov identifier number: NCT02075918) from centres in the USA, Mexico, UK, France and\n",
      "Spain. Inclusion criteria for this study were active alcohol abuse (>50 g/day for men and >40 g/day for women) in the past 3 months, aspartate aminotransferase (AST) > ALT and total bilirubin >3 mg/dl in the past 3 months, and a liver biopsy and/or clinical picture consistent with alco- holic hepatitis. Exclusion criteria were autoimmune liver disease (ANA >1/320), chronic viral hepatitis, hepatocellular carcinoma, complete portal vein thrombosis, extrahepatic terminal disease, pregnancy and a\n",
      "lack of signed informed consent. In all patients, the clinical picture was consistent with alcoholic hepatitis and in patients who underwent liver biopsy, the histology was consistent with the diagnosis of alco- holic hepatitis. Liver biopsies were only done if clinically indicated as part of routine clinical care for diagnostic purposes of alcoholic hepatitis. Bio-specimens were collected during their admission to the hospital. The median time of specimen collection was 4 days fol- lowing\n",
      "admission to the hospital (range 0–24, n = 82). For one patient who underwent liver transplantation, the transplantation date was considered as date of death. Patients were censored at the time point at which they were last seen alive. The baseline characteristics are shown in Supplementary Tables 1, 2. Faecal 16S rRNA sequencing, Enterococcus culture and qPCR were performed. The MELD score, ABIC score and discriminant function were calculated from all alcoholic hepatitis patients from whom\n",
      "respective laboratory values were avail- able. The protocol was approved by the Ethics Committee of Hôpital Huriez, Universidad Autonoma de Nuevo Leon, Hospital Universitari Vall d’Hebron, King’s College London, Yale University, University of North Carolina at Chapel Hill, Weill Cornell Medical College, Columbia University, University of Wisconsin, VA San Diego Healthcare System, University of California San Diego (UCSD) and Université Catholique de Louvain. Patients were enrolled after written\n",
      "informed consent was obtained from each patient.\n",
      "Mice\n",
      "\n",
      "C57BL/6 mice were purchased from Charles River and used in Fig. 2h and Extended Data Fig. 2. C57BL/6 germ-free mice were bred at UCSD and used in Figs. 2a–g, 3, 4, Extended Data Figs. 3, 7 and 9. Sublytic Atp4aSl/Sl\n",
      "8 | Nature | www.nature.com\n",
      "mice on a C57BL/6 background have previously been described11,34 and heterozygous mice were used for breeding; sublytic Atp4aSl/Sl littermate mice and their wild-type littermates were used in Extended Data Fig. 5.\n",
      "Female and male mice (age of 9–12 weeks) were placed on a chronic– binge ethanol diet (NIAAA model) as previously described15. Mice were fed with Lieber–DeCarli diet and the caloric intake from ethanol was 0% on days 1–5 and 36% from day 6 until the end of the study period. At day 16, mice were gavaged with a single dose of ethanol (5 g/kg body weight) in the early morning and killed 9 h later. Pair-fed control mice received a diet with an isocaloric substitution of dextrose.\n",
      "Stool samples from patients with alcoholic hepatitis (Fig. 1) were used for faecal transplantation in germ-free mice. Mice were gavaged with 100 μl of stool samples (1 g stool dissolved in 30 ml Luria–Bertani (LB) medium containing 15% glycerol under anaerobic conditions), starting at an age of 5–6 weeks and repeated 2 weeks later. Two weeks after the second gavage, mice were placed on the ethanol or control (isocaloric) diet.\n",
      "In studies of the effects of cytolysin, 5 × 108 CFUs of a cytolytic E. fae- calis strain (FA2-2(pAM714)), a non-cytolytic E. faecalis strain (FA2- 2(pAM771))5 (E. faecalis Δcytolysin) (kindly provided by M. S. Gilmore), or PBS (vehicle control) were fed to mice by gavage every third day, starting from day 6 through day 15 of ethanol feeding. Administration every third day was necessary, given that E. faecalis does not colonize mice11 (Extended Data Fig. 2o). To determine the effect of phage\n",
      "treat- ment, 1010 PFUs of E. faecalis phages (or C. crescentus phage phiCbK as control)35 were gavaged to the mice 1 day before the ethanol binge (at day 16). All animal studies were reviewed and approved by the Institu- tional Animal Care and Use Committee of UCSD.\n",
      "Bacteriophage isolation and amplification\n",
      "The E. faecalis strain from Atp4aSl/Sl mice faeces has previously been isolated11 and was used to isolate phages Efmus1, Efmus2, Efmus3 and Efmus4 (phages specific to the E. faecalis strain isolated from mouse faeces were named as Efmus with a number (Ef for E. faecalis, mus for mouse, digit for isolation order). E. faecalis strains from human stool samples were isolated using methods described below, and the cor- responding phages were named as Ef with patient number plus a digit (Ef for E.\n",
      "faecalis, last digit for isolation order). All E. faecalis strains were grown statically in brain–heart infusion (BHI) broth or on BHI agar at 37 °C. C. crescentus phage phiCbK was purified as previously described35.\n",
      "E. faecalis phages were isolated from untreated raw sewage water obtained from North City Water Reclamation Plant in San Diego. Fifty millilitres of raw sewage water was centrifuged at 8,000g for 1 min at room temperature to pellet large particles. The supernatant was passed through a 0.45-μm and then a 0.2-μm syringe filter (Whatman, PES membrane). One hundred microlitres of the clarified sewage was mixed with 100 μl overnight E. faecalis culture and then added to BHI broth top agar (0.5% agar)\n",
      "and poured over a BHI plate (1.5% agar). After overnight growth at 37 °C, the resulting plaques were recovered using a sterile pipette tip in 500 μl PBS. Phages were replaqued on E. faecalis three more times to ensure that the phages were clonal isolates.\n",
      "High-titre phage stocks were propagated by infecting 200 ml of expo- nentially growing E. faecalis at a multiplicity of infection of 0.1 in BHI broth containing 10 mM MgSO4. Lysis was allowed to proceed for up to six hours at 37 °C with shaking. The lysates were centrifuged at 10,000g for 20 min at room temperature to remove the remaining bacterial cells and debris. Supernatant was then vacuum-filtered through a 0.2-μm membrane filter and kept at 4 °C until use.\n",
      "Before mice were gavaged, 10–20 ml lysates were concentrated using Corning Spin-X UF Concentrators with 100,000-molecular weight cutoff (MWCO) to a volume of approximately 1 ml. Following concen- tration, the culture medium was replaced with PBS via diafiltration. The resulting lysate was further concentrated to a final volume of 0.5 ml and adjusted to the required PFUs.\n",
      "Whole-genome sequencing for phages\n",
      "For all phages except Efmus4, 10 ml of lysates were treated with 10 μg/ml each of DNase and RNase at 37 °C for 1 h and phages were precipitated by adding 1M NaCl and 10% (w/v) polyethylene glycol 8000 (PEG 8000) and incubated at 4 °C overnight. Precipitated phages were then pel- leted by centrifugation at 10,000g for 10 min at 4 °C and resuspended in 500 μl of resuspension buffer (5 mM MgSO4). Phage DNA was then extracted using Promega Wizard DNA Clean-up kit (Promega). Phage genomes were\n",
      "sequenced using a combination of Illumina and Oxford Nanopore Technologies (ONT) MinION platforms. Illumina sequencing libraries were prepared using the Nextera XT library kit with bead-based size selection before loading onto Illumina flow cells. Sequencing was performed with either Illumina MiSeq Reagent Kit v3 in 2 × 300-bp or NextSeq 500 Mid Output Kit in 2 × 150-bp paired-end formats. ONT MinION sequencing libraries were prepared using the Rapid Barcoding Kit (SQK-RBK004) and loaded onto\n",
      "MinION R9.4 flow cells. ONT reads were basecalled with Albacore v.2.3.4 (ONT). The sequence reads were demultiplexed and adapters trimmed from ONT reads using Porechop v.0.2.336. A hybrid Illumina-ONT de novo assembly was performed using the Unicycler v.0.4.7 pipeline37. Subsequently, Pilon v.1.2238 was used iteratively to polish the assemblies with Illumina reads until no addi- tional corrections could be made.\n",
      "For phage Efmus4, 109 PFUs of the phage was filtered sequentially using 0.45-μm and 0.2-μm filters (VWR) and purified on a caesium chloride (CsCl) density gradient39. One millilitre of the CsCl fraction was purified on Amicon YM-100 protein columns (Millipore) and treated with DNase I. DNA was isolated using a Qiagen UltraSens virus kit (Qia- gen), amplified using GenomiPhi V2 (GE Healthcare), and fragmented to 200 to 400 bp using a Bioruptor (Diagenode). Libraries were created using the Ion\n",
      "Plus fragment library kit and sequenced using a 316 Chip on an Ion Torrent Personal Genome Machine (Life Technologies). Reads were trimmed according to modified Phred scores of 0.5 using CLC Genomics Workbench 4.9 (Cambridge), and the remaining reads were assembled using CLC Genomics Workbench 4.9 based on 98% identity with a minimum of 50% read overlap39. Reads were assembled into a single contig of 18,186 bp (20,118 × coverage).\n",
      "Mapping of ONT reads to the hybrid assemblies was used to deter- mine the orientation and terminal ends of linear phage genomes, and reference genomes served as guides to orient circular phage genomes. Phage genome assemblies were annotated using the NCBI Prokaryotic Genome Annotation Pipeline (PGAP)40,41.\n",
      "Phage raw sequence reads and annotated genomes are available at NCBI under the following consecutive BioSample IDs (SAMN11089809– SAMN11089827). GenBank accession numbers include: Efmus1 (MK721195), Efmus2 (MK721197), Efmus3 (MK721185), Efmus4 (MK721193), Ef2.1 (MK693030), Ef2.2 (MK721189), Ef2.3 (MK721192), Ef5.1 (MK721199), Ef5.2 (MK721186), Ef5.3 (MK721200), Ef5.4 (MK721191), Ef6.1 (MK721187), Ef6.2 (MK721188), Ef6.3 (MK721196), Ef6.4 (MK721190), Ef7.1 (MK721194), Ef7.2 (MK721183), Ef7.3\n",
      "(MK721184) and Ef7.4 (MK721198).\n",
      "Genetic maps of phage genomes were generated by LinearDisplay. pl (https://github.com/JCVenterInstitute/LinearDisplay), a PERL script that uses Xfig (https://sourceforge.net/projects/mcj/) to render high- quality images. Preliminary annotation of genes was derived from the automated annotation and from Phage_Finder42, which uses curated hidden Markov models and databases of core phage gene to annotate core gene functions. Annotation was then manually reviewed to assign the final colours.\n",
      "Phage phylogenetic tree\n",
      "\n",
      "A phage whole-genome phylogeny tree was generated from a pairwise distance matrix calculated with the MASH program, which approxi- mates average nucleotide identity (ANI)43. First, a sketch file was created from all the 19 E. faecalis phage genomes isolated and sequenced in\n",
      "this study plus 54 Enterococcus phage genomes obtained from Gen- Bank, with 5,000 12-mers generated per genome (mash sketch -k 12 -s 5000). The sketch file was then compared to all the initial phage genome sequences to generate the ANI matrix using the mash distance command using default settings. The GGRaSP R package was used to calculate the UPMGA phylogeny from the ANI distance matrix, after redundant phage genomes (genomes ANI >99.985) were removed using the GGRaSP R package with a user\n",
      "defined cutoff of 0.015 (ggrasp. cluster (threshold = 0.015)). The resulting dendrogram was translated into newick format using the APE R package44, loaded into the iTOL tree viewer45, and annotated with taxonomic information and manually entered clade identification.\n",
      "Electron microscopy\n",
      "\n",
      "Phage morphology was examined by transmission electron microscopy of negatively stained grids, prepared using the valentine method46 with either 2% uranyl-acetate or 2% phosphotungstic acid, and examined at an acceleration voltage of 100 kV in the JEOL 1200 EX transmission electron microscope.\n",
      "Bacterial DNA extraction and 16S rRNA sequencing\n",
      "DNA from human stool samples, mouse liver sections or bacterial cul- ture was extracted as previously described11, and DNA from mouse faeces was extracted using QIAamp Fast DNA Stool kit (Qiagen). 16S rRNA PCR was completed using Illumina adaptor and barcode-ligated 16S primers targeting the V4 region of the 16S rRNA gene47,48. Amplicons were purified using the Qiaquick PCR purification kit (Qiagen) using manufacturer’s specifications. Purified amplicons were then quantified via TECAN assay\n",
      "(Tecan), normalized and pooled in preparation for 16S rRNA sequencing. Pooled library was quantified and checked for qual- ity using Agilent 2100 Bioanalyzer (Agilent Technologies). Library was sequenced on Illumina MiSeq (Illumina) using V2 reagent chemistry, 500 cycles, 2 × 250-bp format using manufacturer’s specifications. 16S sequence reads were processed and operational taxonomic units were determined using our MOTHUR-based 16S rDNA analysis workflow as previously described11,49. Raw 16S\n",
      "sequence reads can be found in the NCBI Sequence Read Archive (SRA) associated with Bioproject PRJNA525701.\n",
      "Real-time qPCR\n",
      "Bacterial genomic DNA was extracted from human stool samples and mouse liver11. RNA was extracted from mouse liver and cDNAs were generated11. Primer sequences for mouse genes were obtained from the NIH qPrimerDepot. Primer sequences for E. faecalis 16S rRNA gene, E. faecalis cylLS and cylLL genes have previously been described50,51. All primers used in this study are listed in Supplementary Table 9. Mouse gene expression and amplification of bacterial genes were determined with Sybr Green\n",
      "(Bio-Rad Laboratories) using ABI Ste- pOnePlus real-time PCR system. The qPCR value of mouse genes was normalized to 18S.\n",
      "E. faecalis isolation and whole-genome sequencing\n",
      "To isolate E. faecalis strains from human subjects, 50–300 mg of human stool was resuspended in 500 μl PBS, serial dilutions were made and 100 μl was placed on plates with selective medium, BBL Enterococcosel broth (Becton Dickinson). Enterococci colonies were identified by the production of dark brown or black colour, generated by hydrolysis of esculin to esculetin (which reacts with ferric ammonium citrate). Each Enterococcus colony was then picked, and qPCR was performed to identify E.\n",
      "faecalis, using specific primers against the E. faecalis 16S rRNA gene50. For each subject, between 1 and 6 E. faecalis colonies were analysed and bacterial genomic DNA was then extracted as described in ‘Bacterial DNA extraction and 16S rRNA sequencing’.\n",
      "DNA sequencing was performed on the Illumina HiSeq Ten X generat- ing paired-end reads (2 × 151 bp). Bacterial genomes were assembled\n",
      "Article\n",
      "and annotated using the previously described pipeline52. Antimicrobial resistance and virulence genes including cytolysin (cyl) genes carried by E. faecalis isolates were identified by comparing individual genome assemblies against the CARD and VFDB databases, respectively, using abricate v0.8.10 (https://github.com/tseemann/abricate)53,54.\n",
      "For the phylogeny of E. faecalis, the genome assemblies of the study isolates were annotated with Prokka55, and a pangenome estimated using Roary52. A 95% identity cutoff was used, and core genes were defined as those in 99% of isolates. A maximum likelihood tree of the SNPs in the core genes was created using RAxML56 and 100 bootstraps. The resulting tree was visualized using iTOL45. Genome sequence data of E. faecalis strains isolated in this study have been deposited in the European\n",
      "Nucleotide Archive (ENA) under the accession number PRJEB25007. Sequence reads are available at ENA under run acces- sion identifiers ERR3200171–ERR3200263.\n",
      "E. faecalis culture\n",
      "\n",
      "All E. faecalis strains were grown statically in BHI broth or on BHI agar plate at 37 °C. Fifty micrograms per millilitre erythromycin was added when cytolytic and non-cytolytic E. faecalis strains were grown (Extended Data Fig. 2).\n",
      "Determination of levels of faecal Enterococcus\n",
      "To determine levels of faecal enterococci in mice, 10–30 mg of mouse faeces was resuspended into 500 μl PBS and serial dilutions were made. Five microlitres of each dilution from each sample was spotted onto a plate with a selective medium, BBL enterococcosel broth (Becton Dickinson) and the plates were then incubated at 37 °C overnight. For Extended Data Fig. 2o, agar plates contained 50 μg/ml erythromycin. Enterococci colonies were identified by the production of a dark brown or black colour.\n",
      "Colony numbers of each sample were then counted, and CFUs were calculated.\n",
      "Cytolysin expression and purification\n",
      "To purify bioactive CylLL′′ and CylLS′′, an Escherichia coli heterologous expression system was used. In brief, either 6×His–CylLL or 6×His–CylLS were co-expressed with CylM (the enzyme that performs dehydration and cyclization reactions on cytolysin) in E. coli to yield fully dehydrated and cyclized full-length peptides. The His tag and leader peptide were then removed using recombinant CylA (27–412), the soluble domain of the native peptidase used in cytolysin maturation, to yield bioactive\n",
      "CylLL′′ or CylLS′′. The resulting core peptides were further purified by reversed-phase high-performance liquid chromatography (HPLC).\n",
      "The cylLL and cylLS genes were previously cloned into the MCSI of a pRSFDuet-1 backbone vector that contained the cylM gene in MCSII6. The cylA (27–412) gene was previously cloned into MCSI of a pRSFDuet-1 backbone vector57. E. coli BL21 Star (DE3) cells (50 μl) were transformed with 100 ng of either the cylLL_cylM:pRSFDuet, cylLS_cylM:pRSFDuet or cylA (27–412):pRSFDuet plasmids via KCM chemical transformation. The cells were plated on LB agar plates supplemented with kanamycin (50 μg/ml) and\n",
      "grown at 37 °C overnight. One colony was picked to inoculate 15 ml of LB broth supplemented with kanamycin overnight at 37 °C. The culture was used to inoculate 1.5 l of terrific broth supple- mented with kanamycin. Cultures were grown with shaking at 37 °C to an optical density at 600 nm (OD600) of 0.8. The temperature of the incu- bator was lowered to 18 °C and expression was induced with the addi- tion of 0.3 mM final concentration of isopropyl β-d-thiogalactoside. The cultures were allowed\n",
      "to incubate at 18 °C for 18 h. The cells were collected by centrifugation at 5,000g for 12 min. The cell paste was collected and frozen at -70 °C.\n",
      "For the purification of the protease CylA (27–412), the cell paste was thawed and resuspended in 50 ml LanP buffer (20 mM HEPES, 1 M NaCl, pH 7.5). The cell suspension was lysed by homogenization. The lysate was clarified by centrifugation at 13,000g for 45 min and filtered through a 0.45-μm centrifugal filter (Thermo Scientific). The clarified\n",
      "lysate was applied to a pre-equilibrated HisTrap HP 5 ml column (GE Healthcare) through a peristaltic pump. The loaded column was con- nected to an ÄKTA pure 25 M system. The protein was eluted by a linear gradient of LanP buffer to Elution Buffer (20 mM HEPES, 1 M NaCl, 500 mM imidazole, 10% glycerol, pH 7.5) over 30 min. The purest fractions, as determined by 4–20% SDS–PAGE, were combined, concentrated to 1 mg/ml by Amicon Ultra Centrifugal Filters (30 kDa MWCO), and buffer exchanged into\n",
      "storage buffer (20mM HEPES, 300 mM KCl, 10% glycerol, pH 7.5) by PD-10 desalting column (GE Healthcare). Protein concentration was determined by absorbance at 280 nm.\n",
      "For the purification of CylLL′′ and CylLS′′ peptides, the cell paste was thawed and resuspended in 50 ml of LanA Buffer B1 (6 M guanidine HCl, 20 mM NaH2PO4, 500 mM NaCl, 0.5 mM imidazole, pH 7.5). The cell suspension was lysed via sonication (2-s pulse on, 5-s pulse off, 7 min total pulse on time). The cell lysate was clarified by centrifugation at 13,000g for 45 min. The clarified cell lysate was filtered through a 0.45-μm centrifugal filter and applied via gravity flow to a pre-equili-\n",
      "brated, 2 ml bed volume of His60 Ni Superflow Resin (Clonetech). After the lysate had been applied, the resin was washed with 15 ml of LanA Buffer B2 (4 M guanidine HCl, 20 mM NaH2PO4, 500 mM NaCl, 30 mM imidazole, pH 7.5). The resin was washed again with 15 ml of LanA Wash Buffer (20 mM NaH2PO4, 500 mM NaCl, 30 mM imidazole, pH 7.5) to remove the guanidine HCl. The peptides were eluted with 10 ml of LanA elution buffer (20 mM NaH2PO4, 500 mM NaCl, 500 mM imidazole, pH 7.5). A 0.02 mg/ml final\n",
      "concentration of CylA (27–412) was added to the elution fraction and allowed to incubate at room temperature overnight to remove the leader peptide.\n",
      "The digestion was quenched by adding 2% (v/v) final concentration of trifluoroacetic acid. The solution was centrifuged at 4,500g for 10 min and filtered through a 0.45-μm syringe filter (Thermo Scientific). The core peptides were purified by semi-preparative reverse-phase HPLC using a Phenomenex Jupiter Proteo column (10 mm × 250 mm, 4 μm, 90 Å) connected to an Agilent 1260 Infinity II liquid chromatography system. The peptides were separated using a linear gradient of 3% (v/v) solvent B\n",
      "(acetonitrile + 0.1% trifluoroacetic acid) in solvent A (water + 0.1% trif- luoroacetic acid). The fractions were spotted on a matrix-assisted laser desorption/ionization (MALDI) target plate by mixing 1 μl of sample with 1 μl of a 25 mg/ml solution of Super-DHB (Sigma) in 80% acetonitrile/water + 0.1% trifluoroacetic acid. The fractions were analysed by MALDI–time of flight (TOF) mass spectrometry on a Bruker UltrafleXtreme MALDI–TOF/ TOF operating in positive ionization, reflector mode.\n",
      "Primary mouse hepatocytes\n",
      "Hepatocytes were isolated from C57BL/6 female mice fed the chronic– binge ethanol diet (NIAAA model)15. Livers were perfused in situ with 0.5 mM EGTA containing calcium-free salt solution and then perfused with a solution containing 0.02% (w/v) collagenase D (Roche Applied Science). Livers were then carefully minced and filtered using a 70-μm nylon cell strainer. Hepatocytes were centrifuged at 50g for 1 min after 3 washes. Hepatocyte viability was assessed by Trypan Blue (Thermo Fisher\n",
      "Scientific). Hepatocytes (1.5 × 105) were seeded on 12-well plates coated with rat collagen type I in DMEM-F12 (Thermo Fisher Scientific) with insulin–transferrin–selenium (1% v/v) (Thermo Fisher Scientific) and 40 ng/ml dexamethasone (MP Biomedicals) containing 10% (v/v) fetal bovine serum (FBS) (Gemini Bio-Products) and antibiotics. After 4 h, the culture was washed with DMEM-F12 medium and changed to the same complemented medium without FBS58. Then 16 h later, hepato- cytes were cultured with\n",
      "0 or 25 mM ethanol and stimulated with 0, 200 or 400 nM CylLS′′ and/or CylLL′′ in the same culture medium without FBS. After 3 h stimulation, hepatocyte cytotoxicity was assessed using Pierce LDH cytotoxicity detection kit (Thermo Fisher Scientific).\n",
      "Biochemical analysis\n",
      "Serum levels of ALT were determined using Infinity ALT kit (Thermo Scientific). Hepatic triglyceride levels were measured using Triglyceride\n",
      "Liquid Reagents kit (Pointe Scientific). Levels of serum lipopolysac- charide and faecal albumin were determined by enzyme-linked immu- nosorbent kits (Lifeome Biolabs and Bethyl Labs, respectively). Serum levels of ethanol were measured using ethanol assay kit (BioVision).\n",
      "Staining procedures\n",
      "Formalin-fixed tissue samples were embedded in paraffin and stained with H & E. To determine lipid accumulation, liver sections were embed- ded in OCT compound. Eight-micrometre frozen sections were then cut and stained with Oil Red O (Sigma-Aldrich). Representative images from each group of mice are shown in each figure. The terminal deoxynucleo- tide transferase-mediated dUTP nick-end labelling (TUNEL) assay was performed using an in situ cell death detection kit (Sigma-Aldrich). We randomly\n",
      "selected five high-power fields for counting TUNEL-positive cells and normalized numbers to total cells.\n",
      "Statistical analysis\n",
      "Results are expressed as mean ± s.e.m. (except when stated otherwise). Univariate and multivariate Cox regression analysis was used to detect associations of cytolysin with overall mortality. The multivariate model was adjusted for geographical origin of the patients, antibiotic treat- ment, platelet count, and creatinine, bilirubin and INR as components of the MELD score. Univariate logistic regression analysis of laboratory and clinical parameters associated with the detection of cytolysin and\n",
      "E. faecalis was performed. Univariate linear regression analysis of labo- ratory and clinical parameters associated with the log-transformed total amount of faecal E. faecalis measured with qPCR was performed. To associate log-transformed total E. faecalis and E. faecalis positiv- ity with mortality, univariate Cox regression was used. P values from univariate and multivariate Cox regression, univariate logistic regres- sion and univariate linear regression were determined by Wald test.\n",
      "Multicollinearity was examined using the variance inflation factor. Kaplan–Meier curves were used to compare survival between cytolysin- positive and cytolysin-negative patients with alcoholic hepatitis. Faecal E. faecalis, bacterial diversity and richness from controls and patients were compared using Kruskal–Wallis test with Dunn’s post hoc test. Region- and/or centre-specific clinical characteristics of patients with alcoholic hepatitis were compared with Kruskal–Wallis test for continu- ous\n",
      "and Fisher’s exact test for categorical variables. Faecal E. faecalis in patients with alcoholic hepatitis with or without cytolysin, and with or without cirrhosis, were compared with Mann–Whitney-Wilcoxon rank-sum test. Faecal E. faecalis in patients with alcoholic hepatitis from different region and/or centres were compared with the Kruskal– Wallis test. The percentage of subjects with faecal samples that were positive for E. faecalis and cytolysin was compared using Fisher’s exact test,\n",
      "followed by FDR procedures for multiple group comparisons. Jaccard dissimilarity matrices were used for PCoA, and P values were determined by PERMANOVA followed by FDR procedures to correct for multiple comparisons.\n",
      "For mouse and cell culture studies, the significance of multiple groups was evaluated using one-way or two-way ANOVA with Tukey’s post hoc test. Fisher’s exact test was used in the analysis of liver E. faecalis and cytolysin with FDR correction for multiple comparisons. Kaplan–Meier curves were used to compare survival between experi- mental mouse groups. PCoA based on Jaccard dissimilarity matrices was performed between experimental mouse groups and the P values were determined by PERMANOVA\n",
      "followed by FDR procedures to cor- rect for multiple comparisons.\n",
      "Exact P values for all comparisons, together with group size for each group, were listed in Supplementary Table 10. Statistical analyses were performed using R statistical software, R v.3.5.1 (R Foundation for Sta- tistical Computing) and GraphPad Prism v.6.01. A value of P < 0.05 was considered to be statistically significant (adjusted for multiple comparisons when performing multiple tests).\n",
      "Reporting summary\n",
      "Further information on research design is available in the Nature Research Reporting Summary linked to this paper.\n",
      "Data availability\n",
      "Raw 16S sequence reads can be found in the NCBI SRA associated with Bioproject PRJNA525701. Phage raw sequence reads and annotated genomes are available at NCBI under the following consecutive BioSa- mple identifiers (SAMN11089809–SAMN11089827). Genome sequence data of E. faecalis strains isolated in this study were registered at the ENA under study PRJEB25007.\n",
      "Code availability\n",
      "The PERL script for making the genetic maps of phage genomes can be found at https://github.com/JCVenterInstitute/LinearDisplay.\n",
      "30. Brandl, K. et al. Dysregulation of serum bile acids and FGF19 in alcoholic hepatitis. J. Hepatol. 69, 396–405 (2018).\n",
      "\n",
      "31. Gao, B. et al. Serum and fecal oxylipins in patients with alcohol-related liver disease. Dig. Dis. Sci. 64, 1878–1892 (2019).\n",
      "\n",
      "32. Lang, S. et al. Intestinal fungal dysbiosis and systemic immune response to fungi in patients with alcoholic hepatitis. Hepatology (2019).\n",
      "33. Ball, S. A., Tennen, H., Poling, J. C., Kranzler, H. R. & Rounsaville, B. J. Personality, temperament, and character dimensions and the DSM-IV personality disorders in substance abusers. J. Abnorm. Psychol. 106, 545–553 (1997).\n",
      "34. Krieg, L. et al. Mutation of the gastric hydrogen-potassium ATPase alpha subunit causes iron-deficiency anemia in mice. Blood 118, 6418–6425 (2011).\n",
      "35. Gill, J. J. et al. The Caulobacter crescentus phage phiCbK: genomics of a canonical phage. BMC Genomics 13, 542 (2012).\n",
      "\n",
      "36. Wick, R. R., Judd, L. M., Gorrie, C. L. & Holt, K. E. Completing bacterial genome assemblies with multiplex MinION sequencing. Microb. Genom. 3, e000132 (2017).\n",
      "\n",
      "37. Wick, R. R., Judd, L. M., Gorrie, C. L. & Holt, K. E. Unicycler: resolving bacterial genome assemblies from short and long sequencing reads. PLOS Comput. Biol. 13, e1005595 (2017).\n",
      "38. Walker, B. J. et al. Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement. PLoS ONE 9, e112963 (2014).\n",
      "39. Santiago-Rodriguez, T. M. et al. Transcriptome analysis of bacteriophage communities in periodontal health and disease. BMC Genomics 16, 549 (2015).\n",
      "\n",
      "40. Haft, D. H. et al. RefSeq: an update on prokaryotic genome annotation and curation. Nucleic Acids Res. 46, D851–D860 (2018).\n",
      "\n",
      "41. Tatusova, T. et al. NCBI prokaryotic genome annotation pipeline. Nucleic Acids Res. 44, 6614–6624 (2016).\n",
      "42. Fouts, D. E. Phage_Finder: automated identification and classification of prophage regions in complete bacterial genome sequences. Nucleic Acids Res. 34, 5839–5851 (2006).\n",
      "43. Ondov, B. D. et al. Mash: fast genome and metagenome distance estimation using MinHash. Genome Biol. 17, 132 (2016).\n",
      "44. Paradis, E. & Schliep, K. ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R. Bioinformatics 35, 526–528 (2019).\n",
      "\n",
      "45. Letunic, I. & Bork, P. Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees. Nucleic Acids Res. 44, W242–W245 (2016).\n",
      "46. Valentine, R. C., Shapiro, B. M. & Stadtman, E. R. Regulation of glutamine synthetase. XII. Electron microscopy of the enzyme from Escherichia coli. Biochemistry 7, 2143–2152 (1968).\n",
      "47. Haas, B. J. et al. Chimeric 16S rRNA sequence formation and detection in Sanger and 454-pyrosequenced PCR amplicons. Genome Res. 21, 494–504 (2011).\n",
      "\n",
      "48. Caporaso, J. G. et al. Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample. Proc. Natl Acad. Sci. USA 108 (Suppl 1), 4516–4522 (2011).\n",
      "49. Chen, P. et al. Supplementation of saturated long-chain fatty acids maintains intestinal eubiosis and reduces ethanol-induced liver injury in mice. Gastroenterology 148, 203–214.e16 (2015).\n",
      "50. Ryu, H. et al. Development of quantitative PCR assays targeting the 16S rRNA genes of Enterococcus spp. and their application to the identification of enterococcus species in environmental samples. Appl. Environ. Microbiol. 79, 196–204 (2013).\n",
      "51. Haas, W., Shepard, B. D. & Gilmore, M. S. Two-component regulator of Enterococcus faecalis cytolysin responds to quorum-sensing autoinduction. Nature 415, 84–87 (2002).\n",
      "52. Page, A. J. et al. Robust high-throughput prokaryote de novo assembly and improvement pipeline for Illumina data. Microb. Genom. 2, e000083 (2016).\n",
      "53. Chen, L. et al. VFDB: a reference database for bacterial virulence factors. Nucleic Acids Res. 33, D325–D328 (2005).\n",
      "\n",
      "54. Jia, B. et al. CARD 2017: expansion and model-centric curation of the comprehensive antibiotic resistance database. Nucleic Acids Res. 45, D566–D573 (2017)\n",
      "55. Seemann, T. Prokka: rapid prokaryotic genome annotation. Bioinformatics 30, 2068–2069 (2014).\n",
      "\n",
      "56. Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 30, 1312–1313 (2014).\n",
      "Article\n",
      "57. Tang, W., Bobeica, S. C., Wang, L. & van der Donk, W. A. CylA is a sequence-specific protease involved in toxin biosynthesis. J. Ind. Microbiol. Biotechnol. 46, 537–549 (2019).\n",
      "58. Iwaisako, K. et al. Protection from liver fibrosis by a peroxisome proliferator-activated receptor δ agonist. Proc. Natl Acad. Sci. USA 109, E1369–E1376 (2012).\n",
      "59. Clarke, T. H., Brinkac, L. M., Sutton, G. & Fouts, D. E. GGRaSP: a R-package for selecting representative genomes using Gaussian mixture models. Bioinformatics 34, 3032–3034 (2018).\n",
      "Acknowledgements S. Lang was supported by a DFG fellowship (LA 4286/1-1), C.L. was supported by an AASLD Pinnacle Research Award in Liver Disease and a pilot project award from Southern California Research Center for Alcoholic Liver and Pancreatic Disease (ALPD) and Cirrhosis (P50 AA011999), Y.S. was supported by a Wellcome Trust PhD studentship and I.R.R. was supported by National Institute of General Medical Sciences (NIGMS)–NIH Chemistry–Biology Interface Training Grant (T32-GM070421). This\n",
      "study was supported in part by a Biocodex Microbiota Foundation Grant, NIH grants R01 AA24726, U01 AA026939, by Award Number BX004594 from the Biomedical Laboratory Research & Development Service of the VA Office of Research and Development (to B.S.), the Wellcome Trust (WT098051) (to T.D.L.), NIH grant U01AA021908 (to R.B.) and services provided by P30 DK120515 and P50 AA011999.\n",
      "Author contributions Y.D. was responsible for acquisition, analysis and interpretation of data, and drafting of the manuscript; C.L. was responsible for study concept and design, acquisition,\n",
      "analysis and interpretation of data and key preliminary experiments; S. Lang, K.B., J.L. and X.M.T. provided assistance with statistical analysis; H.C., L.J., B.G., W.S., R.K., F.H. and S. Lee provided assistance with data acquisition; R.C.W., T.H.C., K.N., M.T. and D.E.F. were responsible for 16S rRNA sequencing, phage genome sequencing and data analysis; Y.S. and T.D.L. were responsible for bacterial genome sequencing and data analysis; A.H.-M., L.L. and R.Y. provided assistance with phage\n",
      "studies and were responsible for electron microscopy data; R.Y. provided critical revision of the manuscript; I.R.R. and W.A.v.d.D. were responsible for cytolysin expression and purification; Y.M. and L.E. provided assistance with the design and conduct of the gnotobiotic mouse studies; M.L. and D.P. provided assistance with phage isolation; M.V.-C., F.B.-P., E.C.V., J.G.A., R.S.B. Jr, V.V., J.A., J.C., D.L.S., S.B.H., A.L., M.R.L., P.M., G.G.-T., R.B. and P.S. were responsible for collection of\n",
      "human samples; D.E.F. and B.S. were responsible for the study concept and design, and editing the manuscript; B.S. was responsible for study supervision.\n",
      "Competing interests B.S. consults for the Ferring Research Institute; however, there is no competing interest with regard to this study. All other authors declare no competing interests.\n",
      "Additional information\n",
      "Supplementary information is available for this paper at https://doi.org/10.1038/s41586-019- 1742-x.\n",
      "Correspondence and requests for materials should be addressed to B.S.\n",
      "Reprints and permissions information is available at http://www.nature.com/reprints.\n",
      "[ - 00 Bacterial Genera Bacteroides Unclassified Lachnospiraceae Unclassified Ruminococcaceae 80 Veelllonella Akkermansia 70 Prevotella Blautia 60 Rosebuna Escherichia/Shigella Streptococcus 50 Unclassified Clostndiales 40 Dialister Faecalibacterium Clostndium_XiVa 30 Ruminococcus Enterococcus 20 Megasphaera Mogamonas Ruminococcus2 Relative sequence abundance (%) 10 Bifidobacterium | | = B Q=) = = - B: B ... [remaining 309 | Alcoholic Controls AUD hepatitis\n",
      "o Jede X % % % = N Shannon diversity Controls AUD Alcoholic hepatitis\n",
      "Simpson diversity\n",
      "5\n",
      "o\n",
      "\n",
      "o o\n",
      "(=] »\n",
      "o N\n",
      "\n",
      "o\n",
      "Controls\n",
      "k x\n",
      "AUD\n",
      "%\n",
      "\n",
      "%\n",
      "Alcoholic hepatitis\n",
      "Chao richness\n",
      "— de kK —\n",
      "10,000\n",
      "& 8 oo =34\n",
      "\n",
      "H» , =34 00\n",
      "w &\n",
      "2,000\n",
      "\n",
      "1,000\n",
      "Controls\n",
      "AUD\n",
      "Ll ]\n",
      "Alcoholic hepatitis\n",
      "1.0 0.8 -] » -Cytolysin, AUC=0.81 Sensitivity -MELD, AUC=0.70 0.2 -ABIC, AUC=0.66 -DF, AUC=0.63 0.0 0.0 0.2 0.4 0.6 0.8 1.0 1 - Specificity\n",
      "Enterococcus faecalis ek e K 150,000 * % 100,000 50,000 \" - 300 300 Relative abundance 100 100 . Controls AUD Alcoholic hepatitis\n",
      "\n",
      "@@ E. faecalis negative [ E. faecalis positive 100 80 60 40 20 Percent subjects (%) Controls AUD Alcoholic hepatitis\n",
      "@@\n",
      "[\n",
      "0.50 France Mexico 0.25 Spain 0.00 UK { USA East PC2 (16.74%) USA Midwest 0.25 L 0000 USA West -0.4 -0.2 0.4 PC1 A%)\n",
      "\n",
      "@@ E. faecalis negative 3 E. faecalis positive 100 80 60 40 20 Percent subjects (%) P & « ¥\n",
      "@@\n",
      "3\n",
      "mm Cytolysin negative Bl Cytolysin positive 100 80 60 40 20 Percent subjects (%) Cirrhosis Non- cirrhosis\n",
      "mm\n",
      "Bl\n",
      "Extended Data Fig. 1 | See next page for caption.\n",
      "Enterococcus faecalis 150,000 100,000 - 50,000 - - —— — Relative abundance 1,000 0007 | | Cytolysin Cytolysin negative positive\n",
      "\n",
      "Enterococcus faecalis 150,000 100,000 8 < g Relative abundance _ I - s & & Sl & R & o\n",
      "\n",
      "@@ E. faecalis negative 3 E. faecalis positive 100 80 60 40 20 Percent subjects (%) Cirrhosis Non- cirrhosis\n",
      "@@\n",
      "3\n",
      "Em Cytolysin negative Cytolysin positive s [ o =2 o o Percent subjects (%) S 2 & & W & X\n",
      "Em\n",
      "Enterococcus faecalis\n",
      "80,000\n",
      "Relative abundance\n",
      "60,000\n",
      "\n",
      "40,000\n",
      "\n",
      "20,000\n",
      ")\n",
      "Cirrhosis\n",
      "\n",
      "Non- cirrhosis\n",
      "Article\n",
      "Extended Data Fig. 1 | Intestinal dysbiosis in patients with alcoholic hepatitis. a, 16S rRNA sequencing of faecal samples from controls (n = 14), patients with alcohol-use disorder (n = 43), or alcoholic hepatitis (n = 75). The graph demonstrates the relative abundance of sequence reads in each genus. b, Bacterial diversity (Shannon index and Simpson index) and richness (Chao richness) was calculated in controls (n = 14), patients with alcohol-use disorder (n = 43) or alcoholic hepatitis (n =\n",
      "75). c, E. faecalis in faecal samples from controls (n = 25), patients with alcohol-use disorder (n = 38) or alcoholic hepatitis (n = 82), assessed by qPCR. d, Percentage of faecal samples positive for E. faecalis in controls (n = 25), patients with alcohol-use disorder (n = 38) or alcoholic hepatitis (n = 82), assessed by qPCR. E. faecalis was detected in faeces from 80% of patients with alcoholic hepatitis, versus 36% of controls (P < 0.001). There was also a significant difference between\n",
      "patients with alcohol-use disorder and patients with alcoholic hepatitis (P < 0.01). e, Receiver operating characteristic curves and area under the curve (AUC) for the comparison of 90-day mortality and cytolysin positivity (red; n = 57), MELD score (blue; n = 56), ABIC score (yellow; n = 57) and discriminant function (green; n = 42) in patients with alcoholic hepatitis. f, E. faecalis in faecal samples from patients with alcoholic hepatitis whose faecal samples were cytolysin- positive (n = 25)\n",
      "or cytolysin-negative (n = 54), assessed by qPCR (P = 0.8174). g, 16S rRNA sequencing of faecal samples from patients with alcoholic hepatitis from different centres (France, n = 9; Mexico, n = 6; Spain, n = 5; UK, n = 11; USA (east), n = 16; USA (Midwest), n = 12; USA (west), n = 16 patients). We used PCoA based on Jaccard dissimilarity matrices to show β-diversity among groups at the genus level. The composition of faecal microbiota was significantly different between patients from different\n",
      "regions (P < 0.01). h, Percentage of faecal samples that were positive for cylLL and cylLS DNA sequences (cytolysin- positive), in patients with alcoholic hepatitis from different centres (France,\n",
      "n = 16; Mexico, n = 6; Spain, n = 6; UK, n = 10; USA (east), n = 16; USA (Midwest), n = 13; USA (west), n = 15 patients), assessed by qPCR (P = 0.6094). i, E. faecalis in faecal samples from patients with alcoholic hepatitis from different centres, assessed by qPCR (P = 0.5648). j, Percentage of faecal samples that were positive for E. faecalis in patients with alcoholic hepatitis from different centres (France, n = 16; Mexico, n = 6; Spain, n = 6; UK, n = 10; USA (east), n = 16; USA (Midwest),\n",
      "n = 13; USA (west), n = 15 patients), assessed by qPCR (P = 0.0529). k, Percentage of subjects with faecal samples that were positive for cylLL and cylLS DNA sequences (cytolysin-positive), in patients with alcoholic hepatitis and with (n = 30) or without (n = 18) cirrhosis, assessed by qPCR (P = 0.3431). l, E. faecalis in faecal samples from patients with alcoholic hepatitis and with (n = 30) or without (n = 18) cirrhosis, assessed by qPCR (P = 0.5736). m, Percentage of faecal samples that were\n",
      "positive for E. faecalis in patients with alcoholic hepatitis and with (n = 30) or without (n = 18) cirrhosis, assessed by qPCR (P = 0.2878). Results are expressed as mean ± s.e.m. (c, f, i, l). For the box and whisker plots in b, the box extends from the 25th to 75th percentiles, and the centre line represents the median; for all three groups, the bottom whiskers show the minimum values; for the control group (black), the top whisker shows the maximum value; for the other two groups, the top\n",
      "whiskers represent the 75th percentile plus 1.5× the inter-quartile distance (the distance between the 25th and 75th percentiles); all values greater than this are plotted as individual dots. P values were determined by Kruskal– Wallis test (i) with Dunn’s post hoc test (b, c), two-sided Fisher’s exact test (h, j, k, m) followed by FDR procedures (d), two-sided Mann–Whitney Wilcoxon rank-sum test (f, l) or PERMANOVA (g). The exact group size (n) and P values for each comparison are listed in\n",
      "Supplementary Table 10. *P < 0.05, **P < 0.01, ***P < 0.001, ****P < 0.0001.\n",
      "Serum ALT 400 k 300 200 100 o1 A o Control Ethanol\n",
      "\n",
      "b u Fold change oo H Fod Control Ethanol\n",
      "«\n",
      "« ~ E. faecalis Acytolysin —— E. faecalis 100 920 80 70 Log-rank test 60 (Mantel-Cox) Percent survival (%) P<0.05 50 10 Days\n",
      "~\n",
      "\n",
      "——\n",
      "Fecal albumin\n",
      "\n",
      "Fecal albumin 400 30 - l ~ 200 10 [ [a] il Control Ethanol\n",
      "Serum ethanol 1,000 [T} 800 ] S 600 g —— 400 200 Ethanol\n",
      "Extended Data Fig. 2 | See next page for caption.\n",
      ") Control 3 Y Ethanol PBS E. faecalis E. faecalis Acytolysin\n",
      "\n",
      "Hepatic triglycerides PBS 40 W E. faecalis u B E. faecalis Acytolysin 30 L 20 10 oo [ =] i i Control Ethanol\n",
      "Cxcl1 — Fold change oo e 18 [ o Control Ethanol\n",
      "\n",
      "Liver cyLg 100 k k k, 80 60 40 Percent mice (%) 20 nd nd nd Control Ethanol\n",
      "\n",
      "Serum LPS 60 k —— 40 [N (ng/ml) 20 Qo l 2 [on] Control Ethanol\n",
      "\n",
      "Adht & S . P oo 1] 5 Fold change i S Control Ethanol\n",
      "\n",
      "Cxcl2 x 10 L] Fold change [al Control Ethanol\n",
      "Liver E. faecalis 100 © = = S = Percent mice (%) = nd Control Ethanol\n",
      "\n",
      "Liver Enterococcus\n",
      "1,500\n",
      "L]\n",
      "1,000\n",
      "\n",
      "500\n",
      "Ethanol\n",
      "\n",
      "0.50 PBS E. faecalis Control 0.25 ' E. faecalis Acytolysin PBS B E. faecalis Ethanol 0.00 ‘& B E. faecalis Acytolysin -0.25: -0.4 0.4 PC1 .)\n",
      "\n",
      "Cyp2e1 25 2.0 - o - = Fold change 1F 0.0 Control Ethanol\n",
      "Fecal Enterococcus 1010 108 1 104 102 24 48 72 Time (hrs)\n",
      "Article\n",
      "Extended Data Fig. 2 | Cytolytic E. faecalis causes the progression of ethanol-induced liver disease in mice. a–n, C57BL/6 mice were fed oral isocaloric (control) or chronic–binge ethanol diets and gavaged with vehicle (PBS), a cytolytic E. faecalis strain (FA2-2(pAM714)) (denoted E. faecalis) (5 × 108 CFUs) or a non-cytolytic E. faecalis strain (FA2-2(pAM771))5 (denoted E. faecalis Δcytolysin) (5 × 108 CFUs) every third day. a, Serum levels of ALT. b, Hepatic triglyceride content. c,\n",
      "Representative oil red O-stained liver sections. d–f, Hepatic levels of mRNAs. g, Kaplan–Meier curve of survival of mice on chronic–binge ethanol diets (day 0 denotes the start of ethanol feeding). Mice gavaged with PBS all survived, and are not included in the figure. A higher proportion of mice (n = 15) gavaged with non-cytolytic E. faecalis survived than did mice (n = 25) gavaged with cytolytic E. faecalis. h, Proportions of mice that were positive for cytolysin in the liver, measured by qPCR\n",
      "for cylLS (the gene that encodes cytolysin subunit CylLS′′). i, Proportions of mice that were positive for E. faecalis in the liver, measured by qPCR. About 80% of mice colonized with cytolytic E. faecalis, as well as those colonized with non- cytolytic E. faecalis, were positive for E. faecalis in their livers. j, Liver CFUs of Enterococcus in mice on a chronic–binge ethanol diet. k, Paracellular intestinal permeability was evaluated by measuring faecal albumin content and serum levels of\n",
      "lipopolysaccharide (LPS) by enzyme-linked immunosorbent assays. l, Faecal samples were collected and 16S rRNA genes were sequenced. PCoA based on Jaccard dissimilarity matrices showed no significant differences among mice gavaged with PBS, cytolytic or non-cytolytic E. faecalis following\n",
      "feeding with the control and ethanol diets. Compared to mice fed with a control diet, mice fed with an ethanol diet had significantly different faecal microbiomes after gavaging with E. faecalis (P < 0.05). m, n, Serum levels of ethanol and hepatic levels of Adh1 and Cyp2e1 mRNAs did not differ significantly among mice gavaged with PBS, cytolytic or non-cytolytic E. faecalis after ethanol feeding. o, Mice were gavaged with cytolytic or non- cytolytic E. faecalis strains (carrying the\n",
      "erythromycin resistance gene; 5 × 108 CFUs) at time 0, and faeces were collected 0, 8, 24, 48 and 72 h later. Faecal CFUs of Enterococcus were determined by culturing faecal samples on BBL enterococcosel broth agar plate with 50 μg ml-1 erythromycin. At time 0 and 72 h, five out of five and four out of five mice, respectively, had no detectable erythromycin-resistant Enterococcus in their faeces. These points are not shown on the graph, but have been included in the calculation of mean ± s.e.m.\n",
      "Scale bar, 100 μm. Results are expressed as mean ± s.e.m. (a, b, d–f, j, k, m–o). P values among groups of mice fed with the control or ethanol diet were determined by one-way ANOVA with Tukey’s post hoc test (a, b, d–f, j, k, m, n), two-sided log-rank (Mantel–Cox) test (g), two-sided Fisher’s exact test followed by FDR procedures (h, i) or PERMANOVA followed by FDR procedures (l). All results were generated from at least three independent replicates. The exact group size (n) and P values for\n",
      "each comparison are listed in Supplementary Table 10. P values between mice fed with a control diet and mice fed with an ethanol diet were determined by two-way ANOVA (k). *P < 0.05, **P < 0.01, ***P < 0.001, ****P < 0.0001.\n",
      "TUNEL AH cytolysin positive #1 r AH cytolysin positive #2 AH cytolysin negative #3 [N AH cytolysin negative #4 Percent positive (%) i il l 1l Control Ethanol\n",
      "\n",
      "R Control . 3 /3% ¥ Ethanol F R o Ve, AH cytolysin AH cytolysin AH cytolysin AH cytolysin positive #1 positive #2 negative #3 negative #4\n",
      "\n",
      "Cxcl2 Fold change - o oo 3 i Control Ethanol\n",
      "\n",
      "Acta2 10 Fold change Control Ethanol\n",
      "AH Cytolysin negative AH Cytolysin positive \" 0 90; 80; 70 Log-rank test 60; (Mantel-Cox) Percent survival (%) P<0.05 50 2 10 Days\n",
      "\n",
      "AH cytolysin positive #1 AH cytolysin positive #2 Control | AH cytolysin negative #3 ] AH cytolysin negative #4 H PC2 (24.13%) & 05 '..\n",
      "|\n",
      "\n",
      "]\n",
      "0.4- AH cytolysin pos ive #1 AH cytolysin posi #2 I Ethanol AH cytolysin negative #3 AH cytolysin negative #4 g [mmm|m i PC2 (22.82%) 1 A S | | y/ y 0. 02 0. Be (51%)\n",
      "\n",
      "Cytolysin positive E. faecalis a 8 3 2 8 8 Percent colonies (%) nd nd o Patient Patient Patient Patient #1 # #3 # AH ¢y lysin AH cytolysin po: negative\n",
      "Serum ethanol 1,000 800 600 400 - 200 Ethanol\n",
      "Extended Data Fig. 3 | Transplantation of cytolysin-positive faeces increases ethanol-induced liver disease in gnotobiotic mice. a–f, h, i, C57BL/6 germ- free mice were colonized with faeces from two cytolysin-positive and two cytolysin-negative patients with alcoholic hepatitis, and then fed isocaloric (control) or chronic–binge ethanol diets. a, Percentage of TUNEL-positive hepatic cells. b, Representative oil red O-stained liver sections. c, d, Hepatic levels of mRNAs that encode the\n",
      "inflammatory cytokine Cxcl2 and Acta2 (a marker of activated hepatic stellate cells). e, Kaplan–Meier curve of survival of mice on chronic–binge ethanol diets (day 0 denotes the start of ethanol feeding), gavaged with faeces from cytolysin-positive (n = 48 mice) or cytolysin-negative (n = 32 mice) patients with alcoholic hepatitis. f, Faecal samples were collected and 16S rRNA genes were sequenced. The graph shows PCoA of faecal microbiomes. No significant difference was observed between mice\n",
      "colonized with faeces from cytolysin-positive or cytolysin-negative donors with alcoholic hepatitis, following the control diet. Mice transplanted with faeces from a cytolysin-positive patient with alcoholic hepatitis (patient\n",
      "Ad1 Cyp2et 2.5 2.0 2.0 - 1.5 o 15 e o - % . 1.0{ H - 1.0 H Fold change Fold change 0.5: = W 0.5: 0.0 il 0.0 Control Ethanol Control Ethanol\n",
      "no. 2) showed a microbiota that was significantly different to that of the other mouse groups following ethanol administration (P < 0.01). g, Percentage of cytolysin-positive E. faecalis in four patients with alcoholic hepatitis. Stool samples from the four patients were placed on plates with selective medium, and Enterococcus colonies were identified by the production of a dark brown or black colour. Enterococcus colonies were confirmed to be E. faecalis by qPCR. The cytolysin status of each E.\n",
      "faecalis colony was determined by qPCR. h, Serum levels of ethanol were comparable among colonized mice after ethanol feeding. i, Hepatic levels of Adh1 and Cyp2e1 mRNAs did not differ significantly among colonized mice on control or ethanol diets. Scale bar, 100 μm. Results are expressed as mean ± s.e.m. (a, c, d, h, i). P values were determined by one-way ANOVA with Tukey’s post hoc test (a, c, d, h, i), two- sided log-rank (Mantel–Cox) test (e) or PERMANOVA followed by FDR procedures (f). All\n",
      "results were generated from at least three independent replicates. The exact group size (n) and P values for each comparison are listed in Supplementary Table 10. *P < 0.05, **P < 0.01, ***P < 0.001.\n",
      "Article\n",
      "a Head s v % | . 2 | % o . Tail Siphophage Myophage Podophage (long, flexible, (contractile tail) (short, noncontractile tail) noncontractile tail) S5 or ¥ @ ¥ % Efmus1 Efmus2 Efmus3 Efm us4 Efmus1 (17,927 bp) y b Efmus2 (18,366 bp) Efmus3 (18,286 bp) T SN Efmus4 (18,186 bp) D ol | encapsidation/terminase [ tail fiber DNA polymerase | [l endolysin/lysin | I hypothetical protein - major capsid [ tail components [ DNA binding intron/intein [ holin Il ortal\n",
      "Head\n",
      "\n",
      "Tail\n",
      "major capsid\n",
      "[\n",
      "\n",
      "[\n",
      "\n",
      "[\n",
      "Il\n",
      "ortal\n",
      "Extended Data Fig. 4 | Isolation and amplification of phages against cytolytic E. faecalis isolated from mice. a, BHI agar plate showing phage plaque morphology. The phage cocktail (100 μl) (102–103 PFUs) was mixed with overnight-grown E. faecalis culture (100 μl) and then added to BHI broth top agar (0.5% agar) and poured over a BHI plate (1.5% agar). After overnight growth at 37 °C, images were captured on an Epson Perfection 4990 Photo scanner. b, Simplified illustration of the morphologies\n",
      "of different phages. Siphophages have long, flexible noncontractile tails (left); myophages have contractile tails\n",
      "(middle); and podophages have short noncontractile tails (right). c, Transmission electron microscopy revealed that phages we isolated were all podophages (Efmus1, Efmus2, Efmus3 and Efmus4). d, Genetic map of phage genomes. The linear maps are based on nucleotide sequences of the phage genomes and predicted open reading frames. The name and length (in bp) of each genome are indicated to the left of each phage map. Protein-coding sequences are coloured on the basis of functional role categories.\n",
      "Scale bar, 50 nm. All results were generated from at least three independent replicates.\n",
      "Serum ALT 500 — 400 ~300 200 - 100 ll WT II WT Atpas Control Ethanol\n",
      "\n",
      "Hepatic triglycerides PBS C. crescentus phages 60 E. faecalis phages PBS C. crescentus phages EEEE E. faecalis phages '. i WT WT I/I Control Ethanol\n",
      "\n",
      "S =y 5 Control \" | ey 3 - 3 Ethanol > R - 2 e PBS C. crescentus phages E. faecalis phages PBS C. crescentus phages E. faecalis phages WT Atp4ass\n",
      "\n",
      "Cxcl1 15 es — e, 10 .- \"~ Fold change - o M | i WT WT /I Control Ethanol\n",
      "Cxcl2 pes 3 — i - o - =3 Fold change ] . H - _ B L & WT I/ WT Control Ethanol\n",
      "\n",
      "Liver cylLg 100 — s i 8 3 ] Percent mice (%) 8 2 nd o 3 /I WT / Control Ethanol\n",
      "\n",
      "b S — e - o - o Fold change 5 SUs WT Atpda WT A Control Ethanol\n",
      "\n",
      "Fecal Enterococcus 1010 10° 2108 108 10% il | 104 WT / WT / Control Ethanol\n",
      "\n",
      "PBS a C. crescentus phages o8 E. faecalis phages o o & > Control PBS (10.08%) ] C. crescentus phages Atpdassi PC2 E. faecalis phages -1.0 4 17.1) 0\n",
      "Serum ethanol\n",
      "Adh1\n",
      "800\n",
      "\n",
      "600\n",
      "\n",
      "400\n",
      "20\n",
      "o S il i\n",
      "Fold change\n",
      "H\n",
      "I I\n",
      "\n",
      "|\n",
      "\n",
      ":\n",
      "WT\n",
      "II\n",
      "WT\n",
      "\n",
      "WT\n",
      "/s\n",
      "Ethanol\n",
      "Control\n",
      "Ethanol\n",
      "Extended Data Fig. 5 | See next page for caption.\n",
      "2 PBS C. crescentus phages WT 5 E. faecalis phages Ethanol PBS g (17.92%) C. crescentus phages Atpdasus PC2 E. faecalis phages B 0.4 0 5 PC1 'z.\n",
      "\n",
      "Cyp2e1 S = - S P = 1.0 B Fold change 0.5 il I i 0.0 WT wT I I/I Control Ethanol\n",
      "Article\n",
      "Extended Data Fig. 5 | Phages reduce translocation of cytolysin to the liver and reduce ethanol-induced liver disease in Atp4aSl/Sl mice. a–k, Wild-type (WT) and Atp4aSl/Sl littermates were fed oral isocaloric (control) or chronic– binge ethanol diets, and gavaged with vehicle (PBS), control phages against C. crescentus (1010 PFUs) or a cocktail of four different phages that target cytolytic E. faecalis (1010 PFUs), 1 day before an ethanol binge. a, Serum levels of ALT. b, Hepatic triglyceride\n",
      "content. c, Representative oil red O-stained liver sections. d–f, Hepatic levels of mRNAs. g, Proportions of mice that were positive for cytolysin in the liver, measured by qPCR for cylLS. h, Faecal CFUs of Enterococcus. i, Faecal samples were collected and 16S rRNA genes were sequenced. PCoA based on Jaccard dissimilarity matrices found no significant\n",
      "difference in faecal microbiota among mice given PBS, control phage or phages that target cytolytic E. faecalis in each group. j, k, Serum levels of ethanol and hepatic levels of Adh1 and Cyp2e1 mRNAs did not differ significantly among colonized mice after ethanol feeding. Scale bar, 100 μm. Results are expressed as mean ± s.e.m. (a, b, d–f, h, j, k). P values were determined by two-way ANOVA with Tukey’s post hoc test (a, b, d–f, h, j, k), two-sided Fisher’s exact test followed by FDR\n",
      "procedures (g) or PERMANOVA followed by FDR procedures (i). All results were generated from at least three independent replicates. The exact group size (n) and P values for each comparison are listed in Supplementary Table 10. *P < 0.05, **P < 0.01, ***P < 0.001, ****P < 0.0001.\n",
      "<. 2. - 8s °s - Ef5 Ef2\n",
      "Ef2.1\n",
      "Ef2.3\n",
      "Ef5.1 F)- ) ) o l ) ) o (41,141 bp) Ef5.2 (41,418 bp) ) b o s i D Ef5.3 F)- b ) o ol e ) ) (39,115 bp) Ef5.4 (41,141 bp) W) e om Ef2.1 (140,938 bp) PP o ) ) s o ) ) o D DO O OO 0 0 ¢ Ef2.2 ( W W (58,400 bp) L Ef2.3 i oo ) ) o (147,289 bp) ) ) m) m) o m m Capsid structural components Tail and baseplate and assembly assembly [ encapsidation/terminase [ tail fibers - DNA polymerase - endolysin/lysin - hypothetical protein (] major capsid - tail component [ DNA binding I holin intron/intein I rortal\n",
      "[ tail measure DNA recombination | regulatory [ protease (scaffold protein) I salvage of [ transcription factor nucleosides and Il other nucleotides | other known functions ] DNA restriction/ modification e tRNA || nucleotide degradation\n",
      "Extended Data Fig. 6 | Isolation and amplification of phages against cytolytic E. faecalis strains isolated from patients with alcoholic hepatitis. a, BHI agar plates showing phage plaque morphology. b, Transmission electron microscopy graphs of myophages Ef2.1 and Ef2.3, stained with phosphotungstic acid showing contracted tails. c, Genetic map of phage genomes. The linear maps are based on nucleotide sequences of the phage\n",
      "genomes and predicted open reading frames. The name and length (in bp) of each genome are indicated to the left of each phage map. Protein-coding sequences are coloured on the basis of functional role categories. Sequences that encode tRNA genes are indicated by a cloverleaf structure. Scale bar, 50 nm. All results were generated from at least three independent replicates.\n",
      "Article\n",
      "\n",
      "a\n",
      "TUNEL C. crescentus phages Cytolysin-positive E. faecalis phages K k C. crescentus phages EE Cytolysin-positive E. faecalis phages Percent positive (%) Patient Patient Patient Patient #5 #2 #5 #2 Control Ethanol\n",
      "\n",
      "o e Control e A o ] Ethanol C. crescentus phages Cytolysin-positive C. crescentus phages ylin- ive E. faecalis phages E. faecalis phages Patient #5 Patient #2\n",
      "Patient #2\n",
      "\n",
      "Cxcl2 [T \" Fold change o H al Patient Patient Patient Patient #5 #2 #5 #2 Control Ethanol\n",
      "Acta2 10 I Fold change y B o H ¥ Patient Patient Patient Patient #5 #2 #5 #2 Control Ethanol\n",
      "\n",
      "Fecal Enterococcus 1010 10° 2108 O 107 108 108 | Patient Patient Patient Patient #5 #2 #5 #2 Control Ethanol\n",
      "\n",
      "Control C. crescentus phages 5 Cytolysin-positive E. faecalis phages | Patient #5 =) o C. crescentus phages Cytolysin-positive E. faecalis phages I Patient #2 =] o PC2 (7.13%) -0.5 -1.0 -0.5 c (80.98%) 0\n",
      "Ethanol C. crescentus phages 0.25 Cytolysin-positive E. faecalis phages Patient #5 C. crescentus phages Patient #2 LILILIL Cytolysin-positive E. faecalis phages PC2 (16.3%) -0.25 0.25 1.)\n",
      "\n",
      "Serum ethanol 800 60 = 400 20 Patient Patient #5 #2 Ethanol\n",
      "\n",
      "Adh1 2.0 - o oo o - ] Fold change 0.5 . 0.0 Patient Patient Patient Patient #5 #2 #5 #2 Control Ethanol\n",
      "Extended Data Fig. 7 | Phages that target cytolytic E. faecalis reduce ethanol-induced liver disease in gnotobiotic mice. a–h, C57BL/6 germ-free mice were colonized with faeces from two cytolysin-positive patients with alcoholic hepatitis (faeces from one patient were also used in Fig. 2). The mice were then fed oral isocaloric (control) or chronic–binge ethanol diets, and gavaged with control phages against C. crescentus (1010 PFUs) or a cocktail of 3 or 4 different phages that target cytolytic\n",
      "E. faecalis (1010 PFUs), one day before an ethanol binge. a, Percentage of TUNEL-positive hepatic cells. b, Representative oil red O-stained liver sections. c, d, Hepatic levels of mRNAs that encode the inflammatory cytokine Cxcl2, and Acta2 (a marker of activated hepatic stellate cells). e, Faecal CFUs of Enterococcus. f, Faecal samples were\n",
      "Cyp2e1 s = o Oo T - = o Fold change 0.5 | 0.0 Patient Patient Patient Patient #5 #2 #5 #2 Control Ethanol\n",
      "collected and 16S rRNA genes were sequenced. PCoA based on Jaccard dissimilarity matrices shows no significant differences in the faecal microbiota of mice gavaged with control phage and phages that target cytolytic E. faecalis in each group. g, h, Serum levels of ethanol and hepatic levels of Adh1 and Cyp2e1 mRNAs did not differ significantly among colonized mice after ethanol feeding. Scale bar, 100 μm. Results are expressed as mean ± s.e.m. (a, c–e, g, h). P values were determined by two-way\n",
      "ANOVA with Tukey’s post hoc test (a, c–e, g, h) or PERMANOVA followed by FDR procedures (f). All results were generated from at least three independent replicates. The exact group size (n) and P values for each comparison are listed in Supplementary Table 10. *P < 0.05, ***P < 0.001.\n",
      "%, H % % 84 S X %3 s y 3 3 g H £ S & X3 2 & F o %, 8% , o o Yo, s, 5 2. & e, & B, By ® N ) S <1 G %, A 2. g5eC f54 PhiM1E, fal Ef6 VB-EfaM-£f23 vB-EfaSEf PhiEF17H. PMBT2 B-EfaM-Ef2.1 -vB-EfaS-Ef5.2 iEF24C P2 Si r , EfaP_igg KA s A S S, 103 & 253 o o, 62 ¥ & S, S, - kN N & %, G [ Siphoviridae P ne 5 % & § N N g3 & N % Podoviridae k) Ef7 B Myoviridae Tree scale: 1\n",
      "Ef6.1 ) b e o 4 (40,429 bp) Ef6.2 (17,966 bp) ) o Ef6.3 ) o (18,136 bp) Ef6.4 s ) o (41,133 bp) Ef7.1 D ) | (58,018 bp) - Ef7.2 1 ) o s (18,737 bp) Ef7.3 oo (18,818 bp) Ef7.4 (18,415 bp) C apsid structural components Other N and assembly || encapsidation/terminase [ tail fibers - DNA polymerase [ endolysin/ysin | [ hypothetical protein - tail component I intron/intein major capsid [ DNA binding [ holin I portal [ tail measure [ regulatory DNA recombination [l protease (scaffold protein) I\n",
      "salvage of RNA l other nucleosides and nucleotides [ DNA restriction/ modification (| nucleotide degradation\n",
      "Extended Data Fig. 8 | See next page for caption.\n",
      "Article\n",
      "Extended Data Fig. 8 | Isolation and amplification of phages against non- cytolytic E. faecalis strains isolated from patients with alcoholic hepatitis. a, BHI agar plates showing phage plaque morphology. b, Genetic map of phage genomes. The linear maps are based on nucleotide sequences of the phage genomes and predicted open reading frames. The name and length (in bp) of each genome are indicated to the left of each phage map. Protein-coding sequences are coloured on the basis of functional\n",
      "role categories. Sequences that encode tRNA genes are indicated by a cloverleaf structure. c, Phylogenetic tree of Enterococcus phages. A whole-genome average nucleotide distance tree was constructed for 73 available Enterococcus phage genomes: 54 of these were\n",
      "from GenBank (denoted by black letters) and 19 were from this study (4 phages against cytolysin-positive E. faecalis isolated from mice (shown in blue letters); 7 phages against cytolysin-positive E. faecalis isolated from patients with alcoholic hepatitis (shown in pink letters); and 8 phages against cytolysin- negative E. faecalis isolated from patients with alcoholic hepatitis (shown in green letters)) with Mash43 using a sketch size of s = 5000 and a k-mer size of k = 12 and GGRaSP59\n",
      "(Methods). Coloured branches denote specific phage genera or subfamily: Sap6virus, P68virus and Spounavirinae. The scale bar represents per cent average nucleotide divergence. All results were generated from at least three independent replicates.\n",
      "TUNEL C. crescentus phages Cytolysin-negative E. faecalis phages C. crescentus phages L B | Cytolysin-negative E. faecalis phages Percent positive (%) | Patient Patient Patient Patient #6 #7 #6 #7 Control Ethanol\n",
      "|\n",
      "o o Control 2 . “ . L ‘ ok - -4 & l Ethanol 3 4 C. crescentus phages lyioiv C. crescentus phages lyin- ve E. faecalis phages E. faecalis phages Patient #6 Patient #7\n",
      "\n",
      "Cxcl2 L] L oo Fold change 0 ] i Patient Patient Patient Patient #6 #7 #6 #7 Control Ethanol\n",
      "\n",
      "Acta2 Ll Fold change T l ] g i Patient Patient Patient Patient #6 H#7 #6 H7 Control Ethanol\n",
      "\n",
      "Liver cylLg 100 =3 =3 o o B o Percent mice (%) =3 nd nd nd nd nd nd nd nd [ Patient Patient Patient Patient #6 #7 #6 #7 Control Ethanol\n",
      "Control\n",
      "Ethanol\n",
      "Control 1.0 C. crescentus phages g Cytolysin-positive E. faecalis phages | Patient #6 2 C. crescentus phages Cytolysin-positive E. faecalis phages | Patient #7 g ] PC2 (19.54%) -0.5 -1.0 -1.0 1.0 PC1\n",
      "\n",
      "Ethanol . - C. crescentus phages Cytolysin-positive E. faecalis phages Patient #6 0. C. crescentus phages u|| Cytolysin-positive E. faecalis phages Patient #7 0.0 PC2 (15.2%) -0.3 -0.2 0.2 0.50 (.)\n",
      "\n",
      "Serum ethanol 1,000 800 (1] 600 I (mg/dl) 400 200 Patient Patient #6 #7 Ethanol\n",
      "Adh1 25 20 - LT - Fold change & S Patient Patient Patient Patient #6 #7 #6 #7 Control Ethanol\n",
      "Extended Data Fig. 9 | Phages that target non-cytolytic E. faecalis do not reduce ethanol-induced liver disease in gnotobiotic mice. a–h, C57BL/6 germ-free mice were colonized with faeces from two cytolysin-negative patients with alcoholic hepatitis. Transplanted gnotobiotic mice were fed oral isocaloric (control) or chronic–binge ethanol diets and gavaged with control phages against C. crescentus (1010 PFUs) or a cocktail of four different phages targeting non-cytolytic E. faecalis (1010 PFUs),\n",
      "1 day before an ethanol binge. a, Percentage of TUNEL-positive hepatic cells. b, Representative oil red O-stained liver sections. c, d, Hepatic levels of mRNAs that encode the inflammatory cytokine Cxcl2, and Acta2 (a marker of activated hepatic stellate cells). e, Proportions of mice that were positive for cytolysin in the liver, measured by qPCR for cylLS. f, Faecal samples were collected and 16S rRNA\n",
      "Cyp2e1 25 2.0 O - e - o Fold change = o Patient Patient Patient Patient #6 #7 #6 #7 Control Ethanol\n",
      "genes were sequenced. PCoA based on Jaccard dissimilarity matrices found no significant difference in faecal microbiota among mice gavaged with control phages and phages that target cytolytic E. faecalis in each group. g, h, Serum levels of ethanol and hepatic levels of Adh1 and Cyp2e1 mRNAs did not differ significantly among colonized mice after ethanol feeding. Scale bar, 100 μm. Results are expressed as mean ± s.e.m. (a, c, d, g, h). P values were determined by two-way ANOVA with Tukey’s post\n",
      "hoc test (a, c, d, g, h), two-sided Fisher’s exact test followed by FDR procedures (e) or PERMANOVA followed by FDR procedures (f). All results were generated from at least three independent replicates. The exact group size (n) and P values for each comparison are listed in Supplementary Table 10.\n",
      "natureresearch\n",
      "Reporting Summary\n",
      "Nature Research wishes to improve the reproducibility of the work that we publish. This form provides structure for consistency and transparency in reporting. For further information on Nature Research policies, see Authors & Referees and the Editorial Policy Checklist.\n",
      "Statistics\n",
      "For all statistical analyses, confirm that the following items are present in the figure legend, table legend, main text, or Methods section. n/a Confirmed The exact sample size (n) for each experimental group/condition, given as a discrete number and unit of measurement A statement on whether measurements were taken from distinct samples or whether the same sample was measured repeatedly The statistical test(s) used AND whether they are one- or two-sided Only common tests should be described\n",
      "solely by name; describe more complex techniques in the Methods section. A description of all covariates tested A description of any assumptions or corrections, such as tests of normality and adjustment for multiple comparisons A full description of the statistical parameters including central tendency (e.g. means) or other basic estimates (e.g. regression coefficient) AND variation (e.g. standard deviation) or associated estimates of uncertainty (e.g. confidence intervals) For null hypothesis\n",
      "testing, the test statistic (e.g. F, t, r) with confidence intervals, effect sizes, degrees of freedom and P value noted Give P values as exact values whenever suitable. For Bayesian analysis, information on the choice of priors and Markov chain Monte Carlo settings For hierarchical and complex designs, identification of the appropriate level for tests and full reporting of outcomes Estimates of effect sizes (e.g. Cohen's d, Pearson's r), indicating how they were calculated\n",
      "Our web collection on statistics for biologists contains articles on many of the points above.\n",
      "Software and code\n",
      "\n",
      "Policy information about availability of computer code\n",
      "Data collection All biochemical assays were measured using SoftMax Pro 7.0.3; gPCRs were run with StepOnePlus real-time PCR system; Liver histological pictures were taken with DP Controller and DP Manager (Olympus); Phage electronic microscopy pictures were taken using Maxim DL5; Plates were scanned using EPSON 4990 Photo; All pictures were viewed using Image\n",
      "Data analysis\n",
      "Bacteriophage sequencing and phage tree: Albacore v2.3.4 (ONT), Porechop v0.2.3, Unicycler v0.4.7 pipeline, Pilon v1.22, CLC Genomics Workbench 4.9, NCBI Prokaryotic Genome Annotation Pipeline, in-house PERL script using Xfig, Phage_Finder, MASH program, GGRaSP and APE R-package, iTOL tree viewer 16S sequencing: MOTHUR-based 16S rDNA analysis workflow E. faecalis genome sequencing and tree: abricate v0.8.10 , Prokka, Roary, RAXML, iTOL Statistical analyses: R\n",
      "statistical software 3.5.1, GraphPad Prism v6.01\n",
      "For manuscripts utilizing custom algorithms or software that are central to the research but not yet described in published literature, software must be made available to editors/reviewers. We strongly encourage code deposition in a community repository (e.g. GitHub). See the Nature Research guidelines for submitting code & software for further information.\n",
      "Data\n",
      "\n",
      "Policy information about availability of data\n",
      "All manuscripts must include a data availability statement. This statement should provide the following information, where applicable:\n",
      "\n",
      "- Accession codes, unique identifiers, or web links for publicly available datasets\n",
      "- A list of figures that have associated raw data\n",
      "- A description of any restrictions on data availability\n",
      "Raw 16S sequence reads can be found in the NCBI SRA associated with Bioproject PRJNA525701. Bacteriophage raw sequence reads and annotated genomes are\n",
      "Corresponding author(s): Bernd Schnabl\n",
      "Last updated by author(s): Sep 5, 2019\n",
      "n a t u r e r e s e a r c h | r e p o r t i n g s u m m a r y O c t o b e r 2 0 1 8 1\n",
      "n a t u r e r e s e a r c h | r e p o r t i n g s u m m a r y O c t o b e r 2 0 1 8 2\n",
      "available at NCBI under the following consecutive BioSample IDs (SAMN11089809 – SAMN11089827). Genome sequence data of E. faecalis strains isolated in this study were registered at ENA under Study PRJEB25007.\n",
      "Field-specific reporting\n",
      "Please select the one below that is the best fit for your research. If you are not sure, read the appropriate sections before making your selection. Life sciences Behavioural & social sciences Ecological, evolutionary & environmental sciences For a reference copy of the document with all sections, see nature.com/documents/nr-reporting-summary-flat.pdf\n",
      "Life sciences study design\n",
      "All studies must disclose on these points even when the disclosure is negative.\n",
      "Sample size No power analyses or other calculations were used to predetermine sample sizes. Sample sizes were chosen based on prior literature using similar experimental paradigms (Nat Commun. 2017;8:2137; Gut. 2019;68:1504-1515) Data exclusions No data were excluded Replication In vivo experiments: more than two technical replicates (from different cohorts, on different dates), as well as biological replicates were performed to ensure data reproducibility; In vitro experiments: three\n",
      "independent experiments and also replicates were performed on different dates to ensure data reproducibility. All replications were successful. Randomization Mice of similar age and weight were randomly assigned to experimental and control groups. Blinding The investigators were not blinded during cell and animal experiment assays.\n",
      "Reporting for specific materials, systems and methods\n",
      "\n",
      "We require information from authors about some types of materials, experimental systems and methods used in many studies. Here, indicate whether each material, system or method listed is relevant to your study. If you are not sure if a list item applies to your research, read the appropriate section before selecting a response.\n",
      "Methods n/a Involved in the study Antibodies ChIP-seq Eukaryotic cell lines Flow cytometry Palaeontology MRI-based neuroimaging Animals and other organisms Human research participants Clinical data\n",
      "Animals and other organisms\n",
      "Policy information about studies involving animals; ARRIVE guidelines recommended for reporting animal research Laboratory animals Female and male C57BL/6 mice (age, 9-12 weeks) (strain: wild type, Atpdasl/sl) Wild animals No wild animals were involved in the study. Field-collected samples No field-collected samples were involved in the study\n",
      "Ethics oversight All animal studies were reviewed and approved by the Institutional Animal Care and Use Committee of the University of California, San Diego.\n",
      "Note that full information on the approval of the study protocol must also be provided in the manuscript.\n",
      "Human research participants\n",
      "\n",
      "Policy information about studies involving human research participants\n",
      "Population characteristics\n",
      "Alcoholic hepatitis patients were from multiple centers from United States, Mexico and Europe, with the age ranged from 30 to\n",
      "Population characteristics 75. Alcohol use disorder patients and non-alcoholic controls were from United States and Europe, with the age ranged from 27 to 74. Both genders were included in all populations. Detailed descriptions in Methods, Extended Data Tables 1 and 2\n",
      "Recruitment\n",
      "Patients with alcohol use disorder fulfilling the DSM IV criteria (J Abnorm Psychol. 1997;106:545-553) were recruited from an alcohol withdrawal unit in San Diego, USA and Brussels, Belgium where they followed a detoxification and rehabilitation program. Alcoholic hepatitis patients were enrolled from the InTeam Consortium (ClinicalTrials.gov identifier number: NCT02075918) from centers in the USA, Mexico, United Kingdom, France and Spain. Detailed inclusion and\n",
      "exclusion criteria are listed in Methods\n",
      "Ethics oversight\n",
      "The protocol was approved by the Ethics Committee of Hopital Huriez (Lille, France), Universidad Autonoma de Nuevo Leon (Monterrey, México), Hospital Universitari Vall d'Hebron (Barcelona, Spain), King's College London (London, UK), Yale University (New Haven, USA), University of North Carolina at Chapel Hill (Chapel Hill, USA), Weill Cornell Medical College (New York, USA), Columbia University (New York, USA), University of Wisconsin (Madison, USA), VA San Diego\n",
      "Healthcare System (San Diego, USA) and Université Catholique de Louvain (Brussels, Belgium).\n",
      "Note that full information on the approval of the study protocol must also be provided in the manuscript.\n",
      "n a t u r e r e s e a r c h | r e p o r t i n g s u m m a r y O c t o b e r 2 0 1 8 3\n",
      "RES EARCH\n",
      "MICROBIOME\n",
      "Lactose drives Enterococcus expansion to promote graft-versus-host disease\n",
      "C. K. Stein-Thoeringer1,2,3, K. B. Nichols1,2, A. Lazrak1,2, M. D. Docampo1,2, A. E. Slingerland1,2, J. B. Slingerland1,2, A. G. Clurman4, G. Armijo1,2, A. L. C. Gomes1,2, Y. Shono1,2, A. Staffas1,2, M. Burgos da Silva1,2, S. M. Devlin5, K. A. Markey1,2,4, D. Bajic6, R. Pinedo7, A. Tsakmaklis8,9, E. R. Littmann1,10, A. Pastore1, Y. Taur11, S. Monette12, M. E. Arcila13, A. J. Pickard14, M. Maloy4, R. J. Wright1, L. A. Amoretti1, E. Fontana1, D. Pham15, M. A. Jamal15, D. Weber16, A. D. Sung17, D.\n",
      "Hashimoto18, C. Scheid8, J. B. Xavier19, J. A. Messina20, K. Romero21, M. Lew17, A. Bush17, L. Bohannon17, K. Hayasaka22, Y. Hasegawa18, M. J. G. T. Vehreschild8,9,23, J. R. Cross14, D. M. Ponce2,4, M. A. Perales2,4, S. A. Giralt2,4, R. R. Jenq15, T. Teshima18,22, E. Holler16, N. J. Chao17, E. G. Pamer1,2,10, J. U. Peled1,2,4*†, M. R. M. van den Brink1,2,4*†\n",
      "Disruption of intestinal microbial communities appears to underlie many human illnesses, but the mechanisms that promote this dysbiosis and its adverse consequences are poorly understood. In patients who received allogeneic hematopoietic cell transplantation (allo-HCT), we describe a high incidence of enterococcal expansion, which was associated with graft-versus-host disease (GVHD) and mortality. We found that Enterococcus also expands in the mouse gastrointestinal tract after allo-HCT and\n",
      "exacerbates disease severity in gnotobiotic models. Enterococcus growth is dependent on the disaccharide lactose, and dietary lactose depletion attenuates Enterococcus outgrowth and reduces the severity of GVHD in mice. Allo-HCT patients carrying lactose-nonabsorber genotypes showed compromised clearance of postantibiotic Enterococcus domination. We report lactose as a common nutrient that drives expansion of a commensal bacterium that exacerbates an intestinal and systemic inflammatory disease.\n",
      "community of mostly anaerobic bacteria, and a hallmark of microbial imbalance (dysbiosis) observed in many disease states involves the expansion of facul- tative anaerobic bacteria (1). Enterococci are facultative anaerobes that colonize the intes- tines of almost every species, from insects to mammals (2), and make up a very small proportion (<0.1%) of the gut microbiota in healthy humans (3). However, enterococci are also pathogens; the species Enterococcus faecium and Enterococcus faecalis\n",
      "are impor- tant causes of multidrug-resistant infections in patients (4). In single-center studies, E. faecium has been observed to dominate the fecal microbiota of immunocompromised patients after allogeneic hematopoietic cell transplantation (allo-HCT), a curative-intent therapy for hematological malignancies (5–7). Moreover, fecal domination with vancomycin- resistant enterococci increases the risk of\n",
      "bloodstream infection in allo-HCT patients (5, 8). Patients with severe graft-versus-host disease (GVHD) after allo-HCT have poor outcomes with only ~30% long-term survival (9). Gut microbiota perturbations caused by broad-spectrum antibiotics and a reduction in microbial diversity are associated with in- creased transplant-related mortality and lethal GVHD in humans and mice (10–13). Besides causing infections, experimental studies in gno- tobiotic mice have revealed that enterococci play an\n",
      "important role in colitis (14) by stimulat- ing antigen-presenting cells and CD4+RORg+ T cell infiltration, causing intestinal inflam- mation (15). In this study, we investigated the role of enterococci in the development of acute GVHD, both in allo-HCT patients and preclinical allo-HCT mouse models.\n",
      "We used 16S ribosomal RNA (rRNA) gene sequencing to study the fecal microbiota of 1325 adult allo-HCT recipients at four HCT centers: Memorial Sloan Kettering Cancer\n",
      "Center (MSKCC) (United States), Duke Uni- versity (United States), Hokkaido University (Japan), and University Hospital Regensburg (Germany). Patient characteristics are shown in table S1. We observed high abundance of enterococci soon after transplantation in samples from all four transplant centers (Fig. 1B and fig. S1B). We defined Enterococcus domination as relative genus abundance ≥0.3 (≥30%) in any fecal sample, following a thresh- old we have used previously (5) (materials and methods and\n",
      "fig. S2C). The incidence of domination rose comparably across cen- ters, with up to 65% of patients exhibiting a domination event after allo-HCT (Fig. 1A). E. faecium was the dominant species in both the MSKCC and the multicenter-validation cohort (Duke, Hokkaido, and Regensburg) (Fig. 1B, fig. S1, and table S2), where 40.1% of MSKCC patients (441 of 1101 patients) and 46.0% of multicenter-validation patients (103 of 224 patients) met criteria for domination at any time point between day −20 and\n",
      "day +80 relative to the date of allo-HCT, in which cells are infused on day 0.\n",
      "early posttransplant period (day 0 to +21) was associated with significantly reduced overall survival and increased GVHD-related mortality in both the MSKCC and multicenter- validation cohort, as well as an increased risk of moderate-to-severe acute GVHD in the MSKCC cohort (Fig. 1, C and D, fig. S2, A and B, and table S3). The risk of relapse or disease progression was not associated with enterococcal domination in either cohort. The association of domination by genus Enterococcus with clinical\n",
      "outcomes in the MSKCC cohort remained significant in a multivariate analysis adjusted for graft source, disease, conditioning intensity, gender, and age (table S4). In a subset of MSKCC patients, the vanA operon was found in 152 (37.4%) of 406 patients that had samples available for analysis, indicating the presence of vancomycin-resistant enterococci (VRE) (fig. S2E). Notably, expansions of several different taxa were detected in fecal samples in this study, but the Enterococcus genus was the\n",
      "one most commonly observed dominating the microbiota in all four transplant centers (fig. S3 and tables S5 and S6).\n",
      "1Department of Immunology, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 2Weill Cornell Medical College, New York, NY, USA. 3German Cancer Research Center (DKFZ), Heidelberg, Germany. 4Adult Bone Marrow Transplantation Service, Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 5Epidemiology and Biostatistics, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 6Department of Internal Medicine II, Technical University\n",
      "of Munich, Munich, Germany. 7Gnotobiotic Facility, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 8Department I of Internal Medicine, Center for Integrated Oncology Aachen Bonn Cologne Duesseldorf, University of Cologne, Cologne, Germany. 9German Center for Infection Research, Partner site Bonn-Cologne, Cologne, Germany. 10Department of Medicine, Section of Infectious Medicine and Global Health, University of Chicago, Chicago, IL, USA. 11Infectious Disease Service, Department of\n",
      "Medicine, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 12Laboratory of Comparative Pathology, Memorial Sloan Kettering Cancer Center, The Rockefeller University, Weill Cornell Medicine, New York, NY, USA. 13Diagnostic Molecular Pathology Laboratory, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 14Donald B. and Catherine C. Marron Cancer Metabolism Center, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 15Department of Genomic Medicine, The University of\n",
      "Texas MD Anderson Cancer Center, Houston, TX, USA. 16Internal Medicine III, University Clinic Regensburg, Regensburg, Germany. 17Division of Hematologic Malignancies and Cellular Therapy, Department of Medicine, Duke University Medical Center, Durham, NC, USA. 18Department of Hematology, Hokkaido University, Faculty of Medicine, Sapporo, Japan. 19Computational and Systems Biology Program, Memorial Sloan Kettering Cancer Center, New York, NY, USA. 20Division of Infectious Diseases, Department of\n",
      "Medicine, Duke University, Durham, NC, USA. 21Office of Clinical Research, Duke University School of Medicine, Durham, NC, USA. 22Division of Laboratory and Transfusion Medicine, Hokkaido University Hospital, Sapporo, Japan. 23Department of Internal Medicine, Infectious Diseases, Goethe University Frankfurt, University Hospital Frankfurt, Frankfurt am Main, Germany.\n",
      "*These authors contributed equally to this work.\n",
      "\n",
      "†Corresponding author. Email: peledj@mskcc.org (J.U.P.); vandenbm@mskcc.org (M.R.M.v.d.B.)\n",
      "\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019) 29 November 2019\n",
      "29 November 2019\n",
      "1 of 7\n",
      "\n",
      "D o w n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "RES EARCH | R E P O R T\n",
      "To further investigate these clinical obser- vations, we examined the fecal microbiota of mice early after transplantation using well- established mouse models of allo-HCT. In a major histocompatibility complex (MHC)– matched, minor-antigen-mismatched allo-HCT model [C57BL/6-to-129S1/Sv transplant (C57BL/ 6→129S1/Sv)], we performed 16S rRNA gene sequencing of fecal samples and found that E. faecalis dominated the fecal microbiota at posttransplant day +8 in mice who re- ceived T cell–replete\n",
      "grafts and developed lethal, acute GVHD (Fig. 2A and fig. S4A). In contrast to the patients who had pro- longed antibiotic exposures, this expansion of E. faecalis was independent of antibiotic administration and dependent upon GVHD, as it was not observed in control recipients of T cell–depleted allografts in which GVHD did\n",
      "not develop. This posttransplant expansion of enterococci was consistently found in two additional lethal GVHD models: C57BL/6→ BALB/c mice (MHC-disparate model after ir- radiation conditioning) (Fig. 2B) and LP/ J→C57BL/6 mice [MHC-matched, minor- antigen–mismatched after busulfan and cyclophosphamide conditioning (16)] (Fig. 2C). The expansion of enterococci in murine allo- HCT recipients with GVHD was accompanied by an increase in Enterococcus colony–forming units recovered from mesenteric\n",
      "lymph nodes, consistent with increased bacterial translo- cation (Fig. 2B).\n",
      "Although we observed E. faecium domina- tion in patients and a transient expansion of E. faecalis in GVHD mice, we hypothesized that both members of this genus might be associated with GVHD. Of note, E. faecium\n",
      "only recently became recognized as a major human pathogen; before the 1990s it was E. faecalis that caused >90% of clinical in- fections (17). Because E. faecalis expands in mice with GVHD and is the major Enterococcus species in laboratory mice, we next investigated whether E. faecalis contributes to GVHD. We colonized germ-free C57BL/6 mice with a com- munity of six bacterial strains (Akkermansia muciniphila, Lactobacillus johnsonii, Blautia producta, Bacteroides sartorii, Clostridium bolteae,\n",
      "and Parabacteroides diastonis; see materials and methods) (10, 18, 19) 21 days prior to allo-HCT (LP/J→gnotobiotic C57BL/6). One group of mice was cocolonized on day −21 with E. faecalis OG1RF, which remained detectable in mouse feces on days 0 and +7 (Fig. 2D, right panel, and fig. S4E). GVHD was exacerbated in E. faecalis–harboring mice\n",
      "A\n",
      "Incidence of domination\n",
      "0.75 n f o o i e t a c 0.50 n n i e m d o i c d n i s u e c v c i t o a 0.25 c l o u m r e t u n c E 0 20 0 -20 day relative to HCT\n",
      "\n",
      "Prevalence of domination MSKCC 0.6 Duke s e Hokkaido l p Regensburg m a s d 0.4 e t f o a n n i m o i t o c d a r s 0.2 f u c c o c o r e t n E 0 -20 20 0 day relative to HCT\n",
      "«\n",
      "\n",
      "=\n",
      "\n",
      "«\n",
      "B\n",
      "E. faecium other E. spp. 1.00 E. faecium 0.60 E. cecorum M S E. durans 0.30 K C E. mundtii C 0.10 e c E. rivorum n a d E. moraviensis 0.01 n u E. gallinarum b a v 1.00 a E. lactis e l v i i d t E. malodoratus a a 0.60 l t e i r o E. ureasiticus n 0.30 c E. faecalis o h o 0.10 r t 0.01 20 60 80 −20 0 60 40 20 40 0 −20 80 day relative to HCT\n",
      "®\n",
      "\n",
      "®\n",
      "\n",
      "®\n",
      "%\n",
      "Fig. 1. Enterococcus domination occurs globally and increases risk of GVHD and mortality after allo-HCT. Fecal microbiota were profiled using 16S rRNA gene sequencing of 1325 adult allo-HCT recipients. The patients attended one of four HCT centers in different countries: MSKCC (United States), Duke University (United States), Hokkaido University (Japan), and University Hospital Regensburg (Germany). (A) (Left) Cumulative incidence of patients who experienced at least one instance of genus\n",
      "Enterococcus domination of the gut microbiota [domination defined as a genus relative abundance of ≥0.3 (on a unitless scale from 0 to 1) over the course of allo-HCT (day −20 to +24 relative to HCT; using 7-day sliding windows) at different transplant centers]. (Right) Fraction of fecal specimens with enterococcal domination of the gut microbiota. (B) Relative abundance of different Enterococcus spp. in the microbiota of allo-HCT patients from the MSKCC and multicenter-validation cohort over the\n",
      "course of HCT, determined by 16S rRNA gene sequencing of fecal samples. Each\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019)\n",
      "\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019) 29 November 2019\n",
      "\n",
      "29 November 2019\n",
      "spp.\n",
      "C\n",
      "\n",
      "D\n",
      "Overall survival 1.0 0.8 g n i v i v 0.6 r u s n o i t 0.4 r o p o Enterococcus ≥ 0.3 (n=149) r p < 0.3 (n=389) Enterococcus 0.2 HR, 1.97; p < 0.001 0 15 10 5 0 20 months after HCT\n",
      "\n",
      "1.0 0.8 g n i v i 0.6 v r u s n o i 0.4 t r o p Enterococcus ≥ 0.3 (n=54) o r p Enterococcus < 0.3 (n=113) 0.2 HR, 1.95; p = 0.03 0 0 5 15 20 10 months after HCT\n",
      "GVHD-related mortality\n",
      "Enterococcus ≥ 0.3 (n=148)\n",
      "—\n",
      "Enterococcus < 0.3 (n=389)\n",
      "---\n",
      "0.2\n",
      "\n",
      "HR, 2.04; p = 0.01\n",
      "e c n e d i c n i e v i t a l u m u c\n",
      "\n",
      "4\n",
      "0.15\n",
      "\n",
      "0.1\n",
      "4\n",
      "0.05\n",
      "\n",
      "0\n",
      "o1\n",
      "10\n",
      "\n",
      "5\n",
      "\n",
      "15\n",
      "\n",
      "20\n",
      "months after HCT\n",
      "GVHD-related mortality Enterococcus ≥ 0.3 (n=53) < 0.3 (n=111) Enterococcus 0.2 HR, 5.8; p = 0.002 e c n e 0.15 d i c n i e 0.1 v i t a l u m 0.05 u c 0 20 5 15 10 0\n",
      "point represents a fecal sample, and color indicates the different Enterococcus spp.; the red dotted line indicates the threshold for domination set at a relative abundance ≥0.3. (C) Overall survival (left) and cumulative incidence of GVHD- related mortality (right) in the T cell replete graft recipients in the MSKCC patient cohort (see table S3), stratified into nondominated and Enterococcus- dominated groups (domination is defined as the relative genus abundance ≥0.3 in at least one sample\n",
      "between day 0 and +21). n, number of individuals. (D) Overall survival (left) and cumulative incidence of GVHD-related mortality (right) in Enterococcus-dominated (at genus level) versus nondominated allo-HCT patients in the combined multicenter-validation cohort (table S3). Clinical outcomes in (C) and (D) were analyzed using the R packages survival and cmprsk. Wald values of P <0.05 signify higher risks (HR, hazard ratios) of mortality among patients with Enterococcus domination as compared\n",
      "with those without domination.\n",
      "2 of 7\n",
      "D\n",
      "o\n",
      "n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "RES EARCH | R E P O R T\n",
      "A\n",
      "C57BL/6 = 12951/Sv Survival Enterococcus Akkermansia 100 0.5 M Bacteroides M Blautia x50 rel. abundance 0 Pl i Clostridium 0 10 20 30 0 10 20 30 0O 10 20 30 0 10 20 30 0 10 20 30 [ Lactobacillus 1.0 Oscillospira A p=0013 W Roseburia 0 20 40 60 80 100 days after HCT 0.5 — BM (n=5) — BM+T (n=5) rel. abundance 0 0 10 20 30 10 20 30 O 10 20 30 0 10 20 30 0 10 days after HCT C57BL/6 — BALB/c 16S rRNA feces o before HCT Enterococcus feces Enterococcus mLN Survival e BM -~ 4x10* e BM 10°% e BM+T 8 ®\n",
      "BM+T 100 10\" ° 1 3x10* — BM (n=20) 102 107 2x10* — BM+T ® 50 102 10¢ g Enterococcus el.abundance 10+ A 19 o CFUs/g stool CFUs/mLN 1x10* 10° ° ) 10° o p<0.001 10 -1 4 7 10 10 7 10 0 5 10 15 20 days after HCT days after HCT days after HCT days after HCT c LP/J C57BL/6 16S rRNA feces Survival 14 days rel. to HCT o before bu/cy-cond 100 @ before HCT VVVYVYY v 1074 e BM ® BM+T — BM (n=9) AAA 1074 ¥ 50 g — BM+T (n=10) V busulfan g 'V cyclophosphamide Enterococcus rel. abundance 10° o 8 HCT p<0.01 10\n",
      "-7 -1 10 0 20 40 60 80 100 days after HCT days after HCT D LP/ - C57BL/6 Survival 16S rRNA feces -7 days rel. to HCT B 0 m -21 100 EF] (n=4) - day 0 day 7 v \\AAAA v > o AAA BM [+ EF] (n=5) BM+T [- EF] (n=10) Se 'V colonization V busulfan 50 | BM+T [+ EF] (n=12) o e -1+ EF g WV cyclophosphamide WV HCT p<0.001 Enterococcus rel. abundance 0 0 20 40 60 80 HCT: - BM BM+T days after HCT\n",
      "(n=32)\n",
      "Fig. 2. Enterococcus dominates mouse gut microbiota after HCT and can exacerbate GVHD. (A) (Left) High-density sampling and 16S rRNA gene sequencing of fecal microbiota from 129S1/Sv mice (1 box = 1 mouse) receiving bone marrow (BM; upper row) or T cell–replete bone marrow [BM+T (2 × 106 T cells); lower row]. (Right) BM+T transplanted mice develop lethal GVHD as shown by survival analysis. rel., relative. (B) (Left) Relative abundance of the genus Enterococcus in BALB/c host mice transplanted\n",
      "with C57BL/6 BM or BM+T (1 × 106 T cells) at different time points relative to HCT. (Middle) Colony-forming units (CFUs) of enterococci in fecal samples and in mesenteric lymph nodes (mLN). Scatter plot data show means ± SEM. (Right) Survival of BALB/c recipient mice after HCT [BM versus BM+T (1 × 106 T cells)]. (C) (Left) Schematic showing HCT of LP/J BM versus BM+T (4 × 106 T cells) into C57BL/6 mice after chemotherapy conditioning. (Middle) Relative abundance of the genus Enterococcus in the\n",
      "feces of\n",
      "transplanted mice at different time points relative to HCT. (Right) Comparison of overall survival between BM and BM+T mice. bu/cy-cond., busulfan/ cyclophosphamide conditioning. (D) (Left) Schematic showing colonization of germ-free C57BL/6 mice with a 6-strain minimal microbiota with (+EF) or without (-EF) E. faecalis OG1RF (2 × 107 CFUs per mouse); after 14 days, colonized mice received chemotherapy conditioning with busulfan and cyclophosphamide and, subsequently, an HCT of LP/J BM versus\n",
      "BM+T (4 × 106 T cells). (Middle) Comparison of overall survival. (Right) Relative abundances of E. faecalis spiked to the minimal microbiota in the EF+ group with samples collected at the day of HCT (day 0) and 7 days later (n = 4 to 11 mice per group; P = 0.09, paired testing of relative abundances of enterococci of day 0 versus BM+T day +7). Scatter plot data are presented as means ± SEM **P < 0.01, ***P < 0.001 (independent t test for BM versus BM+T); survival data were statistically analyzed\n",
      "using Mantel-Cox log-rank test.\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019)\n",
      "\n",
      "29 November 2019\n",
      "3 of 7\n",
      "\n",
      "o w n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "RES EARCH | R E P O R T\n",
      "A allo HCTin AML patients == pre HCT/ non-dom. (i == post HCT/ domin. { . . . Pyrimidine déoxyriboncl. salvage [ Superpath. alanine biosyn. ‘Guarlosine ribancl. denovo biosyn. Adenosine riboncl. denovo biosyn. [ Pyuvate feri. to acetate actate N Lactose and galctose degradation [N Calvin-Banson-Bassham cycld GDP-mannose biosyn. Superpath. Lysine, Lhreon, Lmeth. Glycolysis IV P-diacylglycerol biosyn. | CDP-diacylglycerol biosyn. Il Superpath. aromatic AA biosyn. N -2denosyl-L-methionine cycte | I\n",
      "Chorismate biosyn. 2ch degradataion V i hoshote | 3 E B 0 1 2 3 LDA score (log 10)\n",
      "A allo HCTin AML patients pre HCT (diverse) == pre HCT/ non-dom. (i M Enterococcus == post HCT/ domin. { . . . Pyrimidine déoxyriboncl. salvage [ B Akkermansia Superpath. alanine biosyn. W Blautia ‘Guarlosine ribancl. denovo biosyn. Adenosine riboncl. denovo biosyn. [ W Clostridium Pyuvate feri. to acetate actate N Lactose and galctose degradation [N Dorea Calvin-Banson-Bassham cycld Eubacterium GDP-mannose biosyn. Superpath. Lysine, Lhreon, Lmeth. biosyn. ost HCT Glycolysis IV (E. F. domin.) I\n",
      "Lactobacillus P-diacylglycerol biosyn. | Faecalibacterium CDP-diacylglycerol biosyn. Il Superpath. aromatic AA biosyn. 1| Oscillospira N -2denosyl-L-methionine cycte | I Chorismate biosyn. Ruminococcus 2ch degradataion V i hoshote | [ Streptococcus 3 E B 0 1 2 3 W Veillonella LDA score (log 10) I Other\n",
      "pre HCT (diverse) M Enterococcus B Akkermansia W Blautia W Clostridium Dorea Eubacterium Lmeth. biosyn. ost HCT (E. F. domin.) I Lactobacillus Faecalibacterium 1| Oscillospira | Ruminococcus | [ Streptococcus W Veillonella I Other\n",
      "B C57BL/6 — 12951/Sv\n",
      "M\n",
      "\n",
      "B\n",
      "W\n",
      "\n",
      "W\n",
      "I\n",
      "\n",
      "1|\n",
      "day 7 after HCT = BM (n=5) == BM+T (n=5) Lactose and galactose degradation — Thiamine formation from pyrithiamine Pyrimidin deoxyriboncl. salvage Lysine iosynthesis hamnose degradation | ! s-aminoiidazole ribdncl. biosyn supepath 5+ aminoimid. fiboncl.biosyn. Inogine- Inosine- phosph biosyn. | — Peptifloglycanbigsynthesis || hosph biosyn. I I scrine and ycne biosyn. | S-adenosyL-methionne cycle | -1 0 i 2 LDA score (log 10)\n",
      "=\n",
      "\n",
      "==\n",
      "\n",
      "[\n",
      "W\n",
      "I\n",
      "D\n",
      "invitro Enterococcus growth\n",
      "E. faecalis - ° & Incubation in 0D600 nm regular BHI 02462810 hours < Regular BHI -+ Lactase pretreatment\n",
      "15 0.0 —— = 024681 hours\n",
      "E. faecium - Regular BHI -+ Lactase pretreatment 15 - - ° 10 Incubation in 0D600 nm o 0D600 nm 05 regular BHI | [= 0.0 0246 810 024 8 10 hours hours\n",
      "\n",
      "- Regular BHI -+ Lactase pretreatment 15 - 10 0D600 nm 05 0.0 024 8 10 hours\n",
      "\n",
      "- ° Incubation in 0D600 nm o regular BHI | [= 0246 810 hours\n",
      "F Clostridum vs. Enterococcus (genera) in allo-HCT patients Kendall tau =-0.21; p<0.001 1 3 o = 3 2 B o Clostridium 3 2 rel. abundance - 0 107 107 1 Enterococcus rel. abundance\n",
      "Butyrate in C57BL/6 — BALB/c\n",
      "\n",
      "cecal content . 100- * BM BM+T 50- B butyrate pm/mg\n",
      "or in BHI broth pretreated with lactase. (Right) E. faecalis or E. faecium incubated in lactase-pretreated BHI were put into regular BHI broth after 8 hours to assess growth dynamics in regular BHI (gray symbols). Data are results of four experiments combined; values represent means ± SEM. (E) (Left) Fecal butyrate concentrations (means ± SEM) from pre- and posttransplant fecal samples from AML patients from MSKCC, who received allo-HCT and were selected based on a highly diverse pre-HCT\n",
      "microbiota and a posttransplant E. faecium domination [by 16S rRNA gene sequencing; 6 (out of 8) patients are presented in (A)]. (Right) Correlation of butyrate concentrations with relative abundances of the genus Enterococcus (n = 139 patients; 8 patients from left panel), and 131 allo-HCT patients from a dataset published by Haak et al. (28); statistical analysis was performed using Kendall’s tau rank correlation coefficient. (F) Stool samples were collected at the time of engraftment (~24\n",
      "days after allo-HCT). Data show Kendall’s tau rank correlation of relative abundances of the genera Clostridium and Enterococcus from the dataset of Haak et al. (G) Butyrate concentration (mean ± SEM) in cecal contents of BALB/c mice transplanted with C57BL/6 BM or BM+T (1 × 106 T cells) at day +7 after HCT. Statistical analysis: *P < 0.05 [paired t test (E) or independent t test (F)].\n",
      "4 of 7\n",
      "==\n",
      "\n",
      "==\n",
      "C\n",
      "Metagenomic metabolic pathways Enterococcus isolates\n",
      "\n",
      "Amino sugar and nucleotide Pyrimidine metabolism sugar metabolism Purine metabolism Lactose and ‘metabolism alactose AminoacytRNA meta Glycolysis/ gluconeogenesis Pyruvate metabolism Arginine proline metab. Methane metabolism Cysteine methionine metab. Glycine serine threonine metab. Other pathways Oxidative phosphorilation E. faecalis (BALB/c mice, GVHD) Pentose phosphate pathway Fructose mannose metabolism\n",
      "E. faecium (human isolate)\n",
      "Amino sugar and nucleotide sugar metabolism Pyrimidine metabolism Glycolysis/ Purine metabolism gluconeogenesis Sucrose metabolism Methane metabolism Fructose mannose metabolism l Pentose pathway Pyruvate metabolism AminoacyHRNA metab. Other pathways Pentose glucuronate interconversions Glycine serine threonine metabolism\n",
      "E Butyrate in allo-HCT patients\n",
      "feces 15 —_— © preHCT postHCT 10 butyrate mM\n",
      "©\n",
      "Kendall tau =-0.15; p = 0.008 20 0o butyrate mM oo 3 - orts o 0 107 10 _I- Enterococcus rel. abundance\n",
      "Fig. 3. Metagenomic and metabolomic analyses of Enterococcus-dominated fecal specimens in human HCT patients and mice. (A) (Left) Differential abundances of shotgun-sequenced and HUMAnN2-annotated bacterial metabolic pathways between paired pre- and post-HCT fecal samples from MSKCC patients who received allo-HCT for acute myeloid leukemia (AML) analyzed by linear discriminant analysis (LDA) coupled with effect size measurements (LEfSe). pre HCT, day −8 to −1 before allo-HCT; post HCT, day +3 to\n",
      "+25 after allo-HCT. (Right) Pie chart showing mean relative abundances of bacterial genera (analyzed by MetaPhlAn2) found in patient fecal samples pre- and post-HCT; data are aggregated across all patients. non-dom., nondominated; domin., dominated. (B) LEfSe analysis of bacterial metabolic pathway abundances in HCT day +7 fecal samples of 129S1/SV mice transplanted with C57BL/6 BM versus BM+T (2 × 106 T cells) (see Fig. 2A). (C) Pie charts with metabolic pathway abundances determined by\n",
      "whole-genome sequencing of E. faecalis (isolated from feces of a BALB/c GVHD mouse, day +7 after HCT; upper panel) and E. faecium (human isolate; ATCC #700221; lower panel); only pathways with an abundance ≥2% are shown in both panels. (D) (Left) In vitro growth of E. faecalis (mouse GVHD isolate; upper panel) and E. faecium (ATCC #700221; lower panel) in nontreated BHI broth\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019)\n",
      "\n",
      "29 November 2019\n",
      "D o w n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "RES EARCH | R E P O R T\n",
      "vated and proliferating CD4+ T cells (fig. S4D; CD4+CD25+; CD4+Ki67+), and an increased number and percentage of CD4+RORg+ T helper 17 (TH17) cells in colon lamina propria (fig. S4D). Posttransplant administration of E. faecalis\n",
      "Survival 100 += ' - BM [ctr] (n=10) ' - BM [LF] (n=10) — BM+T [ctr] (n=20) 50 — BM+T [LF] (n=20) survival [%] p<0.05 %020 40 60 80 100 days after HCT\n",
      "' -\n",
      "\n",
      "' -\n",
      "\n",
      "—\n",
      "Survival 100 BM [ctr] (n=8) BM [LF] (n=10) BM+T [ctr] (n=19) 50 BM+T [LF] (n=13) survival [%] p<0.01 0 20 40 60 80 100 days after HCT\n",
      "day-7to 0 1.0 0.5 Enterococcus rel. abundance day 25 1.0 N 05 Enterococcus rel. abundance TT+ / T/C\n",
      "T/C\n",
      "OG1RF to conventionally housed, T cell–replete bone marrow (BM+T)–transplanted BALB/c mice also aggravated GVHD (fig. S5A). These findings indicate that E. faecalis can aggravate GVHD severity.\n",
      "16S rRNA feces\n",
      "\n",
      "16S rRNA feces o BM [ctr] © BMI[LF] ¥ 107 © BM+T [ctr] © BM+T[LF] 102 Enterococcus rel. abundance 10° days r HCT\n",
      "o\n",
      "©\n",
      "\n",
      "©\n",
      "\n",
      "©\n",
      "165 rRNA feces 107! o BM [ctr] o BM[LF] 102 © BM+T [ctr] 10, . © BM+T [LF] Enterococcus 10 2, .1 rel. abundance 10 14 days after HCT\n",
      "o\n",
      "\n",
      "o\n",
      "Acute GVHD (grade 2-4) 06 — — T/T+T 04 0.2 cumulative incidence HR, 1.08; p=0.62 20 40 60 80 100 days after HCT\n",
      "—\n",
      "SEM; *P < 0.05 (independent t test). (C) (Left) Relative abundance (log10) of Enterococcus (genus) by days relative to the day of antibiotic cessation (broad- spectrum antibiotics for neutropenic fever: intravenous piperacillin-tazobactam, intravenous imipenem-cilastatin, or intravenous meropenem). Box plot inserts display the median relative abundances of the genus Enterococcus of time binned in the indicated day ranges relative to antibiotic cessation; whiskers represent maximum and minimum.\n",
      "Statistical analysis of box plot data: *P < 0.05 (Wilcoxon rank test). (D) Cumulative incidence of acute GVHD grade 2 to 4 in rs4988235 SNP-genotyped MSKCC patients (T cell–depleted grafts excluded; graft source: BM/PBSC unmodified = 213 patients; cord blood = 102 patients; C/C = 175, T/C+T/T = 140). The cumulative incidence of grade 2 to 4 acute GVHD was compared between genotype groups using the R package cmprsk.\n",
      "5 of 7\n",
      "(Fig. 2D and fig. S4B). Serum interferon-g con- centrations were significantly elevated in E. faecalis–colonized mice (fig. S4C), and we observed a significantly increased num- ber of donor T cells, an increase of acti-\n",
      "A\n",
      "C57BL/6 — BALB/c days rel. to HCT -7 0 14 v W HCT ctr vs. LF chow\n",
      "W\n",
      "B LP/J = C57BL/6\n",
      "days rel. to HCT -7 -3 14 YVVYV V AAA W busulfan W HT W cyclophosphamide ctrvs. LF chow\n",
      "W\n",
      "\n",
      "W\n",
      "C Allo HCT patients: Enterococcus abundance by lactase genotype\n",
      "\n",
      "0 o { - 153 -1 ’ -2 Enterococcus log10 rel. abundance -4 -20 -10 0 10 20 days relative to antibiotic termination T/T+T/C ~ C/C lactose absorber non-absorber\n",
      "\n",
      "day-7to 0 1.0 0.5 Enterococcus rel. abundance day 25 1.0 N 05 Enterococcus rel. abundance TT+ / T/C\n",
      "~\n",
      "Fig. 4. Lactose-free diet reduces experimental GVHD and lactase genotypes associated with microbiota dynamics after allo-HCT in humans. (A) (Left) Schematic showing that BALB/c recipient mice received C57BL/6 BM or BM+T (5 × 105 T cells) and were fed control chow (ctr) versus lactose-free chow (LF) from day −7 to +14 relative to transplant. Comparison of survival between BM and BM+T mice (middle) and relative abundance of the genus Enterococcus in mouse feces (right) are shown. Scatter plot data\n",
      "presented as means ± SEM; *P < 0.05 (independent t test). (B) (Left) Schematic showing HCT of LP/J BM versus BM+T (4 × 106 T cells) into C57BL/6 mice after chemotherapy conditioning. Comparison of survival between BM and BM+T mice (middle) and relative abundance of the genus Enterococcus at different time points relative to HCT (right) are shown. Scatter plot data presented as means ±\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019)\n",
      "\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019) 29 November 2019\n",
      "o w n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "RES EARCH | R E P O R T\n",
      "We next considered whether posttransplant defects of mucosal defense mechanisms facil- itate enterococcal expansion. Immunoglobulin A (IgA) coating of intestinal bacteria can be protective in colitis and is important for maintaining mucosal integrity (20). How- ever, we did not observe members of the genus Enterococcus to be enriched in either IgA- negative or IgA-positive fecal fractions, even though total fecal IgA was significantly re- duced in allo-HCT recipients with GVHD (fig. S5, B to D).\n",
      "Reduction of IgA by transplant- ing IgA-deficient bone marrow (BM) from activation-induced cytidine deaminase knock- out mice did not further increase enterococcal expansion (fig. S5E). Intestinal antimicrobial peptides of the Reg3 family can suppress the growth of VRE (21) and are reported to play a major role in GVHD (22). Accordingly, we found that both Reg3B/G transcripts and interleukin-22 protein, which regulates Reg3 expression (23), were reduced in the ileum of GVHD mice (fig. S5F).\n",
      "Next, we analyzed microbiota-intrinsic fac- tors and used shotgun metagenomic sequenc- ing to characterize the metabolic potential of the Enterococcus-dominated fecal microbiota. Pre- and posttransplant fecal samples from MSKCC patients who received allo-HCT for acute myeloid leukemia were selected for sequencing on the basis of having a highly diverse pre-HCT microbiota and posttrans- plant E. faecium domination (by 16S rRNA gene sequencing). We focused on microbial metabolic pathways that\n",
      "specifically charac- terize domination by comparing them with the highly diverse pretransplant microbiota from the same patients. Pathways involved in DNA synthesis and, notably, in lactose and galactose degradation were enriched in the E. faecium–dominated, posttransplant micro- biota. In contrast, amino acid synthesis and starch-degradation pathways were more prev- alent in pretransplant specimens (Fig. 3A). The lactose-and-galactose degradation path- way was also significantly enriched in the\n",
      "posttransplant E. faecalis–dominated samples of mice with GVHD (Fig. 3B). Comparison of whole-genome sequencing from isolates of E. faecium (from a human allo-HCT patient) and of E. faecalis (from a mouse with GVHD) revealed that genes encoding lactose and galactose metabolism accounted for ~3% of their genomes (Fig. 3C). In silico analysis of these Enterococcus genomes and publicly available genomes of other members of the gnotobiotic six-strain consortium revealed that enterococci are\n",
      "specifically enriched in enzymes of the tagatose-type galactose path- way for galactose-to-glucose degradation (24) (materials and methods and fig. S6, A and B). Enterococcal growth depends on lactose in vitro, as both E. faecalis and E. faecium strains cultured in brain-heart\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019)\n",
      "infusion (BHI) broth depleted of lactose (by lactase; fig. S7A) did not grow (Fig. 3D). Growth was reinstated upon transfer to re- gular BHI, excluding antibacterial effects of lactase treatment (Fig. 3D). Enterococcal expansion after allo-HCT was accompanied by a loss of Clostridium spp. in the microbiota of allo-HCT patients (Fig. 3, A and F) and of mice with GVHD (fig. S8, A to C, and table S7). This may be important for allo-HCT patients, as high abundances of clostridia are associated with\n",
      "better survival and low- er incidence of GVHD (12, 25). Commen- sal clostridia are known to produce large amounts of butyrate (26), which mitigates lethal GVHD in mice through protecting energy homeostasis of enterocytes (27). We observed that posttransplant enterococcal domination and a loss of clostridia were accompanied by a significant reduction in fecal butyrate in both allo-HCT patients and mice with GVHD (Fig. 3, E and G) (28). A loss of this key metabolite may contribute to the poor\n",
      "outcomes in Enterococcus-dominated patients and mice.\n",
      "Given that the optimal growth of enterococci depends on lactose availability in vitro, we investigated whether enterococcal expansion can be mitigated by feeding mice lactose-free chow (fig. S7B and table S8). In the C57BL/ 6→BALB/c model, the absence of dietary lactose significantly reduced posttransplant Enterococcus bloom and mitigated GVHD (Fig. 4A and fig. S9B). Flow cytometric anal- ysis of donor T cells on day +14 revealed a reduction in the percentage of activated and proliferating CD4+\n",
      "T cells (CD4+CD69+; CD4+ Ki67+) as well as a reduction in the percent- age of CD4+Tbet+ (TH1) T cells (fig. S9). The effect of a lactose-free diet on enterococcal outgrowth and GVHD was replicated in the LP/J→C57BL/6 mouse model (Fig. 4B and table S9 for changes in non-enterococcal taxa). Intestinal mucosal damage by irradiation or allo-reactive T cells may affect the expres- sion of lactase, the enzyme found on small- intestine enterocytes that facilitates lactose absorption through\n",
      "disaccharide cleavage. Duodenal lactase transcript abundance pro- gressively declined in BM+T recipients over the course of transplantation (fig. S9C), which may induce a lactose-intolerant–like state in mice, allowing nondigested lactose to reach the lower intestinal tract and serve as a car-\n",
      "Next, we explored whether enterococci ex- pansion is associated with lactose tolerance in human allo-HCT patients by genotyping 602 patients from the MSKCC cohort with available pretransplant germline DNA sam- ples for the gene polymorphism rs4988235 (-13910*T). This single-nucleotide polymorphism (SNP) regulates lactase expression and pre- dicts lactose absorption and/or tolerance (C/T\n",
      "29 November 2019\n",
      "or T/T alleles) and malabsorption (C/C alleles) in the upper gut (29). Although abundance of the genus Enterococcus increased comparably during exposure to broad-spectrum antibiotics in both lactose absorbers and malabsorbers, enterococcal domination was significantly prolonged in malabsorbers after cessation of antibiotics (Fig. 4D and fig. S10). This finding suggests that the maintenance of enterococ- cal domination and microbiota recovery after broad-spectrum antibiotic exposure is signif-\n",
      "icantly modulated by the luminal availability of lactose as a growth substrate.\n",
      "significant risk factor for the development of acute GVHD and for increased overall and GVHD-related mortality after allo-HCT. Our findings extend previous reports from smaller single-center analyses that posttransplant VRE bacteremia and fecal domination are associated with worse outcomes after allo-HCT (7, 8, 30). In gnotobiotic mouse models, enterococci exacerbate GVHD, consistent with previous reports of aggravated colitis in models of in- flammatory bowel disease (14) or systemic autoimmune\n",
      "responses (31). We previously identified Blautia abundance (a genus within class Clostridia) as a predictor of protection from lethal GVHD (12), whereas here we describe Enterococcus domination as a risk factor for GVHD. These two findings are noteworthy in light of our recent observa- tion that a B. producta strain can inhibit VRE growth via the production of a lantibiotic pro- tein (32). We identified a microbiota-intrinsic mechanism that is dependent on lactose uti- lization and favors the\n",
      "expansion of enterococci. This process may be triggered through a loss of lactase produced by enterocytes damaged by conditioning or allo-reactive T cells. We validated this concept experimentally, by showing that depletion of lactose in vitro and in vivo inhibited enterococcal expan- sion and mitigated GVHD, and clinically, by showing that patients harboring a lactose- malabsorption allele experienced prolonged Enterococcus domination after antibiotic ex- posure. These observations in mice and\n",
      "allo- HCT patients provide proof-of-concept for a novel, non–antibiotic-based therapeutic strategy, such as a lactose-free diet, to attenu- ate the outgrowth of pathobionts like enter- ococci and possibly improve clinical outcomes by modulating dietary sources of nutrients for\n",
      "REFERENCES AND NOTES\n",
      "1. Y. Litvak, M. X. Byndloss, A. J. Bäumler, Science 362, eaat9076 (2018).\n",
      "\n",
      "2. M. S. Gilmore, F. Lebreton, W. van Schaik, Curr. Opin. Microbiol. 16, 10–16 (2013).\n",
      "\n",
      "3. S. Schloissnig et al., Nature 493, 45–50 (2013).\n",
      "\n",
      "4. F. Lebreton et al., Cell 169, 849–861.e13 (2017).\n",
      "\n",
      "5. Y. Taur et al., Clin. Infect. Dis. 55, 905–914 (2012).\n",
      "\n",
      "6. C. Ubeda et al., J. Clin. Invest. 120, 4332–4341 (2010).\n",
      "\n",
      "7. E. Holler et al., Biol. Blood Marrow Transplant. 20, 640–645 (2014).\n",
      "6 of 7\n",
      "\n",
      "o w n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "RES EARCH | R E P O R T\n",
      "8. C. D. Ford et al., Biol. Blood Marrow Transplant. 23, 340–346 (2017).\n",
      "\n",
      "9. H. J. Khoury et al., Haematologica 102 , 958–966 (2017).\n",
      "\n",
      "10. Y. Shono et al., Sci. Transl. Med. 8, 339ra71 (2016).\n",
      "\n",
      "11. Y. Taur et al., Blood 124, 1174–1182 (2014).\n",
      "\n",
      "12. R. R. Jenq et al., Biol. Blood Marrow Transplant. 21, 1373–1383 (2015).\n",
      "\n",
      "13. Y. Taur et al., Sci. Transl. Med. 10, eaap9489 (2018).\n",
      "\n",
      "14. N. Steck et al., Gastroenterology 141, 959–971 (2011).\n",
      "\n",
      "15. N. Geva-Zatorsky et al., Cell 168, 928–943.e11 (2017).\n",
      "16. K. Riesner, M. Kalupa, Y. Shi, S. Elezkurtaj, O. Penack, Bone Marrow Transplant. 51, 410–417 (2016).\n",
      "17. K. Dubin, E. G. Pamer, Microbiol. Spectr. doi:10.1128/ microbiolspec.BAD-0014-2016 (2014).\n",
      "\n",
      "18. R. R. Jenq et al., J. Exp. Med. 209, 903–911 (2012).\n",
      "\n",
      "19. S. Caballero et al., Cell Host Microbe 21, 592–602.e4 (2017).\n",
      "\n",
      "20. N. W. Palm et al., Cell 158, 1000–1010 (2014).\n",
      "\n",
      "21. K. Brandl et al., Nature 455, 804–807 (2008).\n",
      "\n",
      "22. D. Zhao et al., J. Clin. Invest. 128, 4970–4979 (2018).\n",
      "\n",
      "23. C. A. Lindemans et al., Nature 528, 560–564 (2015).\n",
      "24. K. H. Schleifer, A. Hartinger, F. Götz, FEMS Microbiol. Lett. 3, 9–11 (1978)\n",
      "\n",
      "25. D. Weber et al., Biol. Blood Marrow Transplant. 23, 845–852 (2017).\n",
      "\n",
      "26. P. A. Gill, M. C. van Zelm, J. G. Muir, P. R. Gibson, Aliment. Pharmacol. Ther. 48 , 15–34 (2018).\n",
      "\n",
      "27. N. D. Mathewson et al., Nat. Immunol. 17, 505–513 (2016).\n",
      "\n",
      "28. B. W. Haak et al., Blood 131, 2978–2986 (2018).\n",
      "29. C. J. Ingram, C. A. Mulcare, Y. Itan, M. G. Thomas,\n",
      "D. M. Swallow, Hum. Genet. 124, 579–591 (2009).\n",
      "\n",
      "30. J. Vydra et al., Clin. Infect. Dis. 55, 764–770 (2012).\n",
      "\n",
      "31. S. Manfredo Vieira et al., Science 359, 1156–1161 (2018).\n",
      "\n",
      "32. S. G. Kim et al., Nature 572, 665–669 (2019).\n",
      "ACKN OW LEDG MEN TS\n",
      "We thank G. M. Dunny and J. Willett (Department of Microbiology, University of Minnesota) as well as R. Zbasnik and V. Schlegel (Department of Food Science, University of Nebraska) for helpful discussion and for providing materials for revision of the manuscript. Funding: This work was supported by a German Research Foundation (DFG) scholarship to\n",
      "Bone Marrow Transplantation to C.K.S.-T.; partially supported by the DFG research consortium TR221 “GvH/GvL” (project B13) to E.H.; by NCI awards R01-CA228358 (M.R.M.v.d.B.), R01-CA228308 (M.R.M.v.d.B.), MSKCC Cancer Center Core Grant P30 CA008748, and Project 4 of P01-CA023766 (M.R.M.v.d.B.); NHLBI award R01-HL125571 (M.R.M.v.d.B.), R01-HL123340 (M.R.M.v.d.B.), and K08-HL143189 (J.U.P.); NIA National Institute of Aging award Project 2 of P01-AG052359 (M.R.M.v.d.B.); NIAID award U01 AI124275\n",
      "(M.R.M.v.d.B.); R01 AI032135 (E.G.P.); AI095706 (E.G.P.); U01 AI124275 (E.G.P. and J.B.X.); Tri-Institutional Stem Cell Initiative award 2016-013 (M.R.M.v.d.B.); the Lymphoma Foundation (M.R.M.v.d.B. and N.K.); the Susan and Peter Solomon Divisional Genomics Program (M.R.M.v.d.B.); the Parker Institute for Cancer Immunotherapy at Memorial Sloan Kettering Cancer Center (M.R.M.v.d.B. and J.U.P.); the Sawiris Foundation (J.U.P.); the Society of Memorial Sloan Kettering Cancer Center (J.U.P.); MSKCC\n",
      "Cancer Systems Immunology Pilot Grant (J.U.P.); Empire Clinical Research Investigator Program (J.U.P.); Seres Therapeutics (M.R.M.v.d.B., J.U.P., J.B.S., A.L.C.G., A.G.C., A.E.S., and A.D.S.); Japan Society for the Promotion of Science KAKENHI (17H04206 to T.T. and 17K09945 to D.H.); the Center of Innovation Program from Japan Science and Technology Agency (T.T.); Mochida Memorial Foundation for Medical and Pharmaceutical Research (D.H.); R56 AI137269-01 (J.B.X.); Conquer Cancer Foundation Young\n",
      "Investigator Award/Gilead Sciences (N.K.); NIH KL2 TR001115-03 (NCATS CTSA to A.D.S.); NIA 2P30AG028716-11 (Claude D. Pepper Older Americans Independence Center to A.D.S.); NCI R01CA203950-01 (to N.J.C., A.D.S., L.B., M.L, and A.B.); NIH 1R01HL124112-01A (A.D.S. and R.R.J.); and NIA R21AG066388- 01 (A.D.S. and N.J.C.). Author contributions: C.K.S.-T., E.G.P., J.U.P, and M.R.M.v.d.B. conceptualized the project; C.K.S.-T., A.L.C.G., A.S., S.D., A.P., J.R.C., Y.T., R.R.J., J.B.X., E.R.L., and\n",
      "J.U.P. were involved in data curation and formal analyses; C.K.S.-T., K.B.N., A.L., M.D.D., A.E.S., J.B.S., A.G.C., G.A., Y.S., M.B.d.S., K.A.M., D.B., R.P., A.T., S.M., M.E.A., A.J.P., M.M., R.J.W., L.A.A., E.F., D.P., M.A.J., D.W., A.D.S., D.H., C.S., J.A.M., K.R., M.L., A.B., L.B., K.H., and Y.H. conducted\n",
      "and M.R.M.v.d.B.\n",
      "\n",
      "the\n",
      "project and provided validation; and C.K.S.-T., J.U.P., A.L.C.G., and M.R.M.v.d.B. prepared data presentation and wrote the manuscript with contributions from all other authors. Competing interests: M.R.M.v.d.B has received research support from Seres Therapeutics; has consulted, received honorarium from, or participated in advisory boards for Seres Therapeutics, Flagship Ventures, Novartis, Evelo, Jazz Pharmaceuticals, Therakos, Amgen, Magenta Therapeutics, Merck & Co, Inc., Acute Leukemia\n",
      "Forum (ALF), and DKMS Medical Council (Board); and has IP Licensing with Seres Therapeutics and Juno Therapeutics and stock options from Smart Immune. J.U.P. reports research funding, intellectual property fees, and travel reimbursement from Seres Therapeutics and consulting fees from Davolterra. A.D.S. received research support from Seres Therapeutics, Merck, and Novartis. E.G.P. has received speaker honoraria from Bristol-Myer Squibb, Celgene, Seres Therapeutics, MedImmune, Novartis, and\n",
      "Ferring Pharmaceuticals; is an inventor on patent application number WPO2015179437A1, entitled “Methods and compositions for reducing Clostridium difficile infection,” and patent number WPO2017091753A1, entitled “Methods and compositions for reducing vancomycin-resistant Enterococci infection or colonization”; and holds patents that receive royalties from Seres Therapeutics Inc. Other authors have no competing interests. Data and materials availability: All data are available in the manuscript\n",
      "or the supplementary materials. Sequencing data have been deposited into Sequence Read Archive under Bioproject number PRJNA545312.\n",
      "SUPPLEMENTARY MATERIALS\n",
      "\n",
      "science.sciencemag.org/content/366/6469/1143/suppl/DC1 Materials and Methods\n",
      "Table S1 to S10\n",
      "\n",
      "Fig S1 to S10\n",
      "References (33–53)\n",
      "View/request a protocol for this paper from Bio-protocol.\n",
      "\n",
      "18 March 2019; accepted 22 October 2019 10.1126/science.aax3760\n",
      "D o w n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "Stein-Thoeringer et al., Science 366, 1143–1149 (2019) 29 November 2019\n",
      "7 of 7\n",
      "Science\n",
      "Lactose drives Enterococcus expansion to promote graft-versus-host disease\n",
      "C. K. Stein-Thoeringer, K. B. Nichols, A. Lazrak, M. D. Docampo, A. E. Slingerland, J. B. Slingerland, A. G. Clurman, G. Armijo, A. L. C. Gomes, Y. Shono, A. Staffas, M. Burgos da Silva, S. M. Devlin, K. A. Markey, D. Bajic, R. Pinedo, A. Tsakmaklis, E. R. Littmann, A. Pastore, Y. Taur, S. Monette, M. E. Arcila, A. J. Pickard, M. Maloy, R. J. Wright, L. A. Amoretti, E. Fontana, D. Pham, M. A. Jamal, D. Weber, A. D. Sung, D. Hashimoto, C. Scheid, J. B. Xavier, J. A. Messina, K. Romero, Lew, A.\n",
      "Bush, L. Bohannon, K. Hayasaka, Y. Hasegawa, M. J. G. T. Vehreschild, J. R. Cross, D. M. Ponce, M. A. Perales, S. A. Giralt, R. R. Jenq, T. Teshima, E. Holler, N. J. Chao, E. G. Pamer, J. U. Peled and M. R. M. van den Brink M.\n",
      "Science 366 (6469), 1143-1149. DOI: 10.1126/science.aax3760\n",
      "Lactose can fuel GVHD\n",
      "Allogeneic hematopoietic cell transplantation (allo-HCT) is used to treat certain hematopoietic malignancies, but patients have a risk of developing graft-versus-host disease (GVHD). Stein-Thoeringer et al. performed a large-scale analysis of more than 1300 patients treated with allo-HCT across four clinical centers (see the Perspective by Zitvogel and Kroemer). High levels of bacteria from the Enterococcus genus were associated with greater incidence of GVHD and mortality. Lactose appears to\n",
      "provide a substrate for Enterococcus growth, and patients with a lactose-malabsorption genotype had a greater abundance of Enterococcus. A lactose-free diet limited Enterococcus growth, reduced the severity of GVHD, and improved survival in gnotobiotic mouse models.\n",
      "Science , this issue p. 1143; see also p. 1077\n",
      "\n",
      "ARTICLE TOOLS http://science.sciencemag.org/content/366/6469/1143 SUPPLEMENTARY MATERIALS http://science.sciencemag.org/content/suppl/2019/11/25/366.6469.1143.DC1 RELATED CONTENT http://science.sciencemag.org/content/sci/366/6469/1077.full REFERENCES This article cites 52 articles, 15 of which you can access for free http://science.sciencemag.org/content/366/6469/1143#BIBL PERMISSIONS http://www.sciencemag.org/help/reprints-and-permissions\n",
      "o w n l o a d e d f r o m h t t p : / / s c i e n c e . s c i e n c e m a g . o r g / o n N o v e m b e r 2 8 , 2 0 1 9\n",
      "Use of this article is subject to the Terms of Service\n",
      "Science (print ISSN 0036-8075; online ISSN 1095-9203) is published by the American Association for the Advancement of Science, 1200 New York Avenue NW, Washington, DC 20005. The title Science is a registered trademark of AAAS.\n",
      "Copyright © 2019 The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works\n",
      "CSV file saved as GVHD-ALD-papers.csv\n"
     ]
    }
   ],
   "source": [
    "import nest_asyncio\n",
    "nest_asyncio.apply()\n",
    "\n",
    "import asyncio\n",
    "import pandas as pd\n",
    "from typing import List\n",
    "\n",
    "semaphore = asyncio.Semaphore(20)\n",
    "\n",
    "async def limited_chunk_review(chunk):\n",
    "    async with semaphore:\n",
    "        return await as_chunk_review(chunk)\n",
    "    \n",
    "async def process_all_chunks(chunks):\n",
    "    tasks = [limited_chunk_review(chunk) for chunk in chunks]\n",
    "    reviewed_chunks = await asyncio.gather(*tasks)\n",
    "    return reviewed_chunks\n",
    "\n",
    "async def main():\n",
    "    reviewed_chunks = await process_all_chunks(chunks)\n",
    "\n",
    "    reviewed_dicts = [rc.model_dump() for rc in reviewed_chunks]\n",
    "\n",
    "    df = pd.DataFrame(reviewed_dicts)\n",
    "    df.to_csv(\"GVHD-ALD-papers.csv\", index=False)\n",
    "    print(\"CSV file saved as GVHD-ALD-papers.csv\")\n",
    "    \n",
    "\n",
    "if __name__ == \"__main__\":\n",
    "    asyncio.run(main())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "relevant\n",
      "False    547\n",
      "True     204\n",
      "Name: count, dtype: int64\n",
      "relevant\n",
      "False    140.307130\n",
      "True     407.897059\n",
      "Name: text_length, dtype: float64\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAABv0AAAJOCAYAAACUQctNAAAAOnRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjEwLjAsIGh0dHBzOi8vbWF0cGxvdGxpYi5vcmcvlHJYcgAAAAlwSFlzAAAPYQAAD2EBqD+naQAAry1JREFUeJzs3Qd8VFX2wPEzk95DD10QBGmK6ArqWgBBwY6urgXcv5XFhnVxbYCCsvbuKqKuBXtZQGlSpCmCSJMOhh5Kes/M+3/OxclmQgIJJHnzZn7fz+ftZN578+bMJOu73HPvuS7LsiwBAAAAAAAAAAAA4FhuuwMAAAAAAAAAAAAAcHRI+gEAAAAAAAAAAAAOR9IPAAAAAAAAAAAAcDiSfgAAAAAAAAAAAIDDkfQDAAAAAAAAAAAAHI6kHwAAAAAAAAAAAOBwJP0AAAAAAAAAAAAAhyPpBwAAAAAAAAAAADgcST8AAAAAAAAAAADA4Uj6AQhqJSUlcv/990vLli3F7XbLJZdcUqfvf/3118sxxxxTp+8JAACcee/esmWLuFwuefrpp8Xp/vOf/0jHjh0lIiJCkpOT6/S933nnHfM96vcJAAAQymbPnm3aRZ999tkRvf6xxx4zr9+7d684DX1yCFUk/YAQsHHjRrnlllukbdu2Eh0dLYmJiXL66afLCy+8IPn5+RIIXn31VdNBU9Pefvtt+de//iWXX365vPvuuzJ8+PBKzz377LNNQ8a3xcTESLdu3eT5558Xr9db47EBAIC6TwT5Nm0THXfccXLbbbfJ7t27a/39ly1bJtdee60ZiBQVFSX169eXvn37yoQJE8Tj8YgdpkyZYjpyatqaNWtMJ8uxxx4rb775pvz73/8+bEeSb9MkoXbO3HHHHZKRkVHjsQEAEAj9H3rPO/XUU+0OJWCUbw9Utmm/TU358MMPTX9PVWn75IILLpBAVd3PE2jokwNqTngNXgtAAJo8ebJcccUVpnNp8ODB0qVLFykqKpJ58+bJfffdJ6tWrTpkR0xdNnobNmxoOohq0vfffy/NmzeX5557rkrnt2jRQsaOHWt+1lFM2mjSROGePXvkiSeeqNHYAABA3Rs1apS0adNGCgoKTHvotddeM8mvlStXSmxsrElS1XTHwltvvSW33nqrNGnSRK677jpp3769ZGdny8yZM+WGG26QnTt3yoMPPih1TT/3K6+8UuOJPx1Rrt+hDjBr165dlV6jv4f4+HjJzc0138tLL70kS5cuNb8jAACCyQcffGASSD/99JNs2LChyvfKYHbZZZf5fQ85OTkydOhQufTSS80xH21L1RTt79H231133SXBIBg+D31yQM0g6QcEsc2bN8tVV10lrVu3Nsmvpk2blh4bNmyYaVxqUjCYpaWlVaukVFJSkhmF76MddFqaSjuetJMwLCysliIFAAB14fzzz5eTTz7Z/HzjjTdKgwYN5Nlnn5Wvv/5a/vrXv5qZZjVp0aJFpj3Rq1cvk2RLSEgoPaadMj///LPpoKlLmliLi4ur1faXqk4bTKsy6AAwpRUqtA378ccfmw7RP/3pT7UWKwAAdd1Ps2DBAvniiy/M/U4TgI8++midxqADc3QwuFY9CBQ6o0s3H034aNJP95Xto0Fwo08OqBmU9wSC2Lhx48zoqPHjx/sl/Hx0FNWdd97pt/7d6NGjTSkmnRmoI8901HlhYaHf63SafUUjwvX8sjP1fGW05s+fL3fffbc0atTIdDDpSC0dpVP2dTrjcM6cOVUu2aCdVffcc09piawOHTqY9W8sy/JbE2fWrFnm2r7r6sjz6tBG8CmnnGJG4/s6sHzef/996dGjhyk5oCW6tHNq69atVWpga3mCzp07m+vrSDVt7Kenp5eeoyUjtBxrRbTT0NdZqbQsWO/evaVx48bmu+jUqZMZLV9ZKQodMa+dZ/re+h7vvffeQedqOS0dTaWv0WvqaCudKVq2hrv+Xeg/TvTvSM/R34Wun1j+7wUAgECm91BfJ1xla39U5d5dmZEjR5o2iHbqlU34+eg9vaJKB1qJwdcm07bI4sWL/Y4vX77cvM5Xvj0lJUX+7//+T/bt21dhuazVq1fL1VdfLfXq1ZMzzjjDvFZn+amypZSqUp1BvweNq1mzZmYgWdkynPrd+Tovte1XWbvxcP785z+Xlqkv68cff5TzzjvPdArpzMyzzjrLtDWr4ttvvzXX1fao/i4GDhxo2ok+2pbUeH///feDXjtixAiJjIws/Z3/8MMPpppGq1atSttB2nYqXzpfv2edwbh9+3aztrT+rN/Lvffee1BZV9/syK5du5rfqZ6nn1UTwzXRBgUA2E/bA3ov1nuQDnjR5z7FxcXmv+t/+9vfDnpdVlaWuTfo/aO6/ybXe5uWM9f38t3Dv/vuu9J732mnnWYGQel9Re8vFa39pvc3Lb2tA3T0HnrRRReZe1tF93ndr20SbS/pe+l76tIrNVVCXL83/Z70+9B21DfffFN6XPtt9P6pfUq+/iGlg971/n/llVea53pcB8HrPd/XBqqptd+qcp/W99dKXNo+O+ecc0ybRqtUaT9eeRqjft8av/b7aHtj6tSpfn1cVfk82s7Q2XLav6PfXZ8+fcz3UlXaH/SXv/zFLBmkfy/an6iVM3y0TXbCCSdU+Frts+vfv79UF31y9MnhyDDTDwhi//3vf80NRBtwVaGj3XXdO21AaUJNO1V0Wv1vv/0mX3755RHHcfvtt5tGrd6MNBmnN1dtcOrobaXP9RztBPnnP/952JIN2nDTBo8m9LQk1oknnmgaPFquVBuXWspTG3n/+c9/TINGE5++8gDHH398teP3JRDLjlbX6z788MOmwaPfmyYxdeTRmWeeKb/88sshR7ZrY0ITotqQ10azdjK+/PLL5nXaaaUzDLQhqjd07eDTBo6PNuB0xoCuU+ijjQltrOh3Eh4ebn7vf//7301DRjviytIGnf5+9XsbMmSIaXhrZ5Q2lPQaSr8v7RDT37s21E866STTsNCG9LZt20wjX6+t76eNlZtvvtl8rytWrDDf/bp16+Srr76q9vcMAIAdfEkl7bw4mnt3RfLy8kypSm0faHKoqrSUkXZu6PtqG0Q7gLS01aZNm0rfa/r06ea5xqQJP1/Jdn3UtkL5BJ4mqLSs6JgxY0xbqnv37rJjxw5zHW0zVYV26mkSU9ci1NH3a9euNe0Qba/4vgdt12nnhbYdfSU7y47cr077S2kb0kcrV+hMTW23aLvS7XaXdrRoEu5QMwL1M2rbRzucnnrqKfO70fg0Aaq/R+1U0XaddpZ88sknpl1Zlu7r169faTyffvqpuYZ+D/q3ozMStS2obSU9VpYm9/R9de0m7VydMWOGPPPMMyapq6/30faZ/p3pZ9T2pQ7I08+lv09f59LRtEEBAPbTxJve03UgiVYY8N1H9d/9eh/VQdI6C/CNN94w5/jov7G1M18TG6q6/ybXe6jey7QvRv9N70sI6WATvc4111xjZv9NnDjRtBkmTZpkEpM+2m+gr9cy5T179jSDtsse99F1kvW4L9GofTM66EbvcZq4PJrSk9rGOf30001y7B//+IdJgmlMOqjm888/N9+dJl70O9XPoPdHbbfpd6Xxa7JSBy8p7XvKzMw0923fcjDaZjla1blPa5JJB/fo34Oer8nWBx54wAz+0baAb8C7tnO0FLwm2bTNp+1E7Q8rqyqf58knnzRtJ00c67navtTfu/b9VYXGqH832r+mbZMXX3zRfAZf0kr/Nm666SZTwUITmj76961/kw899NARfaf0ydEnhyNgAQhKmZmZOqTJuvjii6t0/rJly8z5N954o9/+e++91+z//vvvS/fp80cfffSga7Ru3doaMmRI6fMJEyaYc/v27Wt5vd7S/cOHD7fCwsKsjIyM0n2dO3e2zjrrrCrF+tVXX5nrPv744377L7/8csvlclkbNmwo3afX1GtXhZ7bsWNHa8+ePWZbs2aNdd9995n3GjhwYOl5W7ZsMfE/8cQTfq9fsWKFFR4e7rdfvw/9Xnx++OEHc70PPvjA77Xfffed3379/UVFRVn33HOP33njxo0zn/H3338v3ZeXl3fQZ+nfv7/Vtm1bv30ah77H3LlzS/elpaUd9D6PPPKIOe+LL7446Lq+3+N//vMfy+12m89T1uuvv25eO3/+/INeCwCAnXztkhkzZpj7/NatW62JEydaDRo0sGJiYqxt27Yd1b27Ir/++qs5584776xSjJs3bzbna0z79+8v3f/111+b/f/9738Pef//6KOPDrrXa5tN9/31r3896Pxhw4aZY1WhbYbIyEirX79+lsfjKd3/8ssvm2u8/fbbB72nfs+H4zt37dq15nxtZ+m19HfSqFEjKzc3t7QN0r59e9PGKduu1O+hTZs21rnnnnvQ71q/T5WdnW0lJydbN910k99779q1y0pKSvLb36tXL6tHjx5+5/3000/meu+9957f+5Y3duzYg9pp+vekrx01apTfud27d/d7H21r63l33HHHQdf1fd7qtEEBAIHn559/Nv+tnz59eul/31u0aOHXTpg6depB93w1YMAAv3/jV+ff5Ppcz121atVBMZW/nxUVFVldunSxevfuXbpvyZIl5hp33XWX37nXX3/9Qf1DN9xwg9W0aVNr7969fudeddVV5p5b0f2zItomKH/tPn36WF27drUKCgpK9+l3eNppp5k2Qlna7omNjbXWrVtn/etf/zLX0r6ksrSfp2yb73D03LJ9Q+VV5z6t/U/l2xaFhYVWSkqKNWjQoNJ9zzzzzEGx5+fnm74r3T9r1qzDfh49R889/vjjzXv4vPDCC2a/xleVttpFF13kt//vf/+72a/tXaV9fNHR0dYDDzzgd562beLi4qycnJxDvg99cgfQJ4eaQHlPIEjpCCpVURmpiugaM0rLcJalM/7U0az9p6NOyo421xErOuK5otJJVY1V63jriJzysWp7VkeRHU2pCB2JppvWDdfROzp6RkcB+eioOx1VoyOKdLSNb9MRVzqCvvyIq7J05LeWozr33HP9XqujenQUlu+1Wi5BR3bpqLWyJSl0dqSOmis7W0BLGfjoaC29npZV0NH/+rwsLTPgK5el9HNqmQU910dHyGlJBh0lV57v96ifQ0cS6XdU9nP4SqQd6jsAAMBOOktN739aAkdHy+v9V2el6ajxo7l310R7zEdHF5ed4ea7d5e9X5e9/2tpJY1J2whq6dKlB11T10Q5Gjo7TWcA6Ah9HSXuoyO6td1ytOtEa3tEfy86glxHNWupIm3TabkrtWzZMlm/fr0pUaolTH2/Bx0Br+Wp5s6da9pnFdHZjFomSWdUlP0dantSZ9+V/R3qd79kyRK/sqLa/tKySRdffHGF37/GoNfT6hrabtOR4of7/vV3Wr79pe2sitZ18rW/jqYNCgAIjFl+WtVIyzn6/vuu9x2dXecr+az/ptaZPL7KSEpnU+m9zFea8kj+Ta59BNofUF7Z+5m+j/Yh6D2qbFvCVwpUZy+VpRWbytJ7oN7PLrzwQvNz2bh0xrteu6I2SlXs37/fzFbUe6BWQ/BdV9sEem1tI2jlJx+duaXtN51VpTPCdBZa2ft4bajufVrbkWXXr9OZnVq1oGz7QL97baNqv5SPloXU9ld16cy2srNHK2pfHkr5WWu+37+vP1G/b/2OP/roo9J+LP271r9lnY1ZlfWk6ZOjTw41g/KeQJDSG5TSxlBVaAJOO3C0g6UsvWnqtPgjTdCp8uWsfJ1YVVkHp7JYdQ2Z8h1ovtKdRxOrdjS9+eabpgGhnT1aMkDLBJRd4Fobk3rT18ZERSor8eV7rd70teRERcrWKNcGvU7JX7hwoelE0ni0E0rLZpWl5Qe0g0jP0zJTZel7aYPGp6LSYvr7KPu70PcZNGhQpZ/B9zm01IA2UA73OQAACCS6jt1xxx1nyu9ox5v+Q7tsEuto7t1H2x6rTttJO7+01KZ2FJaPoXwHg2rTpo0cDV/7Sr+vsrTzSMvJH037y9fBod+Xtru0XJSWWirbiaK/B6WlkCqjn7tssrT8a30dIZX9npSWA9NBcNqpo2tba5tPO1a046fseampqfLII4+YUkvl27Tlv3/f+nyHa39p+1bXpKnM0bRBAQD20uSH3rM14edbR1jp4BMt+azlwLWMtLZP9N/jWsJRy3nqoBNNcuh6f2WTftX9N3ll7QAt4/n444+bwTVl1wIrO3Db119U/hrl+4/0Hq6DbLTcuG5ViauqtCyi3gM1gadbZdf2DeLS+6m2J/S+ru09/bm2Vfc+reu0lS/Hru0DXbe57Hev5cDLn1f+u6+Lvrnyn0vj0r8LX0l2pSUxtQ2l5cm11KYOGtOSr5p0rQr65OiTQ80g6QcEKe2U0I4DraVdHeUbEtXhG5lWno6irkjZ0TKBQkce6QwAH60Xr/WztdPH10jUxod+Tzr6vKLPdqg68PpabVyUXay7rLI3bB0dp6PbdWSRNjD0URtU2mgt2xjQ0e06uufZZ581sxa0801HWmkt7/Ij3mvqd6HX1Trz+p4V0TgAAAhEOoLatz5aVVTn3l2edsho552usVEdVblf6+jmBQsWmLXndH1jbX9orLo2TEUz3som0AKRdgzpzAZfG0jbGbrOjHauaPvH95l0xLd+3opU1gbzvVbX9dMBbeXp78hH2886AlvbXdr+03VbNMGn6wCWbfPqCHFNvOraO9oO0zakzjDQdVmq2v6qrqNpgwIA7KWz1HRdNk386VaetjM06ae0EoGu6af/vdcZUnpP0nuNzv450n+TV9QO0MSMzqLSe7Cudde0aVOTMNH1cjXpWF2++5/OXqtskM6RrPNb9tq6Hp3O7KtI+UTY1KlTzaMmVHQttNpe97a69+m67iur6ferqP9QfzeaZH3//ffN35U+aturbD/bodAnVzX0yeFwSPoBQeyCCy4wo6t0tEmvXr0OeW7r1q3NTUNHi/hmzCkdkaMjtfR42VEouq8sLfekDdgjVZ1ko8aio4V01HzZ2X5aBsB3vKZog1QbrNrg1saljsrR0Ux6Q9ZRbjpToDr0tRq7NlwO1/mmjR39Herocr2R62gp7YTSzigfXSBYR+PpKPOyI4aOZiq/xni4ZLGe8+uvv5rGzdEkigEACHTVuXeXpx0FOrtMO/q2bt1aY/8A184rnRGgM/10tln5GW211f5Sa9euNTP7yrYBdcZCVTtzqkI7a3TEtJah0g4W7fzU34NvYFt138v3Wu3kqcprdWS3ljDTz6rtL/09asePjyZx161bJ++++64Z0e6jpdeOlMaonZOaSKxstt/RtEEBAPbSJIPeh7TiQHk6k09Ljb/++uumraHJEk3A6T3ojDPOMO2If/7znzX+b3KdZa8zqPT+ozMKfTTpV1F/kd7vy86u0tl35RMm2kejg2Nqsl2gfG0PTUpW5dpaFvOtt96S+++/33z3moT88ccf/Qb61HRfRm3cp/W7X716tblu2XjLf/eqtvtmtJ1ZdranxqB/Fzo7r2xSS0uxazlOHTClM+W0FOmRDoCiT67yc+iTw6Gwph8QxLRxozepG2+80STvytMRKS+88IL5ecCAAeax/DR136iRgQMH+t1cdN2UsjS5WNlMv6rQOMsnEiujsep7aY32snQUjd7stPxSTX+PWkrD911cdtllpsGiHW3lR+Poc60pXxkdla+xjx49+qBjJSUlB30H2um0Y8cO01jVG3rZch7K13AqG4eWDyjfSK8OLSOg76X/6CjP9z76OXQ0u5ZdKC8/P9+sbQMAQDCo7r27PE1e6f1Tyxrl5OQcdFxnsmnyqDoquv9X1I47HN/aKlVpg2kHm45c1lHWZd93/Pjxpu1Rtq1YE3SWn5a98s2w07VWtA369NNPV/g9aumnyuioc00WjhkzxrTpDvdabQvpd6xr0mhHj3b4lF2HpqLvX3/2tauPhL6nXkPbl+X53udo2qAAAPvov5E1saf3E11jrvx22223mUHNmjhQOptI92tCQWepa3ujfF9ATfybXO8p2odSti9HSzVqoqYs38w6nQ1Y1ksvvXTQ9fR+psnEipIWh7pXH44mTM8++2yT/KlowHnZa2u7RvvBtLqD3vu1P0XXEtSfy9J7e0Ul0Y9Ubdyn9bvX37Pvb8O3lnNFv/ea/jzllU9Y+37/5fvgtM2rA9RuueUW02Yru27hkaBP7n/ok0NVMdMPCGLaMaIlGfSmpLP3dCRyly5dzIhsLQelnRhagkhpmQgd+aTJO73J6aKzP/30k+mE0nISvoWmlTaebr31VnMj0tJGejPSkWG+kkxHQjtyXnvtNVNLXksyaIOusnVXdKS1xqMj3bRBqrFPmzZNvv76a7nrrrtKR3PXFF1oVxONepPX2vF6fY1zxIgR5v31+9HRbDrqTW/KN998sxmBVBH9XrXhM3bsWFMzX8t36Eg1HTGlvw/tLNLGvY++r15br+drQJelr9cOOP1OfA0qvenr93ekMy+1TNhnn31mShb83//9n/nd6KhzbWTqyEP9vrURpyPv9e9ARzDpKCltOOlsS92vfw/VKZ0GAECgqu69uzwtB6SdJDpzTEv/6D1UR8lr597s2bPN/VXbFdWhCSydBTBu3DjTCaLr12hbqOwaQVWh93h1xx13mE4lbWvorLqK6Oh9bftoB4uWENVyYDoTTjsATznllKPu0ClPv+M777zTtEt0tL6+p7bFtGOpc+fOZhagfm7t8NC2iH4n2jlaET2m7Uz97rVElH5G/TxatnPy5MmmHVN2MJm2o7StqZ1L+nsq38Gjv0dtD2r7TN9fr68dnEe6XrXS99P4NKmqf1u+Mq1aek2PaYfw0bRBAQD20Xu93k/03lmRnj17mvuSzkjz3XP0UZMqOnhIy/iVrcikauLf5DpgR+91es/R2Vm6Dpi2WbRPpuy6ctpe0L4IHVykCRWNd86cOWbWuyo70+jJJ5808ehahTrDS/tTtD9Bk246w0p/PlIam8581O9Dr62z/3SAu1a30vKd2jeltP2gcer7adtGP5/2Y+k99OKLLy4tk6qfS2dv6Vq+2pbRSgNlZ/ZXRGe3VdRu6969u/k+a/o+rW1QbaP89a9/NZ9LZ4Dq34lvjbuy3/2RfJ7q0M+hf8P6fep3rqU79e+mbNlZ33ehfY/aTta/W217HQ365OiTwxGwAAS9devWWTfddJN1zDHHWJGRkVZCQoJ1+umnWy+99JJVUFBQel5xcbE1cuRIq02bNlZERITVsmVLa8SIEX7nKI/HYz3wwANWw4YNrdjYWKt///7Whg0brNatW1tDhgwpPW/ChAk6BMVavHix3+tnzZpl9uujz65du6yBAwea2PTYWWeddcjPlJ2dbQ0fPtxq1qyZibV9+/bWv/71L8vr9fqdp9fp3Llzlb6nQ507e/ZsE9ejjz5auu/zzz+3zjjjDCsuLs5sHTt2tIYNG2atXbu29Bz9PvR7Ke/f//631aNHDysmJsZ85q5du1r333+/tWPHjoPOveaaa8x79+3bt8LYvvnmG6tbt25WdHS0+R0/9dRT1ttvv21es3nz5tLzNA79jiv63OW/73379lm33Xab1bx5c/M306JFC/NZ9u7dW3pOUVGReS/9zqKioqx69eqZz6R/Q5mZmRXGCgCAXSprl5RXE/fuiixZssS6+uqrS9suet/s06eP9e6775q2ldL7tsaobZryyrdDtm3bZl166aVWcnKylZSUZF1xxRUmlvLn6c+6b8+ePQdds6SkxLr99tutRo0aWS6Xy5x3OC+//LJp8+hnaNKkiTV06FArPT3d75xDvWd5hzpX2xP62cq2U3755Rfrsssusxo0aGDaH/q7+stf/mLNnDnzoN912XaQ0rantlv1mtpuOvbYY63rr7/e+vnnnw967zfffNNcQ3/X+fn5Bx1fvXq1aZvFx8ebNrG2tX/99VfzGn3/sn9P2k6s7HOX/33o716/X21/6e/l/PPPN387ZVWlDQoACBwXXnihue/k5uZWeo7ej/Te6vs3t/ZtaJ+M3isef/zxCl9T1X+T6zX0PlGR8ePHm/4Ufb3eT/QeVtE9SmPXa9SvX9/c+y655BJz39HznnzySb9zd+/ebc7V+PUzpaSkmDaPtqWqStsF5ds0auPGjdbgwYPNNfXa2mdxwQUXWJ999pk5/vXXX5vXPfPMM36vy8rKMm2GE044wXxvKicnx7TNtC2lr6mo/VeWHtfzKtpuuOGGat2nK+t/qqgdumnTJtOXo21QbRvcc8895j30fRctWlR6XmWfx9cH9+mnn/pd19fuLNtuqYjv70HbPpdffrlpG+nfmvYZVdRGUuPGjTOvGTNmjFVV9Mn9D31yOFou/Z8jSRYCAAAAAAAAAEKPzpTSWV0640vLcqPu6KzL4cOHmxmOWvkg0OiMOY1PZ+KVXesOQN0g6QcAAAAAAAAAqJCuExYTE+O3T5eL0TUHNbHTsmVL22ILte9e1/TTZKuWc/SVWA0kmmrQEpQNGjQwpScB1D3W9AMAAAAAAAAAVEjXEV6yZIlZZzY8PFy+/fZbs+n6aST8atdll11mZsudeOKJkpmZaWZW6tpturZfIMnNzTXrzmmib8WKFfL111/bHRIQspjpBwAAAAAAAACo0PTp02XkyJGyevVqycnJMUmo6667Tv75z3+aJCBqt5TnW2+9ZWZU6uy+Tp06yf333y9XXnmlBBKNr02bNpKcnCx///vf5YknnrA7JCBkkfQDAAAAAAAAAAAAHM5tdwAAAAAAAAAAAAAAjg5JPwAAAAAAAAAAAMDhKLosIl6vV3bs2CEJCQnicrnsDgcAAAQgrYienZ0tzZo1E7ebcVMAAACoGfRLAQCAmuqXIuknYhpWLVu2tDsMAADgAFu3bpUWLVrYHQYAAACCBP1SAACgpvqlSPqJmJFUvi8rMTHR7nAAAEAAysrKMp0xvnYDAAAAUBPolwIAADXVL0XST6S0dII2rGhcAQCAQ6HkEgAAAGoS/VIAAKCm+qVYkAYAAAAAAAAAAABwOJJ+AAAAAAAAAAAAgMOR9AMAAAAAAAAAAAAcjqQfAAAAAAAAAAAA4HAk/QAAAAAAAAAAAACHI+kHAAAAAAAAAAAAOBxJPwAAAAAAAAAAAMDhSPoBAAAAAAAAAAAADkfSDwAAAAAAAAAAAHA4kn4AAAAAAAAAAACAw5H0AwAAAAAAAAAAABwu3O4AAAAAAACBwWtZUuCxJL/E+8dj2Z//91jktcRr6fki1h+vO/AoYlkiTYsypPfCz8Xlcou4XAc29x8/h4WJREaJREWLKyq69GeJijrwPMr3PFpceiw6WiQmTlz6OgAAAABApUj6AQAAAECQyyn2Snqhx2yZRR6TzPMl8PI9lhT88Vjo0dTd0YsvLBTZtcMkAg+l6u/mEomLF1dikkhikrgS/nhMTBZJSDzwqM81SQgAAAAAIYqkHwAAAAAEgbxir+z/I7Gn2/9+PjAzz9kskdxssXKzRXZuqzxZqDME/0gGuho0FmnURFyNGotLH2Pj6zZkAAAAAKhjJP0AAAAAwCF0Rt7+cgk93881NUvP0QoLRPYUiLVnt1gb1/ofi40TV8Mm4mroSwYe+NmVVM+uaAEAAACgRpH0AwAAAIAA5LEs2Z1XIjtyS2SHeSyWjCKv3WE5V16uWKmbzOYnMupAIjClubibtxJXi1YHkoK6HiEAAAAAOAhJPwAAAAAIALrWnknw5RabJJ8m/EqYvFf7igrF2rFVZMdW8Sxd9L9EYPOW4mre2iQBXS1aiysuwe5IAQAAAOCQSPoBAAAAQB0r9lqy08zgKy6dyZdTzCy+gEoEbt5gtlLJ9Q8kAE0isLW4UpqLK5x/UgMAAAAIHPwLBQAAAABqWX6JVzZnFcvW3GLZnlsse/M9QorPYTL2i6XbymUHnoeHi6tVG3G17SDuYzuIK6WZ3RECAAAACHEk/QAAAACgFqTll8jGzCLZmFUk23NLhEqdQaakRKxN683mnTFJJD5BXG2PE3fb48R17HHiik+0O0IAAAAAIYakHwAAAADUUMnO37OLTZJPk31ZlOsMLTnZYi1fIp7lS0TEJdIk5Y8EYAdxtW4rrvAIuyMEAAAAEORI+qHUk7/stTsEBIh/dG9odwgAAACOkFnkKZ3Npwm/EqbzwbBEdu8U7+6dIgvnHCgFqom/jl3FfXxXccUl2B0gAABAwKO/Gj70V1cdST8AAAAAqCKvZZlSnb7ZfHsKPHaHBKeUAt24zmzeKV+Kq3UbcXU64UACkDKgAAAAAGoIST8AAAAAOATLsmRzdrGs2l9okn0FHqbz4ShYXrG2bDSb99svxdXKlwDsJq4EEoAAAAAAjhxJPwAAAACowL6CElmxr1BWpRdKNuvzoTZYlli/bzKb97uvxNVSE4DdxN1JE4BJdkcHAAAAwGFI+gEAAADAHwpKvLI6vVBW7C+UnXkldoeDUEsApm4ym/e7r8XVsrW4TzhFXF27iysyyu7oAAAAADiA2+4AAAAAAMDudfo2ZBbJV5uz5KWV+2XatlwSfrCZJdbWLeKZ9KmUPDNSSr75WLzbfrc7KCCoPfnkk+JyueSuu+4q3VdQUCDDhg2TBg0aSHx8vAwaNEh2797t97rU1FQZOHCgxMbGSuPGjeW+++6TkhLuIQAAwB7M9AMAAAAQkvbkl5gZfav2F0huCev0IUAVFYr1y0/i0a1xiri7nyruE04WV0ys3ZEBQWPx4sXyxhtvSLdu3fz2Dx8+XCZPniyffvqpJCUlyW233SaXXXaZzJ8/3xz3eDwm4ZeSkiILFiyQnTt3yuDBgyUiIkLGjBlj06cBAAChjJl+AAAAAEJGfolXft6TL++syZDxazLkp7R8En5wjrRd4p369YHZf5//R7yb1oll8fcLHI2cnBy55ppr5M0335R69eqV7s/MzJTx48fLs88+K71795YePXrIhAkTTHJv0aJF5pxp06bJ6tWr5f3335cTTzxRzj//fBk9erS88sorUlRUZOOnAgAAoYqkHwAAAICgtz232JTvfHnlfpmxLVd25VN6DQ7mKRFr5TLx/OcNKXlprHh+mCFWdpbdUQGOpOU7dbZe3759/fYvWbJEiouL/fZ37NhRWrVqJQsXLjTP9bFr167SpEmT0nP69+8vWVlZsmrVqjr8FAAAAAdQ3hMAAABAUNIZUBuyiuTH3fmyLZckH4JU+j7xfv+teGdPFVenE8Td6yxxN2tpd1SAI0ycOFGWLl1qynuWt2vXLomMjJTk5GS//Zrg02O+c8om/HzHfccqU1hYaDYfTRICAADUBJJ+AAAAAIJKideSVfsLTenOfYUeu8MB6obXK9bKX8Sz8hfxtmoj7p5niatjZ3G5KPADVGTr1q1y5513yvTp0yU6OrpO33vs2LEycuTIOn1PAAAQGmj9AwAAAAgKBSVeWbArT15btV++3ZpDwg8hy0rdLJ5P3pGSl54Uz0/zxCpmbTGgPC3fmZaWJieddJKEh4ebbc6cOfLiiy+an3XGnq7Ll5GR4fe63bt3S0pKivlZH/V5+eO+Y5UZMWKEWTPQt2kCEgAAoCYw0w8AAACAo2UWeWRxWr4s31coRV7L7nCAwCr9+e2X4p0zTdx/OsNsrphYu6MCAkKfPn1kxYoVfvv+9re/mXX7HnjgAWnZsqVERETIzJkzZdCgQeb42rVrJTU1VXr16mWe6+MTTzxhkoeNGzc2+3TmYGJionTq1KnS946KijIbAABATSPpBwAAAMCRduWVmBKea9ILxWt3MEAgy8s1a/55588S90mnmnX/XEn17I4KsFVCQoJ06dLFb19cXJw0aNCgdP8NN9wgd999t9SvX98k8m6//XaT6OvZs6c53q9fP5Pcu+6662TcuHFmHb+HHnpIhg0bRlIPAADYgqQfAAAAAEfZlFUkP+7Ol99ziu0OBXCW4iLx/viDeBfPF1e3kyXsrHPFlVzf7qiAgPXcc8+J2+02M/0KCwulf//+8uqrr5YeDwsLk0mTJsnQoUNNMlCThkOGDJFRo0bZGjcAAAhdJP0AAAAAOMK6jEL5YWee7ClgrT7gqHi9Yi37SUpWLBH3ST3FfWZfccUn2h0VYLvZs2f7PY+OjpZXXnnFbJVp3bq1TJkypQ6iAwAAODySfgAAAAAC2vbcYpm1PVe25ZbYHQoQXDweM+vPu2yxuE85Xdxn9GbNPwAAAMDBSPoBAAAACEj7Czwye0eurMsssjsUIPjLfi6YJd4lC816f+6eZ4orKtruqAAAAABUE0k/AAAAAAElt9gr83flybK9BeK1OxgglBQWiHf2VPH+NE/cp58j7j+dIa7wCLujAgAAAFBFJP0AAAAABIRiryU/peXLj7vzpchr2R0OELrycsU7fZJ4F80V95/PFfdJp4orLMzuqAAAAAAcBkk/AAAAALbyWpYs31co83blSU4xc/uAgJGdJd4pn4t34WwJ63+xuDt0tjsiAAAAAIdA0g8AAACAbdZnFsqcHXmyt8BjdygAKpO+TzwT3xZvu44Sdt4l4mrQyO6IAAAAAFSApB8AAACAOrczt1i+35ErW3NK7A4FQBVZG9ZIyWv/EnfPM8V95rniioyyOyQAAAAAZZD0AwAAAFBnsoo8Mmt7rvyWUWR3KACOhMcj3vmzxLt8qYSde4G4u55kd0QAAAAA/kDSDwAAAECdrNv3854CmbczT4q8lt3hADha2Zni+eID8S5ZKGHnXyquJs3sjggAAAAIeST9AAAAANSqXXkl8m1qtuzOZ90+INhYv2+SkjeeE/fJvcR9znniiom1OyQAAAAgZJH0AwAAAFArijyWzN2ZK0v2FAhz+4AgZnnFu3i+eFctk7C+A8Xd/VS7IwIAAABCEkk/AAAAADVufWahTN+aK1nFXrtDAVBX8nLF880nB5J/F/5FXEn17I4IAAAACCluuwMAAAAAEDxyi73y5eYs+XxTNgk/IERZG9dJyav/Es/PC+0OBQAAAAgpzPQDAAAAUCNW7y+U6dtyJN9DMU8g5BUVinfyZ2L99uuBWX/J9e2OCAAAAAh6zPQDAAAAcFRyir3y+aYs+eb3bBJ+APxYm9ZLyWtPi+fnBWJZ/PcBAAAAqE3M9AMAAABwxFbsK5CZ23OlgGQfgEPO+vtcrNXLJewiZv0BAAAAtYWZfgAAAACqLbvYI59uzJTJqTkk/ABUibX5j1l/i5n1BwAAANQGZvoBAAAAqJaNmUUyKTVb8kvotAdwBLP+pnx+YK2/i68SV1I9uyMCAAAAggYz/QAAAABUideyZPb2XPl0UxYJPwBHxdq8QUreeFa8a1fZHQoAAAAQNEj6AQAAADis7CKPfLg+Uxal5dsdCoBgkZ8nnolvi2fq12J5PHZHAwAAADge5T0BAAAAHNKmrCL57++U8wRQO7yL5oq1dYuEXX6duJLr2x0OAAAA4Fi2zvR77LHHxOVy+W0dO3YsPV5QUCDDhg2TBg0aSHx8vAwaNEh2797td43U1FQZOHCgxMbGSuPGjeW+++6TkpISGz4NAAAAEITlPHfkyicbKecJoHZZ21MPlPv8bbndoQAAAACOZftMv86dO8uMGTNKn4eH/y+k4cOHy+TJk+XTTz+VpKQkue222+Syyy6T+fPnm+Mej8ck/FJSUmTBggWyc+dOGTx4sERERMiYMWNs+TwAAABAsJTz/HpLtmzLZUAdgDpSkC+eT94V65TTxd3vInGV6R8AAAAAcHi2t6A1yadJu/IyMzNl/Pjx8uGHH0rv3r3NvgkTJsjxxx8vixYtkp49e8q0adNk9erVJmnYpEkTOfHEE2X06NHywAMPmFmEkZGRNnwiAAAAwNko5wnATt7F88W77XcJ13Kf9RvaHQ4AAADgGLaW91Tr16+XZs2aSdu2beWaa64x5TrVkiVLpLi4WPr27Vt6rpb+bNWqlSxcuNA818euXbuahJ9P//79JSsrS1atWmXDpwEAAACci3KeAALGzm0Hyn2u/MXuSAAAAADHsHWm36mnnirvvPOOdOjQwZTmHDlypPz5z3+WlStXyq5du8xMveTkZL/XaIJPjyl9LJvw8x33HatMYWGh2Xw0SQgAAACEMsp5Agg4RYXi+fx9sbZuEXf/i8Xltn3cMgAAABDQbE36nX/++aU/d+vWzSQBW7duLZ988onExMTU2vuOHTvWJBgBAAAAHCjnOen3bMljdh+AAOT9aZ5Ye9Mk7IrB4oquvb4CAAAAwOkCapiczuo77rjjZMOGDWadv6KiIsnIyPA7Z/fu3aVrAOqjPi9/3HesMiNGjDBrBvq2rVu31srnAQAAAALdot15ppwnCT8AgczatE5K3npBrH177A4FAAAACFgBlfTLycmRjRs3StOmTaVHjx4SEREhM2fOLD2+du1as+Zfr169zHN9XLFihaSlpZWeM336dElMTJROnTpV+j5RUVHmnLIbAAAAEGrr932XmiOzd+TZHQoAVM2+PSbx5920zu5IAAAAgIBka9Lv3nvvlTlz5siWLVtkwYIFcumll0pYWJj89a9/laSkJLnhhhvk7rvvllmzZsmSJUvkb3/7m0n09ezZ07y+X79+Jrl33XXXya+//ipTp06Vhx56SIYNG2YSewAAAAAOVuSx5LNNWbJsX4HdoQBA9RTki+eDN8WzeL7dkQAAAAABx9Y1/bZt22YSfPv27ZNGjRrJGWecIYsWLTI/q+eee07cbrcMGjRICgsLpX///vLqq6+Wvl4ThJMmTZKhQ4eaZGBcXJwMGTJERo0aZeOnAgAAAAJXdrFHPtuYJbvzPXaHAgBHxusV75QvRNJ2ifv8S8TlDrM7IgAAACAg2Jr0mzhx4iGPR0dHyyuvvGK2yrRu3VqmTJlSC9EBAAAAwSUtv8Qk/LKKvXaHAgBHzfvzArPGX9gVg8UVE2t3OAAAAIDtAmpNPwAAAAC1Y3NWkXywLpOEH4CgYm1eb9b5s/am2R0KAAAAYDuSfgAAAECQW76vQD7dmCWFXsvuUACg5u3fKyXjXxTvlo12RwIAAADYiqQfAAAAEMTm7siVKak5wvw+AEGtIF88H/xbvGtXSqByuVyH3B577DG7QwQAAIDD2bqmHwAAAIDa4fFaJtm3Kr3Q7lAAoG6UlIjnk3dFLrpS3CecLIFm586dpT9//PHH8sgjj8jatWtL98XHx5f+bFmWeDweCQ+n2wYAAABVx0w/AAAAIMgUlHjl441ZJPwAhB6vVzxfTRTPorkSaFJSUkq3pKQkM7vP93zNmjWSkJAg3377rfTo0UOioqJk3rx5cv3118sll1zid5277rpLzj777NLnXq9Xxo4dK23atJGYmBg54YQT5LPPPrPhEwIAAMBuDBkDAAAAgkhGoUc+3ZQl+wo8docCADaxxDv1a5G8XAnrfb44yT/+8Q95+umnpW3btlKvXr0qvUYTfu+//768/vrr0r59e5k7d65ce+210qhRIznrrLNqPWYAAAAEDpJ+AAAAQJDYk18iEzdkSm6JZXcoAGA77w8zRPLzxD3gUnG5nFHoaNSoUXLuuedW+fzCwkIZM2aMzJgxQ3r16mX2acJQZwm+8cYbJP0AAABCDEk/AAAAIEgSfh9tyJQ8En4AUMr78wKxCvIk7JKrxRUWJoHu5JOrtxbhhg0bJC8v76BEYVFRkXTv3r2GowMAAECgI+kHAAAAONxeEn4AUClr5TLxFBRI2F+GiCsiUgJZXFyc33O32y2W5f/f9uLi4tKfc3JyzOPkyZOlefPmfufpuoAAAAAILST9AAAAAAcj4QcAh2dtWCOe/7whYX+9QVwxseIUui7fypUr/fYtW7ZMIiIizM+dOnUyyb3U1FRKeQIAAECcUdQeAAAAwEH2FhxI+LGGHwAcnrV1i0n8WQX54hS9e/eWn3/+Wd577z1Zv369PProo35JwISEBLn33ntl+PDh8u6778rGjRtl6dKl8tJLL5nnAAAACC0k/QAAAACnJvzWk/ADgOqwdm4TzwdvilVUKE7Qv39/efjhh+X++++XU045RbKzs2Xw4MF+54wePdqcM3bsWDn++OPlvPPOM+U+27RpY1vcAAAAsIfLKl8cPgRlZWVJUlKSZGZmSmJiooSqJ3/Za3cICBD/6N7Q7hAAIODQXkAg2VdQIh+S8EMAa1e4Ty6e+m+7wwAq5WrVVsKuvSng1/hDaKCdCQAVo78aPvRXS5XbC8z0AwAAAByEhB8AHD0rdZN4PnpbrJJiu0MBAAAAagxJPwAAAMAhSPgBQM2xNq8Xz8fviOUpsTsUAAAAoEaQ9AMAAAAcgIQfANQ8a8Ma8Xz6nlgej92hAAAAAEeNpB8AAADggITfR+uzSPgBQC2w1q4Szxfvi+X12h0K6thrr70m3bp1M+vi6NarVy/59ttvS4+fffbZ4nK5/LZbb73V7xqpqakycOBAiY2NlcaNG8t9990nJSXMHgUAAPYIt+l9AQAAAFTB/gKPSfjllNAZDQC1xVq9XDxhH0rYpVeLy8X46FDRokULefLJJ6V9+/ZiWZa8++67cvHFF8svv/winTt3NufcdNNNMmrUqNLXaHLPx+PxmIRfSkqKLFiwQHbu3CmDBw+WiIgIGTNmjC2fCQAAhDaSfgAAAECAyizyyIcbMkn4AUAdsFb8Ip6wcAm76EozowvB78ILL/R7/sQTT5jZf4sWLSpN+mmST5N6FZk2bZqsXr1aZsyYIU2aNJETTzxRRo8eLQ888IA89thjEhkZWSefAwAAwIfhawAAAEAAKvB45dONWZJTTMIPAOqKtWyxeKd+bXcYsIHO2ps4caLk5uaaMp8+H3zwgTRs2FC6dOkiI0aMkLy8vNJjCxculK5du5qEn0///v0lKytLVq1aVeefAQAAgJl+AAAAQIDxWJZ8uSlb9hZ47A4FAEKO98cfRBKTJOy0c+wOBXVgxYoVJslXUFAg8fHx8uWXX0qnTp3Msauvvlpat24tzZo1k+XLl5sZfGvXrpUvvvjCHN+1a5dfwk/5nuuxyhQWFprNR5OEAAAANYGkHwAAABBgvkvNkd9ziu0OAwBClnf6ZHElJIm760l2h4Ja1qFDB1m2bJlkZmbKZ599JkOGDJE5c+aYxN/NN99cep7O6GvatKn06dNHNm7cKMcee+wRv+fYsWNl5MiRNfQJAAAA/ofyngAAAEAAWbArT1bs/9/ofwCAHSzxfD1RvJs32B0Iapmuu9euXTvp0aOHScadcMIJ8sILL1R47qmnnmoeN2w48Heha/3t3r3b7xzf88rWAVRaJlSTjL5t69atNfiJAABAKCPpBwAAAASI1fsLZe7O/60VBACwkccjno8niLV7h92RoA55vV6/0ptl6YxApTP+lJYF1fKgaWlppedMnz5dEhMTS0uEViQqKsqcU3YDAACoCZT3BAAAAALA1pximZyabXcYAICyCguk5MO3JPzGO025TwQXnXF3/vnnS6tWrSQ7O1s+/PBDmT17tkydOtWU8NTnAwYMkAYNGpg1/YYPHy5nnnmmdOvWzby+X79+Jrl33XXXybhx48w6fg899JAMGzbMJPYAAADqGjP9AAAAAJvtL/DI55uyxGPZHQkA4CBZmVLy0dtiFVF6OdjoDL3Bgwebdf10rb7FixebhN+5555ryn7OmDHDJPY6duwo99xzjwwaNEj++9//lr4+LCxMJk2aZB511t+1115rrjdq1ChbPxcAAAhdzPQDAAAAbJRX4pVPNmZKARk/AAhcO7eJ54sPJOzK68XlYvx0sBg/fnylx1q2bClz5sw57DVat24tU6ZMqeHIAAAAjgwtVQAAAMAmJV7LzPDLKPLaHQoA4DCstavEO+1/s7wAAACAQEPSDwAAALCBZVky6fds2Z5bYncoAIAq8i6aK57FC+wOAwAAAKgQST8AAADABnN25MmajCK7wwAAVJP3u6/Em7rZ7jAAAACAg5D0AwAAAOrYsr0Fsigt3+4wAABHwusRz6fviZWTZXckAAAAgB+SfgAAAEAd2ppTLNO25tgdBgDgaORkieez/4jl9dgdCQAAAFCKpB8AAABQR/JKvPLNlmzx2h0IAOCoWb9vEu/0SXaHAQAAAJQi6QcAAADUAcuyZNLv2ZJdTMoPAIKFd9Fc8a78xe4wAAAAAIOkHwAAAFAHfkzLl01ZxXaHAQCoYZ5vPhFrzy67wwAAAABI+gEAAAC1bVtOsczdkWd3GACA2lBcJCUfvyNWYYHdkQAAACDEkfQDAAAAalE+6/gBQPDbt0c8X31kSjkDAAAAdiHpBwAAANSiyb/nSBbr+AFA0LPWrBTv/O/tDgMAAAAhjKQfAAAAUEt+SsuXDVlFdocBAKgj3u+/E++mdXaHAQAAgBBF0g8AAACoBTtyi2X2jly7wwAA1CXLe6DMZz7ruAIAAKDukfQDAAAAalhBiVe+0nX8WNoJAEJPdpZ4Jn1qdxQAAAAIQST9AAAAgBo2OTVHsopYxw8AQpW1erl4f/3Z7jAAAAAQYkj6AQAAADVocVq+rM9kHT8ACHWeb78UK2O/3WEAAAAghJD0AwAAAGrIzjzW8QMA/KGwQDxffiSWxcxvAAAA1A2SfgAAAEANKPR45evN2eJhHT8AwB+s1E3inT/b7jAAAAAQIkj6AQAAADVg1vY8yWAdPwBAOd7Z34m1a7vdYQAAACAEkPQDAAAAjlJqdrEs21dgdxgAgEDk8UjJFx+IVVJsdyQAAAAIciT9AAAAgKNQ4rXku605docBAAhke3aLd/pku6MAAABAkCPpBwAAAByF+bvyZH+hx+4wAAABzvvTPPFuXGt3GAAAAAhiJP0AAACAI5SWXyI/7s63OwwAgCNY4vnmY7GKCu0OBAAAAEGKpB8AAABwBLyWJVNSc8RrdyAAAOfIyhTv99/ZHQUAAACCFEk/AAAA4AgsTsuXXXkldocBAHBgmU9r5za7wwAAAEAQIukHAAAAVFNGoUfm7cqzOwwAgBNZXvFM+kwsi7niAAAAqFkk/QAAAIBq+m5rjhTTVwsAOELWjq3i/Wm+3WEAAAAgyJD0AwAAAKphxb4C2ZJdbHcYAACH8876TqysTLvDAAAAQBAh6QcAAABUUW6xV2Zuz7U7DABAMCgsEM93X9kdBQAAAIIIST8AAACgimZsy5ECj2V3GACAIGH9tly861bbHQYAAACCBEk/AAAAoAo2ZBbJbxlFdocBAAgynilfiFXM/QUAAABHj6QfAAAAcBiFHq9M25pjdxgAgGCUmS7eWVPtjgIAAABBgKQfAAAAcBgLduVLVrHX7jAAAEHK++NcsXbtsDsMAAAAOBxJPwAAAOAQMos8smRPvt1hAACCmdcrnqlf2x0FAAAAHI6kHwAAAHAIP+zMkxLL7igAAMHO2rJBvOtW2x0GAAAAHIykHwAAAFCJ3Xklsmp/od1hAABChGf6JLG8lJMGAADAkSHpBwAAAFRi9o5cYZIfAKDO7N0t1i8/2h0FAAAAHIqkHwAAAFCBLdlFsjm72O4wAAAhxjN7qlhFzDIHAABA9ZH0AwAAAMqxLEtmbc+1OwwAQCjKyRbvgtl2RwEAAAAHIukHAAAAlLM6vVB253vsDgMAEKI06WdlZ9kdBgAAAByGpB8AAABQhsdrydydeXaHAQAIZcVF4pn1nd1RAAAAwGFI+gEAAABlLNlbIJlFXrvDAACEOGvZYrHSdtkdBgAAAByEpB8AAADwhwKPVxbuYpYfACAAWF7xzJhkdxQAAABwEJJ+AAAAwB8W7c6XfI9ldxgAABjW+t/Eu2md3WEAAADAIUj6AQAAACKSVeSRn9Py7Q4DAAA/Xtb2AwAAQBWR9AMAAABE5IedeVLCJD8AQICxtv3ObD8AAABUCUk/AAAAhLw9+SWycn+h3WEAAFAh7w8z7A4BAAAADkDSDwAAACFPZ/kxyQ8AEKisLRvFm7rZ7jAAAAAQ4Ej6AQAAIKTtKyiRdZlFdocBAMAhMduv5r322mvSrVs3SUxMNFuvXr3k22+/LT1eUFAgw4YNkwYNGkh8fLwMGjRIdu/e7XeN1NRUGThwoMTGxkrjxo3lvvvuk5KSEhs+DQAAAEk/AAAAhLhFu/PtDgEAgMOyNqwRa8dWu8MIKi1atJAnn3xSlixZIj///LP07t1bLr74Ylm1apU5Pnz4cPnvf/8rn376qcyZM0d27Nghl112WenrPR6PSfgVFRXJggUL5N1335V33nlHHnnkERs/FQAACGUBk/TTRpbL5ZK77rqrdB8jqgAAAFCbsoo8sjqdtfwAAM7gYbZfjbrwwgtlwIAB0r59eznuuOPkiSeeMP1PixYtkszMTBk/frw8++yzJhnYo0cPmTBhgknu6XE1bdo0Wb16tbz//vty4oknyvnnny+jR4+WV155xSQCAQAAQjLpt3jxYnnjjTdMSYWyGFEFAACA2rQ4LV88LOYHAHAIa80qsdJ22h1GUNI+pokTJ0pubq4p86mz/4qLi6Vv376l53Ts2FFatWolCxcuNM/1sWvXrtKkSZPSc/r37y9ZWVmlswUrUlhYaM4puwEAAARF0i8nJ0euueYaefPNN6VevXql+xlRBQAAgNqUX+KVX/cxyw8A4CQWs/1q2IoVK8zsvqioKLn11lvlyy+/lE6dOsmuXbskMjJSkpOT/c7XBJ8eU/pYNuHnO+47VpmxY8dKUlJS6dayZcta+WwAACD02J700/KdOluv7MgpxYgqAAAA1KYlewqkyMs0PwCAs1irfhVr3x67wwgaHTp0kGXLlsmPP/4oQ4cOlSFDhpgB5rVpxIgRZrC7b9u6lbUaAQBAzQgXG2nZhKVLl5rynuXV9oiqkSNH1tCnAAAAgNMUey1Zsiff7jAA4Ig89cNP8tVv62Xt3v0SEx4uPVs2kzHn/lk6NKxvjm9Jz5TjXhhf4Ws/vOICubzzcQftL/Z45JHv58t36zfL5vRMSYqKkt5tW8kTff8szRLjzTmFJSVyyzfT5b9rNkqT+Fh5aWAf6XNs69JrPDN/sWzNzJbnB/Sutc8OzfpZ4pk3U8IvvsruSIKC9j21a9fO/KxVprSP6oUXXpArr7zSVJHKyMjw65vavXu3pKSkmJ/18aeffvK7nh73HauMzirUDQAAIGhm+ukopjvvvFM++OADiY6OrtP3ZkQVAABAaFuxr0DyWcwPgEP9sGWrDD3lRPnhxr/KlMGXS4nXKwP/87nkFhWb4y2TEiT1nlv8tkfO7iXxkRFyXrtjKrxmXnGJLNuZJg+e2VN+vOVa+eTKC2XdvnS57KOvS895a8kKWbpjt8y98Sq5sUc3Gfz5FLGsA/8t1UTh+CUrZFTv0+voWwht1vIlYmWm2x1GUPJ6vaZClCYAIyIiZObMmaXH1q5dK6mpqWbNP6WPWh40LS2t9Jzp06dLYmKiKREKAAAQMjP9tHynNopOOukkv0WT586dKy+//LJMnTqVEVUAAACocdpB/fOeArvDAIAjNum6QX7P37qkvzT/1+smIffnY1pImNstKQlxfud8vWaDmeEXHxVZ4TWToqPk28GX++17YUBvOe3NDyU1I0taJSfKmj375YIOx0rnxg2lbb0k+cf0ubI3L18axcXK7ZNmmNmGidH8W7tOeL3i/XmhhPUZYHckjqaDws8//3yzlEx2drZ8+OGHMnv2bNMnpWvt3XDDDXL33XdL/fr1TSLv9ttvN4m+nj17mtf369fPJPeuu+46GTdunKk69dBDD5mlbOh3AgAAITXTr0+fPmY0lNZN920nn3yyXHPNNaU/M6IKAAAANW1jVrHsL/TYHQYA1JjMgkLzWC+m4io6mgz8ddce+Vv3rtW+rktEkv9I5HVLaSQLUrdLfnGxTNvwuzSNj5OGsTHy4fLfJCo8XC45vn0NfBpUlfeXH8XylNgdhqNpf9LgwYPNun7aT6WlPTXhd+6555rjzz33nFxwwQUyaNAgOfPMM80A8y+++KL09WFhYTJp0iTzqH1U1157rbneqFGjbPxUAAAglNk20y8hIUG6dOnity8uLk4aNGhQup8RVQAAAKhpi9NYyw9A8PB6Lbn3u9lyWstm0qVJwwrPmbB0pXRsWF96tWpW5esWFJfIgzN+kCu7diydvXd9986yYvceOeGVd6WBJvuuuEDS8wtk1KwFMv36v8gjM+fLpyvXSNv6yfLvi/tJ88SEGvucqEBujlirfhVXtx52R+JY48dXvPaljy5H88orr5itMq1bt5YpU6bUQnQAAAAOSvpVhY6ocrvdZkSV1lPv37+/vPrqqweNqBo6dKhJBmrScMiQIYyoAgAAQIXS8kvk95wDa14BQDC4Y8pMWZW2T2b935UVHtdZeRNXrJEHzzq1ytcs9njkr59OEl2u7+WBfUr3R4SFyYtlnqsbv5oqw07tbtYD/GbNBvl56GB5ev5iGf7tLPnkyouO4pOhKryL54ubpB8AAAACMemnddPLYkQVAAAAatLPzPIDEETunDxTpqzbJDP/dqW0SKp4Vt3nq9dLXnGxXHtCp2ol/FIzs2TakCsOuUbf7M2psjptr7xx0bnyj2lz5bz2bSQuMsKsHfjaT8uO+HOh6qxtv4u1c5u4mrawOxQAAACE8pp+AAAAQF3KLfbKqvQD614BgJNZlmUSfl+v2SBTh1whbeolVXruO0tXygUdjpVGcbFVTvht2Jch3w2+3JTwPFT5zzsmfy+vXHiuhLnd4rEsKfF6/7iOVzxe6wg/HarL89M8u0MAAABAgCDpBwAAgJDwy94C8dAHDSAIaLLtw+Vr5L1BAyQhMlJ2ZeeaTUt5lrVhX7r88Ps2+b+TulR4nS4vTZCvfltfmvC78pNJsnTHbnl30ACTtPNdt6jEc9Brn5i7SM5v30a6N21snvdq2cxca/muPWaW32nVWD8QR8dauUys/Dy7wwAAAEAACKjyngAAAEBtzYpZvr/A7jAAoEa88fOv5rHvO5/67X/r4v4yuHvn0ufv/LJKWiQmyLnHHlPhddbtS5esgiLz8/asHJm0dqP5+ZTX/+N33vQhV8hZbVqWPl+5e698vmqdLL71utJ9gzodJ3O3bJPeEz6W4xrUMwlJ1JGSYvEu/VHCTj/H7kgAAABgM5elPSAhLisrS5KSkiQzM1MSExMlVD35y167Q0CA+Ef3hnaHAAABh/aCs/2eXSQfbciyOwwgZLQr3CcXT/233WEAoSO5voTfMUJcLgo6ORHtTACoGP3V8KG/WqrcXqA1CAAAgKC3cj9r+QEAgljGfrHWr7E7CgAAANiMpB8AAACCWpHHkrUZB8rXAQAQrLyL59sdAgAAAGxG0g8AAABBbV1moRR5Q76iPQAgyFkb1oqVmW53GAAAALARST8AAAAEtRX7KO0JAAgFlniXL7U7CAAAANiIpB8AAACCVlaRR1Jziu0OAwCAOuFdscTuEAAAAGAjkn4AAAAIWiv3FwqFPQEAIWPPbrF2brM7CgAAANiEpB8AAACCOukHAEAo8a6gxCcAAECoIukHAACAoLQ9t1j2F3rsDgMAgDrlXfmLWJbX7jAAAABgA5J+AAAACErM8gMAhKTsLLE2b7A7CgAAANiApB8AAACCTonXkt/SSfoBAEKTd/kSu0MAAACADUj6AQAAIOhsyCySAo9ldxgAANjC+m2FWMXFdocBAACAOkbSDwAAAEFnxf4Cu0MAAMA+RYVirV1pdxQAAACoYyT9AAAAEFRyi72yOYvZDQCA0OZdsdTuEAAAAFDHSPoBAAAgqKxOLxSv3UEAAGAza8NasfJy7A4DAAAAdYikHwAAAILKusxCu0MAAMB+Xo9Z2w8AAAChg6QfAAAAgkZBiVe255TYHQYAAAHBu3aV3SEAAACgDpH0AwAAQNDYnF1MaU8AAP5gbVovVhEz4AEAAEIFST8AAAAEjY1ZRXaHAABA4PCUmLX9AAAAEBpI+gEAACAoWJYlm0j6AQDgx7t2pd0hAAAAoI6Q9AMAAEBQ2JVXInkllt1hAAAQUKz1v4nl9dgdBgAAAOpAeF28CQAAAFDbNjDLDwCAg+XniZW6RVzHHCvBIDc3V5588kmZOXOmpKWlidfrv5rvpk2bbIsNAADAbiT9AAAAEBQ2ZRXbHQIAAAE720+CJOl34403ypw5c+S6666Tpk2bisvlsjskAACAgEHSDwAAAI6XW+yVnXkldocBAEBA8m74TcLOvUCCwbfffiuTJ0+W008/3e5QAAAAAg5r+gEAAMDxNlHaEwCAyqXtEiszXYJBvXr1pH79+naHAQAAEJBI+gEAAMDxNpL0AwDgkLxa4jMIjB49Wh555BHJy8uzOxQAAICAQ3lPAAAAOJrXsmRzNuv5AQBw2HX9Tj5NnKh79+5+a/dt2LBBmjRpIsccc4xERET4nbt06VIbIgQAAAgMJP0AAADgaNtySqTQY9kdBgAAAc3avEEsj0dcYWHiNJdccondIQAAADgCST8AAAA4GqU9AQCoguIisXZuE1eL1uI0jz76qN0hAAAAOAJr+gEAAMDRSPoBAFA11u8bxenatm0r+/btO2h/RkaGOQYAABDKSPoBAADAsTKLPLK3wGN3GAAAOIL1+yZxui1btojHc/C9v7CwULZt22ZLTAAAAIGC8p4AAABwrN+zi+0OAQAAx7C2bhHL8orL5bwx4N98803pz1OnTpWkpKTS55oEnDlzprRp08am6AAAAAIDST8AAAA41o7cErtDAADAOQryRXbvEklpJk5zySWXmEeXyyVDhgzxOxYRESHHHHOMPPPMMzZFBwAAEBhI+gEAAMCxduQx0w8AgOrw/r5RwhyY9PN6veZRZ/MtXrxYGjZsaHdIAAAAAcd59RwAAAAAESn2WrInn/X8AACoDivV2ev6bd68mYQfAABAJZjpBwAAAEfalVcilt1BAADgMNbvm8XJXnzxxQr3a9nP6OhoadeunZx55pkSFhZW57EBAADYjaQfAAAAHGlHLqU9AQCottxssfamiathY3Gi5557Tvbs2SN5eXlSr149sy89PV1iY2MlPj5e0tLSpG3btjJr1ixp2bKl3eECAADUKcp7AgAAwJF25JXYHQIAAI5k/e7cEp9jxoyRU045RdavXy/79u0z27p16+TUU0+VF154QVJTUyUlJUWGDx9ud6gAAAB1jpl+AAAAcKSduST9AAA4Et7UTeLu0VOc6KGHHpLPP/9cjj322NJ9WtLz6aeflkGDBsmmTZtk3Lhx5mcAAIBQw0w/AAAAOE5OsVeyir12hwEAgCM5eabfzp07paTk4IE/um/Xrl3m52bNmkl2drYN0QEAANiLpB8AAAAcZzvr+QEAcOQy08XKyxUnOuecc+SWW26RX375pXSf/jx06FDp3bu3eb5ixQpp06aNjVECAADYg6QfAAAAHIfSngAAHB1r13ZxovHjx0v9+vWlR48eEhUVZbaTTz7Z7NNjKj4+Xp555hm7QwUAAKhzrOkHAAAAx9mRR9IPAICjYe3aIdL2OHGalJQUmT59uqxZs0bWrVtn9nXo0MFsZWcDAgAAhCJm+gEAAMBRLMuSXST9AAA4KtbuHeJkHTt2lIsuushsZRN+1TF27Fg55ZRTJCEhQRo3biyXXHKJrF271u+cs88+W1wul9926623+p2TmpoqAwcOlNjYWHOd++67r8J1BwEAAGobM/0AAADgKHsKPFLktewOAwAA58/0cyCPxyPvvPOOzJw5U9LS0sTr9fod//7776t8rTlz5siwYcNM4k+TdA8++KD069dPVq9eLXFxcaXn3XTTTTJq1KjS55rcKxuPJvx0BuKCBQtk586dMnjwYImIiJAxY8Yc9ecFAACoDpJ+AAAAcBTW8wMAoAbsTRPLUyKuMGd1Dd15550m6aeJti5dupiZd0fqu+++83uu19WZekuWLJEzzzzTL8mnSb2KTJs2zSQJZ8yYIU2aNJETTzxRRo8eLQ888IA89thjEhkZecTxAQAAVJezWnYAAAAIeTvyiu0OAQAA5/N6RNJ2iTRtIU4yceJE+eSTT2TAgAE1fu3MzEzzWL9+fb/9H3zwgbz//vsm8XfhhRfKww8/XDrbb+HChdK1a1eT8PPp37+/DB06VFatWiXdu3c/6H0KCwvN5pOVlVXjnwUAAIQmkn4AAABwlB3M9AMAoMZKfLoclvTTmXPt2rWr8etqmdC77rpLTj/9dDOD0Ofqq6+W1q1bS7NmzWT58uVmBp+u+/fFF1+Y47t27fJL+Cnfcz1W2VqCI0eOrPHPAAAAQNIPAAAAjmFZluwv9NgdBgAAQcHa7bx1/e655x554YUX5OWXXz6q0p7l6dp+K1eulHnz5vntv/nmm0t/1hl9TZs2lT59+sjGjRvl2GOPPaL3GjFihNx9991+M/1atmx5FNEDAAAcQNIPAAAAjpFV7BWPZXcUAAAEz0w/p9Gk3KxZs+Tbb7+Vzp07S0REhN9x3wy86rjttttk0qRJMnfuXGnR4tAzH0899VTzuGHDBpP005KfP/30k985u3fvNo+VrQMYFRVlNgAAgJpG0g8AAACOkc4sPwAAQnqmX3Jyslx66aU1VkHg9ttvly+//FJmz54tbdq0Oexrli1bZh51xp/q1auXPPHEE5KWliaNGzc2+6ZPny6JiYnSqVOnGokTAACgqkj6AQAAwDFI+gEAUIMK8sXK2C+u5PriFBMmTKjRkp4ffvihfP3115KQkFC6Bl9SUpLExMSYEp56fMCAAdKgQQOzpt/w4cPlzDPPlG7duplz+/XrZ5J71113nYwbN85c46GHHjLXZjYfAACoa+46f0cAAADgCKUXeu0OAQCAoGLtOVCK0klKSkpkxowZ8sYbb0h2drbZt2PHDsnJyanWdV577TXJzMyUs88+28zc820ff/yxOR4ZGWneRxN7HTt2NOsJDho0SP773/+WXiMsLMyUBtVHnfV37bXXyuDBg2XUqFE1/KkBAAAOj5l+AAAAcAxm+gEAULOs9H3iJL///rucd955kpqaKoWFhXLuueeaWXpPPfWUef76669Xq7znobRs2VLmzJlz2Ou0bt1apkyZUuX3BQAAqC3M9AMAAIBjkPQDAKCGOSzpd+edd8rJJ58s6enppgSnj67zN3PmTFtjAwAAsBsz/QAAAOAIOho/g6QfAAA1Stf0c5IffvhBFixYYEpvlnXMMcfI9u3bbYsLAAAgEDDTDwAAAI6QXeyVkkNX4QIAAEFe3tPr9YrHc/AgoG3btpkynwAAAKGMpB8AAAAcgdKeAADUgnRnzfTr16+fPP/886XPXS6X5OTkyKOPPioDBgywNTYAAAC7Ud4TAAAAjpBe6LU7BAAAgk9RoVh5OeKKjRcneOaZZ6R///7SqVMnKSgokKuvvlrWr18vDRs2lI8++sju8AAAAGxF0g8AAACOwEw/AABqh5W+3zFJvxYtWsivv/4qH3/8sXnUWX433HCDXHPNNRITE2N3eAAAALYi6QcAAABHIOkHAEAt0XX9mrcSpwgPDzdJPt18Nm3aJLfeeqtMmzbN1tgAAADsxJp+AAAAcASSfgAA1N5MP6fLzs6WmTNn2h0GAACArUj6AQAAIOBZliUZRST9AACoDZbO9AMAAIDjkfQDAABAwMsp9kqx1+4oAAAIUhnOn+kHAAAAkn4AAABwgIwiMn4AANQWi6QfAABAUAi3OwAAAADgcPJKSPoBAFBrsrMk0HXv3l1cLlelx/Py8uo0HgAAgEBE0g8AAAABr8Bj2R0CAADBq6RYrOIicUVESqC65JJL7A4BAAAg4JH0AwAAQMArYKYfAAC1Ky9XJClwk36PPvqo3SEAAAAEPNb0AwAAQMDLZ6YfAAC1n/QDAACAo5H0AwAAQMArKCHpBwBAbbJI+gEAADgeST8AAAAEvHwP5T0BAKhVJP0AAABCM+nXtm1b2bdv30H7MzIyzDEAAACgJjHTDwCA2sVMPwAAgBBN+m3ZskU8Hs9B+wsLC2X79u01ERcAAABQqoCZfgAA1C6HJP3ee+890/9UXlFRkTkGAAAQysKrc/I333xT+vPUqVMlKSmp9LkmAWfOnCnHHHNMzUYIAACAkJfvYaYfAAC1Kj9PnOBvf/ubnHfeedK4cWO//dnZ2ebY4MGDbYsNAADAUTP9LrnkErO5XC4ZMmRI6XPdrrrqKpk+fbo888wzVb7ea6+9Jt26dZPExESz9erVS7799tvS4wUFBTJs2DBp0KCBxMfHy6BBg2T37t1+10hNTZWBAwdKbGysafDdd999UlJSUp2PBQAAgABHeU8AAGqXU8p7WpZl+qXK27Ztm9/gdAAAgFBUrZl+Xu+Bskpt2rSRxYsXS8OGDY/qzVu0aCFPPvmktG/f3jTa3n33Xbn44ovll19+kc6dO8vw4cNl8uTJ8umnn5qG22233SaXXXaZzJ8/v3R2oSb8UlJSZMGCBbJz504zoisiIkLGjBlzVLEBAAAgMHgtS4q8JP0AAKhVAZ706969u0n26danTx8JD/9fl5b2D23evNnMAAQAAAhl1Ur6+WhDqiZceOGFfs+feOIJM/tv0aJFJiE4fvx4+fDDD6V3797m+IQJE+T44483x3v27CnTpk2T1atXy4wZM6RJkyZy4oknyujRo+WBBx6Qxx57TCIjI2skTgAAANiHWX4AANS+QJ/pp1Wm1LJly6R///6mIpSP9v/ocjNaIQoAACCUHVHST+n6fbqlpaWVzgD0efvtt6t9PR2VpTP6cnNzTZnPJUuWSHFxsfTt27f0nI4dO0qrVq1k4cKFJumnj127djUJPx9t+A0dOlRWrVplRoFVRBd8Lrvoc1ZWVrXjBQAAQN3I9/i3NQEAQC0oyJdA9uijj5pHTe5deeWVEh0dbXdIAAAAzl7Tz2fkyJHSr18/k/Tbu3evpKen+23VsWLFCjM6KyoqSm699Vb58ssvpVOnTrJr1y4zUis5OdnvfE3w6TGlj2UTfr7jvmOVGTt2rCkX6ttatmxZrZgBAABQdwo8zPQDAKDWlRSLE+hg8MoSfm+88UadxwMAAOD4mX6vv/66vPPOO3LdddcddQAdOnQwpRkyMzPls88+kyFDhsicOXOkNo0YMULuvvtuv5l+JP4AAAACUz7lPQEAqH0ejziBrtt3xx13yJgxYyQiIsLs0wHpf/vb32TevHlyyy232B0iAACAs2b6FRUVyWmnnVYjAehsvnbt2kmPHj3MDLwTTjhBXnjhBUlJSTHvk5GR4Xf+7t27zTGlj/q8/HHfscrorMLExES/DQAAAIGpgPKeAADUPock/WbNmmWqRJ1yyimyevVqmTx5snTp0sUM6NZB5QAAAKHsiJJ+N954o3z44Yc1H42IWR9Q19vTJKCO2NISoj5r166V1NRUs+af0kctD6rrCvpMnz7dJPG0RCgAAACcr5DyngAA1L6SEnECHYSuyT1N9J100kly6aWXyvDhw2X27NnSunVru8MDAABwXnnPgoIC+fe//y0zZsyQbt26lZZT8Hn22WerXGbz/PPPN/XYs7OzTSJRG2lTp041a+3dcMMNpgxn/fr1TSLv9ttvN4m+nj17mtfruoKa3NMyo+PGjTPr+D300EMybNgwM5sPAAAAzmeR8wMAoPZZXrEsr7hcRzQ+vE6tW7dOfv75Z2nRooXs2LHDDBLPy8uTuLg4u0MDAABwXtJv+fLlcuKJJ5qfV65c6XfM5XJV+To6Q2/w4MGyc+dOk+TTBKIm/M4991xz/LnnnhO32y2DBg0ys//69+8vr776aunrw8LCZNKkSTJ06FCTDNTGna4JOGrUqCP5WAAAAAhEVW9eAgCAoy3xGR7YSb8nn3xSHn30Ubn55pvlX//6l2zYsMEMBtc+pffff7+0OhQAAEAoCj/S+uk1Yfz48Yc8Hh0dLa+88orZKqOlG6ZMmVIj8QAAAAAAAIR20s+/mlOgeeGFF+Srr74ylaOUlvn86aef5MEHH5Szzz7bDBoHAAAIVUeU9AMAAADqChP9AACow3X9Any1lBUrVkjDhg399umyMzrr74ILLrAtLgAAAMcm/c4555xDlvH8/vvvjyYmAAAAAAAA2DHTL8Bpwi8jI0M+++wz2bhxo9x3331Sv359Wbp0qbRr187u8AAAAJyX9POt5+dTXFwsy5YtM+v76Zp6AAAAAAAAcBhPiQS65cuXS9++fSUpKUm2bNkiN910k0n6ffHFF5Kamirvvfee3SECAAA4K+n33HPPVbj/sccek5ycnKONCQAAAAAAAHXNATP9hg8fLtdff72MGzdOEhISSvcPGDBArr76altjAwAAsJu7Ji927bXXyttvv12TlwQAAECIY00/wHkywqIl4/iTpLBFG7ESkvh/MuAUDpjp9/PPP8stt9xy0P7mzZvLrl27bIkJAADA0TP9KrNw4UKJjo6uyUsCAAAAABxmb3icjG/fv/R5lLdEWhRnSUphpjTMz5DkvAyJy94vUVn7xZ2Z7ojZRQACQ1RUlGRlZR20f926ddKoUSNbYgIAAHB00u+yyy7ze25ZluzcudOMtnr44YdrKjYAAAAAQBAodIfLxqj6ZpNE/2Muy5KmJTnSrDBTGhVkSP28DInPSZforP0SlrlfXIUFdoUNhJ6wGh0bXqN0vb4WLVrIRRddJKNGjZJPPvnE7He5XObYAw88IIMGDbI7TAAAAFsdUWtOF0suy+12S4cOHUyjq1+/fjUVGwAAACAuqgICQc1yuWRHRILZJL7FQcfrefKlRVGWNNaEYH6GJOWkS0x2ukRoQjDn4Nk+AI5CeOAm/dq0aWMGnD/zzDNy+eWXS+PGjSU/P1/OOussU9azV69e8sQTT9gdJgAAgK2OqDU3YcKEmo8EAAAAAIBy0sNiJD0mRiSmiUg9/2Mx3uLSsqEN8g6UDY0tWzbU67UrbMCZAjjpp1WmfAPRp0+fLvPmzZPly5dLTk6OnHTSSdK3b1+7QwQAALDdUbXmlixZIr/99pv5uXPnztK9e/eaigsAAAAAgEPKd0fI+qgGZitfNtRteU3Z0OZaNjQ/Q+r9UTY0ylc2tKjQrrCBwBUeIYFMS3n6nHHGGWYDAADAUSb90tLS5KqrrpLZs2dLcnKy2ZeRkSHnnHOOTJw4kYWTAQAAAAC28rrcsj0i0WwS3/Kg4w1K8qR5cZY0zj9QNjQxJ93MEgzP2C+uvBxbYgZsF8Br+qmHH35YYmNjD3nOs88+W2fxAAAABJojas3dfvvtkp2dLatWrZLjjz/e7Fu9erUMGTJE7rjjDvnoo49qOk4AAACEKJewqB+AmrcvPNZsEpNy0LE4LRtalClNCjOloa4jmKsJwXSJzNwv7qwMEYuyoQhS4WESyFasWCGRkZFVmgkIAAAQio4o6ffdd9/JjBkzShN+qlOnTvLKK69Iv379ajI+AAAAAADqVK47QtZGNzSbJPkfC7c80qwkW5oWZEqjgkxTNjQue79EZ6VLWOY+keJiu8IGjo47TFwutwSyL7/8Uho3bmx3GAAAAAHriJJ+Xq9XIiIOrvOu+/QYAAAAUFMYtA8gkJS4wiQ1ItlsknDw8cbFudKsOFMaF2RK/bwMSchJlxgtG6rrCObl2hEyUDXhgV3ak1l8AAAAh3dELbrevXvLnXfeacp4NmvWzOzbvn27DB8+XPr06XMklwQAAAAqFBVGJx8A50iLiDObxDYTqe9/LN5TJC2KMyWlMFMa5GVIoq9saJavbKhlV9hAwCf9LP7/AQAAcFhH1KJ7+eWX5aKLLpJjjjlGWrY8sCD61q1bpUuXLvL+++8fySUBAACACsWQ9AMQJHLCImVNWCNZE93ooLKhEZZHmhdnS9PCDGmYnynJeRkSn7NfojL3S1jmfpGSErvCRqgIC+yk34QJEyQpqdz/cQAAAODniFp0muhbunSpWddvzZo1Zp+u79e3b98juRwAAABQqZjwwF5fCABqQrErTLZEJpvtoLKhliVNPLnSrChTGuf/UTY0N12is/ZLeMY+cRXk2xQ1gkqAz/QbMmSI3SEAAAAEvGq16L7//nu57bbbZNGiRZKYmCjnnnuu2VRmZqZ07txZXn/9dfnzn/9cW/ECAAAgxMSEM9MPQIhzuWR3eLzZJLa5SAP/w0meAmlenCUp+RlSP1/LhmZIbPZ+icjcL+7sLM0a2hU5nCTAk361YezYsfLFF1+YAe0xMTFy2mmnyVNPPSUdOnQoPaegoEDuuecemThxohQWFkr//v3l1VdflSZNmpSek5qaKkOHDpVZs2ZJfHy8SVDqtcND8DsFAAD2qlbr4/nnn5ebbrrJJPzK0xILt9xyizz77LMk/QAAAFBjYsKY6QcAh5IZFm221dGNRer5H4vylkjzkmxJKciQRvkZkqRlQ/9YR9CUDfV47AobAcYVHSOhZs6cOTJs2DA55ZRTpKSkRB588EHp16+frF69WuLi4sw5w4cPl8mTJ8unn35q+r50MPxll10m8+fPN8c9Ho8MHDhQUlJSZMGCBbJz504ZPHiwREREyJgxY2z+hAAAINRUK+n366+/mhFPldGG0dNPP10TcQEAAABGuNslEW6RYq/dkQCA8xS6w2VTZD2zSbnxuy7LkpSSnANlQwsypF5uhiTkpEt0drqEadnQwgK7woYd4srXlQ1+3333nd/zd955Rxo3bixLliyRM88801S1Gj9+vHz44YfSu3fv0rUFdYkbrYLVs2dPmTZtmkkS6hI4OvvvxBNPlNGjR8sDDzwgjz32mERGRtr06QAAQCiqVtJv9+7dZqRSpRcLD5c9e/bURFwAAACA32y/Yi9ZPwCoSZbLJTsjEswmcS0OKhuaXJIvLYqzpEmBriOY7lc21JWTTdnQIOOKPTCzzQkyMjLks88+k40bN8p9990n9evXl6VLl5qkW/PmzY/4uprkU3o9pcm/4uJi6du3b+k5HTt2lFatWsnChQtN0k8fu3bt6lfuU0uAarnPVatWSffu3Q96Hy0TqptPVpaW4QUAAKjjpJ82nFauXCnt2rWr8Pjy5culadOmNRAWAAAA8D/R4S7JKrY7CgAILRnhMWZbGdOkgrKhxdKyOFtSCjOlYX66JOVmSFxOukTpOoKZ6SJeyoY6Tly8OIH2PWkSTkttbtmyxSxDo0k6XZtP19Z77733jui6Xq9X7rrrLjn99NOlS5cuZt+uXbvMTL3k5GS/czXBp8d855RN+PmO+45VRNf7Gzly5BHFCQAAUGNJvwEDBsjDDz8s5513nkRHR/sdy8/Pl0cffVQuuOCC6lwSAAAAqOK6fnQgA0CgKHRHyIao+maTxDZ+x9yWV5pq2dDCTGlUkCn18tIlXsuGZv1RNrTofzOcEEAcMtPv7rvvluuvv17GjRsnCQkJfn1WV1999RFfV9f204Hu8+bNk9o2YsQI8znKzvRr2bJlrb8vAAAIftVK+j300ENm5NRxxx1nFi7u0KGD2b9mzRp55ZVXzOLF//znP2srVgAAAISomHCX3SEAAKrI63LL9ohEs0n8wYmM+p48aV6oZUMzpH5+hiTmpEtMdrpEZO4TV26OLTFDxOWQmX6LFy+WN954o8LqVJXNrDsc7eOaNGmSzJ07V1q0aFG6PyUlRYqKikw50bKz/XT5Gz3mO+enn37yu54e9x2rSFRUlNkAAABsTfppeYIFCxaYuuQ6KsmyDtTvd7lcpl65Jv7KlzQAAAAAjlZMuM70AwAEg/1hsbI/NlZWxB6cEInzFkuzokxpWpgpDfIzDpQNzd4vkVn7xZ2VoTUYbYk5JDgk6afJsorWwFu3bp00atSoWtfSfq3bb79dvvzyS5k9e7a0aeM/a7VHjx4SEREhM2fOlEGDBpl9a9euNWVEe/XqZZ7r4xNPPCFpaWnSuHFjs2/69OmSmJgonTp1OopPCgAAUMtJP9W6dWuZMmWKpKeny4YNG0wDqX379lKvXrkC/wAAAEANiQljph8AhIJcd4Ssj25oNknyPxbm9UqzkmyTFGyUnyHJeRmmbGhU1n4Jz9gnUszir0fDFeuMpN9FF10ko0aNkk8++aR0ILom4R544IHSxFx1Snp++OGH8vXXX5tSob6ZgrpeYExMjHm84YYbTClOXTdQE3maJNREX8+ePc25/fr1M8m96667zpQc1WtopSy9NrP5AABAwCf9fDTJd8opp9RsNAAAAEAFopnpBwAhz+N2y9bIJLNJfKuDjjcsyZUWRVnSuCBD6uVlSGJuusRkpUu4lg3Ny7UlZkeJc8aafs8884xcfvnlZlZdfn6+nHXWWSbR5ptxVx2vvfaaeTz77LP99k+YMMGsG6iee+45cbvdJqFYWFhoKl29+uqrpeeGhYWZ0qBaFUtjiIuLkyFDhpjEJAAAgGOSfgAAAEBdYaYfAOBw9obHmU1im4rU9z8W5ymSFsWZkqJlQ/MOlA2NzdkvkZlaNjRTxKJsqMQ6I+mns++0fOa8efNk+fLlkpOTIyeddJL07du32tfyLVtzKNHR0WY5G90OVxULAADAbiT9AAAAEPBY0w8AcDRywyJlbVgjWRvd6KCyoeGWR5oXZ5t1BMuXDQ3L2C9SEgJlQ6OixRXmrC6iM844w2wAAAD4H2e16AAAABCSYsKZ6QcAqB0lrjD5PTLZbJLQ2v+gZUljT540L8yQxgWZUj8vQxJy0yVa1xHM3C+u/DwJCnHOWM9PvfjiixXu17X9dFZeu3bt5MwzzzRlNwEAAEINST8AAAAEvFhm+gEA7OBySVp4nNkkrrlIA//DCZ5CaVmUJU0KM6RB/oF1BGOz0iUia7+4s7Vs6OHLRwYCl4OSfrrG3p49eyQvL0/q1atn9qWnp0tsbKzEx8dLWlqatG3bVmbNmiUtW7a0O1wAAIA6RdIPAAAAAS8xwi06188ZXacAgFCRHRYlq2MamU2S/Y9FekukRXG2pBRmSMOCTEnOzZC4nP0SlblfwjLTRTwlEjCSDyTPnGDMmDHy73//W9566y059thjzb4NGzbILbfcIjfffLOcfvrpctVVV8nw4cPls88+sztcAACAOkXSDwAAAAEvzO2ShAi3ZBV77Q4FAIAqKXKHy6aoemaTRP9jLsuSlJJcaVaUKY0KMqR+bobE56ZLjK4jqGVDC/LrNFZXcrkpjAHsoYceks8//7w04ae0pOfTTz8tgwYNkk2bNsm4cePMzwAAAKGGpB8AAAAcITkqjKQfACAoWC6X7IyIN1tFZUOTPAXSQsuGFvyvbGhMdrpEakIwO6vG5767HDTTb+fOnVJScvAsSd23a9cu83OzZs0kOzvbhugAAADsRdIPAAAAjpAc6ZZUu4MAAKAOZIZFS2ZMtKyKaSxSLh8X5S3+o2xopjTMz5DkvAyJy94vUbqOoCkb6qn+GybXF6c455xzTClPLe/ZvXt3s++XX36RoUOHSu/evc3zFStWSJs2bWyOFAAAoO6R9AMAAIBjZvoBABDqCt0RsjGqvtkqKhvatCRHmhX+UTY0L0Pic9Il2lc2tLCgwmu6HJT0Gz9+vFx33XXSo0cPiYiIKJ3l16dPH3NMxcfHyzPPPGNzpAAAAHWPpB8AAAAcgaQfAACHLxu6IyLBbBLf4qDj9Tz50qIwU5oUZkr9vHRJzMmQmJx0CU9yTnnPlJQUmT59uqxZs0bWrVtn9nXo0MFsZWcDAgAAhCKSfgAAAHBMeU8AAHDk0sNiJD02RlbEppSWDU2KdMvQMOcNrOnYsaPZAAAA8D8k/QAAAOAI9ZjpBwBAjavvwPvrtm3b5JtvvpHU1FQpKiryO/bss8/aFhcAAIDdSPoBAADAEWLC3RId5pICj2V3KAAABI360c5K+s2cOVMuuugiadu2rSnx2aVLF9myZYtYliUnnXSS3eEBAADYihpJAAAAcIwGDuuYBAAg0Dltpt+IESPk3nvvlRUrVkh0dLR8/vnnsnXrVjnrrLPkiiuusDs8AAAAW5H0AwAAgGM0cFjHJAAAgc5pSb/ffvtNBg8ebH4ODw+X/Px8iY+Pl1GjRslTTz1ld3gAAAC2IukHAAAAx2CmHwAAoV3eMy4urnQdv6ZNm8rGjRtLj+3du9fGyAAAAOzHmn4AAABwjAbRNF8BAKgp4S6RxAhnjQfv2bOnzJs3T44//ngZMGCA3HPPPabU5xdffGGOAQAAhDJ6TQAAAOAYzPQDAKDm1IsKE5fLJU7y7LPPSk5Ojvl55MiR5uePP/5Y2rdvb44BAACEMpJ+AAAAcIzkSLeZlVBi2R0JAADO19Bhg2k8Ho9s27ZNunXrVlrq8/XXX7c7LAAAgIDhrBoOAAAACGk6G0FnJQAAgKOXEuusseBhYWHSr18/SU9PtzsUAACAgETSDwAAAI7itFkJAAAEqmZxEeI0Xbp0kU2bNtkdBgAAQEAi6QcAAABHaerADkoAAAKNy4Ez/dTjjz8u9957r0yaNEl27twpWVlZfhsAAEAoc17rDgAAACGteRxNWAAAamLmfIRbU3/OMmDAAPN40UUXmbLfPpZlmee67h8AAECooscEAAAAjtIkJlzCXCIey+5IAABwrmYOHUQza9Ysu0MAAAAIWM5s4QEAACBkhbtdJvG3I6/E7lAAAHCsprHOLJd91lln2R0CAABAwGJNPwAAADiOU2cnAAAQKJo6cD0/nx9++EGuvfZaOe2002T79u1m33/+8x+ZN2+e3aEBAADYiqQfAAAAHKd5nDNnJwAAEAgi3CKNYsLEiT7//HPp37+/xMTEyNKlS6WwsNDsz8zMlDFjxtgdHgAAgK1I+gEAAMBxmOkHAMCR0zLZbpdLnOjxxx+X119/Xd58802JiPjfIKDTTz/dJAEBAABCGUk/AAAAOE5SZJjE6zQFAAAQUqU9165dK2eeeeZB+5OSkiQjI8OWmAAAAAIFPSUAAABwpGYO7rAEAMBOTR1cJjslJUU2bNhw0H5dz69t27a2xAQAABAoSPoBAADAkZpT4hMAgJAbOHPTTTfJnXfeKT/++KO4XC7ZsWOHfPDBB3LvvffK0KFD7Q4PAADAVs5t5QEAACCkNXPwLAUAAOwSE+6S5Kgwcap//OMf4vV6pU+fPpKXl2dKfUZFRZmk3+233253eAAAALYi6QcAAABHSokNF7dLxGvZHQkAAM7h5PX8lM7u++c//yn33XefKfOZk5MjnTp1kvj4eLtDAwAAsB3lPQEAAOBIEW6XNI5xdsclAAB1zelJv/fff9/M8IuMjDTJvj/96U8k/AAAAP5A0g8AAACO5eQ1iQAAsEOzWGeXxx4+fLg0btxYrr76apkyZYp4PB67QwIAAAgYJP0AAADgWM3jSPoBAFCdTqDm8c6+d+7cuVMmTpxoynz+5S9/kaZNm8qwYcNkwYIFdocGAABgO5J+AAAAcKzmcc6erQAAQF3ShF90mLO7gsLDw+WCCy6QDz74QNLS0uS5556TLVu2yDnnnCPHHnus3eEBAADYytnDuwAAABDSkqPCJDHCLVnFXrtDAQAg4B2bGCnBJDY2Vvr37y/p6eny+++/y2+//WZ3SAAAALZy9vAuAAAAhLxjk4KrAxMAgNrSNkiSfnl5eWam34ABA6R58+by/PPPy6WXXiqrVq2yOzQAAABbMdMPAAAAjp+18MveArvDAAAgoCVEuKVxjPO7ga666iqZNGmSmeWna/o9/PDD0qtXL7vDAgAACAjOb+0BAAAgpB2TECERbhEqfAIAULm2icGxDm5YWJh88sknpqyn/lzWypUrpUuXLrbFBgAAYDeSfgAAAHC0cLdLWsVHyMasYrtDAQAgYAVLaU8t61lWdna2fPTRR/LWW2/JkiVLxOPx2BYbAACA3VjTDwAAAI7XjnX9AAColNt1YGZ8MJk7d64MGTJEmjZtKk8//bT07t1bFi1aZHdYAAAAtmKmHwAAAByvXWKkTJVcu8MAACAgtYiLkKgw54/73rVrl7zzzjsyfvx4ycrKMmv6FRYWyldffSWdOnWyOzwAAADbOb/FBwAAgJCXEBkmjWP81/UBAAAHHBsE6/ldeOGF0qFDB1m+fLk8//zzsmPHDnnppZfsDgsAACCgMNMPAAAAQTPbLy0/3+4wAAAIOMGwnt+3334rd9xxhwwdOlTat29vdzgAAAABiZl+AAAACAqs6wcAwMESI9zSKMb5Y77nzZsn2dnZ0qNHDzn11FPl5Zdflr1799odFgAAQEAh6QcAAICg0DQ2XGLDXXaHAQBAQAmGWX6qZ8+e8uabb8rOnTvllltukYkTJ0qzZs3E6/XK9OnTTUIQAAAg1JH0AwAAQFBwuVxybJB0bAIAUFPaBsF6fmXFxcXJ//3f/5mZfytWrJB77rlHnnzySWncuLFcdNFFdocHAABgK5J+AAAACBrHUuITAIBSYS6RYxKC997YoUMHGTdunGzbtk0++uijar9+7ty5cuGFF5oZgzp46KuvvvI7fv3115v9ZbfzzjvP75z9+/fLNddcI4mJiZKcnCw33HCD5OTkHPVnAwAAcFzSb+zYsXLKKadIQkKCGZF1ySWXyNq1a/3OKSgokGHDhkmDBg0kPj5eBg0aJLt37/Y7JzU1VQYOHCixsbHmOvfdd5+UlJTU8acBAACA3dokRJgOTgAAINIiLkIiQ+DGGBYWZvqUvvnmm2q9Ljc3V0444QR55ZVXKj1Hk3xaUtS3lU8uasJv1apVpsTopEmTTCLx5ptvPuLPAgAAcDRsXcl5zpw5JqGniT9N0j344IPSr18/Wb16tSnXoIYPHy6TJ0+WTz/9VJKSkuS2226Tyy67TObPn2+Oezwek/BLSUmRBQsWmAbY4MGDJSIiQsaMGWPnxwMAxyseeY/dISBARDz6jN0hAFUSFeaWlvERsiW72O5QAACwXTtmwB/S+eefb7ZDiYqKMn1OFfntt9/ku+++k8WLF8vJJ59s9r300ksyYMAAefrpp80MQgAAgJCZ6acNIy2V0LlzZzOy6p133jGz9pYsWWKOZ2Zmyvjx4+XZZ5+V3r17S48ePWTChAkmubdo0SJzzrRp00yS8P3335cTTzzRNNZGjx5tRmkVFRXZ+fEAAABgg3as6wcAgOj8vk71ouwOw/Fmz55tqkppKdGhQ4fKvn37So8tXLjQlPT0JfxU3759xe12y48//ljpNQsLCyUrK8tvAwAACLo1/TTJp+rXr28eNflXXFxsGkw+HTt2lFatWpmGldLHrl27SpMmTUrP6d+/v2kwaXmFitC4AgAACF7MagAAQKRNYoTERQRUt4/jaGnP9957T2bOnClPPfWUqVilg8216pTatWuXSQiWFR4ebvq19NihlrvRala+rWXLlrX+WQAAQGgImNaf1+uVu+66S04//XTp0qWL2acNpMjISDNqqixN8PkaT/pYNuHnO+47VhEaVwAAAMErOSpMWsTZWsUeAADbdakfbXcIjnfVVVfJRRddZAab65qBumaflvLU2X9HY8SIEWbgu2/bunVrjcUMAABCW8Ak/XRtv5UrV8rEiRNr/b1oXAEAAAS3rg3o6AQAhK4ot0vaM/O9xrVt21YaNmwoGzZsMM91rb+0tDS/c0pKSmT//v2VrgPoWycwMTHRbwMAAAiapN9tt91mRkvNmjVLWrRoUbpfG0i6Ll9GRobf+bt37y5tPOmjPi9/3HesIjSuAAAAgtvxyVFCRTMAQKjqkBwpEW5d1Q81adu2bWZNv6ZNm5rnvXr1Mn1WujyNz/fff2+qWZ166qk2RgoAAEKVrV0hlmWZhN+XX35pGkVt2rTxO96jRw+JiIgwtdN91q5dK6mpqaZhpfRxxYoVfiOrpk+fbhJ5nTp1qsNPAwAAgEARGeaSDslRdocBAIAtKO1ZNTk5ObJs2TKzqc2bN5uftd9Jj913332yaNEi2bJli+mbuvjii6Vdu3bSv39/c/7xxx9v1v276aab5KeffpL58+ebfi4tC9qsWTObPx0AAAhF4XaX9Pzwww/l66+/loSEhNI1+HSdvZiYGPN4ww03yN13320WQdZE3u23324SfT179jTn9uvXzyT3rrvuOhk3bpy5xkMPPWSurTP6AAAAEJq61o+SlfsL7Q4DAIA6lRjplpbxrG1bFT///LOcc845pc+1/0kNGTJEXnvtNVm+fLm8++67ZjafJvG0D2r06NF+/U0ffPCBSfT16dNH3G63DBo0SF588UVbPg8AAICtrUBtQKmzzz7bb/+ECRPk+uuvNz8/99xzpY2mwsJCM5rq1VdfLT03LCzMlAYdOnSoSQbGxcWZxtmoUaPq+NMAAAAgkLSKj5CkSLdkFnntDgUAgDrTpV6UuFyU9qwK7Y/SKlSVmTp16mGvoYPUdUA7AACAhHrS71ANK5/o6Gh55ZVXzFaZ1q1by5QpU2o4OgAAADiZdnh2qR8l83fl2x0KAAB1htKeAAAAocvWNf0AAACA2tSVjk8AQAhpGhsu9aPD7A4DAAAANiHpBwAAgKCVHBVmynwCABAKdIY7AAAAQhdJPwAAAAS1rnSAAgBCgNsl0qke9zwAAIBQZuuafgAAAEBt65AcJdO35UqR9/DrSQMA4FTHJkZKTDhju+FcxSPvsTsEBIiIR5+xOwQAcCxagwAAAAhqkWEu6ZAcaXcYAADUKkp7AgAAgJl+AAAACHpdG0TLiv2FdocRUp4aeJJk7Nx60P6eV/xNzv37CJnx+lOyftFsydi1XeLqNZBOZ58v/YaOkOiExEqvWZiXI9+9OFpWz/5W8jLTpX6zVnLaX2+SUy+/vvScSc88LEv/O1EiY2Kl/+0PS/cBl5ceWzH9a1k66RMZ8sIHtfCJAcA+0WEuaZfIABcAAIBQR9IPAAAAQa9VfIQkR7olo8hrdyghY9j708TyeEqf7964RsYPvVy6nnuxZO3ZZbYBd42Uxm2Pk4yd2+TLMfdK9p5dcs2/JlR6zcnPPCIbF/8gVz7+mtRr1lLWL5wtXz95vyQ0SpFOZ50nv82ZKr9+97n836ufyt7UTfL5yDvluF7nmKRiQXaWTH1ljNz42ud19A0AQN3pXD9KwnRRPwAAAIQ0ynsCAAAgZGb7oe7E12soCQ2blG6/zZ0m9VscI216nCYp7Y6Xa59+R44/q780aNlGjv3Tn6X/sAfNOZ6Skkqvmbp8sZx04VXS9uTTpV6zVvKnQYMlpX1n2bZyqTmetnmdtO1xurTodKKceN5lEh2XIPt3pJpj374wUnpe/jdJbtqizr4DAKgLmuo7uVGM3WEAAAAgAJD0AwAAQEjoWj+Kxq9NSoqLZNm3n8nJF18tLlfFM1EKcrJMki4svPJiJK26nSK/zflOMtN2imVZsnHxPNmbulHa9zzbHG96XGfZtnqZ5GdlyPbVv0pxYb40bNlGtvyySLavWW5KgQJAsGmfFCn1osLsDgMAAAABgPKeAAAACAmJkWHSsV6UrE5nbb+6tnrWFCnIzpQeF/21wuO56fvk+zeflVMuu+6Q17nogbHyxeN3y5PndRN3eLi4XG657OFnzexBddxpvaX7gCvk5WvPlYjoGLli5MsSERMrX429X6547CVZ9OkEWfjxWxKXXF8ufehZaXJsx1r5vABQl/7UmFl+AAAAOICkHwAAAELGqY1jSPrZ4OevPpDjTusjiY1SDjpWkJMt79x5tVnbr+8t9x/yOgsmviVbVyyRwc+9b8p0bl66UL5+8gFz3XannmXO6Xvr/WbzmfHGv6Tdn840ScJZ45+VOz+ZK2vmTpNPHh4mt384sxY+LQDUnWax4dIiPsLuMAAAABAgqHAEAACAkNEkNlyOSaBztC6l79gqG36aK6dceu1Bxwpzc2TCbVdKVGy8XPvMuxIWUfnvprggX6a9/IQMvHuUWQtQS3medtWN0q3fJTL3vVcqfE3a5vWybMqncu7f/yGblsyXNif1MmsNdut3sexYs9y8PwA4GbP8AAAAUBZJPwAAAISUnnSQ1qkl33wk8fUbSoczzj1oht/4v19hEn2Dn/uPRERFH/I6npIS8ZQUi8vt/08YtzvMrO9Xnu776ol7ZODdo01S0fJ4zTV811Jer6cGPiEA2CMp0i3HJUfaHQYAAAACCEk/AAAAhJRjEiOlSUyY3WGEBK/Xa5J+J11wpYSFh/sl/N7++xVSnJ8ngx55XgpzsyV7726zeT3/S8Q9e1kvWfX9ZPNzdHyCWbvv2+dHyqaf58v+7b+bay+d/Il0PmfAQe+9+Mv3Ja5eAzMrULU+8U+ycfEPkrr8Z5n3/uvSuG0HiUlIqpPvAQBqw8mNYsTtctkdBgAAAAIIa/oBAAAg5JzaJFa+2ZJtdxhBb8OPcyRj1zbpcfE1fvu1tObWlUvMz09f/Ce/Y/dPWiL1mrUyP+/ZskEKcrJKj/117L9l6kuPy8f/vFXysjKkXtMW0m/Yg3Lq5df7XSN7X5rMGv+cDH1nSum+ll1Okj9f+3ezfqCW+Lxi1Mu18pkBoC5EhbnkhAaHniENAACA0EPSDwAAACGnY3KkzIl0S2aR1+5Qgtpxvc6RsUv3HLS/7cmnV7i/vPLnJDRsIpePfOmwr0to0FgemLz0oP19br7XbADgdCc2iJbIMGb5AQAAwB/lPQEAABBytBzaKaztBwBwILdLS3syyw8AAAAHI+kHAACAkKRl0WKYJQEAcJjjk6MkIZK1aQEAAHAwkn4AAAAISRFul5zETAkAgMP8iZnqAAAAqARJPwAAAISsHo1iJJzJfgAAh2gdHyFNYsPtDgMAAAABiqQfAAAAQlZsuFu6NWC2HwDAGZjlBwAAgEMh6QcAAAAJ9Q5UJvsBAAJdg+gwaZsYYXcYAAAACGAk/QAAABDSkqPCpENypN1hAABwSGekxIrLxTAVAAAAVI6kHwAAAEJezyaxdocAAEClUmLDpSMDVAAAAHAYJP0AAAAQ8rQztX0SnakAgMB0djNm+QEAAODwSPoBAAAAInJOszgaxwCAgNMmIUKOSWBgCgAAAA6Pfg0AAABAROpHh8mJDaPtDgMAAD9nNYuzOwQAAAA4BEk/AAAA4A+np8RKpJvyaQCAwNCpXpQpQQ0AAABUBUk/AAAA4A9xEW45tUmM3WEAACA6BuXMprF2hwEAAAAHIekHAAAAlPGnxjESH0EzGQBgrxMbREtyVJjdYQAAAMBB6M0AAAAAyohwu+TPzKwAANhIS01ryWkAAACgOkj6AQAAAOV0qx8ljaKZXQEAsMcpjaNNyWkAAACgOmhBAgAAAOW4XC45u1mc3WEAAEJQbLhLTm3MLD8AAABUH0k/AAAAoALHJkVK6/gIu8MAAISY01JiJTLMZXcYAAAAcCCSfgAAAEAlzmnObD8AQN1JjnRL94bRdocBAAAAhyLpBwAAAFQiJTZcOteLsjsMAECIOLNpnIS5mOUHAACAI0PSDwAAADiEM5vFSjj9rwCAWtYkJkyOrxdpdxgAAABwMJJ+AAAAwCEkRYZJj0YxdocBAAhiOrakX8t4cTHLDwAAAEeBpB8AAABwGL1SYiSG6X4AgFqi6/g1j4uwOwwAAAA4HEk/AAAA4DCiw9zSp3mc3WEAAIJQQoRbzmoWa3cYAAAACAIk/QAAAIAq6FI/WtomMAsDAFCz+rWMk6gwumcAAABw9GhVAgAAAFXUv1W8RLop8wkAqBnHJUVK+6Qou8MAAABAkCDpBwAAAFRRUmQYJdgAADUiKswl57akdDQAAABqDkk/AAAAoBpOahgtLeLC7Q4DAOBwZzWNlYSIMLvDAAAAQBAh6QcAAABUg8vlkvNbxUsYVT4BAEdIB490bxhtdxgAAAAIMiT9AAAAgGpqEB0up6VQ5hMAUH06aOS8lvFmEAkAAABQk0j6AQAAAEegZ5MYaRRNWTYAQPWc2iRGGsZQJjoQzJ07Vy688EJp1qyZScJ+9dVXfscty5JHHnlEmjZtKjExMdK3b19Zv3693zn79++Xa665RhITEyU5OVluuOEGycnJqeNPAgAAcABJPwAAAOAIhLlcMqB1vDBPAwBQVfWjwuS0JswUDxS5ublywgknyCuvvFLh8XHjxsmLL74or7/+uvz4448SFxcn/fv3l4KCgtJzNOG3atUqmT59ukyaNMkkEm+++eY6/BQAAAD/w9AyAAAA4Ag1jY2QUxrHyE9p+XaHAgBwgPNaxUu4m+EigeL88883W0V0lt/zzz8vDz30kFx88cVm33vvvSdNmjQxMwKvuuoq+e233+S7776TxYsXy8knn2zOeemll2TAgAHy9NNPmxmEAAAAdYmZfgAAAMBR+HPTWEmOpFkNADi0ExpESav4CLvDQBVt3rxZdu3aZUp6+iQlJcmpp54qCxcuNM/1UUt6+hJ+Ss93u91mZiAAAEBdo3cCAAAAOAoRbpec3yre7jAAAAEsLtwl5zSLszsMVIMm/JTO7CtLn/uO6WPjxo39joeHh0v9+vVLz6lIYWGhZGVl+W0AAAA1gaQfAAAAcJRaJ0SaGRwAAFSkf8t4iQ6nCwYHjB071swa9G0tW7a0OyQAABAkaHECAAAANeCc5nESH0HzGgDgr0ejaDkumYEhTpOSkmIed+/e7bdfn/uO6WNaWprf8ZKSEtm/f3/pORUZMWKEZGZmlm5bt26tlc8AAABCD70SAAAAQA2IDnPLwFbx4rI7EABAwGgSE0ZZT4dq06aNSdzNnDmzdJ+W4dS1+nr16mWe62NGRoYsWbKk9Jzvv/9evF6vWfuvMlFRUZKYmOi3AQAA1ITwGrkKAAAAAGmTGCm9UmJkwa58u0MBANgs0u2SS9okSrib4SCBKicnRzZs2FD6fPPmzbJs2TKzJl+rVq3krrvukscff1zat29vkoAPP/ywNGvWTC655BJz/vHHHy/nnXee3HTTTfL6669LcXGx3HbbbXLVVVeZ8wAAAOoaST8AAID/b+9OgOSqzvthv909S88+2vcNCSE2CRBYkCCzWAREIBbGbIYADpvjYMrowxBStliMLeOKsYEC5MRGGAJhcSohwTHfZ6BQ/mwGiT9g9tUGBxCLQKNtRjOa+epeMYMEEiBZ0u2eeZ7KrZ7b93bPaZy5On1+97wHtqDpQ2vjj8s74tXl7Vk3BYAMHTyqLvpVF7JuBp9g4cKFccABB/Tsz549O3086aST4rrrrotzzz03VqxYEaeffno6o2/fffeNO++8M4rFYs9rbrzxxjTo+8IXvhD5fD6OPPLIuOKKKzL5PAAAQj8AANiCcrlc/NXYhpj/7HuxoqMr6+YAkIHJ/atj5/4fBkOUpv333z+6uro+8d/0iy++ON02JpkVeNNNN22lFgIAbBpr+gEAwBZWX5mPw8c2WN8PoA8aWCzEQaPqs24GAAB9kNAPAAC2grENVfHnQ2uzbgYA21BFLuKLYxui0jp+AABkQOgHAABbyZ8PrYmxDZVZNwOAbWTGyPoYVGMlFQAAsiH0AwCArSRZC+jwMQ1RX6HbDdDbTWquit0GWscPAIDsGH0AAICtqK4yH39lfT+AXq25Kh8zR1vHDwCAbAn9AABgKxvdUBnTh1nfD6A3yn+wjl91wRALAADZ0iMFAIBtYJ8hNTHO+n4Avc7+w+tiWJ3rOwAA2RP6AQDAtlrfb2xDNFTqggP0FuMbK2OvQdbxAwCgNBhxAACAbaS2wvp+AL1F/+pCHD6mIb2pAwAASoHQDwAAtqFR9ZWx33Dr+wGUs2IhF1/erjGKFYZVAAAoHXqnAACwje09pDZ27leddTMA2MyBlFnjGqJ/sZB1UwAAYD1CPwAAyMCho+tjZF1F1s0AYBMdNKouxjZUZd0MAAD4GKEfAABkoJDPxZHbNUa/al1ygHKxx8Bi7D6wJutmAADABhlhAACAjNRU5NeuCVXIZd0UAD7FuIbKmDGyLutmAADARgn9AAAgQwOKFXHEuIbIy/0AStaAYiG+mF6rXawBAChdQj8AAMjYmIaqOHhUfdbNAGAD6ipycfT4ZFa2IRQAAEqbHisAAJSAKQOK8WdDrBMFUEqq8rk4anxTNFUVsm4KAAB8KqEfAACUiM8Pr4td+1dn3QwAPhgw+eLYhhhaW5F1UwAA4DMR+gEAQAmZObo+tmuszLoZAH3eX4yqj/FNVVk3AwAAyiP0+5//+Z84/PDDY/jw4ZHL5eI//uM/1jve1dUVc+bMiWHDhkVNTU3MmDEjXnjhhfXOWbJkSRx//PHR2NgYzc3Nccopp8Ty5cu38ScBAIAtI5/LxayxjTHMzBKAzCTllncbWMy6GQAAUD6h34oVK2LKlClx1VVXbfD4D3/4w7jiiiti3rx58dvf/jbq6uri4IMPjtbW1p5zksDvqaeeit/85jdxxx13pEHi6aefvg0/BQAAbFlVhVwctV1j9KtWmANgW9u5X3VabhkAAMpNprcPz5w5M902JJnl95Of/CS+/e1vxxe/+MX0ueuvvz6GDBmSzgg89thj45lnnok777wzHnnkkdhzzz3Tc6688so49NBD4x//8R/TGYQAAFCOaivzccz4prj++fdjZUdX1s0B6BMmNlXFoWPqs24GAABslpK9dfiVV16JN998My3p2a2pqSmmTZsWDz74YLqfPCYlPbsDv0Ryfj6fT2cGAgBAOWuuLsTR45uiOp/LuikAvd72TVXxxXENUci55gIAUJ5KNvRLAr9EMrNvXcl+97HkcfDgwesdr6ioiP79+/ecsyFtbW3R0tKy3gYAAKVoaG1FHDOhMaoLBqEBtpbxjZUxa6zADwCA8layod/WNHfu3HTWYPc2atSorJsEAAAbNbyuMo4ZL/gD2FqB35fGNUbBrGoAAMpcyYZ+Q4cOTR8XL1683vPJfvex5PGtt95a73hHR0csWbKk55wNOf/882Pp0qU922uvvbZVPgMAAKXruuuuS0vFl1Pwd6zgD2CL2q6hMo4Q+AEA0EuUbOg3bty4NLi7++67e55LynAma/Xts88+6X7y+P7778eiRYt6zrnnnnuis7MzXftvY6qrq6OxsXG9DQCA8nTyySdHLpf72Pbiiy9GbzOsrjKOm9AURcEfwJ9sXENlfGm7xqgQ+AEA0EtUZPnLly9fvt5gzCuvvBKPPfZYuibf6NGj45vf/GZccsklsf3226ch4He+850YPnx4zJo1Kz1/xx13jEMOOSROO+20mDdvXrS3t8eZZ54Zxx57bHoeAAB9Q9InnD9//nrPDRo0KHrrGn/HTmiKm19cGq1rurJuDkBZGlMv8AMAoPfJdKbfwoULY/fdd0+3xOzZs9Of58yZk+6fe+658Y1vfCNOP/302GuvvdKQ8M4774xisdjzHjfeeGNMmjQpvvCFL8Shhx4a++67b/zTP/1TZp8JAIBtL6nkkFSJWHe7/PLLY9ddd426urp0Deevf/3raX9yYx5//PE44IADoqGhIa0EMXXq1LS/2u2+++6L6dOnR01NTfp+Z511VqxYsSKyDP7M+APYdKPrK+PL4xujUuAHAEAvk+lMv/333z+6ujZ+d3JSluniiy9Ot41JZgXedNNNW6mFAACUq3w+H1dccUVaMeLll19OQ7/kprKrr756g+cff/zx6Q1o11xzTRQKhbQCRWVlZXrspZdeSmcTJlUorr322nj77bfTChPJ9tEZhtsy+Dvugxl/q8z4A/hMRtVXxFECPwAAeqlMQz8AANgS7rjjjqivr+/ZnzlzZtx22209+2PHjk0Du6997WsbDf1effXV+Na3vpVWkUgkJea7zZ07Nw0Fk/Lz3ceSQHG//fZLQ8J1K1FsS0OS4G/7pvjXJPjrEPwBfJKRdRVx1HZNAj8AAHotoR8AAGUvKcuZhG/dkpKed911VxrWPfvss9HS0hIdHR3R2toaK1eujNra2o+9R1Jq/tRTT40bbrghZsyYEUcddVSMHz++p/TnE088kZaW75ZUrOjs7EzXpU7Wms7K4JoPZ/ytFPwBbDTwO3p8U1QpiwwAQC+W6Zp+AACwJSQh34QJE3q2tra2OOyww2Ly5Mnxb//2b7Fo0aK46qqr0nNXr169wfe48MIL46mnnoq//Mu/jHvuuSd22mmn+Pd///f0WLIW4BlnnJGW/OzekiDwhRde6AkGs9Qd/NVWGMwG+KgRyQy/8Y0CPwAAej0z/QAA6HWSkC+ZhfejH/0oXdsvceutt37q6yZOnJhuZ599dhx33HHpen1HHHFE7LHHHvH000+ngWKpGvRB8JeU+jTjD2CtcQ2VMWtcQ1QX3PMMAEDvp9cLAECvk4Rz7e3tceWVV8bLL7+cluycN2/eRs9ftWpVnHnmmXHvvffGH/7wh7j//vvjkUce6Snbed5558UDDzyQnpPM8ktm+N1+++3pfqkFf1+Z0BT1lbr5AJP7V6cz/AR+AAD0FXq+AAD0OlOmTInLLrssLr300thll13StfiS9f02plAoxLvvvhsnnnhiOtPv6KOPjpkzZ8ZFF12UHk/KhC5YsCCef/75mD59euy+++4xZ86cGD58eJSagTUVceLEphhULGTdFIDM7Du0Ng4d0xD5nJKeAAD0Hbmurq4+X/unpaUlmpqaYunSpdHY2Bh91Q/+7ztZN4ES8fe7D8y6CZSI9ov+n6ybQImovOBH0dfpL1Bu2tZ0xu2vLIuXl7Vn3RSAbSafi5g5qj52HVDMuinwmelnruX7J918/6Sb8Wq6Ga+Oz9xfMNMPAAB6oaSc3ZfHN8buAw18A31DdT4XR2/XKPADAKDPqsi6AQAAwNaRlLU7eFR9NFfl497XV0afL/EB9FoNlfl0/b7BNYY5AADou/SGAQCgl5s2pDaaqwtxxx+WRXtn1q0B2LKSNUyTwK+xylqmAAD0bUI/AADoA3Zoro7Gynz88uWWWNFhzh/QO4xtqIwjxjWkJY0BAKCv0ysGAIA+YlhdZZy4Q3MMLJoNA5S/XftXpzP8BH4AALCWnjEAAPQhTVWFOGFiUzo7BqBc/fnQmvjLMQ1RyOWybgoAAJQMoR8AAPQxxUI+jh7fGFMGVGfdFIBNHsQ4dHR9TB9Wl3VTAACg5FjTDwAA+qB8LhczRzdEv+pC3Pv6yqybA/Cpaity8VdjG2JsQ1XWTQEAgJIk9AMAgD5s7yG10VxViF+9uizaO7NuDcCGjayriC+ObYiGKmuSAgDAxgj9AACgj5vUrzoGFgvxH79fFu+0rsm6OQDrmTa4JvYbXpvOUAYAADbOmn4AAEAMrKmIk3ZojsnW+QNKRLGQiy9v1xgHjKgT+AEAwGcg9AMAAFKV+VwcOrohDh9TH1V5A+xAdobVVsRXJzXHhCbr95GdCy+8MHK53HrbpEmTeo63trbG3/3d38WAAQOivr4+jjzyyFi8eHGmbQYA+jahHwAAsJ6d+xfj5B2aY3CNtbOAbW/qoGKcsH1TNFm/jxKw8847xxtvvNGz3XfffT3Hzj777Piv//qvuO2222LBggXx+uuvx5e+9KVM2wsA9G3W9AMAAD6mf7EQJ05sjnv+d0U8+k5r1s0B+oDqfC5mjqmPSc3KDFM6KioqYujQoR97funSpfHzn/88brrppjjwwAPT5+bPnx877rhjPPTQQ7H33ntn0FoAoK8z0w8AANiginwu/mJUfcwa1xDVBeU+ga1nSE0hTp7ULPCj5LzwwgsxfPjw2G677eL444+PV199NX1+0aJF0d7eHjNmzOg5Nyn9OXr06HjwwQczbDEA0JeZ6QcAAHyiZBB+aE1F3P77ZfHGyo6smwP0MrsNKMaMkXXpjQZQSqZNmxbXXXdd7LDDDmlpz4suuiimT58eTz75ZLz55ptRVVUVzc3N671myJAh6bFP0tbWlm7dWlpattpnAAD6FqEfAADwqZqrC3HCxKZY8PrKePitVVk3B+gFqvK5OHhUXbqOKJSimTNn9vw8efLkNAQcM2ZM3HrrrVFTU7PZ7zt37tw0QAQA2NKU9wQAAD6TQi4XB46oiy9v1xg1yn0Cf4LBNYU4aYcmgR9lJZnVN3HixHjxxRfTdf5Wr14d77///nrnLF68eINrAK7r/PPPT9cE7N5ee+21rdxyAKCvEPoBAACbZEJTVXx1UnOMrFM4BNg0yf0Cnx9WGyfv0BwDiq4hlJfly5fHSy+9FMOGDYupU6dGZWVl3H333T3Hn3vuuXTNv3322ecT36e6ujoaGxvX2wAAtgQ9bAAAYJM1VhXi+O2bYtE7rfE/r6+M1Z1dWTcJKHEj6iri0NH1wj7KxjnnnBOHH354WtLz9ddfjwsuuCAKhUIcd9xx0dTUFKecckrMnj07+vfvnwZ33/jGN9LAb++998666QBAH6WnDQAAbJZcLhd7DqqJiU1V8f++tjxeamnPuklACarMR+w3rC6mDiqm1w0oF3/84x/TgO/dd9+NQYMGxb777hsPPfRQ+nPixz/+ceTz+TjyyCOjra0tDj744Lj66quzbjYA0IcJ/QAAgD951t9R45vimffa4q4/Lo8VHWb9AWuNbaiMQ0bVR3N1IeumwCa7+eabP/F4sViMq666Kt0AAEqB0A8AANgiduxXHeMaKuOe11fEE++2Zd0cIEPVhVx8YURdTB5QzLopAADQZwj9AACALaZYkY9DRzfEzv2q487Xlsd7bZ1ZNwnYxpKSv38xqj7qk7qeAADANiP0AwAAtrgxDVVxyqR+cf+bK+O3i1eF6A96v7qKXBw0sj4m9avOuikAANAnCf0AAICtoiKfi/2G16VlP3/96vJ4Y2VH1k0CtpJkdu+MkXVRU2F2HwAAZEXoBwAAbFWDayrixIlNsfDt1vg/b6yM1Z1dWTcJ2EIaK/Nx8Kj6GN9UlXVTAACgzxP6AQAAW10ul4u9BtfExOaq+P9eWx4vtbRn3STgT1CVz8XeQ2rSv+vKfC7r5gAAAEI/AABgW2qqKsRR45vixaWrY8HrK+Lt1jVZNwnYBEnxzt0GFuPPh9ZGXaVSngAAUEqEfgAAwDY3oakqxjdWxu+WtMV9b6yMlvbOrJsEfIa/2wOG18aAoqEEAAAoRXrqAABAZiU/Jw8oxk79qmPh26viwcWrom2N9f6g1AyrrYgDRtTF6PrKrJsCAAB8AqEfAACQqYp0bbDa2G1AMR5YvCoefXtVdMj+IHONVfnYb1htGswnIT0AAFDahH4AAEBJKFbk48ARdTF1UDH+zxsr46klbSH7g22vupCLPxtSE1MH1aShPAAAUB6EfgAAQElpqirEYWMa4nODa2LB6yvipZb2rJsEfUKS7+0+sBj7Dq2Nmop81s0BAAA2kdAPAAAoSYNrKuKo8U3xh2Wr497XV8YbKzuybhL0Wjs0V8X+w+uiX3Uh66YAAACbSegHAACUtDENVXHSDlXx7HttseCNFfFeW2fWTYJeIfdB2DdtSE0Mq63MujkAAMCfSOgHAACUhUn9qmNic1U8/V5bPPzWqnhr1ZqsmwRlqSIXsUv/Yhr2mdkHAAC9h9APAAAoG/lcLg0rku2VltVp+PfKMmv+wWdRXcjFHgOLseegmqirtGYfAAD0NkI/AACgLI1rrEq3t1Z1pOHfM++1xZqurFsFpaehMh97Da6J3QYUo6qQFPUEAAB6I6EfAABQ1gbXVMRhYxpiv+G1seit1vi/77ZGm/QPYmCxEJ8bXBM796+OQk7YBwAAvZ3QDwAA6BUaKgux/4i6+LOhtfH4u62x8O1VsXR1Z9bNgm1uZF1Ful7fhMaqyAn7AACgzxD6AQAAvUpSvjApZTh1UDGee3/tun9vrOzIulmw1U1oqoq9B9fEyPrKrJsCAABkQOgHAAD0SvlcLnbsV51ury5vj4cXr4oXW1Zn3SzYomorcrFzv+rYbWAxBhR9xQcAgL7MNwIAAKDXG11fmW7vta2J3y1pjaeWtCn9SdnKR8T4pqrYtX91OrsvCbgBAACEfgAAQJ/Rr7oQnx9WF9OH1sZryzviySWt8ez7q2N1Z1fWTYNPNahYiF0HFNOZfXWVSfQHAADwIaEfAADQ5+RyuRjdUJluB43qiuffb4snl7TF75e1h/iPUlJdyMVO/apjcv/qGFZnrT4AAGDjhH4AAECfVpnPxc79i+m2rH1NPL1kbQD4duuarJtGH5UU6xzbUJnO6pvYVBUVeeU7AQCATyf0AwAA+EBDZSGmDalNtzdXri3/+fR7bbGyw/w/tr5+1fnYtX8xdulfHY1VhaybAwAAlBmhHwAAwAYMra2IobX1ceCIunipZXU6++/FpatjjfyPLaixKh/bN1XFpObqGFWvfCcAALD5hH4AAACfIJ/LxfZN1enWtqYzXm5pT8O/JAhslQCyGQYVCzGxuSr9/6kkXAYAANgSfLsAAAD4jKoL+dixX3W6dXZ1xR9XdKQBYLItabMGIBuWrMg3sr4iDfmSNfqaq5XuBAAAtjyhHwAAwGbOABxdX5luSQnQJa1r4oWlbfHKsvZ4bXm7MqB9XF1FLrZrrEq3cQ2VUazIZ90kAACglxP6AQAAbAH9i4WYVqyNaUMi2ju70uDv5ZbVaQj4bqtZgH1hNt+IuoqeoG9ITSFyueRZAACAbUPoBwAAsIVV5j+c5ZVoWb0mDf9eaVmdhoErOkwDLHeV+UjX4xtRWxnD6yrSGZ9m8wEAAFkS+gEAAGxljVWFmDIg2Yo9IeDrKzrijZUd8frK9nhzZUe0d2bdSj5JU1U+RtRVprP5ksfBNYW0xCsAAECpEPoBAABkEAIm26R+1el+Z1dXvNO6Jt5YsTYETALBZN98wGxU5D6YxbdOyFeXTO0DAAAoYUI/AACAjCUzxgbXVKTblFg7GzBZFzCZAfj6iva1MwJXdESL6YBbdRZfUqYzCfmS/x0KZvEBAABlRugHAABQousCjqqvTLduK9o70wDw3daOeH91Z7zXtibdWlZ3mhX4KYqFXPSvLkS/6kL0LxbSn7v3qwoCPgAAoPwJ/QAAAMpEUmJyQlNVuq1rTVdXLG3rjPdXrw0Bu7ckGHy/bU2s6eo7ZTnXDfXSnz/Yr61QnhMAAOjdhH4AAABlLilFmQZdxcLHjnV1dcWy9rWzAt9v++Bx9ZpY1dEVrWs6o3VN8tgVbSWcDCbL6dUU8lFTkYuainwa4CUz95LHuspc9KsqRL9iIRor85FTlhMAAOijhH4AAAC9WBKCNVYV0m1Mw8bPS8LB7gAwDQM7NvJz8tjRFW2dXZHWFM2l/5dua3/fhz9/uL/2mY+ek88lZTc/DPNqCh88JvuFJNxbu1+RnAgAAMAnEvoBAACQhoNrw7dk7+MzBgEAAChtFjUAAAAAAACAMif0AwAAAAAAgDIn9AMAAAAAAIAyJ/QDAAAAAACAMif0AwAAAAAAgDIn9AMAAAAAAIAyJ/QDAAAAAACAMif0AwAAAAAAgDLXa0K/q666KsaOHRvFYjGmTZsWDz/8cNZNAgAAAKAPMC4FAJSCXhH63XLLLTF79uy44IIL4tFHH40pU6bEwQcfHG+99VbWTQMAAACgFzMuBQCUil4R+l122WVx2mmnxVe/+tXYaaedYt68eVFbWxvXXntt1k0DAAAAoBczLgUAlIqyD/1Wr14dixYtihkzZvQ8l8/n0/0HH3ww07YBAAAA0HsZlwIASklFlLl33nkn1qxZE0OGDFnv+WT/2Wef3eBr2tra0q3b0qVL08eWlpboy1qXL8u6CZSIlpaqrJtAiWhv/fBaSd9W2cf/jVy3n9DV1ZV1UwAAKBHGpbYc3z/p5vsn3YxX0814dXzmcamyD/02x9y5c+Oiiy762POjRo3KpD1Qaj7+1wH0eT+4KusWlIxly5ZFU1NT1s0AAKBMGZeCT+H7J/ARxqs/+7hU2Yd+AwcOjEKhEIsXL17v+WR/6NChG3zN+eefny6w3K2zszOWLFkSAwYMiFwut9XbTGmn5Ukn+7XXXovGxsasmwOUANcFuiV3UiUdq+HDh2fdFAAASoRxKbYk3z+Bj3JdYFPHpco+9KuqqoqpU6fG3XffHbNmzerpLCX7Z5555gZfU11dnW7ram5u3ibtpTwkF1AXUWBdrgskzPADAGBdxqXYGnz/BD7KdYHPOi5V9qFfIrk76qSTToo999wzPve5z8VPfvKTWLFiRXz1q1/NumkAAAAA9GLGpQCAUtErQr9jjjkm3n777ZgzZ068+eabsdtuu8Wdd975sUWUAQAAAGBLMi4FAJSKXhH6JZKSCRsrmwCfVVJe44ILLvhYmQ2g73JdAAAAPo1xKbYE3z+Bj3JdYFPlupLV/wAAAAAAAICylc+6AQAAAAAAAMCfRugHAAAAAAAAZU7oBx+47rrrorm5OetmAAAAANDHGJcCYEsQ+tHrnHzyyZHL5T62vfjii1k3DcjQhq4L624XXnhh1k0EAACgzBmXAjbEuBTbSsU2+02wDR1yyCExf/789Z4bNGhQZu0BsvfGG2/0/HzLLbfEnDlz4rnnnut5rr6+vufnrq6uWLNmTVRU+GcSAACATWNcCvgo41JsK2b60StVV1fH0KFD19suv/zy2HXXXaOuri5GjRoVX//612P58uUbfY/HH388DjjggGhoaIjGxsaYOnVqLFy4sOf4fffdF9OnT4+ampr0/c4666xYsWLFNvqEwKZa93rQ1NSU3kXVvf/ss8+mf+u//vWv07/15BqS/I0nd2jOmjVrvff55je/Gfvvv3/PfmdnZ8ydOzfGjRuXXg+mTJkSv/zlLzP4hAAAAJQC41LARxmXYlsR+tFn5PP5uOKKK+Kpp56KX/ziF3HPPffEueeeu9Hzjz/++Bg5cmQ88sgjsWjRovj7v//7qKysTI+99NJL6V1bRx55ZDzxxBPp3RnJhfjMM8/chp8I2NKSv/Mf/OAH8cwzz8TkyZM/02uSjtX1118f8+bNS68vZ599dpxwwgmxYMGCrd5eAAAAyoNxKeDTGJdiSzA/lF7pjjvuWG9K9MyZM+O2227r2R87dmxccskl8bWvfS2uvvrqDb7Hq6++Gt/61rdi0qRJ6f7222+/3sU06Xwld1Z0H0s6bvvtt19cc801USwWt+KnA7aWiy++OA466KDPfH5bW1t8//vfj7vuuiv22Wef9Lntttsu/bL105/+NL0mAAAA0LcYlwI2h3EptgShH71SUv4g6eR0S0onJBe/pFOUTJduaWmJjo6OaG1tjZUrV0Ztbe3H3mP27Nlx6qmnxg033BAzZsyIo446KsaPH99TYiG5k+rGG29cr9ZyMp36lVdeiR133HEbfVJgS9pzzz036fxkIfbkGvLRDtnq1atj991338KtAwAAoBwYlwI2h3EptgShH71S0pmaMGFCz/7vf//7OOyww+Jv//Zv43vf+170798/vePhlFNOSS+CG+pcXXjhhfGVr3wlfvWrX6X1lC+44IK4+eab44gjjkhrrp9xxhlpvfSPGj169Fb/fMDWu3Z8tPxK8sVpXe3t7T0/d6+/kFwnRowYsd55Sf11AAAA+h7jUsDmMC7FliD0o09Iap8ndzv96Ec/Si+WiVtvvfVTXzdx4sR0S2ohH3fccTF//vy0c7XHHnvE008/vV4HDuh9Bg0aFE8++eR6zz322GM96yjstNNOaScqKbuiZAIAAAAbYlwK2BzGpdgca/+VgV4u6QQld0FceeWV8fLLL6elEZLFTTdm1apV6eLH9957b/zhD3+I+++/P104ubs8wnnnnRcPPPBAek5yoX3hhRfi9ttvt2Ay9DIHHnhgLFy4MF0QOfk7T+6sXLez1dDQEOecc076BSxZiD1ZTP3RRx9NrzXJPgAAABiXAjaHcSk2h9CPPmHKlClx2WWXxaWXXhq77LJLWvM8qaO+MYVCId5999048cQT0zuqjj766HTR5Ysuuig9Pnny5FiwYEE8//zzMX369LRG8pw5c2L48OHb8FMBW9vBBx8c3/nOd+Lcc8+NvfbaK5YtW5ZeF9b13e9+Nz0nuaYkX8AOOeSQtKzCuHHjMms3AAAApcO4FLA5jEuxOXJdHy0KCwAAAAAAAJQVM/0AAAAAAACgzAn9AAAAAAAAoMwJ/QAAAAAAAKDMCf0AAAAAAACgzAn9AAAAAAAAoMwJ/QAAAAAAAKDMCf0AAAAAAACgzAn9AAAAAAAAoMwJ/YBe5d57741cLhfvv/9+1k0BAAAAoA8xLgVkTegHlJSTTz457RwlW2VlZYwbNy7OPffcaG1tzbppAAAAAPRixqWAcleRdQMAPuqQQw6J+fPnR3t7eyxatChOOumktLN16aWXZt00AAAAAHox41JAOTPTDyg51dXVMXTo0Bg1alTMmjUrZsyYEb/5zW/SY52dnTF37tz0TquampqYMmVK/PKXv/zE97vvvvti+vTp6fnJe5511lmxYsWK9Ng//MM/xLRp0z72muR9L7744vTnRx55JA466KAYOHBgNDU1xX777RePPvroeucnnb+f/exnccQRR0RtbW1sv/328Z//+Z/rnfPUU0/FYYcdFo2NjdHQ0JC26aWXXuo5nrx+xx13jGKxGJMmTYqrr776T/ivCAAAAMCmMi5lXArKmdAPKGlPPvlkPPDAA1FVVZXuJx2r66+/PubNm5d2Vs4+++w44YQTYsGCBRt8fdJ5Se7QOvLII+OJJ56IW265Je1snXnmmenx448/Ph5++OH1OjnJ+ybnfuUrX0n3ly1blt7VlbzuoYceSjtOhx56aPr8ui666KI4+uij09cmx5P3XrJkSXrsf//3f+Pzn/982nG855570jvF/uZv/iY6OjrS4zfeeGPMmTMnvve978UzzzwT3//+9+M73/lO/OIXv9hK/2UBAAAA+CTGpYxLQbnJdXV1dWXdCIB1a6f/y7/8S3pXUdLxaGtri3w+H7feemt6N1L//v3jrrvuin322afnNaeeemqsXLkybrrppnTB5AMOOCDee++9aG5uTo8VCoX46U9/2nN+0klK7opK7qpKfs9uu+2Wdr6Szkz3XVZJByjpSG1IcldX8t7J70va1H1H1be//e347ne/m+4n711fXx+//vWv085d8p4333xzPPfcc2lN+I+aMGFC+trjjjuu57lLLrkk/vu//zvtXAIAAACwdRmXMi4F5c6afkDJSTpH11xzTdpB+fGPfxwVFRVp5ye50ynpRCUlDda1evXq2H333Tf4Xo8//nh6h1Nyx1K35F6HpIP0yiuvpGULkjufrr322rRzlRz713/915g9e3bP+YsXL047TknH7a233oo1a9ak7Xj11VfX+12TJ0/u+bmuri4tl5Ccn3jsscfSsgkb6lglnzO5o+uUU06J0047ref5pHOZlG0AAAAAYNswLrWWcSkoT0I/oOQkHZPkDqNE0ulJ6pj//Oc/j1122SV97le/+lWMGDFivdck5Qk2ZPny5XHGGWek9dI/avTo0eljchfTeeedl9ZDX7VqVbz22mtxzDHH9JyXlFB499134/LLL48xY8akvyu5oyvp1K3rox2n5C6rpBOXSOq2b0zSxsQ///M/f6yOe3I3GAAAAADbhnGpDxmXgvIj9ANKWlJCISlBkNzh9Pzzz6cdm+ROpqQMwmexxx57xNNPP93TWduQkSNHpu+X3HWVdK6SO7YGDx7cc/z+++9PFy9O6qEnks7XO++8s0mfI7nbKqmD3t7e/rFO2JAhQ2L48OHx8ssvp3d3AQAAAJA941JAucln3QCAT3PUUUf11D8/55xz0kWSk45KUnoguQvqyiuv3OjCwsmdUknt8WSB5KSUwQsvvBC33357z4LJ3ZJOTVLb/LbbbvtYBydZIPmGG25IFzL+7W9/mx7/pDukNiT5fS0tLXHsscfGwoUL03Yk75nUUu9ebDlZDPqKK65IO5G/+93vYv78+XHZZZdt8n8vAAAAALYM41JAORH6ASUvqZ2edE5++MMfxvnnn5/WOE86Iknd82Qx4qSswrhx4zZ6J9OCBQvSDktSuzypsT5nzpz0DqZ1ffnLX05LJSQ10WfNmrXesaSEQ7IAc3J31l//9V+nJRnWvePqsxgwYEC6CHNSMiG5e2vq1Klp2YTuu6uShZ1/9rOfpR2qXXfdNT3nuuuu2+jnAgAAAGDrMy4FlJNcV7I6KAAAAAAAAFC2zPQDAAAAAACAMif0AwAAAAAAgDIn9AMAAAAAAIAyJ/QDAAAAAACAMif0AwAAAAAAgDIn9AMAAAAAAIAyJ/QDAAAAAACAMif0AwAAAAAAgDIn9AMAAAAAAIAyJ/QDAAAAAACAMif0AwAAAAAAgDIn9AMAAAAAAIAob/8/RE2yGP5eVQEAAAAASUVORK5CYII=",
      "text/plain": [
       "<Figure size 1800x600 with 3 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import matplotlib.pyplot as plt\n",
    "\n",
    "df = pd.read_csv(rf\"C:\\Users\\vtorr\\Work\\Projects\\aipatent\\evals\\GVHD-ALD-papers.csv\")\n",
    "\n",
    "relevance = df[\"relevant\"].value_counts()\n",
    "print(relevance)\n",
    "\n",
    "\n",
    "df[\"text_length\"] = df[\"text\"].str.len()\n",
    "\n",
    "\n",
    "avg_text_length = df.groupby(\"relevant\")[\"text_length\"].mean()\n",
    "print(avg_text_length)\n",
    "\n",
    "\n",
    "fig, axs = plt.subplots(1, 3, figsize=(18, 6))\n",
    "\n",
    "\n",
    "relevance.plot(kind='bar', color=['skyblue', 'salmon'], ax=axs[0])\n",
    "axs[0].set_xlabel(\"Relevance\")\n",
    "axs[0].set_ylabel(\"Count\")\n",
    "axs[0].set_title(\"Count of Relevance\")\n",
    "axs[0].set_xticklabels(relevance.index, rotation=0)\n",
    "\n",
    "\n",
    "axs[1].pie(\n",
    "    relevance, \n",
    "    labels=relevance.index, \n",
    "    autopct='%1.1f%%', \n",
    "    colors=['skyblue', 'salmon'],\n",
    "    startangle=90\n",
    ")\n",
    "axs[1].set_title(\"Pie Chart of Relevance\")\n",
    "axs[1].axis('equal')\n",
    "\n",
    "avg_text_length.plot(kind='bar', color=['salmon', 'skyblue'], ax=axs[2])\n",
    "axs[2].set_xlabel(\"Relevance\")\n",
    "axs[2].set_ylabel(\"Average Text Length\")\n",
    "axs[2].set_title(\"Average Text Length by Relevance\")\n",
    "axs[2].set_xticklabels(avg_text_length.index, rotation=0)\n",
    "\n",
    "plt.tight_layout()\n",
    "plt.show()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Filtered CSV saved as GVHD-ALD-papers-not-false.csv\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Read the original CSV\n",
    "df = pd.read_csv(r\"C:\\Users\\vtorr\\Work\\Projects\\aipatent\\evals\\GVHD-ALD-papers.csv\")\n",
    "\n",
    "# Filter rows where 'relevant' is not False (i.e. True or any value other than False)\n",
    "filtered_df = df[df[\"relevant\"] != False]\n",
    "\n",
    "# Save the filtered DataFrame to a new CSV\n",
    "filtered_df.to_csv(r\"C:\\Users\\vtorr\\Work\\Projects\\aipatent\\evals\\GVHD-ALD-papers-not-false.csv\", index=False)\n",
    "\n",
    "print(\"Filtered CSV saved as GVHD-ALD-papers-not-false.csv\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
>>>>>>> 5b6e3e1f6bb904635df1f05e870b8aeeed94cf1b
